{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Background on rounds of sequencing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Two families were chosen for sequencing, offspring from female 101 and female 110.  Both gynogenetic diploids and haploids were included in library prep and sequencing with the following sample sizes: \n",
    "101: haploid=49, gyno diploid=23\n",
    "107: haploid=118\n",
    "Together with dams 101 and 102 we prepared two libraries of 96 individuals each.\n",
    "\n",
    "Libraries were mistakenly swapped on the instrument during the first round of sequencing, resulting in incorrect titrations for the second round of sequencing and inbalanced coverage among individuals.  A third round of sequencing was conducted but didn't result in adequate coverage for the smaller group of 101 haploid individuals to be useful in estimating recombination frequencies.  Finally, we used the instrument training run on ADF&G's NextSeq 500 to try and resurrect 101 haploid individuals and incorporate variation from dam 101 on the linkage map."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Processing .fastq from final round of sequencing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing paired-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 32 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Undetermined_S0_R1_001.fastq.gz]\n",
      "  Reading data from:\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R1_001.fastq.gz and\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R2_001.fastq.gz\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...159M...160M...161M...162M...163M...164M...165M...166M...167M...168M...169M...170M...171M...172M...173M...174M...175M...176M...177M...178M...179M...180M...181M...182M...183M...184M...185M...186M...187M...188M...189M...190M...191M...192M...193M...194M...195M...196M...197M...198M...199M...200M...201M...202M...203M...204M...205M...206M...207M...208M...209M...210M...211M...212M...213M...214M...215M...216M...217M...218M...219M...220M...221M...222M...223M...224M...225M...226M...227M...228M...229M...230M...231M...232M...233M...234M...235M...236M...237M...238M...239M...240M...241M...242M...243M...244M...245M...246M...247M...248M...249M...250M...251M...252M...253M...254M...255M...256M...257M...258M...259M...260M...261M...262M...263M...264M...265M...266M...267M...268M...269M...270M...271M...272M...273M...274M...275M...276M...277M...278M...279M...280M...281M...282M...283M...284M...285M...286M...287M...288M...289M...290M...291M...292M...293M...294M...295M...296M...297M...298M...299M...300M...301M...302M...303M...304M...305M...306M...307M...308M...309M...310M...311M...312M...313M...314M...315M...316M...317M...318M...319M...320M...321M...322M...323M...324M...325M...326M...327M...328M...329M...330M...331M...332M...333M...334M...335M...336M...337M...338M...339M...340M...341M...342M...343M...344M...345M...346M...347M...348M...349M...350M...351M...352M...353M...354M...355M...356M...357M...358M...359M...360M...361M...362M...363M...364M...365M...366M...367M...368M...369M...370M...371M...372M...373M...374M...375M...376M...377M...378M...379M...380M...381M...382M...383M...384M...385M...386M...387M...388M...389M...390M...391M...392M...393M...394M...395M...396M...397M...398M...399M...400M...401M...402M...403M...404M...405M...406M...407M...408M...409M...410M...411M...412M...413M...414M...415M...416M...417M...418M...419M...420M...421M...422M...423M...424M...425M...426M...427M...428M...429M...430M...431M...432M...433M...434M...435M...436M...437M...438M...439M...440M...441M...442M...443M...444M...445M...446M...447M...448M...449M...450M...451M...452M...453M...454M...455M...456M...457M...458M...459M...460M...461M...462M...463M...464M...465M...466M...467M...468M...469M...470M...471M...472M...473M...474M...475M...476M...477M...478M...479M...480M...481M...482M...483M...484M...485M...486M...487M...488M...489M...490M...491M...492M...493M...494M...495M...496M...497M...\n",
      "  995296694 total reads; -0 failed Illumina reads; -789393602 ambiguous barcodes; -73115971 ambiguous RAD-Tags; +132998688 recovered; -755123 low quality reads; 132031998 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/process_radtags.fastq.log'\n",
      "\n",
      "995296694 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      "789393602 ambiguous barcode drops (79.3%)\n",
      "   755123 low quality read drops (0.1%)\n",
      " 73115971 ambiguous RAD-Tag drops (7.3%)\n",
      "132031998 retained reads (13.3%)\n"
     ]
    }
   ],
   "source": [
    "! process_radtags -P -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/Bird2014TrainingRunBarcodes.txt' -E phred33 -e sbfI -i gzfastq -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Same process radtags but trimming to 94 bases to align with previously collected data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing paired-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 32 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Undetermined_S0_R1_001.fastq.gz]\n",
      "  Reading data from:\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R1_001.fastq.gz and\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R2_001.fastq.gz\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...159M...160M...161M...162M...163M...164M...165M...166M...167M...168M...169M...170M...171M...172M...173M...174M...175M...176M...177M...178M...179M...180M...181M...182M...183M...184M...185M...186M...187M...188M...189M...190M...191M...192M...193M...194M...195M...196M...197M...198M...199M...200M...201M...202M...203M...204M...205M...206M...207M...208M...209M...210M...211M...212M...213M...214M...215M...216M...217M...218M...219M...220M...221M...222M...223M...224M...225M...226M...227M...228M...229M...230M...231M...232M...233M...234M...235M...236M...237M...238M...239M...240M...241M...242M...243M...244M...245M...246M...247M...248M...249M...250M...251M...252M...253M...254M...255M...256M...257M...258M...259M...260M...261M...262M...263M...264M...265M...266M...267M...268M...269M...270M...271M...272M...273M...274M...275M...276M...277M...278M...279M...280M...281M...282M...283M...284M...285M...286M...287M...288M...289M...290M...291M...292M...293M...294M...295M...296M...297M...298M...299M...300M...301M...302M...303M...304M...305M...306M...307M...308M...309M...310M...311M...312M...313M...314M...315M...316M...317M...318M...319M...320M...321M...322M...323M...324M...325M...326M...327M...328M...329M...330M...331M...332M...333M...334M...335M...336M...337M...338M...339M...340M...341M...342M...343M...344M...345M...346M...347M...348M...349M...350M...351M...352M...353M...354M...355M...356M...357M...358M...359M...360M...361M...362M...363M...364M...365M...366M...367M...368M...369M...370M...371M...372M...373M...374M...375M...376M...377M...378M...379M...380M...381M...382M...383M...384M...385M...386M...387M...388M...389M...390M...391M...392M...393M...394M...395M...396M...397M...398M...399M...400M...401M...402M...403M...404M...405M...406M...407M...408M...409M...410M...411M...412M...413M...414M...415M...416M...417M...418M...419M...420M...421M...422M...423M...424M...425M...426M...427M...428M...429M...430M...431M...432M...433M...434M...435M...436M...437M...438M...439M...440M...441M...442M...443M...444M...445M...446M...447M...448M...449M...450M...451M...452M...453M...454M...455M...456M...457M...458M...459M...460M...461M...462M...463M...464M...465M...466M...467M...468M...469M...470M...471M...472M...473M...474M...475M...476M...477M...478M...479M...480M...481M...482M...483M...484M...485M...486M...487M...488M...489M...490M...491M...492M...493M...494M...495M...496M...497M...\n",
      "  995296694 total reads; -0 failed Illumina reads; -789393602 ambiguous barcodes; -73115971 ambiguous RAD-Tags; +132998688 recovered; -675118 low quality reads; 132112003 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/process_radtags.fastq.log'\n",
      "\n",
      "995296694 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      "789393602 ambiguous barcode drops (79.3%)\n",
      "   675118 low quality read drops (0.1%)\n",
      " 73115971 ambiguous RAD-Tag drops (7.3%)\n",
      "132112003 retained reads (13.3%)\n"
     ]
    }
   ],
   "source": [
    "! process_radtags -P -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/Bird2014TrainingRunBarcodes.txt' -E phred33 -e sbfI -i gzfastq -t 94 -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Description of the four rounds of sequencing for 2014 Bird Creek haploids"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Protocol when the two 2014 Bird Creek libraries (PHap1 and PHap2) were created was to conduct two rounds of sequencing to equalize coverage among individuals, titrating before the second round according to retained reads from the first round.\n",
    "\n",
    "Unfortunately the two libraries were swapped when sequenced at the Core lab for the first round, resulting in incorrect titrations and no equalizing of coverage.  A third round was done on a subset of individuals to rescue them from too few reads.  The final round (unpacked above) was acquired during ADF&G's training run.\n",
    "\n",
    "\n",
    "|         |                        | Barcode key used in: |               |                     |                 |\n",
    "|---------|------------------------|:--------------------:|:-------------:|:-------------------:|:---------------:|\n",
    "| Library | Originally planned key | Round 1              | ReRun         | ReReRun             | Round 4         |\n",
    "| PHap1   | PHap1Barcodes          | PHap2Barcodes        | PHap1Barcodes | PHapReReSeqBarcodes | PHap1 (partial) |\n",
    "| PHap2   | PHap2Barcodes          | PHap1Barcodes        | PHap2Barcodes |                     |                 |"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 59 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Phaps2014_NoIndex_S97_L007_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...\n",
      "  158449390 total reads; -0 failed Illumina reads; -56117100 ambiguous barcodes; -20981990 ambiguous RAD-Tags; +19354516 recovered; -996062 low quality reads; 80354238 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/process_radtags.PHapReReSeq.log'\n",
      "\n",
      "158449390 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      " 56117100 ambiguous barcode drops (35.4%)\n",
      "   996062 low quality read drops (0.6%)\n",
      " 20981990 ambiguous RAD-Tag drops (13.2%)\n",
      " 80354238 retained reads (50.7%)\n"
     ]
    }
   ],
   "source": [
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeq' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqBarcodes.txt' -E phred33 -e sbfI -i gzfastq -t 94 -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 15 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 15 [PHap1_lane5_NoIndex_L005_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1841729 failed Illumina reads; -4861645 ambiguous barcodes; -3275420 ambiguous RAD-Tags; +3998931 recovered; -413159 low quality reads; 5608047 retained reads.\n",
      "Processing file 2 of 15 [PHap1_lane5_NoIndex_L005_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1442672 failed Illumina reads; -4856855 ambiguous barcodes; -3385188 ambiguous RAD-Tags; +3817593 recovered; -507293 low quality reads; 5807992 retained reads.\n",
      "Processing file 3 of 15 [PHap1_lane5_NoIndex_L005_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1768721 failed Illumina reads; -4849842 ambiguous barcodes; -3297460 ambiguous RAD-Tags; +3877673 recovered; -437433 low quality reads; 5646544 retained reads.\n",
      "Processing file 4 of 15 [PHap1_lane5_NoIndex_L005_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1525150 failed Illumina reads; -4870371 ambiguous barcodes; -3362990 ambiguous RAD-Tags; +3910295 recovered; -496853 low quality reads; 5744636 retained reads.\n",
      "Processing file 5 of 15 [PHap1_lane5_NoIndex_L005_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1574810 failed Illumina reads; -4867386 ambiguous barcodes; -3351296 ambiguous RAD-Tags; +3831048 recovered; -447826 low quality reads; 5758682 retained reads.\n",
      "Processing file 6 of 15 [PHap1_lane5_NoIndex_L005_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1750301 failed Illumina reads; -4871056 ambiguous barcodes; -3295240 ambiguous RAD-Tags; +4018245 recovered; -449776 low quality reads; 5633627 retained reads.\n",
      "Processing file 7 of 15 [PHap1_lane5_NoIndex_L005_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1458112 failed Illumina reads; -4859683 ambiguous barcodes; -3387927 ambiguous RAD-Tags; +3818661 recovered; -454998 low quality reads; 5839280 retained reads.\n",
      "Processing file 8 of 15 [PHap1_lane5_NoIndex_L005_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1268941 failed Illumina reads; -4847234 ambiguous barcodes; -3460987 ambiguous RAD-Tags; +3968631 recovered; -423914 low quality reads; 5998924 retained reads.\n",
      "Processing file 9 of 15 [PHap1_lane5_NoIndex_L005_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1630656 failed Illumina reads; -4849660 ambiguous barcodes; -3350808 ambiguous RAD-Tags; +4133675 recovered; -394710 low quality reads; 5774166 retained reads.\n",
      "Processing file 10 of 15 [PHap1_lane5_NoIndex_L005_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1701429 failed Illumina reads; -4868733 ambiguous barcodes; -3313981 ambiguous RAD-Tags; +4164742 recovered; -395174 low quality reads; 5720683 retained reads.\n",
      "Processing file 11 of 15 [PHap1_lane5_NoIndex_L005_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1673024 failed Illumina reads; -4840081 ambiguous barcodes; -3327779 ambiguous RAD-Tags; +3879115 recovered; -502256 low quality reads; 5656860 retained reads.\n",
      "Processing file 12 of 15 [PHap1_lane5_NoIndex_L005_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -4067913 failed Illumina reads; -4852106 ambiguous barcodes; -2491807 ambiguous RAD-Tags; +3955297 recovered; -1973888 low quality reads; 2614286 retained reads.\n",
      "Processing file 13 of 15 [PHap1_lane5_NoIndex_L005_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1274445 failed Illumina reads; -4846972 ambiguous barcodes; -3462894 ambiguous RAD-Tags; +3799797 recovered; -404760 low quality reads; 6010929 retained reads.\n",
      "Processing file 14 of 15 [PHap1_lane5_NoIndex_L005_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1648099 failed Illumina reads; -4853786 ambiguous barcodes; -3342015 ambiguous RAD-Tags; +4134085 recovered; -387209 low quality reads; 5768891 retained reads.\n",
      "Processing file 15 of 15 [PHap1_lane5_NoIndex_L005_R1_015.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...\n",
      "  3486172 total reads; -490779 failed Illumina reads; -1067090 ambiguous barcodes; -681703 ambiguous RAD-Tags; +1053402 recovered; -91708 low quality reads; 1154892 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/process_radtags.PHap1.log'\n",
      "\n",
      "227486172 total sequences\n",
      " 25116781 failed Illumina filtered reads (11.0%)\n",
      " 69062500 ambiguous barcode drops (30.4%)\n",
      "  7780957 low quality read drops (3.4%)\n",
      " 46787495 ambiguous RAD-Tag drops (20.6%)\n",
      " 78738439 retained reads (34.6%)\n",
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 14 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 14 [PHap2_lane6_NoIndex_L006_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1330704 failed Illumina reads; -4599828 ambiguous barcodes; -3268344 ambiguous RAD-Tags; +3626926 recovered; -492588 low quality reads; 6308536 retained reads.\n",
      "Processing file 2 of 14 [PHap2_lane6_NoIndex_L006_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1678389 failed Illumina reads; -4601133 ambiguous barcodes; -3184186 ambiguous RAD-Tags; +3678462 recovered; -436206 low quality reads; 6100086 retained reads.\n",
      "Processing file 3 of 14 [PHap2_lane6_NoIndex_L006_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1436785 failed Illumina reads; -4609882 ambiguous barcodes; -3243865 ambiguous RAD-Tags; +3636098 recovered; -507671 low quality reads; 6201797 retained reads.\n",
      "Processing file 4 of 14 [PHap2_lane6_NoIndex_L006_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1490283 failed Illumina reads; -4603818 ambiguous barcodes; -3231925 ambiguous RAD-Tags; +3635438 recovered; -426563 low quality reads; 6247411 retained reads.\n",
      "Processing file 5 of 14 [PHap2_lane6_NoIndex_L006_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1536959 failed Illumina reads; -4616159 ambiguous barcodes; -3214975 ambiguous RAD-Tags; +3638869 recovered; -467713 low quality reads; 6164194 retained reads.\n",
      "Processing file 6 of 14 [PHap2_lane6_NoIndex_L006_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1455947 failed Illumina reads; -4608911 ambiguous barcodes; -3242646 ambiguous RAD-Tags; +3632713 recovered; -450161 low quality reads; 6242335 retained reads.\n",
      "Processing file 7 of 14 [PHap2_lane6_NoIndex_L006_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1590042 failed Illumina reads; -4594226 ambiguous barcodes; -3215995 ambiguous RAD-Tags; +3749648 recovered; -440666 low quality reads; 6159071 retained reads.\n",
      "Processing file 8 of 14 [PHap2_lane6_NoIndex_L006_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1233899 failed Illumina reads; -4584488 ambiguous barcodes; -3315046 ambiguous RAD-Tags; +3609987 recovered; -405495 low quality reads; 6461072 retained reads.\n",
      "Processing file 9 of 14 [PHap2_lane6_NoIndex_L006_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1673124 failed Illumina reads; -4586831 ambiguous barcodes; -3202959 ambiguous RAD-Tags; +3834485 recovered; -391928 low quality reads; 6145158 retained reads.\n",
      "Processing file 10 of 14 [PHap2_lane6_NoIndex_L006_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1098221 failed Illumina reads; -4579024 ambiguous barcodes; -3361518 ambiguous RAD-Tags; +3616755 recovered; -430492 low quality reads; 6530745 retained reads.\n",
      "Processing file 11 of 14 [PHap2_lane6_NoIndex_L006_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1315593 failed Illumina reads; -4577851 ambiguous barcodes; -3303501 ambiguous RAD-Tags; +3622165 recovered; -396292 low quality reads; 6406763 retained reads.\n",
      "Processing file 12 of 14 [PHap2_lane6_NoIndex_L006_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1390550 failed Illumina reads; -4589564 ambiguous barcodes; -3272464 ambiguous RAD-Tags; +3726163 recovered; -421485 low quality reads; 6325937 retained reads.\n",
      "Processing file 13 of 14 [PHap2_lane6_NoIndex_L006_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1376376 failed Illumina reads; -4589221 ambiguous barcodes; -3280385 ambiguous RAD-Tags; +3678069 recovered; -412692 low quality reads; 6341326 retained reads.\n",
      "Processing file 14 of 14 [PHap2_lane6_NoIndex_L006_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...\n",
      "  11577285 total reads; -1334678 failed Illumina reads; -3324862 ambiguous barcodes; -2278383 ambiguous RAD-Tags; +2842609 recovered; -313524 low quality reads; 4325838 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/process_radtags.PHap2.log'\n",
      "\n",
      "219577285 total sequences\n",
      " 19941550 failed Illumina filtered reads (9.1%)\n",
      " 63065798 ambiguous barcode drops (28.7%)\n",
      "  5993476 low quality read drops (2.7%)\n",
      " 44616192 ambiguous RAD-Tag drops (20.3%)\n",
      " 85960269 retained reads (39.1%)\n",
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 15 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 15 [PHap1rr_lane8_NoIndex_L008_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1602479 failed Illumina reads; -4695437 ambiguous barcodes; -3277284 ambiguous RAD-Tags; +3870641 recovered; -485705 low quality reads; 5939095 retained reads.\n",
      "Processing file 2 of 15 [PHap1rr_lane8_NoIndex_L008_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1612693 failed Illumina reads; -4693554 ambiguous barcodes; -3266974 ambiguous RAD-Tags; +3746812 recovered; -609702 low quality reads; 5817077 retained reads.\n",
      "Processing file 3 of 15 [PHap1rr_lane8_NoIndex_L008_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2021641 failed Illumina reads; -4721926 ambiguous barcodes; -3166730 ambiguous RAD-Tags; +4012619 recovered; -490793 low quality reads; 5598910 retained reads.\n",
      "Processing file 4 of 15 [PHap1rr_lane8_NoIndex_L008_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2008058 failed Illumina reads; -4739371 ambiguous barcodes; -3158722 ambiguous RAD-Tags; +4143499 recovered; -607534 low quality reads; 5486315 retained reads.\n",
      "Processing file 5 of 15 [PHap1rr_lane8_NoIndex_L008_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1875257 failed Illumina reads; -4709012 ambiguous barcodes; -3200580 ambiguous RAD-Tags; +3878907 recovered; -571554 low quality reads; 5643597 retained reads.\n",
      "Processing file 6 of 15 [PHap1rr_lane8_NoIndex_L008_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2276737 failed Illumina reads; -4739102 ambiguous barcodes; -3088549 ambiguous RAD-Tags; +4396773 recovered; -503786 low quality reads; 5391826 retained reads.\n",
      "Processing file 7 of 15 [PHap1rr_lane8_NoIndex_L008_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1642162 failed Illumina reads; -4689234 ambiguous barcodes; -3265012 ambiguous RAD-Tags; +3738107 recovered; -590719 low quality reads; 5812873 retained reads.\n",
      "Processing file 8 of 15 [PHap1rr_lane8_NoIndex_L008_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2059459 failed Illumina reads; -4718699 ambiguous barcodes; -3152874 ambiguous RAD-Tags; +4007598 recovered; -492553 low quality reads; 5576415 retained reads.\n",
      "Processing file 9 of 15 [PHap1rr_lane8_NoIndex_L008_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1632234 failed Illumina reads; -4702433 ambiguous barcodes; -3277596 ambiguous RAD-Tags; +3748186 recovered; -434261 low quality reads; 5953476 retained reads.\n",
      "Processing file 10 of 15 [PHap1rr_lane8_NoIndex_L008_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1944027 failed Illumina reads; -4723975 ambiguous barcodes; -3190628 ambiguous RAD-Tags; +4100146 recovered; -468172 low quality reads; 5673198 retained reads.\n",
      "Processing file 11 of 15 [PHap1rr_lane8_NoIndex_L008_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1418732 failed Illumina reads; -4682878 ambiguous barcodes; -3339668 ambiguous RAD-Tags; +3703497 recovered; -509951 low quality reads; 6048771 retained reads.\n",
      "Processing file 12 of 15 [PHap1rr_lane8_NoIndex_L008_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1822112 failed Illumina reads; -4710022 ambiguous barcodes; -3230969 ambiguous RAD-Tags; +4060340 recovered; -437409 low quality reads; 5799488 retained reads.\n",
      "Processing file 13 of 15 [PHap1rr_lane8_NoIndex_L008_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1738507 failed Illumina reads; -4701593 ambiguous barcodes; -3241922 ambiguous RAD-Tags; +3955733 recovered; -490667 low quality reads; 5827311 retained reads.\n",
      "Processing file 14 of 15 [PHap1rr_lane8_NoIndex_L008_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1586142 failed Illumina reads; -4691086 ambiguous barcodes; -3292685 ambiguous RAD-Tags; +3739036 recovered; -453503 low quality reads; 5976584 retained reads.\n",
      "Processing file 15 of 15 [PHap1rr_lane8_NoIndex_L008_R1_015.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...\n",
      "  14018093 total reads; -1873726 failed Illumina reads; -4139698 ambiguous barcodes; -2744203 ambiguous RAD-Tags; +3952326 recovered; -361325 low quality reads; 4899141 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/process_radtags.PHap1Rerun.log'\n",
      "\n",
      "238018093 total sequences\n",
      " 27113966 failed Illumina filtered reads (11.4%)\n",
      " 70058020 ambiguous barcode drops (29.4%)\n",
      "  7507634 low quality read drops (3.2%)\n",
      " 47894396 ambiguous RAD-Tag drops (20.1%)\n",
      " 85444077 retained reads (35.9%)\n",
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 15 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 15 [PHap2rr_lane7_NoIndex_L007_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1227217 failed Illumina reads; -4737586 ambiguous barcodes; -3412125 ambiguous RAD-Tags; +3821279 recovered; -382618 low quality reads; 6240454 retained reads.\n",
      "Processing file 2 of 15 [PHap2rr_lane7_NoIndex_L007_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1401461 failed Illumina reads; -4758477 ambiguous barcodes; -3355864 ambiguous RAD-Tags; +3769144 recovered; -446761 low quality reads; 6037437 retained reads.\n",
      "Processing file 3 of 15 [PHap2rr_lane7_NoIndex_L007_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1742549 failed Illumina reads; -4768938 ambiguous barcodes; -3243897 ambiguous RAD-Tags; +4143079 recovered; -407387 low quality reads; 5837229 retained reads.\n",
      "Processing file 4 of 15 [PHap2rr_lane7_NoIndex_L007_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1624298 failed Illumina reads; -4752055 ambiguous barcodes; -3280535 ambiguous RAD-Tags; +3913643 recovered; -491493 low quality reads; 5851619 retained reads.\n",
      "Processing file 5 of 15 [PHap2rr_lane7_NoIndex_L007_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1764312 failed Illumina reads; -4760553 ambiguous barcodes; -3237523 ambiguous RAD-Tags; +4024161 recovered; -452630 low quality reads; 5784982 retained reads.\n",
      "Processing file 6 of 15 [PHap2rr_lane7_NoIndex_L007_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1887797 failed Illumina reads; -4789054 ambiguous barcodes; -3188939 ambiguous RAD-Tags; +4276285 recovered; -434640 low quality reads; 5699570 retained reads.\n",
      "Processing file 7 of 15 [PHap2rr_lane7_NoIndex_L007_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1663021 failed Illumina reads; -4752589 ambiguous barcodes; -3265568 ambiguous RAD-Tags; +3920300 recovered; -458684 low quality reads; 5860138 retained reads.\n",
      "Processing file 8 of 15 [PHap2rr_lane7_NoIndex_L007_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2019554 failed Illumina reads; -4804656 ambiguous barcodes; -3143677 ambiguous RAD-Tags; +4455055 recovered; -411885 low quality reads; 5620228 retained reads.\n",
      "Processing file 9 of 15 [PHap2rr_lane7_NoIndex_L007_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1528303 failed Illumina reads; -4753294 ambiguous barcodes; -3318134 ambiguous RAD-Tags; +4017673 recovered; -354918 low quality reads; 6045351 retained reads.\n",
      "Processing file 10 of 15 [PHap2rr_lane7_NoIndex_L007_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1494370 failed Illumina reads; -4782647 ambiguous barcodes; -3305228 ambiguous RAD-Tags; +4104806 recovered; -410521 low quality reads; 6007234 retained reads.\n",
      "Processing file 11 of 15 [PHap2rr_lane7_NoIndex_L007_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1422756 failed Illumina reads; -4743479 ambiguous barcodes; -3349364 ambiguous RAD-Tags; +3891987 recovered; -394776 low quality reads; 6089625 retained reads.\n",
      "Processing file 12 of 15 [PHap2rr_lane7_NoIndex_L007_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1899278 failed Illumina reads; -4857943 ambiguous barcodes; -3161427 ambiguous RAD-Tags; +4813193 recovered; -366728 low quality reads; 5714624 retained reads.\n",
      "Processing file 13 of 15 [PHap2rr_lane7_NoIndex_L007_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1386825 failed Illumina reads; -4740097 ambiguous barcodes; -3355731 ambiguous RAD-Tags; +3799814 recovered; -395816 low quality reads; 6121531 retained reads.\n",
      "Processing file 14 of 15 [PHap2rr_lane7_NoIndex_L007_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1882720 failed Illumina reads; -4786366 ambiguous barcodes; -3192114 ambiguous RAD-Tags; +4553515 recovered; -379263 low quality reads; 5759537 retained reads.\n",
      "Processing file 15 of 15 [PHap2rr_lane7_NoIndex_L007_R1_015.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...\n",
      "  3096382 total reads; -460699 failed Illumina reads; -931010 ambiguous barcodes; -583537 ambiguous RAD-Tags; +923079 recovered; -66931 low quality reads; 1054205 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/process_radtags.PHap2Rerun.log'\n",
      "\n",
      "227096382 total sequences\n",
      " 23405160 failed Illumina filtered reads (10.3%)\n",
      " 67718744 ambiguous barcode drops (29.8%)\n",
      "  5855051 low quality read drops (2.6%)\n",
      " 46393663 ambiguous RAD-Tag drops (20.4%)\n",
      " 83723764 retained reads (36.9%)\n"
     ]
    }
   ],
   "source": [
    "## Also rerunning process radtags on initial and resequence rounds for both libraries so that everything starts fresh from stacks 1.46\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Rerun' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Rerun' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 32 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Undetermined_S0_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...159M...160M...161M...162M...163M...164M...165M...166M...167M...168M...169M...170M...171M...172M...173M...174M...175M...176M...177M...178M...179M...180M...181M...182M...183M...184M...185M...186M...187M...188M...189M...190M...191M...192M...193M...194M...195M...196M...197M...198M...199M...200M...201M...202M...203M...204M...205M...206M...207M...208M...209M...210M...211M...212M...213M...214M...215M...216M...217M...218M...219M...220M...221M...222M...223M...224M...225M...226M...227M...228M...229M...230M...231M...232M...233M...234M...235M...236M...237M...238M...239M...240M...241M...242M...243M...244M...245M...246M...247M...248M...249M...250M...251M...252M...253M...254M...255M...256M...257M...258M...259M...260M...261M...262M...263M...264M...265M...266M...267M...268M...269M...270M...271M...272M...273M...274M...275M...276M...277M...278M...279M...280M...281M...282M...283M...284M...285M...286M...287M...288M...289M...290M...291M...292M...293M...294M...295M...296M...297M...298M...299M...300M...301M...302M...303M...304M...305M...306M...307M...308M...309M...310M...311M...312M...313M...314M...315M...316M...317M...318M...319M...320M...321M...322M...323M...324M...325M...326M...327M...328M...329M...330M...331M...332M...333M...334M...335M...336M...337M...338M...339M...340M...341M...342M...343M...344M...345M...346M...347M...348M...349M...350M...351M...352M...353M...354M...355M...356M...357M...358M...359M...360M...361M...362M...363M...364M...365M...366M...367M...368M...369M...370M...371M...372M...373M...374M...375M...376M...377M...378M...379M...380M...381M...382M...383M...384M...385M...386M...387M...388M...389M...390M...391M...392M...393M...394M...395M...396M...397M...398M...399M...400M...401M...402M...403M...404M...405M...406M...407M...408M...409M...410M...411M...412M...413M...414M...415M...416M...417M...418M...419M...420M...421M...422M...423M...424M...425M...426M...427M...428M...429M...430M...431M...432M...433M...434M...435M...436M...437M...438M...439M...440M...441M...442M...443M...444M...445M...446M...447M...448M...449M...450M...451M...452M...453M...454M...455M...456M...457M...458M...459M...460M...461M...462M...463M...464M...465M...466M...467M...468M...469M...470M...471M...472M...473M...474M...475M...476M...477M...478M...479M...480M...481M...482M...483M...484M...485M...486M...487M...488M...489M...490M...491M...492M...493M...494M...495M...496M...497M...\n",
      "  497648347 total reads; -0 failed Illumina reads; -394696801 ambiguous barcodes; -73115971 ambiguous RAD-Tags; +66959453 recovered; -124959 low quality reads; 29710616 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/SingleEndt94/process_radtags.fastq.log'\n",
      "\n",
      "497648347 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      "394696801 ambiguous barcode drops (79.3%)\n",
      "   124959 low quality read drops (0.0%)\n",
      " 73115971 ambiguous RAD-Tag drops (14.7%)\n",
      " 29710616 retained reads (6.0%)\n"
     ]
    }
   ],
   "source": [
    "## Paired end reads have two .fq along with .rem for each sample corresponding to read 1 and 2 (and reads thrown out .rem).\n",
    "## Running process_radtags with single end to only look at read 1 for comparison purposes and maybe make concatenation easier.\n",
    "! process_radtags -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R1_001.fastq.gz' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/SingleEndt94' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/Bird2014TrainingRunBarcodes.txt' -E phred33 -e sbfI -i gzfastq -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Concatenate multiple rounds of sequence - single-end and trimmed to 94 bp data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X_0101.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0013.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0014.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0015.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0016.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0017.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0018.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0019.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0020.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0021.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0021.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0022.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0022.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0023.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0002.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0003.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0005.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0007.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0009.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0010.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0011.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0012.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0012.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0013.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0014.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0015.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0016.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0017.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0018.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0019.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0020.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0023.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0024.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0024.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0024.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0025.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0026.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0026.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0026.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0027.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0027.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0027.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0029.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0029.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0029.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0031.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0031.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0031.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0032.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0032.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0033.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0033.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0034.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0034.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0035.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0035.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0035.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0038.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0038.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0038.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0039.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0039.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0039.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0040.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0040.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0040.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0041.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0041.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0041.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0047.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0047.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0047.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0049.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0050.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0050.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0050.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0053.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0053.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0053.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0056.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0056.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0056.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0058.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0058.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0058.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0059.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0059.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0059.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0062.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0066.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0066.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0066.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0067.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0067.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0067.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0069.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0070.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0073.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0075.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0075.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0075.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0078.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0078.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0078.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X101H_0101.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0101.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0104.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0104.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0001.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0001.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0002.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0003.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0004.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0004.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0004.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0005.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0006.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0006.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0006.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0007.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0008.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0009.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0010.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0011.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0012.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0012.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0013.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0014.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0015.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0016.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0017.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0018.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0019.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0020.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0021.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0021.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0021.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0022.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0022.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0022.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0023.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0024.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0024.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0025.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0026.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0026.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0027.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0027.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0028.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0028.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0028.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0029.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0029.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0030.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0030.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0030.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0031.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0031.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0032.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0032.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0032.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0033.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0033.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0034.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0034.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0045.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0045.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X_0107.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X_0107.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/PBIRD14X_0107.1.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0001.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0001.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0002.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0003.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0004.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0004.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0005.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0006.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0006.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0007.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0008.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0009.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0010.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0011.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0012.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0012.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0035.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0035.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0036.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0036.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0037.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0037.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0038.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0038.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0039.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0039.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0040.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0040.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0041.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0041.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0042.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0042.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0043.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0043.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0044.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0044.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0046.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0046.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0047.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0047.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0048.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0048.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0049.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0050.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0050.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0051.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0051.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0052.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0052.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0052.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0053.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0053.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0054.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0054.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0055.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0056.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0056.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0057.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0057.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0059.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0059.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0060.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0060.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0061.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0061.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0062.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0063.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0063.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0063.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0064.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0064.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0064.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0065.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0065.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0065.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0066.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0066.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0067.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0067.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0068.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0068.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0068.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0069.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0070.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0071.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0071.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0072.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0072.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0072.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0073.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0074.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0074.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0074.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0075.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0075.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0076.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0076.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0076.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0077.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0077.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0077.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0078.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0078.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0079.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0079.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0079.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0080.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0080.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0080.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0081.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0081.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0081.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0082.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0082.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0082.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0083.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0083.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0083.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0084.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0084.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0085.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0085.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0086.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0086.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0087.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0087.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0088.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0088.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0089.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0089.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0090.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0090.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0091.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0091.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0092.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0092.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0093.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0093.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0094.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0094.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0095.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0095.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0096.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0096.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0097.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0097.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0098.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0098.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0099.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0099.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0100.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0100.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0101.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0102.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0102.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0103.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0103.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0104.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0104.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0105.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0105.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0106.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0106.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0107.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0107.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0109.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0109.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0110.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0110.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0111.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0111.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0111.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0113.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0113.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0114.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0114.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0115.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0115.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0115.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0116.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0116.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0117.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0117.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0122.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0122.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0123.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0123.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0124.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0124.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0124.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0126.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0126.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0126.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## ustacks with single-end trimmed data - using standard parameter values for m (min depth coverage to create a stack) and M (maximum distance allowed between stacks) and enabling the deleveraging algorithm to remove highly repetitive stacks."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ustacks parameters selected:\n",
      "  Sample ID: 5000\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3059977 RAD-Tags.\n",
      "  Inserted 353231 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110646 initial stacks were populated; 242585 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.4631; Std Dev: 74.6973; Max: 11914\n",
      "Deleveraging trigger: 100; Removal trigger: 175\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4044 stacks.\n",
      "  106731 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.1421; Std Dev: 15.6575; Max: 175\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  106731 stacks merged into 90523 loci; deleveraged 596 loci; blacklisted 444 loci.\n",
      "After merging, coverage depth Mean: 28.131; Std Dev: 16.0765; Max: 287\n",
      "Merging remainder radtags\n",
      "  242585 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 201498 remainder reads; unable to match 41087 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 30.3711; Std Dev: 17.0568; Max: 310\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3018890\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz... read 3059977 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5001\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1031745 RAD-Tags.\n",
      "  Inserted 177370 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "91641 initial stacks were populated; 85729 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.3231; Std Dev: 28.5769; Max: 4881\n",
      "Deleveraging trigger: 39; Removal trigger: 67\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2516 stacks.\n",
      "  89236 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.71501; Std Dev: 5.4505; Max: 67\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  89236 stacks merged into 81740 loci; deleveraged 314 loci; blacklisted 179 loci.\n",
      "After merging, coverage depth Mean: 10.5094; Std Dev: 5.86445; Max: 107\n",
      "Merging remainder radtags\n",
      "  85729 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 70492 remainder reads; unable to match 15237 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.3749; Std Dev: 6.2311; Max: 118\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1016508\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz... read 1031745 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5002\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4374437 RAD-Tags.\n",
      "  Inserted 448121 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114369 initial stacks were populated; 333752 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 35.3302; Std Dev: 109.286; Max: 16613\n",
      "Deleveraging trigger: 145; Removal trigger: 254\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4931 stacks.\n",
      "  109572 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 33.5856; Std Dev: 23.3247; Max: 254\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109572 stacks merged into 89960 loci; deleveraged 704 loci; blacklisted 672 loci.\n",
      "After merging, coverage depth Mean: 40.3945; Std Dev: 22.8681; Max: 395\n",
      "Merging remainder radtags\n",
      "  333752 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 285797 remainder reads; unable to match 47955 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 43.6001; Std Dev: 24.2303; Max: 412\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4326482\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz... read 4374437 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5003\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 978484 RAD-Tags.\n",
      "  Inserted 181740 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "92480 initial stacks were populated; 89260 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.61531; Std Dev: 25.0123; Max: 3557\n",
      "Deleveraging trigger: 35; Removal trigger: 60\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2577 stacks.\n",
      "  90029 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.03532; Std Dev: 5.27153; Max: 60\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  90029 stacks merged into 82157 loci; deleveraged 337 loci; blacklisted 177 loci.\n",
      "After merging, coverage depth Mean: 9.80937; Std Dev: 5.68472; Max: 140\n",
      "Merging remainder radtags\n",
      "  89260 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 68549 remainder reads; unable to match 20711 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 10.6468; Std Dev: 6.07841; Max: 162\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 957773\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz... read 978484 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5004\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1956763 RAD-Tags.\n",
      "  Inserted 254123 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100295 initial stacks were populated; 153828 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.9763; Std Dev: 59.8584; Max: 10840\n",
      "Deleveraging trigger: 78; Removal trigger: 138\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3022 stacks.\n",
      "  97381 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.8841; Std Dev: 10.5931; Max: 134\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97381 stacks merged into 86320 loci; deleveraged 481 loci; blacklisted 286 loci.\n",
      "After merging, coverage depth Mean: 18.8289; Std Dev: 11.064; Max: 196\n",
      "Merging remainder radtags\n",
      "  153828 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 127085 remainder reads; unable to match 26743 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.3079; Std Dev: 11.759; Max: 223\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1930020\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz... read 1956763 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5005\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 506333 RAD-Tags.\n",
      "  Inserted 131793 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "76331 initial stacks were populated; 55462 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.90679; Std Dev: 15.2129; Max: 2259\n",
      "Deleveraging trigger: 21; Removal trigger: 36\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1943 stacks.\n",
      "  74499 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.52228; Std Dev: 3.32774; Max: 36\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  74499 stacks merged into 69751 loci; deleveraged 225 loci; blacklisted 154 loci.\n",
      "After merging, coverage depth Mean: 5.83299; Std Dev: 3.6096; Max: 67\n",
      "Merging remainder radtags\n",
      "  55462 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 34482 remainder reads; unable to match 20980 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.32923; Std Dev: 3.8724; Max: 73\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 485353\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz... read 506333 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5006\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1900416 RAD-Tags.\n",
      "  Inserted 248943 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99705 initial stacks were populated; 149238 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.5636; Std Dev: 55.7311; Max: 9852\n",
      "Deleveraging trigger: 73; Removal trigger: 129\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3035 stacks.\n",
      "  96776 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.5224; Std Dev: 10.2821; Max: 129\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96776 stacks merged into 85473 loci; deleveraged 469 loci; blacklisted 323 loci.\n",
      "After merging, coverage depth Mean: 18.4768; Std Dev: 10.7603; Max: 212\n",
      "Merging remainder radtags\n",
      "  149238 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 126592 remainder reads; unable to match 22646 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.9653; Std Dev: 11.4858; Max: 256\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1877770\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz... read 1900416 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5007\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 836692 RAD-Tags.\n",
      "  Inserted 160895 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88546 initial stacks were populated; 72349 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.63216; Std Dev: 23.124; Max: 3316\n",
      "Deleveraging trigger: 32; Removal trigger: 55\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2309 stacks.\n",
      "  86341 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.12498; Std Dev: 4.58888; Max: 54\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  86341 stacks merged into 79628 loci; deleveraged 271 loci; blacklisted 176 loci.\n",
      "After merging, coverage depth Mean: 8.72674; Std Dev: 4.95749; Max: 87\n",
      "Merging remainder radtags\n",
      "  72349 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 58669 remainder reads; unable to match 13680 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.46612; Std Dev: 5.3269; Max: 166\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 823012\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz... read 836692 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5008\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 485790 RAD-Tags.\n",
      "  Inserted 130995 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "78795 initial stacks were populated; 52200 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.50276; Std Dev: 14.329; Max: 1874\n",
      "Deleveraging trigger: 20; Removal trigger: 34\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1924 stacks.\n",
      "  76974 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.16711; Std Dev: 2.87281; Max: 34\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  76974 stacks merged into 72362 loci; deleveraged 206 loci; blacklisted 139 loci.\n",
      "After merging, coverage depth Mean: 5.44139; Std Dev: 3.09748; Max: 56\n",
      "Merging remainder radtags\n",
      "  52200 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 33634 remainder reads; unable to match 18566 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.90775; Std Dev: 3.33161; Max: 74\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 467224\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz... read 485790 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5009\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 768704 RAD-Tags.\n",
      "  Inserted 170315 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88079 initial stacks were populated; 82236 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.79378; Std Dev: 23.8505; Max: 3971\n",
      "Deleveraging trigger: 32; Removal trigger: 55\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2255 stacks.\n",
      "  85930 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.28483; Std Dev: 4.63917; Max: 54\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  85930 stacks merged into 78359 loci; deleveraged 312 loci; blacklisted 218 loci.\n",
      "After merging, coverage depth Mean: 7.89501; Std Dev: 5.02011; Max: 84\n",
      "Merging remainder radtags\n",
      "  82236 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 56892 remainder reads; unable to match 25344 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.62407; Std Dev: 5.40188; Max: 95\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 743360\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz... read 768704 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5010\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 742970 RAD-Tags.\n",
      "  Inserted 161982 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "85796 initial stacks were populated; 76186 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.77174; Std Dev: 20.8619; Max: 3276\n",
      "Deleveraging trigger: 29; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2208 stacks.\n",
      "  83693 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.26026; Std Dev: 4.60837; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83693 stacks merged into 77020 loci; deleveraged 316 loci; blacklisted 192 loci.\n",
      "After merging, coverage depth Mean: 7.80217; Std Dev: 4.95565; Max: 84\n",
      "Merging remainder radtags\n",
      "  76186 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 51673 remainder reads; unable to match 24513 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.47567; Std Dev: 5.32942; Max: 119\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 718457\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz... read 742970 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5011\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 490221 RAD-Tags.\n",
      "  Inserted 134844 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "74131 initial stacks were populated; 60713 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.79391; Std Dev: 15.5144; Max: 2642\n",
      "Deleveraging trigger: 21; Removal trigger: 37\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1799 stacks.\n",
      "  72424 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.41283; Std Dev: 3.49579; Max: 37\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  72424 stacks merged into 67430 loci; deleveraged 251 loci; blacklisted 163 loci.\n",
      "After merging, coverage depth Mean: 5.744; Std Dev: 3.80805; Max: 101\n",
      "Merging remainder radtags\n",
      "  60713 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 33090 remainder reads; unable to match 27623 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.23659; Std Dev: 4.09238; Max: 106\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 462598\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz... read 490221 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5012\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2179812 RAD-Tags.\n",
      "  Inserted 285013 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102970 initial stacks were populated; 182043 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4015; Std Dev: 55.4339; Max: 9215\n",
      "Deleveraging trigger: 75; Removal trigger: 130\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3362 stacks.\n",
      "  99743 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.2712; Std Dev: 12.7894; Max: 130\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  99743 stacks merged into 88774 loci; deleveraged 520 loci; blacklisted 353 loci.\n",
      "After merging, coverage depth Mean: 20.2917; Std Dev: 13.2624; Max: 181\n",
      "Merging remainder radtags\n",
      "  182043 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 144391 remainder reads; unable to match 37652 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.9272; Std Dev: 14.1415; Max: 189\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2142160\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz... read 2179812 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5013\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7611497 RAD-Tags.\n",
      "  Inserted 749141 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "147044 initial stacks were populated; 602097 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 47.6687; Std Dev: 158.222; Max: 25063\n",
      "Deleveraging trigger: 206; Removal trigger: 364\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6852 stacks.\n",
      "  140344 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 45.4758; Std Dev: 41.7967; Max: 364\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  140344 stacks merged into 103885 loci; deleveraged 1179 loci; blacklisted 2020 loci.\n",
      "After merging, coverage depth Mean: 60.2211; Std Dev: 40.8289; Max: 538\n",
      "Merging remainder radtags\n",
      "  602097 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 467620 remainder reads; unable to match 134477 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 64.8185; Std Dev: 43.3259; Max: 567\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7477020\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz... read 7611497 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5014\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1852808 RAD-Tags.\n",
      "  Inserted 260332 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99405 initial stacks were populated; 160927 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.0201; Std Dev: 44.5494; Max: 6704\n",
      "Deleveraging trigger: 62; Removal trigger: 106\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3123 stacks.\n",
      "  96423 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.0783; Std Dev: 9.92684; Max: 106\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96423 stacks merged into 86724 loci; deleveraged 414 loci; blacklisted 271 loci.\n",
      "After merging, coverage depth Mean: 17.6936; Std Dev: 10.2448; Max: 157\n",
      "Merging remainder radtags\n",
      "  160927 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 130329 remainder reads; unable to match 30598 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.2035; Std Dev: 10.9503; Max: 179\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1822210\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz... read 1852808 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5015\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4015590 RAD-Tags.\n",
      "  Inserted 456199 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "119820 initial stacks were populated; 336379 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.7062; Std Dev: 105.345; Max: 18819\n",
      "Deleveraging trigger: 136; Removal trigger: 241\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4855 stacks.\n",
      "  115092 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 29.036; Std Dev: 21.4321; Max: 241\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  115092 stacks merged into 95054 loci; deleveraged 724 loci; blacklisted 719 loci.\n",
      "After merging, coverage depth Mean: 34.6303; Std Dev: 20.9615; Max: 309\n",
      "Merging remainder radtags\n",
      "  336379 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 266908 remainder reads; unable to match 69471 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 37.4634; Std Dev: 22.3101; Max: 333\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3946119\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz... read 4015590 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5016\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6098729 RAD-Tags.\n",
      "  Inserted 641107 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "137160 initial stacks were populated; 503947 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 40.7902; Std Dev: 133.061; Max: 23319\n",
      "Deleveraging trigger: 174; Removal trigger: 307\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6080 stacks.\n",
      "  131241 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 38.7999; Std Dev: 33.7687; Max: 307\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  131241 stacks merged into 101358 loci; deleveraged 1079 loci; blacklisted 1472 loci.\n",
      "After merging, coverage depth Mean: 49.4223; Std Dev: 33.082; Max: 514\n",
      "Merging remainder radtags\n",
      "  503947 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 394219 remainder reads; unable to match 109728 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 53.3753; Std Dev: 35.1911; Max: 538\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5989001\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz... read 6098729 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5017\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2079077 RAD-Tags.\n",
      "  Inserted 284802 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100082 initial stacks were populated; 184720 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.928; Std Dev: 53.7126; Max: 8526\n",
      "Deleveraging trigger: 73; Removal trigger: 126\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3309 stacks.\n",
      "  96906 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.8137; Std Dev: 10.9668; Max: 126\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96906 stacks merged into 87331 loci; deleveraged 435 loci; blacklisted 257 loci.\n",
      "After merging, coverage depth Mean: 19.5678; Std Dev: 11.3908; Max: 200\n",
      "Merging remainder radtags\n",
      "  184720 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 140896 remainder reads; unable to match 43824 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.1884; Std Dev: 12.1298; Max: 217\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2035253\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz... read 2079077 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5018\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 4112 RAD-Tags.\n",
      "  Inserted 3258 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "459 initial stacks were populated; 2799 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.86057; Std Dev: 10.7957; Max: 232\n",
      "Deleveraging trigger: 14; Removal trigger: 24\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 23 stacks.\n",
      "  437 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.22706; Std Dev: 0.583647; Max: 8\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  437 stacks merged into 425 loci; deleveraged 4 loci; blacklisted 0 loci.\n",
      "After merging, coverage depth Mean: 2.29009; Std Dev: 0.820317; Max: 10\n",
      "Merging remainder radtags\n",
      "  2799 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 306 remainder reads; unable to match 2493 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.01179; Std Dev: 4.89343; Max: 77\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1619\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz... read 4112 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5019\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2030819 RAD-Tags.\n",
      "  Inserted 259829 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98522 initial stacks were populated; 161307 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.9756; Std Dev: 53.9717; Max: 7248\n",
      "Deleveraging trigger: 73; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3087 stacks.\n",
      "  95551 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.9409; Std Dev: 10.5636; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95551 stacks merged into 86137 loci; deleveraged 384 loci; blacklisted 257 loci.\n",
      "After merging, coverage depth Mean: 19.6819; Std Dev: 10.8804; Max: 212\n",
      "Merging remainder radtags\n",
      "  161307 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 131025 remainder reads; unable to match 30282 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.2097; Std Dev: 11.5593; Max: 224\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2000537\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz... read 2030819 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5020\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1553564 RAD-Tags.\n",
      "  Inserted 264585 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103917 initial stacks were populated; 160668 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.4039; Std Dev: 43.8741; Max: 7807\n",
      "Deleveraging trigger: 57; Removal trigger: 101\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2952 stacks.\n",
      "  101067 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.5618; Std Dev: 8.28762; Max: 101\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101067 stacks merged into 91534 loci; deleveraged 417 loci; blacklisted 275 loci.\n",
      "After merging, coverage depth Mean: 13.697; Std Dev: 8.67993; Max: 154\n",
      "Merging remainder radtags\n",
      "  160668 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 108339 remainder reads; unable to match 52329 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.8855; Std Dev: 9.32993; Max: 165\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1501235\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz... read 1553564 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5021\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1904041 RAD-Tags.\n",
      "  Inserted 274090 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103262 initial stacks were populated; 170828 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.7846; Std Dev: 53.4207; Max: 8922\n",
      "Deleveraging trigger: 70; Removal trigger: 124\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3038 stacks.\n",
      "  100327 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.8296; Std Dev: 10.0212; Max: 124\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100327 stacks merged into 90239 loci; deleveraged 416 loci; blacklisted 292 loci.\n",
      "After merging, coverage depth Mean: 17.3859; Std Dev: 10.4147; Max: 188\n",
      "Merging remainder radtags\n",
      "  170828 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 127702 remainder reads; unable to match 43126 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.8073; Std Dev: 11.1607; Max: 233\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1860915\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz... read 1904041 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5022\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2216770 RAD-Tags.\n",
      "  Inserted 292651 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "104551 initial stacks were populated; 188100 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4036; Std Dev: 56.7134; Max: 9707\n",
      "Deleveraging trigger: 76; Removal trigger: 133\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3305 stacks.\n",
      "  101373 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.3334; Std Dev: 11.9307; Max: 131\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101373 stacks merged into 90260 loci; deleveraged 471 loci; blacklisted 327 loci.\n",
      "After merging, coverage depth Mean: 20.3668; Std Dev: 12.3181; Max: 243\n",
      "Merging remainder radtags\n",
      "  188100 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 146806 remainder reads; unable to match 41294 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.0015; Std Dev: 13.1113; Max: 259\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2175476\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz... read 2216770 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5023\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1681593 RAD-Tags.\n",
      "  Inserted 248411 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100543 initial stacks were populated; 147868 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.2544; Std Dev: 44.2099; Max: 6938\n",
      "Deleveraging trigger: 59; Removal trigger: 104\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3032 stacks.\n",
      "  97630 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.3549; Std Dev: 8.96744; Max: 103\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97630 stacks merged into 88061 loci; deleveraged 407 loci; blacklisted 259 loci.\n",
      "After merging, coverage depth Mean: 15.7425; Std Dev: 9.34816; Max: 160\n",
      "Merging remainder radtags\n",
      "  147868 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 112100 remainder reads; unable to match 35768 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.021; Std Dev: 9.95483; Max: 177\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1645825\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz... read 1681593 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5024\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1827433 RAD-Tags.\n",
      "  Inserted 315474 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "104803 initial stacks were populated; 210671 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.4267; Std Dev: 45.4549; Max: 6408\n",
      "Deleveraging trigger: 61; Removal trigger: 106\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3173 stacks.\n",
      "  101763 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.459; Std Dev: 10.2703; Max: 106\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101763 stacks merged into 91668 loci; deleveraged 515 loci; blacklisted 307 loci.\n",
      "After merging, coverage depth Mean: 15.8519; Std Dev: 10.7908; Max: 147\n",
      "Merging remainder radtags\n",
      "  210671 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 134767 remainder reads; unable to match 75904 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.3292; Std Dev: 11.6458; Max: 184\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1751529\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz... read 1827433 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5025\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1554895 RAD-Tags.\n",
      "  Inserted 225273 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95448 initial stacks were populated; 129825 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.9303; Std Dev: 45.0373; Max: 7331\n",
      "Deleveraging trigger: 60; Removal trigger: 105\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2814 stacks.\n",
      "  92745 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.0744; Std Dev: 8.10533; Max: 105\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92745 stacks merged into 84560 loci; deleveraged 379 loci; blacklisted 228 loci.\n",
      "After merging, coverage depth Mean: 15.2736; Std Dev: 8.47389; Max: 164\n",
      "Merging remainder radtags\n",
      "  129825 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 105007 remainder reads; unable to match 24818 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.5204; Std Dev: 9.01663; Max: 176\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1530077\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz... read 1554895 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5026\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1772600 RAD-Tags.\n",
      "  Inserted 304840 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100325 initial stacks were populated; 204515 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.6301; Std Dev: 42.7244; Max: 6108\n",
      "Deleveraging trigger: 58; Removal trigger: 101\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3042 stacks.\n",
      "  97407 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.7369; Std Dev: 8.91372; Max: 101\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97407 stacks merged into 89182 loci; deleveraged 375 loci; blacklisted 220 loci.\n",
      "After merging, coverage depth Mean: 15.9434; Std Dev: 9.31206; Max: 165\n",
      "Merging remainder radtags\n",
      "  204515 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 129550 remainder reads; unable to match 74965 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.4017; Std Dev: 9.99243; Max: 179\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1697635\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz... read 1772600 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5027\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1888167 RAD-Tags.\n",
      "  Inserted 286895 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "104781 initial stacks were populated; 182114 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.2821; Std Dev: 50.5416; Max: 8742\n",
      "Deleveraging trigger: 67; Removal trigger: 117\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3343 stacks.\n",
      "  101563 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.3173; Std Dev: 9.80098; Max: 117\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101563 stacks merged into 90479 loci; deleveraged 484 loci; blacklisted 305 loci.\n",
      "After merging, coverage depth Mean: 16.9894; Std Dev: 10.1573; Max: 196\n",
      "Merging remainder radtags\n",
      "  182114 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 131066 remainder reads; unable to match 51048 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.4449; Std Dev: 10.8607; Max: 210\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1837119\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz... read 1888167 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5028\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2064354 RAD-Tags.\n",
      "  Inserted 266712 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98713 initial stacks were populated; 167999 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.2108; Std Dev: 53.8159; Max: 7855\n",
      "Deleveraging trigger: 73; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3132 stacks.\n",
      "  95697 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.1581; Std Dev: 11.2448; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95697 stacks merged into 85986 loci; deleveraged 403 loci; blacklisted 254 loci.\n",
      "After merging, coverage depth Mean: 20.0025; Std Dev: 11.5659; Max: 191\n",
      "Merging remainder radtags\n",
      "  167999 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 139402 remainder reads; unable to match 28597 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.6307; Std Dev: 12.3067; Max: 198\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2035757\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz... read 2064354 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5029\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2029055 RAD-Tags.\n",
      "  Inserted 410929 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110435 initial stacks were populated; 300494 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.6523; Std Dev: 83.3423; Max: 22883\n",
      "Deleveraging trigger: 99; Removal trigger: 182\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3065 stacks.\n",
      "  107439 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.6507; Std Dev: 10.6004; Max: 182\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107439 stacks merged into 95163 loci; deleveraged 580 loci; blacklisted 375 loci.\n",
      "After merging, coverage depth Mean: 16.2655; Std Dev: 10.8485; Max: 239\n",
      "Merging remainder radtags\n",
      "  300494 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 182666 remainder reads; unable to match 117828 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.194; Std Dev: 11.8902; Max: 255\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1911227\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz... read 2029055 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5030\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3068096 RAD-Tags.\n",
      "  Inserted 405660 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121379 initial stacks were populated; 284281 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.9349; Std Dev: 81.4306; Max: 13949\n",
      "Deleveraging trigger: 104; Removal trigger: 186\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4162 stacks.\n",
      "  117339 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.5759; Std Dev: 16.1339; Max: 185\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117339 stacks merged into 100620 loci; deleveraged 704 loci; blacklisted 559 loci.\n",
      "After merging, coverage depth Mean: 24.8065; Std Dev: 16.4598; Max: 283\n",
      "Merging remainder radtags\n",
      "  284281 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 206999 remainder reads; unable to match 77282 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 26.8778; Std Dev: 17.6346; Max: 306\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2990814\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz... read 3068096 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5031\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1953660 RAD-Tags.\n",
      "  Inserted 340210 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "106440 initial stacks were populated; 233770 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.1583; Std Dev: 51.7279; Max: 9022\n",
      "Deleveraging trigger: 68; Removal trigger: 120\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3169 stacks.\n",
      "  103377 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.2079; Std Dev: 10.4463; Max: 120\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  103377 stacks merged into 92189 loci; deleveraged 537 loci; blacklisted 332 loci.\n",
      "After merging, coverage depth Mean: 16.852; Std Dev: 10.8484; Max: 173\n",
      "Merging remainder radtags\n",
      "  233770 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 149778 remainder reads; unable to match 83992 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.4844; Std Dev: 11.6677; Max: 193\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1869668\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz... read 1953660 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5032\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2450179 RAD-Tags.\n",
      "  Inserted 421120 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111045 initial stacks were populated; 310075 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.2724; Std Dev: 57.3858; Max: 8586\n",
      "Deleveraging trigger: 77; Removal trigger: 134\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3495 stacks.\n",
      "  107665 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.1979; Std Dev: 12.7017; Max: 134\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107665 stacks merged into 95289 loci; deleveraged 613 loci; blacklisted 367 loci.\n",
      "After merging, coverage depth Mean: 20.3087; Std Dev: 13.0208; Max: 261\n",
      "Merging remainder radtags\n",
      "  310075 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 183745 remainder reads; unable to match 126330 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.2468; Std Dev: 13.9238; Max: 275\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2323849\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz... read 2450179 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5033\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2054374 RAD-Tags.\n",
      "  Inserted 370001 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111220 initial stacks were populated; 258781 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.1445; Std Dev: 82.8759; Max: 21774\n",
      "Deleveraging trigger: 99; Removal trigger: 182\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3002 stacks.\n",
      "  108290 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.0841; Std Dev: 10.5686; Max: 180\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  108290 stacks merged into 96610 loci; deleveraged 519 loci; blacklisted 380 loci.\n",
      "After merging, coverage depth Mean: 16.6134; Std Dev: 10.8816; Max: 269\n",
      "Merging remainder radtags\n",
      "  258781 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 149543 remainder reads; unable to match 109238 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.1686; Std Dev: 11.7055; Max: 282\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1945136\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz... read 2054374 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5034\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3047244 RAD-Tags.\n",
      "  Inserted 497292 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "117042 initial stacks were populated; 380250 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.7866; Std Dev: 70.8433; Max: 11614\n",
      "Deleveraging trigger: 94; Removal trigger: 164\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3981 stacks.\n",
      "  113181 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.5419; Std Dev: 15.6169; Max: 164\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  113181 stacks merged into 97587 loci; deleveraged 725 loci; blacklisted 517 loci.\n",
      "After merging, coverage depth Mean: 24.6604; Std Dev: 15.8622; Max: 252\n",
      "Merging remainder radtags\n",
      "  380250 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 231304 remainder reads; unable to match 148946 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.0462; Std Dev: 17.0698; Max: 339\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2898298\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz... read 3047244 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5035\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1704806 RAD-Tags.\n",
      "  Inserted 261857 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "105852 initial stacks were populated; 156005 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6318; Std Dev: 42.0104; Max: 6832\n",
      "Deleveraging trigger: 57; Removal trigger: 99\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3025 stacks.\n",
      "  102957 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.7113; Std Dev: 9.57485; Max: 99\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  102957 stacks merged into 92747 loci; deleveraged 517 loci; blacklisted 311 loci.\n",
      "After merging, coverage depth Mean: 15.0408; Std Dev: 10.0889; Max: 169\n",
      "Merging remainder radtags\n",
      "  156005 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 111608 remainder reads; unable to match 44397 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.2499; Std Dev: 10.7865; Max: 176\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1660409\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz... read 1704806 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5036\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1814074 RAD-Tags.\n",
      "  Inserted 268407 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102937 initial stacks were populated; 165470 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.0157; Std Dev: 50.1006; Max: 9176\n",
      "Deleveraging trigger: 66; Removal trigger: 116\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2928 stacks.\n",
      "  100121 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.0732; Std Dev: 9.57085; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100121 stacks merged into 90597 loci; deleveraged 402 loci; blacklisted 277 loci.\n",
      "After merging, coverage depth Mean: 16.4594; Std Dev: 9.99458; Max: 194\n",
      "Merging remainder radtags\n",
      "  165470 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 119664 remainder reads; unable to match 45806 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.786; Std Dev: 10.654; Max: 208\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1768268\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz... read 1814074 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5037\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1586004 RAD-Tags.\n",
      "  Inserted 239398 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99643 initial stacks were populated; 139755 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.5143; Std Dev: 46.8163; Max: 7434\n",
      "Deleveraging trigger: 61; Removal trigger: 108\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2886 stacks.\n",
      "  96872 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.6067; Std Dev: 8.33083; Max: 108\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96872 stacks merged into 88994 loci; deleveraged 356 loci; blacklisted 239 loci.\n",
      "After merging, coverage depth Mean: 14.6485; Std Dev: 8.81666; Max: 185\n",
      "Merging remainder radtags\n",
      "  139755 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 101470 remainder reads; unable to match 38285 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.7932; Std Dev: 9.39067; Max: 193\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1547719\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz... read 1586004 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5038\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1421988 RAD-Tags.\n",
      "  Inserted 290604 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102470 initial stacks were populated; 188134 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.0411; Std Dev: 40.1957; Max: 6827\n",
      "Deleveraging trigger: 52; Removal trigger: 92\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2834 stacks.\n",
      "  99731 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.2743; Std Dev: 7.11887; Max: 92\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  99731 stacks merged into 91241 loci; deleveraged 388 loci; blacklisted 233 loci.\n",
      "After merging, coverage depth Mean: 12.1819; Std Dev: 7.51352; Max: 150\n",
      "Merging remainder radtags\n",
      "  188134 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 107264 remainder reads; unable to match 80870 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.3618; Std Dev: 8.10545; Max: 165\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1341118\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz... read 1421988 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5039\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1109462 RAD-Tags.\n",
      "  Inserted 248629 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "93782 initial stacks were populated; 154847 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.1791; Std Dev: 32.2598; Max: 5131\n",
      "Deleveraging trigger: 42; Removal trigger: 75\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2340 stacks.\n",
      "  91552 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.46603; Std Dev: 7.08102; Max: 75\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91552 stacks merged into 83580 loci; deleveraged 410 loci; blacklisted 267 loci.\n",
      "After merging, coverage depth Mean: 10.224; Std Dev: 7.5145; Max: 150\n",
      "Merging remainder radtags\n",
      "  154847 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 79603 remainder reads; unable to match 75244 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.1808; Std Dev: 8.09141; Max: 152\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1034218\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz... read 1109462 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5040\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2482351 RAD-Tags.\n",
      "  Inserted 452366 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "112768 initial stacks were populated; 339598 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.0014; Std Dev: 61.5816; Max: 8788\n",
      "Deleveraging trigger: 81; Removal trigger: 142\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3778 stacks.\n",
      "  109110 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.859; Std Dev: 12.6522; Max: 142\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109110 stacks merged into 96139 loci; deleveraged 613 loci; blacklisted 380 loci.\n",
      "After merging, coverage depth Mean: 20.0179; Std Dev: 13.0536; Max: 225\n",
      "Merging remainder radtags\n",
      "  339598 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 204374 remainder reads; unable to match 135224 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.1548; Std Dev: 14.1544; Max: 248\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2347127\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz... read 2482351 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5041\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2659304 RAD-Tags.\n",
      "  Inserted 490017 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "126442 initial stacks were populated; 363575 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.1564; Std Dev: 112.966; Max: 32095\n",
      "Deleveraging trigger: 131; Removal trigger: 244\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3237 stacks.\n",
      "  123271 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.8692; Std Dev: 13.4628; Max: 240\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  123271 stacks merged into 107833 loci; deleveraged 765 loci; blacklisted 536 loci.\n",
      "After merging, coverage depth Mean: 18.895; Std Dev: 13.7489; Max: 291\n",
      "Merging remainder radtags\n",
      "  363575 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 192249 remainder reads; unable to match 171326 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.6879; Std Dev: 14.798; Max: 324\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2487978\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz... read 2659304 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5042\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1710608 RAD-Tags.\n",
      "  Inserted 294638 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103654 initial stacks were populated; 190984 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6605; Std Dev: 46.0465; Max: 7950\n",
      "Deleveraging trigger: 61; Removal trigger: 107\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3123 stacks.\n",
      "  100647 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.74; Std Dev: 9.05831; Max: 107\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100647 stacks merged into 90586 loci; deleveraged 461 loci; blacklisted 257 loci.\n",
      "After merging, coverage depth Mean: 15.0967; Std Dev: 9.44976; Max: 214\n",
      "Merging remainder radtags\n",
      "  190984 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 123697 remainder reads; unable to match 67287 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.4679; Std Dev: 10.1157; Max: 233\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1643321\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz... read 1710608 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5043\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1771836 RAD-Tags.\n",
      "  Inserted 311174 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110020 initial stacks were populated; 201154 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.2763; Std Dev: 44.2745; Max: 7186\n",
      "Deleveraging trigger: 59; Removal trigger: 103\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2975 stacks.\n",
      "  107167 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.3566; Std Dev: 9.68527; Max: 103\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107167 stacks merged into 97121 loci; deleveraged 525 loci; blacklisted 316 loci.\n",
      "After merging, coverage depth Mean: 14.5521; Std Dev: 10.2227; Max: 154\n",
      "Merging remainder radtags\n",
      "  201154 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 119363 remainder reads; unable to match 81791 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.7867; Std Dev: 10.9756; Max: 165\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1690045\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz... read 1771836 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5044\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1896594 RAD-Tags.\n",
      "  Inserted 341857 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "109799 initial stacks were populated; 232058 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.1598; Std Dev: 48.6174; Max: 9256\n",
      "Deleveraging trigger: 64; Removal trigger: 112\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3312 stacks.\n",
      "  106620 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.191; Std Dev: 11.1663; Max: 112\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  106620 stacks merged into 91057 loci; deleveraged 700 loci; blacklisted 402 loci.\n",
      "After merging, coverage depth Mean: 16.3898; Std Dev: 11.9365; Max: 195\n",
      "Merging remainder radtags\n",
      "  232058 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 139481 remainder reads; unable to match 92577 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.9306; Std Dev: 12.8159; Max: 206\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1804017\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz... read 1896594 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5045\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 867186 RAD-Tags.\n",
      "  Inserted 241301 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "89035 initial stacks were populated; 152266 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.02965; Std Dev: 23.3755; Max: 3899\n",
      "Deleveraging trigger: 31; Removal trigger: 55\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2188 stacks.\n",
      "  86972 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.47558; Std Dev: 5.87025; Max: 55\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  86972 stacks merged into 80816 loci; deleveraged 334 loci; blacklisted 224 loci.\n",
      "After merging, coverage depth Mean: 7.95235; Std Dev: 6.26989; Max: 77\n",
      "Merging remainder radtags\n",
      "  152266 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 61087 remainder reads; unable to match 91179 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.71151; Std Dev: 6.79191; Max: 87\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 776007\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz... read 867186 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5046\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 883301 RAD-Tags.\n",
      "  Inserted 263170 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "90306 initial stacks were populated; 172864 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.867; Std Dev: 25.7828; Max: 4401\n",
      "Deleveraging trigger: 34; Removal trigger: 59\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2416 stacks.\n",
      "  88028 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.29563; Std Dev: 6.24468; Max: 59\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  88028 stacks merged into 80674 loci; deleveraged 408 loci; blacklisted 249 loci.\n",
      "After merging, coverage depth Mean: 7.86127; Std Dev: 6.70958; Max: 88\n",
      "Merging remainder radtags\n",
      "  172864 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 65371 remainder reads; unable to match 107493 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.67549; Std Dev: 7.3172; Max: 93\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 775808\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz... read 883301 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5047\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2487047 RAD-Tags.\n",
      "  Inserted 416491 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "123221 initial stacks were populated; 293270 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.8036; Std Dev: 54.4893; Max: 8917\n",
      "Deleveraging trigger: 72; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3906 stacks.\n",
      "  119474 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.722; Std Dev: 13.2018; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  119474 stacks merged into 104137 loci; deleveraged 731 loci; blacklisted 450 loci.\n",
      "After merging, coverage depth Mean: 18.9351; Std Dev: 13.908; Max: 240\n",
      "Merging remainder radtags\n",
      "  293270 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 173551 remainder reads; unable to match 119719 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.6114; Std Dev: 14.9607; Max: 258\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2367328\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz... read 2487047 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5048\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1389202 RAD-Tags.\n",
      "  Inserted 336611 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "108699 initial stacks were populated; 227912 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.6835; Std Dev: 34.3522; Max: 5262\n",
      "Deleveraging trigger: 45; Removal trigger: 79\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2893 stacks.\n",
      "  105936 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.92968; Std Dev: 7.65552; Max: 79\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  105936 stacks merged into 95728 loci; deleveraged 531 loci; blacklisted 325 loci.\n",
      "After merging, coverage depth Mean: 10.8457; Std Dev: 8.17396; Max: 146\n",
      "Merging remainder radtags\n",
      "  227912 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 106953 remainder reads; unable to match 120959 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.9683; Std Dev: 8.90373; Max: 166\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1268243\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz... read 1389202 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5049\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1059219 RAD-Tags.\n",
      "  Inserted 256850 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98781 initial stacks were populated; 158069 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.12271; Std Dev: 27.8305; Max: 4344\n",
      "Deleveraging trigger: 37; Removal trigger: 65\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2466 stacks.\n",
      "  96437 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.50412; Std Dev: 6.24444; Max: 65\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96437 stacks merged into 87572 loci; deleveraged 387 loci; blacklisted 248 loci.\n",
      "After merging, coverage depth Mean: 9.24982; Std Dev: 6.71858; Max: 105\n",
      "Merging remainder radtags\n",
      "  158069 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 75966 remainder reads; unable to match 82103 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 10.121; Std Dev: 7.24917; Max: 121\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 977116\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz... read 1059219 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5050\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 397036 RAD-Tags.\n",
      "  Inserted 122399 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "65205 initial stacks were populated; 57194 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.2119; Std Dev: 12.2623; Max: 1842\n",
      "Deleveraging trigger: 17; Removal trigger: 30\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1693 stacks.\n",
      "  63624 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.86253; Std Dev: 3.03252; Max: 30\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  63624 stacks merged into 60221 loci; deleveraged 185 loci; blacklisted 137 loci.\n",
      "After merging, coverage depth Mean: 5.07809; Std Dev: 3.26686; Max: 50\n",
      "Merging remainder radtags\n",
      "  57194 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 26768 remainder reads; unable to match 30426 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.52443; Std Dev: 3.55679; Max: 87\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 366610\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz... read 397036 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5051\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1951227 RAD-Tags.\n",
      "  Inserted 333946 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114043 initial stacks were populated; 219903 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.1813; Std Dev: 87.825; Max: 24829\n",
      "Deleveraging trigger: 103; Removal trigger: 191\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3045 stacks.\n",
      "  111067 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.0564; Std Dev: 11.8719; Max: 187\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111067 stacks merged into 97148 loci; deleveraged 682 loci; blacklisted 448 loci.\n",
      "After merging, coverage depth Mean: 15.7512; Std Dev: 12.3592; Max: 231\n",
      "Merging remainder radtags\n",
      "  219903 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 135075 remainder reads; unable to match 84828 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.149; Std Dev: 13.2806; Max: 251\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1866399\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz... read 1951227 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5052\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 844278 RAD-Tags.\n",
      "  Inserted 233746 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "93112 initial stacks were populated; 140634 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.55696; Std Dev: 120.178; Max: 36029\n",
      "Deleveraging trigger: 128; Removal trigger: 248\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1430 stacks.\n",
      "  91713 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.78322; Std Dev: 5.42605; Max: 200\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91713 stacks merged into 82672 loci; deleveraged 364 loci; blacklisted 373 loci.\n",
      "After merging, coverage depth Mean: 7.29516; Std Dev: 5.40626; Max: 185\n",
      "Merging remainder radtags\n",
      "  140634 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 61581 remainder reads; unable to match 79053 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.0437; Std Dev: 5.9026; Max: 198\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 765225\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz... read 844278 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5053\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1306176 RAD-Tags.\n",
      "  Inserted 291916 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101711 initial stacks were populated; 190205 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.972; Std Dev: 56.0867; Max: 14323\n",
      "Deleveraging trigger: 67; Removal trigger: 123\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2396 stacks.\n",
      "  99379 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.162; Std Dev: 7.84484; Max: 123\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  99379 stacks merged into 88297 loci; deleveraged 490 loci; blacklisted 336 loci.\n",
      "After merging, coverage depth Mean: 11.2305; Std Dev: 8.28821; Max: 173\n",
      "Merging remainder radtags\n",
      "  190205 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 100823 remainder reads; unable to match 89382 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.3776; Std Dev: 8.99294; Max: 186\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1216794\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz... read 1306176 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5054\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2105681 RAD-Tags.\n",
      "  Inserted 380588 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "116394 initial stacks were populated; 264194 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.8212; Std Dev: 49.9445; Max: 8249\n",
      "Deleveraging trigger: 66; Removal trigger: 116\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3523 stacks.\n",
      "  112998 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.8207; Std Dev: 10.9271; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112998 stacks merged into 98578 loci; deleveraged 687 loci; blacklisted 387 loci.\n",
      "After merging, coverage depth Mean: 16.771; Std Dev: 11.4667; Max: 175\n",
      "Merging remainder radtags\n",
      "  264194 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 156899 remainder reads; unable to match 107295 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.3709; Std Dev: 12.3745; Max: 257\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1998386\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz... read 2105681 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5055\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1713947 RAD-Tags.\n",
      "  Inserted 367339 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110663 initial stacks were populated; 256676 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.1685; Std Dev: 42.2282; Max: 6772\n",
      "Deleveraging trigger: 55; Removal trigger: 98\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3182 stacks.\n",
      "  107609 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.3078; Std Dev: 9.50872; Max: 98\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107609 stacks merged into 94514 loci; deleveraged 606 loci; blacklisted 355 loci.\n",
      "After merging, coverage depth Mean: 13.8296; Std Dev: 10.0313; Max: 171\n",
      "Merging remainder radtags\n",
      "  256676 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 138019 remainder reads; unable to match 118657 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.2974; Std Dev: 10.8589; Max: 182\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1595290\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz... read 1713947 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5056\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 891113 RAD-Tags.\n",
      "  Inserted 207321 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88483 initial stacks were populated; 118838 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.72795; Std Dev: 171.18; Max: 43185\n",
      "Deleveraging trigger: 180; Removal trigger: 351\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1273 stacks.\n",
      "  87234 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.58826; Std Dev: 7.13215; Max: 341\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  87234 stacks merged into 78283 loci; deleveraged 322 loci; blacklisted 442 loci.\n",
      "After merging, coverage depth Mean: 8.03689; Std Dev: 6.09706; Max: 288\n",
      "Merging remainder radtags\n",
      "  118838 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 61263 remainder reads; unable to match 57575 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.82416; Std Dev: 6.62863; Max: 306\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 833538\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz... read 891113 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5057\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1149593 RAD-Tags.\n",
      "  Inserted 283563 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101233 initial stacks were populated; 182330 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.55482; Std Dev: 31.2712; Max: 4684\n",
      "Deleveraging trigger: 41; Removal trigger: 72\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2577 stacks.\n",
      "  98762 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.8804; Std Dev: 6.18798; Max: 72\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98762 stacks merged into 89581 loci; deleveraged 449 loci; blacklisted 251 loci.\n",
      "After merging, coverage depth Mean: 9.66742; Std Dev: 6.56704; Max: 116\n",
      "Merging remainder radtags\n",
      "  182330 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 90606 remainder reads; unable to match 91724 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 10.6829; Std Dev: 7.12912; Max: 171\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1057869\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz... read 1149593 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5058\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1843793 RAD-Tags.\n",
      "  Inserted 359514 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110144 initial stacks were populated; 249370 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.4758; Std Dev: 46.1814; Max: 8086\n",
      "Deleveraging trigger: 61; Removal trigger: 107\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3315 stacks.\n",
      "  106962 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.5537; Std Dev: 10.8221; Max: 107\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  106962 stacks merged into 93340 loci; deleveraged 637 loci; blacklisted 360 loci.\n",
      "After merging, coverage depth Mean: 15.3409; Std Dev: 11.5247; Max: 186\n",
      "Merging remainder radtags\n",
      "  249370 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 140883 remainder reads; unable to match 108487 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.8582; Std Dev: 12.4259; Max: 194\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1735306\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz... read 1843793 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5059\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 298885 RAD-Tags.\n",
      "  Inserted 99767 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "54344 initial stacks were populated; 45423 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.66403; Std Dev: 10.4604; Max: 1706\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1329 stacks.\n",
      "  53116 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.36533; Std Dev: 2.61869; Max: 26\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  53116 stacks merged into 50655 loci; deleveraged 133 loci; blacklisted 107 loci.\n",
      "After merging, coverage depth Mean: 4.52871; Std Dev: 2.85011; Max: 56\n",
      "Merging remainder radtags\n",
      "  45423 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 19891 remainder reads; unable to match 25532 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.92301; Std Dev: 3.11164; Max: 62\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 273353\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz... read 298885 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5060\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 6958 RAD-Tags.\n",
      "  Inserted 5698 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "766 initial stacks were populated; 4932 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.64491; Std Dev: 3.40201; Max: 63\n",
      "Deleveraging trigger: 6; Removal trigger: 9\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 57 stacks.\n",
      "  718 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.31876; Std Dev: 0.805447; Max: 9\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  718 stacks merged into 691 loci; deleveraged 2 loci; blacklisted 1 loci.\n",
      "After merging, coverage depth Mean: 2.37885; Std Dev: 0.954359; Max: 12\n",
      "Merging remainder radtags\n",
      "  4932 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 463 remainder reads; unable to match 4469 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.05874; Std Dev: 3.09081; Max: 45\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2489\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz... read 6958 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5061\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3081893 RAD-Tags.\n",
      "  Inserted 347536 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "107445 initial stacks were populated; 240091 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 26.4489; Std Dev: 103.673; Max: 18824\n",
      "Deleveraging trigger: 130; Removal trigger: 234\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3824 stacks.\n",
      "  103709 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.9993; Std Dev: 16.6233; Max: 232\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  103709 stacks merged into 88600 loci; deleveraged 525 loci; blacklisted 488 loci.\n",
      "After merging, coverage depth Mean: 28.8091; Std Dev: 16.2315; Max: 348\n",
      "Merging remainder radtags\n",
      "  240091 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 201060 remainder reads; unable to match 39031 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 31.0932; Std Dev: 17.2349; Max: 364\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3042862\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz... read 3081893 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5062\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4604151 RAD-Tags.\n",
      "  Inserted 508362 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "124870 initial stacks were populated; 383492 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 33.8004; Std Dev: 102.245; Max: 22016\n",
      "Deleveraging trigger: 136; Removal trigger: 238\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5427 stacks.\n",
      "  119627 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 32.0211; Std Dev: 30.9846; Max: 238\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  119627 stacks merged into 93769 loci; deleveraged 972 loci; blacklisted 1263 loci.\n",
      "After merging, coverage depth Mean: 40.0248; Std Dev: 31.8342; Max: 338\n",
      "Merging remainder radtags\n",
      "  383492 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 306580 remainder reads; unable to match 76912 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 43.3456; Std Dev: 34.0126; Max: 378\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4527239\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz... read 4604151 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5063\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1919446 RAD-Tags.\n",
      "  Inserted 260778 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95281 initial stacks were populated; 165497 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.4082; Std Dev: 55.6882; Max: 7712\n",
      "Deleveraging trigger: 74; Removal trigger: 130\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3003 stacks.\n",
      "  92398 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.3291; Std Dev: 10.8348; Max: 130\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92398 stacks merged into 83242 loci; deleveraged 394 loci; blacklisted 278 loci.\n",
      "After merging, coverage depth Mean: 19.0084; Std Dev: 11.2053; Max: 245\n",
      "Merging remainder radtags\n",
      "  165497 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 139450 remainder reads; unable to match 26047 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.6917; Std Dev: 12.0626; Max: 265\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1893399\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz... read 1919446 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5064\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4543578 RAD-Tags.\n",
      "  Inserted 472914 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "115267 initial stacks were populated; 357647 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 36.3151; Std Dev: 107.57; Max: 16956\n",
      "Deleveraging trigger: 144; Removal trigger: 251\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4624 stacks.\n",
      "  110784 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 34.6901; Std Dev: 24.3695; Max: 251\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  110784 stacks merged into 91844 loci; deleveraged 657 loci; blacklisted 726 loci.\n",
      "After merging, coverage depth Mean: 41.3119; Std Dev: 23.4913; Max: 423\n",
      "Merging remainder radtags\n",
      "  357647 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 295061 remainder reads; unable to match 62586 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 44.5552; Std Dev: 24.9063; Max: 482\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4480992\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz... read 4543578 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5065\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6825612 RAD-Tags.\n",
      "  Inserted 650052 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "136654 initial stacks were populated; 513398 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 46.1912; Std Dev: 187.723; Max: 34416\n",
      "Deleveraging trigger: 234; Removal trigger: 422\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5694 stacks.\n",
      "  131084 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 43.852; Std Dev: 37.4926; Max: 418\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  131084 stacks merged into 104275 loci; deleveraged 906 loci; blacklisted 1355 loci.\n",
      "After merging, coverage depth Mean: 54.087; Std Dev: 36.7908; Max: 490\n",
      "Merging remainder radtags\n",
      "  513398 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 402528 remainder reads; unable to match 110870 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 58.0028; Std Dev: 38.9406; Max: 515\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6714742\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz... read 6825612 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5066\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4911562 RAD-Tags.\n",
      "  Inserted 490647 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "116358 initial stacks were populated; 374289 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 38.9941; Std Dev: 146.935; Max: 24573\n",
      "Deleveraging trigger: 186; Removal trigger: 333\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4626 stacks.\n",
      "  111839 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 37.0535; Std Dev: 27.0819; Max: 331\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111839 stacks merged into 91876 loci; deleveraged 654 loci; blacklisted 839 loci.\n",
      "After merging, coverage depth Mean: 44.3275; Std Dev: 25.8628; Max: 523\n",
      "Merging remainder radtags\n",
      "  374289 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 308746 remainder reads; unable to match 65543 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 47.7229; Std Dev: 27.377; Max: 550\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4846019\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz... read 4911562 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5067\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1425539 RAD-Tags.\n",
      "  Inserted 216515 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94123 initial stacks were populated; 122392 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.8451; Std Dev: 50.4703; Max: 8345\n",
      "Deleveraging trigger: 64; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2597 stacks.\n",
      "  91610 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.0328; Std Dev: 7.56272; Max: 114\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91610 stacks merged into 83734 loci; deleveraged 315 loci; blacklisted 234 loci.\n",
      "After merging, coverage depth Mean: 14.0671; Std Dev: 7.88567; Max: 194\n",
      "Merging remainder radtags\n",
      "  122392 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 98367 remainder reads; unable to match 24025 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.2463; Std Dev: 8.41531; Max: 224\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1401514\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz... read 1425539 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5068\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6407382 RAD-Tags.\n",
      "  Inserted 618927 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "129756 initial stacks were populated; 489171 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 45.6103; Std Dev: 145.409; Max: 24193\n",
      "Deleveraging trigger: 191; Removal trigger: 336\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5764 stacks.\n",
      "  124138 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 43.6747; Std Dev: 35.2806; Max: 335\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  124138 stacks merged into 95832 loci; deleveraged 937 loci; blacklisted 1341 loci.\n",
      "After merging, coverage depth Mean: 55.64; Std Dev: 33.4168; Max: 483\n",
      "Merging remainder radtags\n",
      "  489171 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 399316 remainder reads; unable to match 89855 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 59.8725; Std Dev: 35.3787; Max: 511\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6317527\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz... read 6407382 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5069\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2933020 RAD-Tags.\n",
      "  Inserted 334889 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103691 initial stacks were populated; 231198 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 26.0565; Std Dev: 79.0776; Max: 13954\n",
      "Deleveraging trigger: 105; Removal trigger: 184\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3559 stacks.\n",
      "  100246 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.7426; Std Dev: 15.3207; Max: 183\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100246 stacks merged into 88418 loci; deleveraged 462 loci; blacklisted 363 loci.\n",
      "After merging, coverage depth Mean: 27.7492; Std Dev: 15.4456; Max: 254\n",
      "Merging remainder radtags\n",
      "  231198 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 191143 remainder reads; unable to match 40055 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 29.9227; Std Dev: 16.4106; Max: 269\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2892965\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz... read 2933020 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5070\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1600529 RAD-Tags.\n",
      "  Inserted 234550 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95419 initial stacks were populated; 139131 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.3156; Std Dev: 48.8172; Max: 9052\n",
      "Deleveraging trigger: 64; Removal trigger: 113\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2784 stacks.\n",
      "  92743 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.4462; Std Dev: 8.15225; Max: 113\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92743 stacks merged into 84476 loci; deleveraged 380 loci; blacklisted 235 loci.\n",
      "After merging, coverage depth Mean: 15.6867; Std Dev: 8.48162; Max: 157\n",
      "Merging remainder radtags\n",
      "  139131 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 114126 remainder reads; unable to match 25005 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.0431; Std Dev: 9.06901; Max: 167\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1575524\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz... read 1600529 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5071\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2689632 RAD-Tags.\n",
      "  Inserted 305197 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98817 initial stacks were populated; 206380 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.1298; Std Dev: 77.388; Max: 13476\n",
      "Deleveraging trigger: 103; Removal trigger: 180\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3357 stacks.\n",
      "  95567 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.7787; Std Dev: 13.9319; Max: 179\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95567 stacks merged into 86044 loci; deleveraged 383 loci; blacklisted 320 loci.\n",
      "After merging, coverage depth Mean: 26.0891; Std Dev: 14.1033; Max: 284\n",
      "Merging remainder radtags\n",
      "  206380 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 175256 remainder reads; unable to match 31124 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 28.1361; Std Dev: 14.97; Max: 310\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2658508\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz... read 2689632 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5072\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3369853 RAD-Tags.\n",
      "  Inserted 370431 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "105063 initial stacks were populated; 265368 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 29.5488; Std Dev: 113.087; Max: 18278\n",
      "Deleveraging trigger: 143; Removal trigger: 256\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3734 stacks.\n",
      "  101425 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 27.9825; Std Dev: 17.7199; Max: 256\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101425 stacks merged into 88813 loci; deleveraged 470 loci; blacklisted 449 loci.\n",
      "After merging, coverage depth Mean: 31.4879; Std Dev: 17.5939; Max: 340\n",
      "Merging remainder radtags\n",
      "  265368 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 219668 remainder reads; unable to match 45700 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 33.9765; Std Dev: 18.6968; Max: 366\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3324153\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz... read 3369853 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5073\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7142863 RAD-Tags.\n",
      "  Inserted 707813 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "141175 initial stacks were populated; 566638 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 46.5821; Std Dev: 188.617; Max: 34227\n",
      "Deleveraging trigger: 235; Removal trigger: 424\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5965 stacks.\n",
      "  135321 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 44.3693; Std Dev: 39.0858; Max: 423\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  135321 stacks merged into 102706 loci; deleveraged 1006 loci; blacklisted 1717 loci.\n",
      "After merging, coverage depth Mean: 57.2179; Std Dev: 37.6088; Max: 539\n",
      "Merging remainder radtags\n",
      "  566638 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 448027 remainder reads; unable to match 118611 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 61.6592; Std Dev: 39.9118; Max: 577\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7024252\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz... read 7142863 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5074\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...done\n",
      "Loaded 11307187 RAD-Tags.\n",
      "  Inserted 1126269 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "202368 initial stacks were populated; 923901 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 51.3089; Std Dev: 268.191; Max: 60988\n",
      "Deleveraging trigger: 319; Removal trigger: 588\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 8715 stacks.\n",
      "  193797 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 48.4024; Std Dev: 58.9723; Max: 584\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  193797 stacks merged into 135490 loci; deleveraged 1741 loci; blacklisted 4547 loci.\n",
      "After merging, coverage depth Mean: 66.0477; Std Dev: 61.316; Max: 602\n",
      "Merging remainder radtags\n",
      "  923901 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 637031 remainder reads; unable to match 286870 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 70.918; Std Dev: 65.2159; Max: 678\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 11020317\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz... read 11307187 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5075\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4420847 RAD-Tags.\n",
      "  Inserted 507345 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "128242 initial stacks were populated; 379103 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 31.5165; Std Dev: 118.744; Max: 24827\n",
      "Deleveraging trigger: 150; Removal trigger: 269\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4958 stacks.\n",
      "  123401 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 29.8442; Std Dev: 23.4157; Max: 268\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  123401 stacks merged into 99886 loci; deleveraged 864 loci; blacklisted 931 loci.\n",
      "After merging, coverage depth Mean: 36.2192; Std Dev: 22.9281; Max: 346\n",
      "Merging remainder radtags\n",
      "  379103 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 286991 remainder reads; unable to match 92112 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 39.1229; Std Dev: 24.3953; Max: 369\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4328735\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz... read 4420847 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5076\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5119263 RAD-Tags.\n",
      "  Inserted 580668 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "134482 initial stacks were populated; 446186 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 34.7487; Std Dev: 130.43; Max: 27745\n",
      "Deleveraging trigger: 165; Removal trigger: 296\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5369 stacks.\n",
      "  129241 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 32.9112; Std Dev: 27.3141; Max: 295\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  129241 stacks merged into 100671 loci; deleveraged 938 loci; blacklisted 1302 loci.\n",
      "After merging, coverage depth Mean: 41.4026; Std Dev: 26.4226; Max: 349\n",
      "Merging remainder radtags\n",
      "  446186 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 341058 remainder reads; unable to match 105128 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 44.8393; Std Dev: 28.1551; Max: 366\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5014135\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz... read 5119263 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5077\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...done\n",
      "Loaded 8012485 RAD-Tags.\n",
      "  Inserted 781283 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "151707 initial stacks were populated; 629576 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 48.6656; Std Dev: 184.009; Max: 35942\n",
      "Deleveraging trigger: 233; Removal trigger: 417\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 7058 stacks.\n",
      "  144797 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 46.4397; Std Dev: 43.7197; Max: 417\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  144797 stacks merged into 102828 loci; deleveraged 1251 loci; blacklisted 2600 loci.\n",
      "After merging, coverage depth Mean: 63.6737; Std Dev: 41.561; Max: 585\n",
      "Merging remainder radtags\n",
      "  629576 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 488463 remainder reads; unable to match 141113 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 68.5544; Std Dev: 44.1249; Max: 618\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7871372\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz... read 8012485 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5078\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...done\n",
      "Loaded 9213735 RAD-Tags.\n",
      "  Inserted 810326 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143200 initial stacks were populated; 667126 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 59.683; Std Dev: 202.853; Max: 35621\n",
      "Deleveraging trigger: 263; Removal trigger: 465\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 7371 stacks.\n",
      "  135979 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 57.2662; Std Dev: 51.3889; Max: 459\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  135979 stacks merged into 97045 loci; deleveraged 1060 loci; blacklisted 2451 loci.\n",
      "After merging, coverage depth Mean: 78.5335; Std Dev: 47.5394; Max: 588\n",
      "Merging remainder radtags\n",
      "  667126 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 539317 remainder reads; unable to match 127809 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 84.2439; Std Dev: 50.1805; Max: 605\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 9085926\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz... read 9213735 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5079\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2512638 RAD-Tags.\n",
      "  Inserted 322546 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "105469 initial stacks were populated; 217077 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 21.7653; Std Dev: 66.8036; Max: 11828\n",
      "Deleveraging trigger: 89; Removal trigger: 155\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3480 stacks.\n",
      "  102101 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 20.6337; Std Dev: 12.9555; Max: 153\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  102101 stacks merged into 88966 loci; deleveraged 511 loci; blacklisted 380 loci.\n",
      "After merging, coverage depth Mean: 23.371; Std Dev: 12.8622; Max: 241\n",
      "Merging remainder radtags\n",
      "  217077 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 174993 remainder reads; unable to match 42084 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 25.3489; Std Dev: 13.7473; Max: 257\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2470554\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz... read 2512638 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5080\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6037770 RAD-Tags.\n",
      "  Inserted 624757 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "134193 initial stacks were populated; 490564 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 41.3375; Std Dev: 139.399; Max: 23549\n",
      "Deleveraging trigger: 181; Removal trigger: 320\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5394 stacks.\n",
      "  128939 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 39.3008; Std Dev: 32.8999; Max: 315\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128939 stacks merged into 105800 loci; deleveraged 847 loci; blacklisted 1146 loci.\n",
      "After merging, coverage depth Mean: 47.1004; Std Dev: 32.9808; Max: 447\n",
      "Merging remainder radtags\n",
      "  490564 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 372514 remainder reads; unable to match 118050 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 50.6647; Std Dev: 35.0657; Max: 472\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5919720\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz... read 6037770 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5081\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1656408 RAD-Tags.\n",
      "  Inserted 231201 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94922 initial stacks were populated; 136279 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.0145; Std Dev: 55.3617; Max: 8234\n",
      "Deleveraging trigger: 71; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2728 stacks.\n",
      "  92284 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.0729; Std Dev: 8.7188; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92284 stacks merged into 84611 loci; deleveraged 341 loci; blacklisted 239 loci.\n",
      "After merging, coverage depth Mean: 16.2156; Std Dev: 9.08391; Max: 228\n",
      "Merging remainder radtags\n",
      "  136279 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 108851 remainder reads; unable to match 27428 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.5071; Std Dev: 9.65618; Max: 240\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1628980\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz... read 1656408 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5082\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3920230 RAD-Tags.\n",
      "  Inserted 423322 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111932 initial stacks were populated; 311390 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 32.2414; Std Dev: 122.192; Max: 19579\n",
      "Deleveraging trigger: 154; Removal trigger: 277\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4164 stacks.\n",
      "  107867 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 30.5559; Std Dev: 20.7521; Max: 277\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107867 stacks merged into 91275 loci; deleveraged 582 loci; blacklisted 623 loci.\n",
      "After merging, coverage depth Mean: 35.5514; Std Dev: 20.1329; Max: 355\n",
      "Merging remainder radtags\n",
      "  311390 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 251516 remainder reads; unable to match 59874 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 38.329; Std Dev: 21.343; Max: 383\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3860356\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz... read 3920230 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5083\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1949083 RAD-Tags.\n",
      "  Inserted 266387 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98552 initial stacks were populated; 167835 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.0742; Std Dev: 49.8637; Max: 7676\n",
      "Deleveraging trigger: 68; Removal trigger: 118\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2975 stacks.\n",
      "  95700 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.1139; Std Dev: 9.87616; Max: 118\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95700 stacks merged into 86689 loci; deleveraged 381 loci; blacklisted 252 loci.\n",
      "After merging, coverage depth Mean: 18.6881; Std Dev: 10.228; Max: 197\n",
      "Merging remainder radtags\n",
      "  167835 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 133015 remainder reads; unable to match 34820 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.2292; Std Dev: 10.9103; Max: 265\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1914263\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz... read 1949083 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5084\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5815230 RAD-Tags.\n",
      "  Inserted 560435 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "127358 initial stacks were populated; 433077 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 42.26; Std Dev: 147.621; Max: 27899\n",
      "Deleveraging trigger: 190; Removal trigger: 338\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5133 stacks.\n",
      "  122346 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 40.2691; Std Dev: 32.4043; Max: 337\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  122346 stacks merged into 98791 loci; deleveraged 746 loci; blacklisted 1055 loci.\n",
      "After merging, coverage depth Mean: 49.0188; Std Dev: 31.6725; Max: 462\n",
      "Merging remainder radtags\n",
      "  433077 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 355135 remainder reads; unable to match 77942 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 52.6569; Std Dev: 33.6038; Max: 489\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5737288\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz... read 5815230 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5085\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5136394 RAD-Tags.\n",
      "  Inserted 512901 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "122283 initial stacks were populated; 390618 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 38.8098; Std Dev: 122.022; Max: 20973\n",
      "Deleveraging trigger: 161; Removal trigger: 283\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4955 stacks.\n",
      "  117470 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 36.9897; Std Dev: 27.5257; Max: 282\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117470 stacks merged into 95026 loci; deleveraged 833 loci; blacklisted 883 loci.\n",
      "After merging, coverage depth Mean: 45.1012; Std Dev: 26.396; Max: 411\n",
      "Merging remainder radtags\n",
      "  390618 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 312800 remainder reads; unable to match 77818 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 48.4288; Std Dev: 27.8725; Max: 448\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5058576\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz... read 5136394 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5086\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 793285 RAD-Tags.\n",
      "  Inserted 155600 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "85379 initial stacks were populated; 70221 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.46887; Std Dev: 27.5428; Max: 4089\n",
      "Deleveraging trigger: 36; Removal trigger: 64\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2060 stacks.\n",
      "  83403 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.97205; Std Dev: 4.4201; Max: 64\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83403 stacks merged into 79022 loci; deleveraged 201 loci; blacklisted 176 loci.\n",
      "After merging, coverage depth Mean: 8.30623; Std Dev: 4.71073; Max: 97\n",
      "Merging remainder radtags\n",
      "  70221 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 52963 remainder reads; unable to match 17258 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.97867; Std Dev: 5.0818; Max: 160\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 776027\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz... read 793285 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5087\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1814441 RAD-Tags.\n",
      "  Inserted 243104 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95239 initial stacks were populated; 147865 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.4989; Std Dev: 51.9822; Max: 8892\n",
      "Deleveraging trigger: 69; Removal trigger: 121\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2866 stacks.\n",
      "  92478 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.5369; Std Dev: 9.34765; Max: 121\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92478 stacks merged into 84720 loci; deleveraged 309 loci; blacklisted 227 loci.\n",
      "After merging, coverage depth Mean: 17.8492; Std Dev: 9.7078; Max: 214\n",
      "Merging remainder radtags\n",
      "  147865 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 123026 remainder reads; unable to match 24839 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.3071; Std Dev: 10.3346; Max: 229\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1789602\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz... read 1814441 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5088\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1997292 RAD-Tags.\n",
      "  Inserted 276469 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99953 initial stacks were populated; 176516 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.2163; Std Dev: 65.6011; Max: 9934\n",
      "Deleveraging trigger: 84; Removal trigger: 149\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2936 stacks.\n",
      "  97098 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.1735; Std Dev: 10.2774; Max: 149\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97098 stacks merged into 87644 loci; deleveraged 375 loci; blacklisted 314 loci.\n",
      "After merging, coverage depth Mean: 18.7557; Std Dev: 10.6156; Max: 234\n",
      "Merging remainder radtags\n",
      "  176516 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 132511 remainder reads; unable to match 44005 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.2745; Std Dev: 11.2837; Max: 247\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1953287\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz... read 1997292 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5089\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 622668 RAD-Tags.\n",
      "  Inserted 142149 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "82427 initial stacks were populated; 59722 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.82963; Std Dev: 18.401; Max: 2791\n",
      "Deleveraging trigger: 25; Removal trigger: 44\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1940 stacks.\n",
      "  80585 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.4294; Std Dev: 3.72493; Max: 44\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80585 stacks merged into 76305 loci; deleveraged 212 loci; blacklisted 146 loci.\n",
      "After merging, coverage depth Mean: 6.71844; Std Dev: 3.99407; Max: 73\n",
      "Merging remainder radtags\n",
      "  59722 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 42347 remainder reads; unable to match 17375 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.27519; Std Dev: 4.26638; Max: 76\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 605293\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz... read 622668 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5090\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2493911 RAD-Tags.\n",
      "  Inserted 316013 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102533 initial stacks were populated; 213480 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.2409; Std Dev: 76.5681; Max: 12427\n",
      "Deleveraging trigger: 99; Removal trigger: 175\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3325 stacks.\n",
      "  99308 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.0153; Std Dev: 12.8196; Max: 173\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  99308 stacks merged into 88933 loci; deleveraged 436 loci; blacklisted 333 loci.\n",
      "After merging, coverage depth Mean: 23.158; Std Dev: 13.0758; Max: 278\n",
      "Merging remainder radtags\n",
      "  213480 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 167500 remainder reads; unable to match 45980 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 25.0507; Std Dev: 13.9035; Max: 298\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2447931\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz... read 2493911 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5091\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1087223 RAD-Tags.\n",
      "  Inserted 187887 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "90926 initial stacks were populated; 96961 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.8909; Std Dev: 35.687; Max: 5512\n",
      "Deleveraging trigger: 47; Removal trigger: 82\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2343 stacks.\n",
      "  88684 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.2473; Std Dev: 5.91571; Max: 81\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  88684 stacks merged into 83027 loci; deleveraged 261 loci; blacklisted 199 loci.\n",
      "After merging, coverage depth Mean: 10.8195; Std Dev: 6.27796; Max: 122\n",
      "Merging remainder radtags\n",
      "  96961 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 73882 remainder reads; unable to match 23079 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.7126; Std Dev: 6.68774; Max: 130\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1064144\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz... read 1087223 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5092\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1896600 RAD-Tags.\n",
      "  Inserted 256419 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "97152 initial stacks were populated; 159267 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.8826; Std Dev: 48.5869; Max: 7312\n",
      "Deleveraging trigger: 66; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3015 stacks.\n",
      "  94260 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.912; Std Dev: 9.61107; Max: 115\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  94260 stacks merged into 85830 loci; deleveraged 344 loci; blacklisted 246 loci.\n",
      "After merging, coverage depth Mean: 18.3871; Std Dev: 9.99528; Max: 182\n",
      "Merging remainder radtags\n",
      "  159267 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 126197 remainder reads; unable to match 33070 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.8638; Std Dev: 10.6036; Max: 189\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1863530\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz... read 1896600 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5093\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1870044 RAD-Tags.\n",
      "  Inserted 254371 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "97502 initial stacks were populated; 156869 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.5707; Std Dev: 63.1856; Max: 9762\n",
      "Deleveraging trigger: 81; Removal trigger: 144\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2797 stacks.\n",
      "  94784 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.5826; Std Dev: 9.81023; Max: 142\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  94784 stacks merged into 85964 loci; deleveraged 388 loci; blacklisted 264 loci.\n",
      "After merging, coverage depth Mean: 18.0393; Std Dev: 10.1164; Max: 210\n",
      "Merging remainder radtags\n",
      "  156869 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 126475 remainder reads; unable to match 30394 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.5164; Std Dev: 10.7586; Max: 229\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1839650\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz... read 1870044 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5094\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 576672 RAD-Tags.\n",
      "  Inserted 142211 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "81921 initial stacks were populated; 60290 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.30341; Std Dev: 22.0349; Max: 3203\n",
      "Deleveraging trigger: 28; Removal trigger: 50\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1879 stacks.\n",
      "  80120 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.89893; Std Dev: 3.35509; Max: 50\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80120 stacks merged into 75928 loci; deleveraged 203 loci; blacklisted 156 loci.\n",
      "After merging, coverage depth Mean: 6.14989; Std Dev: 3.61669; Max: 76\n",
      "Merging remainder radtags\n",
      "  60290 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 41036 remainder reads; unable to match 19254 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.69202; Std Dev: 3.93184; Max: 155\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 557418\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz... read 576672 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5095\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 345734 RAD-Tags.\n",
      "  Inserted 114093 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "66986 initial stacks were populated; 47107 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.45805; Std Dev: 11.4881; Max: 1630\n",
      "Deleveraging trigger: 16; Removal trigger: 27\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1588 stacks.\n",
      "  65487 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.18582; Std Dev: 2.27503; Max: 27\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  65487 stacks merged into 62794 loci; deleveraged 140 loci; blacklisted 117 loci.\n",
      "After merging, coverage depth Mean: 4.31912; Std Dev: 2.44578; Max: 43\n",
      "Merging remainder radtags\n",
      "  47107 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 22553 remainder reads; unable to match 24554 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.67946; Std Dev: 2.68199; Max: 84\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 321180\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz... read 345734 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5096\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3719119 RAD-Tags.\n",
      "  Inserted 626425 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "128760 initial stacks were populated; 497665 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.0191; Std Dev: 94.7504; Max: 15467\n",
      "Deleveraging trigger: 120; Removal trigger: 215\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4754 stacks.\n",
      "  124124 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.6256; Std Dev: 18.6993; Max: 211\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  124124 stacks merged into 93828 loci; deleveraged 1003 loci; blacklisted 1108 loci.\n",
      "After merging, coverage depth Mean: 30.6185; Std Dev: 18.4284; Max: 356\n",
      "Merging remainder radtags\n",
      "  497665 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 335270 remainder reads; unable to match 162395 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 34.239; Std Dev: 19.9187; Max: 438\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3556724\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz... read 3719119 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5097\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3710826 RAD-Tags.\n",
      "  Inserted 536075 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "159995 initial stacks were populated; 376080 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 20.8428; Std Dev: 88.1965; Max: 19858\n",
      "Deleveraging trigger: 109; Removal trigger: 197\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5213 stacks.\n",
      "  154916 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.4755; Std Dev: 18.775; Max: 197\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  154916 stacks merged into 94459 loci; deleveraged 1431 loci; blacklisted 3184 loci.\n",
      "After merging, coverage depth Mean: 30.3864; Std Dev: 17.9024; Max: 228\n",
      "Merging remainder radtags\n",
      "  376080 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 311442 remainder reads; unable to match 64638 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 33.8035; Std Dev: 19.3274; Max: 256\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3646188\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz... read 3710826 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5098\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 763190 RAD-Tags.\n",
      "  Inserted 207860 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "86454 initial stacks were populated; 121406 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.42342; Std Dev: 19.9638; Max: 3534\n",
      "Deleveraging trigger: 27; Removal trigger: 47\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2738 stacks.\n",
      "  83841 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.8825; Std Dev: 4.91692; Max: 47\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83841 stacks merged into 69701 loci; deleveraged 474 loci; blacklisted 361 loci.\n",
      "After merging, coverage depth Mean: 8.13062; Std Dev: 5.37231; Max: 70\n",
      "Merging remainder radtags\n",
      "  121406 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 94734 remainder reads; unable to match 26672 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.49931; Std Dev: 6.02041; Max: 86\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 736518\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz... read 763190 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5099\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3082824 RAD-Tags.\n",
      "  Inserted 468282 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "141961 initial stacks were populated; 326321 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4173; Std Dev: 68.1116; Max: 12736\n",
      "Deleveraging trigger: 88; Removal trigger: 156\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4730 stacks.\n",
      "  137372 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.2244; Std Dev: 15.9704; Max: 155\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  137372 stacks merged into 93230 loci; deleveraged 1133 loci; blacklisted 1857 loci.\n",
      "After merging, coverage depth Mean: 25.9547; Std Dev: 15.4517; Max: 190\n",
      "Merging remainder radtags\n",
      "  326321 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 268636 remainder reads; unable to match 57685 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 28.8992; Std Dev: 16.7007; Max: 211\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3025139\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz... read 3082824 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5100\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1586287 RAD-Tags.\n",
      "  Inserted 301800 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111749 initial stacks were populated; 190051 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.4944; Std Dev: 39.7975; Max: 7403\n",
      "Deleveraging trigger: 52; Removal trigger: 92\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3351 stacks.\n",
      "  108515 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7385; Std Dev: 7.93013; Max: 92\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  108515 stacks merged into 87313 loci; deleveraged 662 loci; blacklisted 488 loci.\n",
      "After merging, coverage depth Mean: 14.3723; Std Dev: 8.03375; Max: 121\n",
      "Merging remainder radtags\n",
      "  190051 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 159761 remainder reads; unable to match 30290 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.2148; Std Dev: 8.74592; Max: 150\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1555997\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz... read 1586287 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5101\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5002086 RAD-Tags.\n",
      "  Inserted 607446 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "148120 initial stacks were populated; 459326 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.6695; Std Dev: 120.787; Max: 27181\n",
      "Deleveraging trigger: 151; Removal trigger: 272\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5762 stacks.\n",
      "  142502 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.8762; Std Dev: 26.2133; Max: 272\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  142502 stacks merged into 98569 loci; deleveraged 1263 loci; blacklisted 2104 loci.\n",
      "After merging, coverage depth Mean: 40.4105; Std Dev: 25.0868; Max: 298\n",
      "Merging remainder radtags\n",
      "  459326 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 368861 remainder reads; unable to match 90465 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 44.24; Std Dev: 26.9028; Max: 320\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4911621\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz... read 5002086 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5102\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1623565 RAD-Tags.\n",
      "  Inserted 308907 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114507 initial stacks were populated; 194400 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.481; Std Dev: 40.0923; Max: 7685\n",
      "Deleveraging trigger: 53; Removal trigger: 93\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3359 stacks.\n",
      "  111265 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7479; Std Dev: 8.1716; Max: 93\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111265 stacks merged into 85130 loci; deleveraged 716 loci; blacklisted 698 loci.\n",
      "After merging, coverage depth Mean: 15.0494; Std Dev: 8.04099; Max: 117\n",
      "Merging remainder radtags\n",
      "  194400 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 171278 remainder reads; unable to match 23122 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.0808; Std Dev: 8.78024; Max: 204\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1600443\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz... read 1623565 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5103\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2896047 RAD-Tags.\n",
      "  Inserted 408188 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121649 initial stacks were populated; 286539 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 21.4511; Std Dev: 74.1222; Max: 14447\n",
      "Deleveraging trigger: 96; Removal trigger: 170\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4446 stacks.\n",
      "  117317 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 20.2031; Std Dev: 15.0121; Max: 170\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117317 stacks merged into 89692 loci; deleveraged 833 loci; blacklisted 843 loci.\n",
      "After merging, coverage depth Mean: 25.9101; Std Dev: 14.3482; Max: 241\n",
      "Merging remainder radtags\n",
      "  286539 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 243487 remainder reads; unable to match 43052 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 28.6541; Std Dev: 15.4208; Max: 259\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2852995\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz... read 2896047 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5104\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6303429 RAD-Tags.\n",
      "  Inserted 714652 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "159674 initial stacks were populated; 554978 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 36.0012; Std Dev: 125.21; Max: 21973\n",
      "Deleveraging trigger: 161; Removal trigger: 286\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6273 stacks.\n",
      "  153550 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 34.2091; Std Dev: 32.7654; Max: 286\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  153550 stacks merged into 98402 loci; deleveraged 1410 loci; blacklisted 3102 loci.\n",
      "After merging, coverage depth Mean: 51.4805; Std Dev: 30.4667; Max: 401\n",
      "Merging remainder radtags\n",
      "  554978 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 450469 remainder reads; unable to match 104509 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 56.2148; Std Dev: 32.5499; Max: 421\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6198920\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz... read 6303429 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5105\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1203467 RAD-Tags.\n",
      "  Inserted 247055 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95946 initial stacks were populated; 151109 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.9682; Std Dev: 31.8843; Max: 5845\n",
      "Deleveraging trigger: 43; Removal trigger: 75\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2956 stacks.\n",
      "  93123 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.2174; Std Dev: 7.72888; Max: 75\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  93123 stacks merged into 81175 loci; deleveraged 516 loci; blacklisted 355 loci.\n",
      "After merging, coverage depth Mean: 11.549; Std Dev: 8.26436; Max: 102\n",
      "Merging remainder radtags\n",
      "  151109 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 110886 remainder reads; unable to match 40223 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.9232; Std Dev: 9.08933; Max: 166\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1163244\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz... read 1203467 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5106\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3914709 RAD-Tags.\n",
      "  Inserted 531747 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "142489 initial stacks were populated; 389258 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 24.7419; Std Dev: 88.74; Max: 16115\n",
      "Deleveraging trigger: 113; Removal trigger: 202\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5098 stacks.\n",
      "  137518 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.3242; Std Dev: 20.0302; Max: 202\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  137518 stacks merged into 96903 loci; deleveraged 1195 loci; blacklisted 1692 loci.\n",
      "After merging, coverage depth Mean: 32.1779; Std Dev: 19.1646; Max: 215\n",
      "Merging remainder radtags\n",
      "  389258 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 314857 remainder reads; unable to match 74401 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 35.4893; Std Dev: 20.6469; Max: 316\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3840308\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz... read 3914709 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5107\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3379814 RAD-Tags.\n",
      "  Inserted 454560 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "127610 initial stacks were populated; 326950 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 23.9234; Std Dev: 81.022; Max: 13757\n",
      "Deleveraging trigger: 105; Removal trigger: 186\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4728 stacks.\n",
      "  123009 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 22.4512; Std Dev: 18.7677; Max: 185\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  123009 stacks merged into 95676 loci; deleveraged 954 loci; blacklisted 1002 loci.\n",
      "After merging, coverage depth Mean: 28.2876; Std Dev: 19.0774; Max: 246\n",
      "Merging remainder radtags\n",
      "  326950 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 257742 remainder reads; unable to match 69208 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 31.0137; Std Dev: 20.5403; Max: 264\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3310606\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz... read 3379814 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5108\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1275204 RAD-Tags.\n",
      "  Inserted 261937 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103697 initial stacks were populated; 158240 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.7714; Std Dev: 35.9146; Max: 6726\n",
      "Deleveraging trigger: 47; Removal trigger: 83\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3072 stacks.\n",
      "  100741 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.0429; Std Dev: 6.93387; Max: 83\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100741 stacks merged into 84422 loci; deleveraged 566 loci; blacklisted 397 loci.\n",
      "After merging, coverage depth Mean: 11.785; Std Dev: 7.24191; Max: 121\n",
      "Merging remainder radtags\n",
      "  158240 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 127461 remainder reads; unable to match 30779 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.304; Std Dev: 7.9361; Max: 137\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1244425\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz... read 1275204 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5109\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 260255 RAD-Tags.\n",
      "  Inserted 117227 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "52078 initial stacks were populated; 65149 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.74642; Std Dev: 9.22062; Max: 1261\n",
      "Deleveraging trigger: 13; Removal trigger: 22\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1376 stacks.\n",
      "  50773 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.51041; Std Dev: 1.84097; Max: 22\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  50773 stacks merged into 48074 loci; deleveraged 132 loci; blacklisted 88 loci.\n",
      "After merging, coverage depth Mean: 3.66735; Std Dev: 2.01912; Max: 46\n",
      "Merging remainder radtags\n",
      "  65149 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 29721 remainder reads; unable to match 35428 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.28763; Std Dev: 2.4356; Max: 73\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 224827\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz... read 260255 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5110\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1835238 RAD-Tags.\n",
      "  Inserted 322067 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110959 initial stacks were populated; 211108 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6372; Std Dev: 53.3996; Max: 8901\n",
      "Deleveraging trigger: 68; Removal trigger: 121\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3188 stacks.\n",
      "  107867 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.7938; Std Dev: 9.54429; Max: 121\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107867 stacks merged into 85343 loci; deleveraged 604 loci; blacklisted 615 loci.\n",
      "After merging, coverage depth Mean: 17.073; Std Dev: 9.17223; Max: 140\n",
      "Merging remainder radtags\n",
      "  211108 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 182247 remainder reads; unable to match 28861 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.2264; Std Dev: 9.99235; Max: 154\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1806377\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz... read 1835238 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5111\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 694417 RAD-Tags.\n",
      "  Inserted 182702 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "87040 initial stacks were populated; 95662 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.87908; Std Dev: 20.4777; Max: 3562\n",
      "Deleveraging trigger: 27; Removal trigger: 48\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2432 stacks.\n",
      "  84705 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.44133; Std Dev: 3.81094; Max: 48\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  84705 stacks merged into 77648 loci; deleveraged 282 loci; blacklisted 180 loci.\n",
      "After merging, coverage depth Mean: 6.9453; Std Dev: 4.07879; Max: 68\n",
      "Merging remainder radtags\n",
      "  95662 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 74698 remainder reads; unable to match 20964 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.91075; Std Dev: 4.51394; Max: 88\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 673453\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz... read 694417 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5112\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 275462 RAD-Tags.\n",
      "  Inserted 120774 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "51654 initial stacks were populated; 69120 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.9947; Std Dev: 8.30906; Max: 1176\n",
      "Deleveraging trigger: 12; Removal trigger: 21\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1466 stacks.\n",
      "  50297 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.73213; Std Dev: 2.11554; Max: 21\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  50297 stacks merged into 47087 loci; deleveraged 178 loci; blacklisted 105 loci.\n",
      "After merging, coverage depth Mean: 3.94302; Std Dev: 2.33208; Max: 35\n",
      "Merging remainder radtags\n",
      "  69120 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 32066 remainder reads; unable to match 37054 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.62712; Std Dev: 2.76819; Max: 43\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 238408\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz... read 275462 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5113\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 668697 RAD-Tags.\n",
      "  Inserted 165274 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "84264 initial stacks were populated; 81010 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.97435; Std Dev: 21.0159; Max: 3718\n",
      "Deleveraging trigger: 28; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2145 stacks.\n",
      "  82206 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.55506; Std Dev: 3.80895; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  82206 stacks merged into 76391 loci; deleveraged 233 loci; blacklisted 170 loci.\n",
      "After merging, coverage depth Mean: 6.9756; Std Dev: 4.09405; Max: 85\n",
      "Merging remainder radtags\n",
      "  81010 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 64091 remainder reads; unable to match 16919 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.81741; Std Dev: 4.48244; Max: 96\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 651778\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz... read 668697 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5114\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1302591 RAD-Tags.\n",
      "  Inserted 291653 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110588 initial stacks were populated; 181065 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.1415; Std Dev: 38.4471; Max: 6147\n",
      "Deleveraging trigger: 49; Removal trigger: 87\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2989 stacks.\n",
      "  107692 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.48069; Std Dev: 6.5784; Max: 87\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107692 stacks merged into 82605 loci; deleveraged 615 loci; blacklisted 708 loci.\n",
      "After merging, coverage depth Mean: 12.0583; Std Dev: 6.57393; Max: 91\n",
      "Merging remainder radtags\n",
      "  181065 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 158227 remainder reads; unable to match 22838 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.9925; Std Dev: 7.25103; Max: 130\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1279753\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz... read 1302591 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5115\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1212671 RAD-Tags.\n",
      "  Inserted 241754 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98459 initial stacks were populated; 143295 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.8611; Std Dev: 33.2779; Max: 5850\n",
      "Deleveraging trigger: 44; Removal trigger: 77\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2819 stacks.\n",
      "  95744 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.228; Std Dev: 6.30104; Max: 76\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95744 stacks merged into 83726 loci; deleveraged 427 loci; blacklisted 273 loci.\n",
      "After merging, coverage depth Mean: 11.5331; Std Dev: 6.47645; Max: 125\n",
      "Merging remainder radtags\n",
      "  143295 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 118224 remainder reads; unable to match 25071 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.9515; Std Dev: 7.06307; Max: 141\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1187600\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz... read 1212671 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5116\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1239350 RAD-Tags.\n",
      "  Inserted 238590 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "96957 initial stacks were populated; 141633 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.3217; Std Dev: 39.3885; Max: 5946\n",
      "Deleveraging trigger: 51; Removal trigger: 90\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2710 stacks.\n",
      "  94345 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.6453; Std Dev: 6.49649; Max: 90\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  94345 stacks merged into 84198 loci; deleveraged 384 loci; blacklisted 249 loci.\n",
      "After merging, coverage depth Mean: 11.7659; Std Dev: 6.69383; Max: 117\n",
      "Merging remainder radtags\n",
      "  141633 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 114476 remainder reads; unable to match 27157 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.1312; Std Dev: 7.31392; Max: 236\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1212193\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz... read 1239350 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5117\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 902867 RAD-Tags.\n",
      "  Inserted 199315 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "91448 initial stacks were populated; 107867 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.69347; Std Dev: 31.0241; Max: 4784\n",
      "Deleveraging trigger: 40; Removal trigger: 71\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2376 stacks.\n",
      "  89160 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.14547; Std Dev: 4.9087; Max: 71\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  89160 stacks merged into 81216 loci; deleveraged 314 loci; blacklisted 230 loci.\n",
      "After merging, coverage depth Mean: 8.81124; Std Dev: 5.16188; Max: 111\n",
      "Merging remainder radtags\n",
      "  107867 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 85670 remainder reads; unable to match 22197 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.87023; Std Dev: 5.65213; Max: 127\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 880670\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz... read 902867 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5118\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1047551 RAD-Tags.\n",
      "  Inserted 220253 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "93765 initial stacks were populated; 126488 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.8231; Std Dev: 32.8457; Max: 4921\n",
      "Deleveraging trigger: 43; Removal trigger: 76\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2557 stacks.\n",
      "  91300 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.23249; Std Dev: 5.62677; Max: 76\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91300 stacks merged into 81564 loci; deleveraged 317 loci; blacklisted 264 loci.\n",
      "After merging, coverage depth Mean: 10.1761; Std Dev: 5.86408; Max: 127\n",
      "Merging remainder radtags\n",
      "  126488 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 104557 remainder reads; unable to match 21931 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.4636; Std Dev: 6.42883; Max: 160\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1025620\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz... read 1047551 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5119\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1031154 RAD-Tags.\n",
      "  Inserted 214529 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "93616 initial stacks were populated; 120913 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.72313; Std Dev: 31.1092; Max: 5571\n",
      "Deleveraging trigger: 41; Removal trigger: 72\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2591 stacks.\n",
      "  91128 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.10687; Std Dev: 5.44326; Max: 72\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91128 stacks merged into 82631 loci; deleveraged 332 loci; blacklisted 200 loci.\n",
      "After merging, coverage depth Mean: 9.9239; Std Dev: 5.72995; Max: 111\n",
      "Merging remainder radtags\n",
      "  120913 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 96904 remainder reads; unable to match 24009 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.1009; Std Dev: 6.26637; Max: 122\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1007145\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz... read 1031154 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5120\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2016001 RAD-Tags.\n",
      "  Inserted 334983 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111053 initial stacks were populated; 223930 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.1371; Std Dev: 49.6815; Max: 8427\n",
      "Deleveraging trigger: 66; Removal trigger: 116\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3530 stacks.\n",
      "  107647 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.2311; Std Dev: 10.4329; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107647 stacks merged into 88110 loci; deleveraged 603 loci; blacklisted 495 loci.\n",
      "After merging, coverage depth Mean: 18.3102; Std Dev: 10.2182; Max: 167\n",
      "Merging remainder radtags\n",
      "  223930 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 186799 remainder reads; unable to match 37131 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.4452; Std Dev: 11.1205; Max: 188\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1978870\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz... read 2016001 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5121\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1568923 RAD-Tags.\n",
      "  Inserted 271328 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101652 initial stacks were populated; 169676 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.7651; Std Dev: 47.8531; Max: 7420\n",
      "Deleveraging trigger: 62; Removal trigger: 109\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2988 stacks.\n",
      "  98760 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.9389; Std Dev: 8.29948; Max: 107\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98760 stacks merged into 83665 loci; deleveraged 502 loci; blacklisted 398 loci.\n",
      "After merging, coverage depth Mean: 15.0162; Std Dev: 8.26096; Max: 145\n",
      "Merging remainder radtags\n",
      "  169676 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 146304 remainder reads; unable to match 23372 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.7753; Std Dev: 8.9702; Max: 199\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1545551\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz... read 1568923 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5122\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1559365 RAD-Tags.\n",
      "  Inserted 266430 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100108 initial stacks were populated; 166322 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.9154; Std Dev: 50.505; Max: 8502\n",
      "Deleveraging trigger: 64; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3022 stacks.\n",
      "  97175 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.0503; Std Dev: 8.32486; Max: 115\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97175 stacks merged into 85251 loci; deleveraged 422 loci; blacklisted 328 loci.\n",
      "After merging, coverage depth Mean: 14.6441; Std Dev: 8.41717; Max: 155\n",
      "Merging remainder radtags\n",
      "  166322 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 134733 remainder reads; unable to match 31589 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.2323; Std Dev: 9.11639; Max: 162\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1527776\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz... read 1559365 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5123\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 753640 RAD-Tags.\n",
      "  Inserted 170680 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "86064 initial stacks were populated; 84616 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.77356; Std Dev: 22.0022; Max: 3458\n",
      "Deleveraging trigger: 30; Removal trigger: 52\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2268 stacks.\n",
      "  83890 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.31924; Std Dev: 4.22418; Max: 52\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83890 stacks merged into 77981 loci; deleveraged 268 loci; blacklisted 165 loci.\n",
      "After merging, coverage depth Mean: 7.79086; Std Dev: 4.49695; Max: 83\n",
      "Merging remainder radtags\n",
      "  84616 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 67904 remainder reads; unable to match 16712 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.66454; Std Dev: 4.94098; Max: 144\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 736928\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz... read 753640 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5124\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1193901 RAD-Tags.\n",
      "  Inserted 227708 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94427 initial stacks were populated; 133281 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.2322; Std Dev: 33.6524; Max: 5904\n",
      "Deleveraging trigger: 45; Removal trigger: 79\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2711 stacks.\n",
      "  91811 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.5808; Std Dev: 6.28818; Max: 79\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91811 stacks merged into 81640 loci; deleveraged 349 loci; blacklisted 255 loci.\n",
      "After merging, coverage depth Mean: 11.7466; Std Dev: 6.50451; Max: 112\n",
      "Merging remainder radtags\n",
      "  133281 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 114853 remainder reads; unable to match 18428 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.1595; Std Dev: 7.09222; Max: 125\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1175473\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz... read 1193901 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5125\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2852852 RAD-Tags.\n",
      "  Inserted 464194 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121948 initial stacks were populated; 342246 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 20.5875; Std Dev: 66.3196; Max: 12308\n",
      "Deleveraging trigger: 87; Removal trigger: 153\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4644 stacks.\n",
      "  117453 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.4649; Std Dev: 14.8362; Max: 152\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117453 stacks merged into 91115 loci; deleveraged 764 loci; blacklisted 879 loci.\n",
      "After merging, coverage depth Mean: 24.6142; Std Dev: 14.3215; Max: 241\n",
      "Merging remainder radtags\n",
      "  342246 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 276520 remainder reads; unable to match 65726 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.6837; Std Dev: 15.6385; Max: 265\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2787126\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz... read 2852852 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5126\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 599052 RAD-Tags.\n",
      "  Inserted 162372 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "82840 initial stacks were populated; 79532 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.27137; Std Dev: 20.3208; Max: 2748\n",
      "Deleveraging trigger: 27; Removal trigger: 47\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2083 stacks.\n",
      "  80838 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.88135; Std Dev: 3.49327; Max: 47\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80838 stacks merged into 75255 loci; deleveraged 235 loci; blacklisted 167 loci.\n",
      "After merging, coverage depth Mean: 6.23514; Std Dev: 3.74342; Max: 96\n",
      "Merging remainder radtags\n",
      "  79532 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 59652 remainder reads; unable to match 19880 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.03042; Std Dev: 4.19372; Max: 161\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 579172\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz... read 599052 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5127\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2153734 RAD-Tags.\n",
      "  Inserted 324477 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "107301 initial stacks were populated; 217176 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.0479; Std Dev: 68.1696; Max: 10604\n",
      "Deleveraging trigger: 86; Removal trigger: 154\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3377 stacks.\n",
      "  104012 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.9957; Std Dev: 11.3124; Max: 154\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  104012 stacks merged into 86070 loci; deleveraged 569 loci; blacklisted 510 loci.\n",
      "After merging, coverage depth Mean: 20.1786; Std Dev: 10.8736; Max: 167\n",
      "Merging remainder radtags\n",
      "  217176 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 184806 remainder reads; unable to match 32370 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.3408; Std Dev: 11.7288; Max: 180\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2121364\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz... read 2153734 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5128\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4331420 RAD-Tags.\n",
      "  Inserted 559654 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143612 initial stacks were populated; 416042 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 27.2636; Std Dev: 118.108; Max: 21468\n",
      "Deleveraging trigger: 145; Removal trigger: 263\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5109 stacks.\n",
      "  138613 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 25.657; Std Dev: 22.8069; Max: 262\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  138613 stacks merged into 95128 loci; deleveraged 1044 loci; blacklisted 2033 loci.\n",
      "After merging, coverage depth Mean: 36.1221; Std Dev: 20.9158; Max: 356\n",
      "Merging remainder radtags\n",
      "  416042 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 339198 remainder reads; unable to match 76844 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 39.77; Std Dev: 22.5258; Max: 376\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4254576\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz... read 4331420 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5129\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2481542 RAD-Tags.\n",
      "  Inserted 382670 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121438 initial stacks were populated; 261232 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.2835; Std Dev: 68.769; Max: 12100\n",
      "Deleveraging trigger: 87; Removal trigger: 156\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3773 stacks.\n",
      "  117775 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.1421; Std Dev: 13.418; Max: 156\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117775 stacks merged into 90812 loci; deleveraged 757 loci; blacklisted 890 loci.\n",
      "After merging, coverage depth Mean: 21.7063; Std Dev: 13.0063; Max: 227\n",
      "Merging remainder radtags\n",
      "  261232 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 213289 remainder reads; unable to match 47943 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.0811; Std Dev: 14.0765; Max: 239\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2433599\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz... read 2481542 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5130\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2367282 RAD-Tags.\n",
      "  Inserted 367507 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "119022 initial stacks were populated; 248485 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.8017; Std Dev: 69.8902; Max: 11856\n",
      "Deleveraging trigger: 88; Removal trigger: 158\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3592 stacks.\n",
      "  115531 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.7122; Std Dev: 12.4615; Max: 158\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  115531 stacks merged into 87087 loci; deleveraged 720 loci; blacklisted 882 loci.\n",
      "After merging, coverage depth Mean: 21.6431; Std Dev: 11.6539; Max: 165\n",
      "Merging remainder radtags\n",
      "  248485 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 210547 remainder reads; unable to match 37938 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.0884; Std Dev: 12.5762; Max: 190\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2329344\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz... read 2367282 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5131\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1333709 RAD-Tags.\n",
      "  Inserted 236639 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95971 initial stacks were populated; 140668 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.4313; Std Dev: 33.6965; Max: 5138\n",
      "Deleveraging trigger: 46; Removal trigger: 80\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2910 stacks.\n",
      "  93179 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7447; Std Dev: 6.97781; Max: 80\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  93179 stacks merged into 82666 loci; deleveraged 355 loci; blacklisted 228 loci.\n",
      "After merging, coverage depth Mean: 13.0853; Std Dev: 7.20816; Max: 136\n",
      "Merging remainder radtags\n",
      "  140668 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 120416 remainder reads; unable to match 20252 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.5481; Std Dev: 7.81843; Max: 147\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1313457\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz... read 1333709 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5132\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1491573 RAD-Tags.\n",
      "  Inserted 264713 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99946 initial stacks were populated; 164767 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.2752; Std Dev: 48.862; Max: 7978\n",
      "Deleveraging trigger: 62; Removal trigger: 111\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2830 stacks.\n",
      "  97196 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.4822; Std Dev: 7.7658; Max: 111\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97196 stacks merged into 84827 loci; deleveraged 423 loci; blacklisted 314 loci.\n",
      "After merging, coverage depth Mean: 14.0889; Std Dev: 7.91207; Max: 146\n",
      "Merging remainder radtags\n",
      "  164767 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 133741 remainder reads; unable to match 31026 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.6729; Std Dev: 8.58702; Max: 169\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1460547\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz... read 1491573 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5133\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1899144 RAD-Tags.\n",
      "  Inserted 334275 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114520 initial stacks were populated; 219755 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6646; Std Dev: 49.6495; Max: 9270\n",
      "Deleveraging trigger: 64; Removal trigger: 114\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3730 stacks.\n",
      "  110918 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.7629; Std Dev: 9.7135; Max: 114\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  110918 stacks merged into 89889 loci; deleveraged 666 loci; blacklisted 591 loci.\n",
      "After merging, coverage depth Mean: 16.6688; Std Dev: 9.62736; Max: 134\n",
      "Merging remainder radtags\n",
      "  219755 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 176699 remainder reads; unable to match 43056 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.6504; Std Dev: 10.4899; Max: 146\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1856088\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz... read 1899144 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5134\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4112057 RAD-Tags.\n",
      "  Inserted 522182 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "132820 initial stacks were populated; 389362 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 28.0281; Std Dev: 117.959; Max: 20916\n",
      "Deleveraging trigger: 146; Removal trigger: 264\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4857 stacks.\n",
      "  128074 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 26.4128; Std Dev: 21.7374; Max: 260\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128074 stacks merged into 94934 loci; deleveraged 953 loci; blacklisted 1399 loci.\n",
      "After merging, coverage depth Mean: 34.6887; Std Dev: 20.073; Max: 293\n",
      "Merging remainder radtags\n",
      "  389362 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 315159 remainder reads; unable to match 74203 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 38.0621; Std Dev: 21.5889; Max: 318\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4037854\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz... read 4112057 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5135\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4679003 RAD-Tags.\n",
      "  Inserted 587341 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "140135 initial stacks were populated; 447206 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.198; Std Dev: 119.487; Max: 26462\n",
      "Deleveraging trigger: 150; Removal trigger: 269\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5379 stacks.\n",
      "  134873 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.5301; Std Dev: 24.6159; Max: 267\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  134873 stacks merged into 98161 loci; deleveraged 1087 loci; blacklisted 1627 loci.\n",
      "After merging, coverage depth Mean: 38.1711; Std Dev: 23.102; Max: 310\n",
      "Merging remainder radtags\n",
      "  447206 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 355695 remainder reads; unable to match 91511 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 41.8602; Std Dev: 24.8567; Max: 326\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4587492\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz... read 4679003 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5136\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 4223 RAD-Tags.\n",
      "  Inserted 3562 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "418 initial stacks were populated; 3144 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.58134; Std Dev: 3.15536; Max: 47\n",
      "Deleveraging trigger: 6; Removal trigger: 9\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 13 stacks.\n",
      "  410 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.25679; Std Dev: 0.742525; Max: 9\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  410 stacks merged into 401 loci; deleveraged 1 loci; blacklisted 0 loci.\n",
      "After merging, coverage depth Mean: 2.30808; Std Dev: 0.937995; Max: 11\n",
      "Merging remainder radtags\n",
      "  3144 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 182 remainder reads; unable to match 2962 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 2.76768; Std Dev: 1.94795; Max: 21\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1261\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz... read 4223 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5137\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3895374 RAD-Tags.\n",
      "  Inserted 504099 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "135941 initial stacks were populated; 368158 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.9467; Std Dev: 105.884; Max: 18152\n",
      "Deleveraging trigger: 132; Removal trigger: 238\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4806 stacks.\n",
      "  131259 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.3801; Std Dev: 20.8246; Max: 235\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  131259 stacks merged into 91974 loci; deleveraged 929 loci; blacklisted 1632 loci.\n",
      "After merging, coverage depth Mean: 33.7412; Std Dev: 19.0732; Max: 270\n",
      "Merging remainder radtags\n",
      "  368158 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 303998 remainder reads; unable to match 64160 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 37.1108; Std Dev: 20.4439; Max: 282\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3831214\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz... read 3895374 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5138\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1553798 RAD-Tags.\n",
      "  Inserted 267297 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101411 initial stacks were populated; 165886 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.686; Std Dev: 43.9009; Max: 6049\n",
      "Deleveraging trigger: 58; Removal trigger: 101\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2995 stacks.\n",
      "  98517 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.8775; Std Dev: 8.57162; Max: 101\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98517 stacks merged into 85892 loci; deleveraged 487 loci; blacklisted 305 loci.\n",
      "After merging, coverage depth Mean: 14.5621; Std Dev: 8.78328; Max: 148\n",
      "Merging remainder radtags\n",
      "  165886 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 137456 remainder reads; unable to match 28430 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.1701; Std Dev: 9.53979; Max: 155\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1525368\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz... read 1553798 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5139\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1896666 RAD-Tags.\n",
      "  Inserted 312963 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "109379 initial stacks were populated; 203584 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.479; Std Dev: 46.8864; Max: 7783\n",
      "Deleveraging trigger: 62; Removal trigger: 109\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3488 stacks.\n",
      "  106028 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.557; Std Dev: 9.98119; Max: 108\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  106028 stacks merged into 86286 loci; deleveraged 578 loci; blacklisted 518 loci.\n",
      "After merging, coverage depth Mean: 17.5979; Std Dev: 9.80166; Max: 135\n",
      "Merging remainder radtags\n",
      "  203584 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 172843 remainder reads; unable to match 30741 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.6163; Std Dev: 10.6401; Max: 143\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1865925\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz... read 1896666 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5140\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2764667 RAD-Tags.\n",
      "  Inserted 403461 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "117398 initial stacks were populated; 286063 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 21.1128; Std Dev: 88.4809; Max: 15607\n",
      "Deleveraging trigger: 110; Removal trigger: 198\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3964 stacks.\n",
      "  113521 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.8342; Std Dev: 14.5182; Max: 197\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  113521 stacks merged into 89114 loci; deleveraged 707 loci; blacklisted 788 loci.\n",
      "After merging, coverage depth Mean: 24.7053; Std Dev: 13.4974; Max: 207\n",
      "Merging remainder radtags\n",
      "  286063 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 239007 remainder reads; unable to match 47056 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.414; Std Dev: 14.6485; Max: 328\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2717611\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz... read 2764667 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5141\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5096752 RAD-Tags.\n",
      "  Inserted 658811 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "153255 initial stacks were populated; 505556 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 29.9579; Std Dev: 114.796; Max: 23027\n",
      "Deleveraging trigger: 145; Removal trigger: 260\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5810 stacks.\n",
      "  147583 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.2726; Std Dev: 26.8445; Max: 257\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  147583 stacks merged into 108157 loci; deleveraged 1255 loci; blacklisted 1963 loci.\n",
      "After merging, coverage depth Mean: 37.4053; Std Dev: 26.5335; Max: 463\n",
      "Merging remainder radtags\n",
      "  505556 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 373075 remainder reads; unable to match 132481 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 40.923; Std Dev: 28.6129; Max: 492\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4964271\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz... read 5096752 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5142\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6508843 RAD-Tags.\n",
      "  Inserted 709441 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "147534 initial stacks were populated; 561907 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 40.3089; Std Dev: 139.292; Max: 25625\n",
      "Deleveraging trigger: 180; Removal trigger: 319\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6401 stacks.\n",
      "  141283 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 38.3861; Std Dev: 35.2331; Max: 319\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  141283 stacks merged into 96043 loci; deleveraged 1144 loci; blacklisted 2482 loci.\n",
      "After merging, coverage depth Mean: 54.7761; Std Dev: 32.107; Max: 403\n",
      "Merging remainder radtags\n",
      "  561907 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 458322 remainder reads; unable to match 103585 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 59.6826; Std Dev: 34.3076; Max: 421\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6405258\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz... read 6508843 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5143\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2167327 RAD-Tags.\n",
      "  Inserted 352872 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "115801 initial stacks were populated; 237071 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.6687; Std Dev: 49.2712; Max: 8111\n",
      "Deleveraging trigger: 66; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3640 stacks.\n",
      "  112285 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.7721; Std Dev: 11.3; Max: 115\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112285 stacks merged into 87323 loci; deleveraged 689 loci; blacklisted 667 loci.\n",
      "After merging, coverage depth Mean: 19.9049; Std Dev: 10.8075; Max: 176\n",
      "Merging remainder radtags\n",
      "  237071 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 200206 remainder reads; unable to match 36865 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.2186; Std Dev: 11.7083; Max: 193\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2130462\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz... read 2167327 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5144\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4120581 RAD-Tags.\n",
      "  Inserted 530652 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "132617 initial stacks were populated; 398035 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 28.0699; Std Dev: 122.732; Max: 22011\n",
      "Deleveraging trigger: 151; Removal trigger: 274\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4709 stacks.\n",
      "  128006 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 26.4284; Std Dev: 21.7059; Max: 272\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128006 stacks merged into 98171 loci; deleveraged 884 loci; blacklisted 1173 loci.\n",
      "After merging, coverage depth Mean: 33.6433; Std Dev: 20.6201; Max: 328\n",
      "Merging remainder radtags\n",
      "  398035 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 311826 remainder reads; unable to match 86209 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 36.8613; Std Dev: 22.1914; Max: 359\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4034372\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz... read 4120581 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5145\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5980737 RAD-Tags.\n",
      "  Inserted 684038 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "153132 initial stacks were populated; 530906 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 35.5891; Std Dev: 134.919; Max: 28034\n",
      "Deleveraging trigger: 171; Removal trigger: 305\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6235 stacks.\n",
      "  147045 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 33.6914; Std Dev: 31.9874; Max: 301\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  147045 stacks merged into 99247 loci; deleveraged 1210 loci; blacklisted 2678 loci.\n",
      "After merging, coverage depth Mean: 48.2238; Std Dev: 29.8734; Max: 359\n",
      "Merging remainder radtags\n",
      "  530906 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 420637 remainder reads; unable to match 110269 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 52.5863; Std Dev: 31.9573; Max: 373\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5870468\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz... read 5980737 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5146\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3835136 RAD-Tags.\n",
      "  Inserted 481311 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "123696 initial stacks were populated; 357615 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 28.1134; Std Dev: 109.236; Max: 18584\n",
      "Deleveraging trigger: 137; Removal trigger: 247\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4562 stacks.\n",
      "  119236 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 26.5693; Std Dev: 20.263; Max: 246\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  119236 stacks merged into 91867 loci; deleveraged 793 loci; blacklisted 996 loci.\n",
      "After merging, coverage depth Mean: 33.7353; Std Dev: 18.6887; Max: 328\n",
      "Merging remainder radtags\n",
      "  357615 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 295414 remainder reads; unable to match 62201 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 36.9899; Std Dev: 20.1072; Max: 385\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3772935\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz... read 3835136 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5147\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1895004 RAD-Tags.\n",
      "  Inserted 325940 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111724 initial stacks were populated; 214216 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.0441; Std Dev: 50.751; Max: 8030\n",
      "Deleveraging trigger: 66; Removal trigger: 117\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3409 stacks.\n",
      "  108424 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.1436; Std Dev: 10.2125; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  108424 stacks merged into 88510 loci; deleveraged 667 loci; blacklisted 593 loci.\n",
      "After merging, coverage depth Mean: 16.9732; Std Dev: 10.1471; Max: 127\n",
      "Merging remainder radtags\n",
      "  214216 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 175569 remainder reads; unable to match 38647 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.9727; Std Dev: 11.0421; Max: 135\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1856357\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz... read 1895004 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5148\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5683067 RAD-Tags.\n",
      "  Inserted 657970 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "146837 initial stacks were populated; 511133 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 35.2223; Std Dev: 135.524; Max: 29735\n",
      "Deleveraging trigger: 171; Removal trigger: 306\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6114 stacks.\n",
      "  140853 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 33.3666; Std Dev: 30.4553; Max: 306\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  140853 stacks merged into 99044 loci; deleveraged 1157 loci; blacklisted 2110 loci.\n",
      "After merging, coverage depth Mean: 46.1023; Std Dev: 28.6098; Max: 324\n",
      "Merging remainder radtags\n",
      "  511133 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 408158 remainder reads; unable to match 102975 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 50.3187; Std Dev: 30.7076; Max: 422\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5580092\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz... read 5683067 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5149\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2639636 RAD-Tags.\n",
      "  Inserted 353907 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "107472 initial stacks were populated; 246435 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.2681; Std Dev: 80.8767; Max: 13567\n",
      "Deleveraging trigger: 103; Removal trigger: 184\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3558 stacks.\n",
      "  104005 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.0395; Std Dev: 13.65; Max: 184\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  104005 stacks merged into 87174 loci; deleveraged 582 loci; blacklisted 483 loci.\n",
      "After merging, coverage depth Mean: 24.6901; Std Dev: 12.9824; Max: 221\n",
      "Merging remainder radtags\n",
      "  246435 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 206702 remainder reads; unable to match 39733 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.0769; Std Dev: 13.9117; Max: 236\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2599903\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz... read 2639636 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5150\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2467187 RAD-Tags.\n",
      "  Inserted 363074 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "113842 initial stacks were populated; 249232 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4827; Std Dev: 73.9901; Max: 12625\n",
      "Deleveraging trigger: 93; Removal trigger: 167\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3438 stacks.\n",
      "  110495 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.3734; Std Dev: 13.1227; Max: 167\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  110495 stacks merged into 88026 loci; deleveraged 666 loci; blacklisted 689 loci.\n",
      "After merging, coverage depth Mean: 22.5731; Std Dev: 12.3941; Max: 192\n",
      "Merging remainder radtags\n",
      "  249232 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 211417 remainder reads; unable to match 37815 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.9963; Std Dev: 13.3861; Max: 243\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2429372\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz... read 2467187 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5151\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7233016 RAD-Tags.\n",
      "  Inserted 774730 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "160090 initial stacks were populated; 614640 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 41.3416; Std Dev: 189.588; Max: 36908\n",
      "Deleveraging trigger: 231; Removal trigger: 421\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6450 stacks.\n",
      "  153743 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 39.148; Std Dev: 39.1268; Max: 418\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  153743 stacks merged into 97386 loci; deleveraged 1281 loci; blacklisted 3494 loci.\n",
      "After merging, coverage depth Mean: 59.2434; Std Dev: 35.2355; Max: 456\n",
      "Merging remainder radtags\n",
      "  614640 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 497250 remainder reads; unable to match 117390 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 64.5452; Std Dev: 37.6391; Max: 456\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7115626\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz... read 7233016 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5152\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4714496 RAD-Tags.\n",
      "  Inserted 596188 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143427 initial stacks were populated; 452761 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 29.7136; Std Dev: 131.396; Max: 24672\n",
      "Deleveraging trigger: 161; Removal trigger: 293\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5208 stacks.\n",
      "  138334 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.0044; Std Dev: 25.0002; Max: 293\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  138334 stacks merged into 98384 loci; deleveraged 1012 loci; blacklisted 1928 loci.\n",
      "After merging, coverage depth Mean: 38.082; Std Dev: 23.3266; Max: 335\n",
      "Merging remainder radtags\n",
      "  452761 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 360419 remainder reads; unable to match 92342 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 41.8231; Std Dev: 25.1187; Max: 382\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4622154\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz... read 4714496 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5153\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4140989 RAD-Tags.\n",
      "  Inserted 500613 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "125033 initial stacks were populated; 375580 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.1153; Std Dev: 99.9064; Max: 18720\n",
      "Deleveraging trigger: 130; Removal trigger: 230\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4945 stacks.\n",
      "  120220 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.5077; Std Dev: 22.5491; Max: 230\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  120220 stacks merged into 93103 loci; deleveraged 847 loci; blacklisted 1012 loci.\n",
      "After merging, coverage depth Mean: 36.0878; Std Dev: 21.4014; Max: 247\n",
      "Merging remainder radtags\n",
      "  375580 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 307689 remainder reads; unable to match 67891 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 39.4338; Std Dev: 22.9313; Max: 344\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4073098\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz... read 4140989 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5154\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2627262 RAD-Tags.\n",
      "  Inserted 388524 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "116085 initial stacks were populated; 272439 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 20.2853; Std Dev: 72.6414; Max: 11107\n",
      "Deleveraging trigger: 93; Removal trigger: 166\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3997 stacks.\n",
      "  112206 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.0887; Std Dev: 13.9354; Max: 165\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112206 stacks merged into 90159 loci; deleveraged 701 loci; blacklisted 672 loci.\n",
      "After merging, coverage depth Mean: 23.3183; Std Dev: 13.5169; Max: 187\n",
      "Merging remainder radtags\n",
      "  272439 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 223555 remainder reads; unable to match 48884 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 25.8197; Std Dev: 14.6014; Max: 206\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2578378\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz... read 2627262 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5155\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6540120 RAD-Tags.\n",
      "  Inserted 721798 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "154107 initial stacks were populated; 567691 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 38.7551; Std Dev: 182.968; Max: 34767\n",
      "Deleveraging trigger: 222; Removal trigger: 405\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6250 stacks.\n",
      "  147975 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 36.5664; Std Dev: 35.1331; Max: 404\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  147975 stacks merged into 98808 loci; deleveraged 1178 loci; blacklisted 2863 loci.\n",
      "After merging, coverage depth Mean: 52.7945; Std Dev: 32.316; Max: 382\n",
      "Merging remainder radtags\n",
      "  567691 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 453004 remainder reads; unable to match 114687 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 57.5218; Std Dev: 34.559; Max: 472\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6425433\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz... read 6540120 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5156\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1307424 RAD-Tags.\n",
      "  Inserted 295472 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "112919 initial stacks were populated; 182553 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.96175; Std Dev: 29.435; Max: 4891\n",
      "Deleveraging trigger: 39; Removal trigger: 69\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3244 stacks.\n",
      "  109800 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.35522; Std Dev: 6.59143; Max: 69\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109800 stacks merged into 84914 loci; deleveraged 680 loci; blacklisted 634 loci.\n",
      "After merging, coverage depth Mean: 11.8486; Std Dev: 6.76669; Max: 98\n",
      "Merging remainder radtags\n",
      "  182553 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 154303 remainder reads; unable to match 28250 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.6822; Std Dev: 7.47306; Max: 132\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1279174\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz... read 1307424 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5157\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 956204 RAD-Tags.\n",
      "  Inserted 245955 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101342 initial stacks were populated; 144613 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.00844; Std Dev: 31.355; Max: 5324\n",
      "Deleveraging trigger: 39; Removal trigger: 71\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2688 stacks.\n",
      "  98742 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.44267; Std Dev: 4.91297; Max: 70\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98742 stacks merged into 82154 loci; deleveraged 509 loci; blacklisted 408 loci.\n",
      "After merging, coverage depth Mean: 8.75923; Std Dev: 5.11782; Max: 81\n",
      "Merging remainder radtags\n",
      "  144613 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 119453 remainder reads; unable to match 25160 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 10.2221; Std Dev: 5.72484; Max: 115\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 931044\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz... read 956204 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5158\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 802147 RAD-Tags.\n",
      "  Inserted 219034 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94331 initial stacks were populated; 124703 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.18156; Std Dev: 26.7488; Max: 4199\n",
      "Deleveraging trigger: 34; Removal trigger: 61\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2362 stacks.\n",
      "  92046 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.69084; Std Dev: 4.26211; Max: 61\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92046 stacks merged into 78987 loci; deleveraged 384 loci; blacklisted 355 loci.\n",
      "After merging, coverage depth Mean: 7.64426; Std Dev: 4.48893; Max: 75\n",
      "Merging remainder radtags\n",
      "  124703 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 102402 remainder reads; unable to match 22301 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.94783; Std Dev: 5.02838; Max: 131\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 779846\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz... read 802147 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5159\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1634885 RAD-Tags.\n",
      "  Inserted 326835 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "119647 initial stacks were populated; 207188 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.9326; Std Dev: 42.4978; Max: 7329\n",
      "Deleveraging trigger: 54; Removal trigger: 97\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3293 stacks.\n",
      "  116470 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.1749; Std Dev: 8.38054; Max: 95\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  116470 stacks merged into 83229 loci; deleveraged 703 loci; blacklisted 973 loci.\n",
      "After merging, coverage depth Mean: 15.1971; Std Dev: 8.13582; Max: 118\n",
      "Merging remainder radtags\n",
      "  207188 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 182945 remainder reads; unable to match 24243 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.4244; Std Dev: 8.86095; Max: 163\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1610642\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz... read 1634885 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5160\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 859782 RAD-Tags.\n",
      "  Inserted 226459 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95450 initial stacks were populated; 131009 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.63513; Std Dev: 23.3481; Max: 3242\n",
      "Deleveraging trigger: 31; Removal trigger: 54\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2591 stacks.\n",
      "  92967 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.12536; Std Dev: 4.76055; Max: 54\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92967 stacks merged into 78612 loci; deleveraged 430 loci; blacklisted 333 loci.\n",
      "After merging, coverage depth Mean: 8.29109; Std Dev: 5.11268; Max: 70\n",
      "Merging remainder radtags\n",
      "  131009 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 108556 remainder reads; unable to match 22453 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.67979; Std Dev: 5.74457; Max: 146\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 837329\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz... read 859782 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5161\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 564008 RAD-Tags.\n",
      "  Inserted 177164 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "83003 initial stacks were populated; 94161 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.6606; Std Dev: 19.6591; Max: 3134\n",
      "Deleveraging trigger: 25; Removal trigger: 45\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2110 stacks.\n",
      "  80973 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.2691; Std Dev: 3.23826; Max: 45\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80973 stacks merged into 72353 loci; deleveraged 282 loci; blacklisted 213 loci.\n",
      "After merging, coverage depth Mean: 5.80332; Std Dev: 3.51499; Max: 84\n",
      "Merging remainder radtags\n",
      "  94161 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 72158 remainder reads; unable to match 22003 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.80468; Std Dev: 4.00524; Max: 90\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 542005\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz... read 564008 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5162\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 660511 RAD-Tags.\n",
      "  Inserted 197572 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88463 initial stacks were populated; 109109 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.23314; Std Dev: 20.9213; Max: 3312\n",
      "Deleveraging trigger: 27; Removal trigger: 48\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2418 stacks.\n",
      "  86144 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.79288; Std Dev: 3.6575; Max: 48\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  86144 stacks merged into 75893 loci; deleveraged 326 loci; blacklisted 247 loci.\n",
      "After merging, coverage depth Mean: 6.47576; Std Dev: 3.97283; Max: 98\n",
      "Merging remainder radtags\n",
      "  109109 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 84289 remainder reads; unable to match 24820 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.59147; Std Dev: 4.48103; Max: 123\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 635691\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz... read 660511 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5163\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2361466 RAD-Tags.\n",
      "  Inserted 354753 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114950 initial stacks were populated; 239803 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.4573; Std Dev: 54.4792; Max: 9208\n",
      "Deleveraging trigger: 73; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3830 stacks.\n",
      "  111264 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.4266; Std Dev: 12.6506; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111264 stacks merged into 85882 loci; deleveraged 702 loci; blacklisted 713 loci.\n",
      "After merging, coverage depth Mean: 22.1706; Std Dev: 12.037; Max: 183\n",
      "Merging remainder radtags\n",
      "  239803 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 208888 remainder reads; unable to match 30915 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.6274; Std Dev: 12.9712; Max: 203\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2330551\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz... read 2361466 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5164\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 727486 RAD-Tags.\n",
      "  Inserted 207800 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "92037 initial stacks were populated; 115763 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.64649; Std Dev: 20.4026; Max: 3792\n",
      "Deleveraging trigger: 27; Removal trigger: 47\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2490 stacks.\n",
      "  89650 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.21273; Std Dev: 3.86293; Max: 47\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  89650 stacks merged into 77785 loci; deleveraged 369 loci; blacklisted 292 loci.\n",
      "After merging, coverage depth Mean: 7.04631; Std Dev: 4.15514; Max: 87\n",
      "Merging remainder radtags\n",
      "  115763 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 93930 remainder reads; unable to match 21833 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.26003; Std Dev: 4.67995; Max: 117\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 705653\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz... read 727486 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5165\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1204794 RAD-Tags.\n",
      "  Inserted 277751 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "108361 initial stacks were populated; 169390 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.55514; Std Dev: 28.5506; Max: 5136\n",
      "Deleveraging trigger: 38; Removal trigger: 67\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3060 stacks.\n",
      "  105417 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.95443; Std Dev: 6.26871; Max: 67\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  105417 stacks merged into 82880 loci; deleveraged 581 loci; blacklisted 607 loci.\n",
      "After merging, coverage depth Mean: 11.1367; Std Dev: 6.47252; Max: 86\n",
      "Merging remainder radtags\n",
      "  169390 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 143897 remainder reads; unable to match 25493 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.8882; Std Dev: 7.14236; Max: 98\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1179301\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz... read 1204794 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5166\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2025667 RAD-Tags.\n",
      "  Inserted 344768 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "115923 initial stacks were populated; 228845 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.5001; Std Dev: 64.3174; Max: 11570\n",
      "Deleveraging trigger: 80; Removal trigger: 144\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3511 stacks.\n",
      "  112508 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.5131; Std Dev: 10.4138; Max: 143\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112508 stacks merged into 85474 loci; deleveraged 685 loci; blacklisted 836 loci.\n",
      "After merging, coverage depth Mean: 18.6353; Std Dev: 9.69305; Max: 141\n",
      "Merging remainder radtags\n",
      "  228845 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 199288 remainder reads; unable to match 29557 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.9926; Std Dev: 10.5318; Max: 156\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1996110\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz... read 2025667 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5167\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1062265 RAD-Tags.\n",
      "  Inserted 261092 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "104448 initial stacks were populated; 156644 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.67054; Std Dev: 26.1257; Max: 4593\n",
      "Deleveraging trigger: 35; Removal trigger: 61\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2989 stacks.\n",
      "  101580 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.13468; Std Dev: 5.33628; Max: 61\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101580 stacks merged into 82009 loci; deleveraged 539 loci; blacklisted 459 loci.\n",
      "After merging, coverage depth Mean: 9.8959; Std Dev: 5.53738; Max: 71\n",
      "Merging remainder radtags\n",
      "  156644 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 134242 remainder reads; unable to match 22402 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.5445; Std Dev: 6.15543; Max: 132\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1039863\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz... read 1062265 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5168\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1334222 RAD-Tags.\n",
      "  Inserted 301046 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "112904 initial stacks were populated; 188142 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.1509; Std Dev: 30.6625; Max: 5778\n",
      "Deleveraging trigger: 41; Removal trigger: 71\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3330 stacks.\n",
      "  109709 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.53168; Std Dev: 6.73823; Max: 71\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109709 stacks merged into 83803 loci; deleveraged 710 loci; blacklisted 669 loci.\n",
      "After merging, coverage depth Mean: 12.2179; Std Dev: 6.83783; Max: 79\n",
      "Merging remainder radtags\n",
      "  188142 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 160993 remainder reads; unable to match 27149 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.1576; Std Dev: 7.53667; Max: 141\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1307073\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz... read 1334222 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5169\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 230003 RAD-Tags.\n",
      "  Inserted 108588 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "45962 initial stacks were populated; 62626 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.64164; Std Dev: 7.58952; Max: 867\n",
      "Deleveraging trigger: 11; Removal trigger: 19\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1266 stacks.\n",
      "  44789 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.40127; Std Dev: 1.77912; Max: 19\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  44789 stacks merged into 42188 loci; deleveraged 125 loci; blacklisted 93 loci.\n",
      "After merging, coverage depth Mean: 3.56719; Std Dev: 1.95863; Max: 32\n",
      "Merging remainder radtags\n",
      "  62626 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 25871 remainder reads; unable to match 36755 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.18313; Std Dev: 2.42565; Max: 75\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 193248\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz... read 230003 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5170\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 117340 RAD-Tags.\n",
      "  Inserted 66247 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "22757 initial stacks were populated; 43490 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.24516; Std Dev: 18.8866; Max: 1947\n",
      "Deleveraging trigger: 22; Removal trigger: 41\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 397 stacks.\n",
      "  22383 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.91418; Std Dev: 1.71198; Max: 41\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  22383 stacks merged into 21132 loci; deleveraged 73 loci; blacklisted 76 loci.\n",
      "After merging, coverage depth Mean: 2.99139; Std Dev: 1.73795; Max: 44\n",
      "Merging remainder radtags\n",
      "  43490 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 10420 remainder reads; unable to match 33070 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.48681; Std Dev: 2.5031; Max: 140\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 84270\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz... read 117340 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5171\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 333059 RAD-Tags.\n",
      "  Inserted 124806 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "59185 initial stacks were populated; 65621 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.51868; Std Dev: 10.0473; Max: 1410\n",
      "Deleveraging trigger: 15; Removal trigger: 25\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1626 stacks.\n",
      "  57657 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.23049; Std Dev: 2.52857; Max: 25\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  57657 stacks merged into 54146 loci; deleveraged 172 loci; blacklisted 121 loci.\n",
      "After merging, coverage depth Mean: 4.4545; Std Dev: 2.76978; Max: 42\n",
      "Merging remainder radtags\n",
      "  65621 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 32651 remainder reads; unable to match 32970 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.05997; Std Dev: 3.15386; Max: 49\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 300089\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz... read 333059 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5172\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 302940 RAD-Tags.\n",
      "  Inserted 107650 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "47416 initial stacks were populated; 60234 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.11865; Std Dev: 15.8535; Max: 2043\n",
      "Deleveraging trigger: 21; Removal trigger: 37\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1351 stacks.\n",
      "  46152 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.70628; Std Dev: 3.20968; Max: 37\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  46152 stacks merged into 42800 loci; deleveraged 216 loci; blacklisted 137 loci.\n",
      "After merging, coverage depth Mean: 4.99274; Std Dev: 3.48643; Max: 49\n",
      "Merging remainder radtags\n",
      "  60234 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 30840 remainder reads; unable to match 29394 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.71709; Std Dev: 4.06763; Max: 138\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 273546\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz... read 302940 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5173\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 107578 RAD-Tags.\n",
      "  Inserted 61780 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "21615 initial stacks were populated; 40165 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.11881; Std Dev: 5.29906; Max: 497\n",
      "Deleveraging trigger: 8; Removal trigger: 14\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 663 stacks.\n",
      "  21020 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.89514; Std Dev: 1.33206; Max: 14\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  21020 stacks merged into 20176 loci; deleveraged 56 loci; blacklisted 29 loci.\n",
      "After merging, coverage depth Mean: 2.99477; Std Dev: 1.50505; Max: 20\n",
      "Merging remainder radtags\n",
      "  40165 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 10041 remainder reads; unable to match 30124 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.49485; Std Dev: 2.03632; Max: 42\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 77454\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz... read 107578 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5174\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 438876 RAD-Tags.\n",
      "  Inserted 155487 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "71462 initial stacks were populated; 84025 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.96559; Std Dev: 13.7633; Max: 2043\n",
      "Deleveraging trigger: 19; Removal trigger: 32\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2058 stacks.\n",
      "  69505 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.61401; Std Dev: 2.74787; Max: 32\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  69505 stacks merged into 63261 loci; deleveraged 254 loci; blacklisted 157 loci.\n",
      "After merging, coverage depth Mean: 5.00643; Std Dev: 3.03585; Max: 48\n",
      "Merging remainder radtags\n",
      "  84025 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 56252 remainder reads; unable to match 27773 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.89927; Std Dev: 3.51923; Max: 105\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 411103\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz... read 438876 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5175\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 833361 RAD-Tags.\n",
      "  Inserted 204266 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "91094 initial stacks were populated; 113172 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.906; Std Dev: 24.1186; Max: 4218\n",
      "Deleveraging trigger: 32; Removal trigger: 56\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2459 stacks.\n",
      "  88739 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.41294; Std Dev: 4.51588; Max: 56\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  88739 stacks merged into 79685 loci; deleveraged 335 loci; blacklisted 215 loci.\n",
      "After merging, coverage depth Mean: 8.14994; Std Dev: 4.82543; Max: 74\n",
      "Merging remainder radtags\n",
      "  113172 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 89308 remainder reads; unable to match 23864 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.27521; Std Dev: 5.32936; Max: 81\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 809497\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz... read 833361 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5176\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 535704 RAD-Tags.\n",
      "  Inserted 177923 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "80275 initial stacks were populated; 97648 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.45694; Std Dev: 14.8148; Max: 1818\n",
      "Deleveraging trigger: 20; Removal trigger: 35\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2194 stacks.\n",
      "  78192 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.10187; Std Dev: 3.12626; Max: 35\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  78192 stacks merged into 70320 loci; deleveraged 270 loci; blacklisted 192 loci.\n",
      "After merging, coverage depth Mean: 5.5982; Std Dev: 3.41856; Max: 50\n",
      "Merging remainder radtags\n",
      "  97648 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 69277 remainder reads; unable to match 28371 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.58763; Std Dev: 3.9335; Max: 108\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 507333\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz... read 535704 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5177\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 802239 RAD-Tags.\n",
      "  Inserted 192538 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88309 initial stacks were populated; 104229 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.90418; Std Dev: 23.8235; Max: 3674\n",
      "Deleveraging trigger: 32; Removal trigger: 56\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2452 stacks.\n",
      "  85973 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.37324; Std Dev: 4.88195; Max: 56\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  85973 stacks merged into 77864 loci; deleveraged 331 loci; blacklisted 246 loci.\n",
      "After merging, coverage depth Mean: 8.02334; Std Dev: 5.22729; Max: 112\n",
      "Merging remainder radtags\n",
      "  104229 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 78455 remainder reads; unable to match 25774 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.03564; Std Dev: 5.77215; Max: 125\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 776465\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz... read 802239 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5178\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 162073 RAD-Tags.\n",
      "  Inserted 75360 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "31750 initial stacks were populated; 43610 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.73112; Std Dev: 7.40212; Max: 934\n",
      "Deleveraging trigger: 11; Removal trigger: 19\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 865 stacks.\n",
      "  30983 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.478; Std Dev: 1.95656; Max: 19\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  30983 stacks merged into 29548 loci; deleveraged 101 loci; blacklisted 57 loci.\n",
      "After merging, coverage depth Mean: 3.61062; Std Dev: 2.14148; Max: 27\n",
      "Merging remainder radtags\n",
      "  43610 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 14121 remainder reads; unable to match 29489 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.09104; Std Dev: 2.5981; Max: 91\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 132584\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz... read 162073 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5179\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 392299 RAD-Tags.\n",
      "  Inserted 130893 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "61840 initial stacks were populated; 69053 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.22713; Std Dev: 12.7484; Max: 2149\n",
      "Deleveraging trigger: 18; Removal trigger: 31\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1769 stacks.\n",
      "  60183 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.8739; Std Dev: 3.21028; Max: 31\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  60183 stacks merged into 56431 loci; deleveraged 214 loci; blacklisted 134 loci.\n",
      "After merging, coverage depth Mean: 5.1382; Std Dev: 3.472; Max: 54\n",
      "Merging remainder radtags\n",
      "  69053 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 35565 remainder reads; unable to match 33488 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.7712; Std Dev: 3.88584; Max: 58\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 358811\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz... read 392299 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5180\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 63151 RAD-Tags.\n",
      "  Inserted 40664 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "11919 initial stacks were populated; 28745 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.88665; Std Dev: 4.45767; Max: 327\n",
      "Deleveraging trigger: 7; Removal trigger: 12\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 388 stacks.\n",
      "  11583 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.66395; Std Dev: 1.11497; Max: 12\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  11583 stacks merged into 11187 loci; deleveraged 39 loci; blacklisted 15 loci.\n",
      "After merging, coverage depth Mean: 2.73849; Std Dev: 1.28521; Max: 18\n",
      "Merging remainder radtags\n",
      "  28745 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 4941 remainder reads; unable to match 23804 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.18282; Std Dev: 1.85837; Max: 25\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 39347\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz... read 63151 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5181\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 212945 RAD-Tags.\n",
      "  Inserted 91820 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "40275 initial stacks were populated; 51545 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.00745; Std Dev: 8.90369; Max: 1284\n",
      "Deleveraging trigger: 13; Removal trigger: 22\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1126 stacks.\n",
      "  39249 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.72127; Std Dev: 2.15912; Max: 22\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  39249 stacks merged into 37068 loci; deleveraged 111 loci; blacklisted 92 loci.\n",
      "After merging, coverage depth Mean: 3.89055; Std Dev: 2.36683; Max: 45\n",
      "Merging remainder radtags\n",
      "  51545 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 19750 remainder reads; unable to match 31795 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.42613; Std Dev: 2.7756; Max: 49\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 181150\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz... read 212945 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5182\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3710092 RAD-Tags.\n",
      "  Inserted 498792 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "133466 initial stacks were populated; 365326 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.0608; Std Dev: 94.6474; Max: 15352\n",
      "Deleveraging trigger: 120; Removal trigger: 214\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4892 stacks.\n",
      "  128706 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.5574; Std Dev: 20.837; Max: 212\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128706 stacks merged into 92517 loci; deleveraged 987 loci; blacklisted 1574 loci.\n",
      "After merging, coverage depth Mean: 31.7824; Std Dev: 19.9396; Max: 222\n",
      "Merging remainder radtags\n",
      "  365326 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 299454 remainder reads; unable to match 65872 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 35.08; Std Dev: 21.5311; Max: 277\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3644220\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz... read 3710092 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5183\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1284589 RAD-Tags.\n",
      "  Inserted 255372 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100124 initial stacks were populated; 155248 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.2794; Std Dev: 35.9385; Max: 6430\n",
      "Deleveraging trigger: 47; Removal trigger: 83\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2983 stacks.\n",
      "  97246 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.5969; Std Dev: 6.5846; Max: 82\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97246 stacks merged into 83350 loci; deleveraged 477 loci; blacklisted 341 loci.\n",
      "After merging, coverage depth Mean: 12.1728; Std Dev: 6.65132; Max: 122\n",
      "Merging remainder radtags\n",
      "  155248 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 132569 remainder reads; unable to match 22679 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.7718; Std Dev: 7.27698; Max: 131\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1261910\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz... read 1284589 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5184\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 220105 RAD-Tags.\n",
      "  Inserted 96337 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "43546 initial stacks were populated; 52791 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.84224; Std Dev: 8.6024; Max: 1184\n",
      "Deleveraging trigger: 12; Removal trigger: 21\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1240 stacks.\n",
      "  42389 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.57921; Std Dev: 2.00283; Max: 21\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  42389 stacks merged into 40306 loci; deleveraged 122 loci; blacklisted 94 loci.\n",
      "After merging, coverage depth Mean: 3.721; Std Dev: 2.21291; Max: 42\n",
      "Merging remainder radtags\n",
      "  52791 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 19309 remainder reads; unable to match 33482 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.20217; Std Dev: 2.64337; Max: 79\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 186623\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz... read 220105 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5185\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1796847 RAD-Tags.\n",
      "  Inserted 299399 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101961 initial stacks were populated; 197438 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.6865; Std Dev: 43.6576; Max: 8693\n",
      "Deleveraging trigger: 59; Removal trigger: 103\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3465 stacks.\n",
      "  98650 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.7159; Std Dev: 12.3597; Max: 103\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98650 stacks merged into 85014 loci; deleveraged 620 loci; blacklisted 494 loci.\n",
      "After merging, coverage depth Mean: 16.8081; Std Dev: 12.9903; Max: 145\n",
      "Merging remainder radtags\n",
      "  197438 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 147237 remainder reads; unable to match 50201 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.5533; Std Dev: 14.1228; Max: 154\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1746646\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz... read 1796847 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5186\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 338291 RAD-Tags.\n",
      "  Inserted 123920 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "59362 initial stacks were populated; 64558 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.61125; Std Dev: 10.6282; Max: 1635\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1688 stacks.\n",
      "  57783 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.30801; Std Dev: 2.61315; Max: 26\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  57783 stacks merged into 54470 loci; deleveraged 169 loci; blacklisted 124 loci.\n",
      "After merging, coverage depth Mean: 4.51939; Std Dev: 2.85419; Max: 58\n",
      "Merging remainder radtags\n",
      "  64558 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 30932 remainder reads; unable to match 33626 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.0897; Std Dev: 3.23727; Max: 71\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 304665\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz... read 338291 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5187\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1052123 RAD-Tags.\n",
      "  Inserted 230779 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95965 initial stacks were populated; 134814 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.55879; Std Dev: 29.6576; Max: 4367\n",
      "Deleveraging trigger: 39; Removal trigger: 69\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2737 stacks.\n",
      "  93339 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.93712; Std Dev: 6.2416; Max: 69\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  93339 stacks merged into 83257 loci; deleveraged 408 loci; blacklisted 288 loci.\n",
      "After merging, coverage depth Mean: 9.86518; Std Dev: 6.60152; Max: 106\n",
      "Merging remainder radtags\n",
      "  134814 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 99947 remainder reads; unable to match 34867 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.0714; Std Dev: 7.2665; Max: 112\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1017256\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz... read 1052123 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5188\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 428009 RAD-Tags.\n",
      "  Inserted 125872 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "59824 initial stacks were populated; 66048 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.05043; Std Dev: 14.5745; Max: 2580\n",
      "Deleveraging trigger: 21; Removal trigger: 35\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1758 stacks.\n",
      "  58194 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.65701; Std Dev: 3.96925; Max: 35\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  58194 stacks merged into 54269 loci; deleveraged 231 loci; blacklisted 142 loci.\n",
      "After merging, coverage depth Mean: 5.99754; Std Dev: 4.27438; Max: 57\n",
      "Merging remainder radtags\n",
      "  66048 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 37814 remainder reads; unable to match 28234 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.69781; Std Dev: 4.74299; Max: 75\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 399775\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz... read 428009 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5189\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1385987 RAD-Tags.\n",
      "  Inserted 284446 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "106557 initial stacks were populated; 177889 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.3376; Std Dev: 34.0343; Max: 5535\n",
      "Deleveraging trigger: 45; Removal trigger: 79\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3326 stacks.\n",
      "  103374 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.5978; Std Dev: 7.58167; Max: 79\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  103374 stacks merged into 86040 loci; deleveraged 566 loci; blacklisted 443 loci.\n",
      "After merging, coverage depth Mean: 12.5304; Std Dev: 7.88129; Max: 104\n",
      "Merging remainder radtags\n",
      "  177889 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 144098 remainder reads; unable to match 33791 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.2167; Std Dev: 8.69127; Max: 115\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1352196\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz... read 1385987 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5190\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4945581 RAD-Tags.\n",
      "  Inserted 608778 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143599 initial stacks were populated; 465179 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 31.2008; Std Dev: 103.623; Max: 19446\n",
      "Deleveraging trigger: 135; Removal trigger: 238\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6097 stacks.\n",
      "  137665 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 29.4956; Std Dev: 29.0856; Max: 238\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  137665 stacks merged into 93262 loci; deleveraged 1135 loci; blacklisted 2435 loci.\n",
      "After merging, coverage depth Mean: 41.9757; Std Dev: 28.2604; Max: 281\n",
      "Merging remainder radtags\n",
      "  465179 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 378566 remainder reads; unable to match 86613 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 46.1512; Std Dev: 30.4053; Max: 310\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4858968\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz... read 4945581 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5191\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7125415 RAD-Tags.\n",
      "  Inserted 801140 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "162653 initial stacks were populated; 638487 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 39.882; Std Dev: 184.125; Max: 35655\n",
      "Deleveraging trigger: 224; Removal trigger: 408\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 7001 stacks.\n",
      "  155777 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 37.7556; Std Dev: 38.333; Max: 408\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  155777 stacks merged into 100832 loci; deleveraged 1398 loci; blacklisted 3563 loci.\n",
      "After merging, coverage depth Mean: 55.8481; Std Dev: 35.714; Max: 557\n",
      "Merging remainder radtags\n",
      "  638487 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 498971 remainder reads; unable to match 139516 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 60.9845; Std Dev: 38.2609; Max: 684\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6985899\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz... read 7125415 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n"
     ]
    }
   ],
   "source": [
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz' -i  5000 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz' -i  5001 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz' -i  5002 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz' -i  5003 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz' -i  5004 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz' -i  5005 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz' -i  5006 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz' -i  5007 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz' -i  5008 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz' -i  5009 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz' -i  5010 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz' -i  5011 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz' -i  5012 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz' -i  5013 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz' -i  5014 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz' -i  5015 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz' -i  5016 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz' -i  5017 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz' -i  5018 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz' -i  5019 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz' -i  5020 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz' -i  5021 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz' -i  5022 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz' -i  5023 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz' -i  5024 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz' -i  5025 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz' -i  5026 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz' -i  5027 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz' -i  5028 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz' -i  5029 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz' -i  5030 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz' -i  5031 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz' -i  5032 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz' -i  5033 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz' -i  5034 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz' -i  5035 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz' -i  5036 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz' -i  5037 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz' -i  5038 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz' -i  5039 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz' -i  5040 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz' -i  5041 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz' -i  5042 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz' -i  5043 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz' -i  5044 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz' -i  5045 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz' -i  5046 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz' -i  5047 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz' -i  5048 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz' -i  5049 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz' -i  5050 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz' -i  5051 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz' -i  5052 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz' -i  5053 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz' -i  5054 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz' -i  5055 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz' -i  5056 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz' -i  5057 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz' -i  5058 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz' -i  5059 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz' -i  5060 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz' -i  5061 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz' -i  5062 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz' -i  5063 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz' -i  5064 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz' -i  5065 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz' -i  5066 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz' -i  5067 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz' -i  5068 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz' -i  5069 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz' -i  5070 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz' -i  5071 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz' -i  5072 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz' -i  5073 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz' -i  5074 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz' -i  5075 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz' -i  5076 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz' -i  5077 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz' -i  5078 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz' -i  5079 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz' -i  5080 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz' -i  5081 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz' -i  5082 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz' -i  5083 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz' -i  5084 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz' -i  5085 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz' -i  5086 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz' -i  5087 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz' -i  5088 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz' -i  5089 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz' -i  5090 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz' -i  5091 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz' -i  5092 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz' -i  5093 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz' -i  5094 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz' -i  5095 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz' -i  5096 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz' -i  5097 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz' -i  5098 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz' -i  5099 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz' -i  5100 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz' -i  5101 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz' -i  5102 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz' -i  5103 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz' -i  5104 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz' -i  5105 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz' -i  5106 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz' -i  5107 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz' -i  5108 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz' -i  5109 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz' -i  5110 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz' -i  5111 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz' -i  5112 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz' -i  5113 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz' -i  5114 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz' -i  5115 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz' -i  5116 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz' -i  5117 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz' -i  5118 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz' -i  5119 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz' -i  5120 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz' -i  5121 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz' -i  5122 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz' -i  5123 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz' -i  5124 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz' -i  5125 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz' -i  5126 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz' -i  5127 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz' -i  5128 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz' -i  5129 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz' -i  5130 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz' -i  5131 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz' -i  5132 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz' -i  5133 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz' -i  5134 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz' -i  5135 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz' -i  5136 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz' -i  5137 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz' -i  5138 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz' -i  5139 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz' -i  5140 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz' -i  5141 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz' -i  5142 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz' -i  5143 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz' -i  5144 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz' -i  5145 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz' -i  5146 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz' -i  5147 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz' -i  5148 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz' -i  5149 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz' -i  5150 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz' -i  5151 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz' -i  5152 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz' -i  5153 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz' -i  5154 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz' -i  5155 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz' -i  5156 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz' -i  5157 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz' -i  5158 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz' -i  5159 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz' -i  5160 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz' -i  5161 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz' -i  5162 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz' -i  5163 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz' -i  5164 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz' -i  5165 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz' -i  5166 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz' -i  5167 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz' -i  5168 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz' -i  5169 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz' -i  5170 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz' -i  5171 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz' -i  5172 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz' -i  5173 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz' -i  5174 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz' -i  5175 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz' -i  5176 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz' -i  5177 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz' -i  5178 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz' -i  5179 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz' -i  5180 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz' -i  5181 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz' -i  5182 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz' -i  5183 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz' -i  5184 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz' -i  5185 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz' -i  5186 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz' -i  5187 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz' -i  5188 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz' -i  5189 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz' -i  5190 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz' -i  5191 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## I'd already added variation from populations running cstacks before (catalog 3 -> catalog 4). Since we've added 2.8M reads sequence to one even-year dam I'm going to add it to the catalog again (catalog 4 -> 5) just to be safe."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cstacks parameters selected:\n",
      "  Database/Batch ID: 5\n",
      "  Loci matched based on sequence identity.\n",
      "  Number of mismatches allowed between stacks: 2\n",
      "  Gapped alignments: disabled\n",
      "Constructing catalog from 1 samples.\n",
      "\n",
      "Initializing existing catalog...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4.catalog.alleles.tsv\n",
      "\n",
      "Processing sample /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107 [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "  854646 loci in the catalog, 37732116 kmers in the catalog hash.\n",
      "Merging matches into catalog...\n",
      "  85852 loci were matched to a catalog locus.\n",
      "  0 loci were matched to a catalog locus using gapped alignments.\n",
      "  5501 loci were newly added to the catalog.\n",
      "  1249 loci matched more than one catalog locus and were excluded.\n",
      "\n",
      "Writing catalog in directory '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'.\n",
      "cstacks is done.\n"
     ]
    }
   ],
   "source": [
    "! cstacks -p 7 -b 5 -n 2 -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' --catalog '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## sstacks to match sets of stacks to the catalog"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101' [1 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89950 stacks compared against the catalog containing 860147 loci.\n",
      "  88653 matching loci, 395 contained no verified haplotypes.\n",
      "  221 loci matched more than one catalog locus and were excluded.\n",
      "  174 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  96765 total haplotypes examined from matching loci, 96075 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107' [2 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92602 stacks compared against the catalog containing 860147 loci.\n",
      "  91529 matching loci, 180 contained no verified haplotypes.\n",
      "  173 loci matched more than one catalog locus and were excluded.\n",
      "  7 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  100846 total haplotypes examined from matching loci, 100453 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001' [3 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91141 stacks compared against the catalog containing 860147 loci.\n",
      "  82716 matching loci, 562 contained no verified haplotypes.\n",
      "  176 loci matched more than one catalog locus and were excluded.\n",
      "  386 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87306 total haplotypes examined from matching loci, 86479 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002' [4 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "69215 stacks compared against the catalog containing 860147 loci.\n",
      "  66159 matching loci, 1008 contained no verified haplotypes.\n",
      "  160 loci matched more than one catalog locus and were excluded.\n",
      "  848 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  69126 total haplotypes examined from matching loci, 67892 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003' [5 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91232 stacks compared against the catalog containing 860147 loci.\n",
      "  83097 matching loci, 549 contained no verified haplotypes.\n",
      "  181 loci matched more than one catalog locus and were excluded.\n",
      "  369 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88087 total haplotypes examined from matching loci, 87281 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004' [6 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86708 stacks compared against the catalog containing 860147 loci.\n",
      "  81719 matching loci, 578 contained no verified haplotypes.\n",
      "  181 loci matched more than one catalog locus and were excluded.\n",
      "  397 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86204 total haplotypes examined from matching loci, 85372 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005' [7 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96321 stacks compared against the catalog containing 860147 loci.\n",
      "  84643 matching loci, 546 contained no verified haplotypes.\n",
      "  190 loci matched more than one catalog locus and were excluded.\n",
      "  356 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89643 total haplotypes examined from matching loci, 88816 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006' [8 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84315 stacks compared against the catalog containing 860147 loci.\n",
      "  81191 matching loci, 574 contained no verified haplotypes.\n",
      "  195 loci matched more than one catalog locus and were excluded.\n",
      "  379 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85775 total haplotypes examined from matching loci, 84950 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007' [9 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88735 stacks compared against the catalog containing 860147 loci.\n",
      "  83264 matching loci, 483 contained no verified haplotypes.\n",
      "  222 loci matched more than one catalog locus and were excluded.\n",
      "  261 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88328 total haplotypes examined from matching loci, 87542 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008' [10 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95151 stacks compared against the catalog containing 860147 loci.\n",
      "  84683 matching loci, 549 contained no verified haplotypes.\n",
      "  184 loci matched more than one catalog locus and were excluded.\n",
      "  365 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89798 total haplotypes examined from matching loci, 88956 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009' [11 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "80687 stacks compared against the catalog containing 860147 loci.\n",
      "  74403 matching loci, 610 contained no verified haplotypes.\n",
      "  172 loci matched more than one catalog locus and were excluded.\n",
      "  438 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78517 total haplotypes examined from matching loci, 77663 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010' [12 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95084 stacks compared against the catalog containing 860147 loci.\n",
      "  84784 matching loci, 533 contained no verified haplotypes.\n",
      "  191 loci matched more than one catalog locus and were excluded.\n",
      "  342 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90062 total haplotypes examined from matching loci, 89250 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011' [13 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94547 stacks compared against the catalog containing 860147 loci.\n",
      "  84135 matching loci, 549 contained no verified haplotypes.\n",
      "  199 loci matched more than one catalog locus and were excluded.\n",
      "  350 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89821 total haplotypes examined from matching loci, 88988 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012' [14 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83909 stacks compared against the catalog containing 860147 loci.\n",
      "  79029 matching loci, 644 contained no verified haplotypes.\n",
      "  183 loci matched more than one catalog locus and were excluded.\n",
      "  461 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83140 total haplotypes examined from matching loci, 82244 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013' [15 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81450 stacks compared against the catalog containing 860147 loci.\n",
      "  79637 matching loci, 372 contained no verified haplotypes.\n",
      "  169 loci matched more than one catalog locus and were excluded.\n",
      "  203 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84125 total haplotypes examined from matching loci, 83531 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014' [16 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89154 stacks compared against the catalog containing 860147 loci.\n",
      "  84489 matching loci, 450 contained no verified haplotypes.\n",
      "  198 loci matched more than one catalog locus and were excluded.\n",
      "  252 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89934 total haplotypes examined from matching loci, 89194 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015' [17 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81854 stacks compared against the catalog containing 860147 loci.\n",
      "  79339 matching loci, 415 contained no verified haplotypes.\n",
      "  175 loci matched more than one catalog locus and were excluded.\n",
      "  240 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83896 total haplotypes examined from matching loci, 83257 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016' [18 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85926 stacks compared against the catalog containing 860147 loci.\n",
      "  81931 matching loci, 425 contained no verified haplotypes.\n",
      "  185 loci matched more than one catalog locus and were excluded.\n",
      "  240 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87041 total haplotypes examined from matching loci, 86369 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017' [19 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "69486 stacks compared against the catalog containing 860147 loci.\n",
      "  67544 matching loci, 465 contained no verified haplotypes.\n",
      "  155 loci matched more than one catalog locus and were excluded.\n",
      "  310 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  70169 total haplotypes examined from matching loci, 69505 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018' [20 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85044 stacks compared against the catalog containing 860147 loci.\n",
      "  82078 matching loci, 426 contained no verified haplotypes.\n",
      "  187 loci matched more than one catalog locus and were excluded.\n",
      "  239 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87170 total haplotypes examined from matching loci, 86485 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019' [21 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "79348 stacks compared against the catalog containing 860147 loci.\n",
      "  77646 matching loci, 409 contained no verified haplotypes.\n",
      "  163 loci matched more than one catalog locus and were excluded.\n",
      "  246 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81465 total haplotypes examined from matching loci, 80852 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020' [22 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "72120 stacks compared against the catalog containing 860147 loci.\n",
      "  70435 matching loci, 354 contained no verified haplotypes.\n",
      "  113 loci matched more than one catalog locus and were excluded.\n",
      "  241 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  73069 total haplotypes examined from matching loci, 72577 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021' [23 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78035 stacks compared against the catalog containing 860147 loci.\n",
      "  74337 matching loci, 616 contained no verified haplotypes.\n",
      "  159 loci matched more than one catalog locus and were excluded.\n",
      "  457 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78048 total haplotypes examined from matching loci, 77214 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022' [24 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "76723 stacks compared against the catalog containing 860147 loci.\n",
      "  73544 matching loci, 492 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  326 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77085 total haplotypes examined from matching loci, 76375 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023' [25 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "67175 stacks compared against the catalog containing 860147 loci.\n",
      "  63483 matching loci, 581 contained no verified haplotypes.\n",
      "  130 loci matched more than one catalog locus and were excluded.\n",
      "  451 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  65897 total haplotypes examined from matching loci, 65144 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002' [26 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88286 stacks compared against the catalog containing 860147 loci.\n",
      "  81340 matching loci, 424 contained no verified haplotypes.\n",
      "  150 loci matched more than one catalog locus and were excluded.\n",
      "  274 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84184 total haplotypes examined from matching loci, 83544 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003' [27 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "101713 stacks compared against the catalog containing 860147 loci.\n",
      "  84787 matching loci, 575 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  426 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87696 total haplotypes examined from matching loci, 86873 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005' [28 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86312 stacks compared against the catalog containing 860147 loci.\n",
      "  81034 matching loci, 371 contained no verified haplotypes.\n",
      "  143 loci matched more than one catalog locus and were excluded.\n",
      "  228 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83713 total haplotypes examined from matching loci, 83141 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007' [29 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94208 stacks compared against the catalog containing 860147 loci.\n",
      "  83750 matching loci, 403 contained no verified haplotypes.\n",
      "  152 loci matched more than one catalog locus and were excluded.\n",
      "  251 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86634 total haplotypes examined from matching loci, 86003 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009' [30 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "99725 stacks compared against the catalog containing 860147 loci.\n",
      "  84784 matching loci, 516 contained no verified haplotypes.\n",
      "  156 loci matched more than one catalog locus and were excluded.\n",
      "  360 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87755 total haplotypes examined from matching loci, 86985 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010' [31 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86941 stacks compared against the catalog containing 860147 loci.\n",
      "  81394 matching loci, 363 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  214 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84152 total haplotypes examined from matching loci, 83565 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011' [32 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "424 stacks compared against the catalog containing 860147 loci.\n",
      "  33 matching loci, 3 contained no verified haplotypes.\n",
      "  0 loci matched more than one catalog locus and were excluded.\n",
      "  3 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  37 total haplotypes examined from matching loci, 34 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012' [33 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85764 stacks compared against the catalog containing 860147 loci.\n",
      "  80808 matching loci, 322 contained no verified haplotypes.\n",
      "  130 loci matched more than one catalog locus and were excluded.\n",
      "  192 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83478 total haplotypes examined from matching loci, 82970 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013' [34 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91157 stacks compared against the catalog containing 860147 loci.\n",
      "  81216 matching loci, 529 contained no verified haplotypes.\n",
      "  155 loci matched more than one catalog locus and were excluded.\n",
      "  374 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83882 total haplotypes examined from matching loci, 83141 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014' [35 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89844 stacks compared against the catalog containing 860147 loci.\n",
      "  81896 matching loci, 399 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  242 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84699 total haplotypes examined from matching loci, 84066 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015' [36 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89806 stacks compared against the catalog containing 860147 loci.\n",
      "  81553 matching loci, 404 contained no verified haplotypes.\n",
      "  138 loci matched more than one catalog locus and were excluded.\n",
      "  266 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84281 total haplotypes examined from matching loci, 83664 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016' [37 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87683 stacks compared against the catalog containing 860147 loci.\n",
      "  80657 matching loci, 428 contained no verified haplotypes.\n",
      "  134 loci matched more than one catalog locus and were excluded.\n",
      "  294 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83196 total haplotypes examined from matching loci, 82574 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017' [38 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91228 stacks compared against the catalog containing 860147 loci.\n",
      "  80539 matching loci, 558 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  409 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83087 total haplotypes examined from matching loci, 82319 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018' [39 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84221 stacks compared against the catalog containing 860147 loci.\n",
      "  80329 matching loci, 366 contained no verified haplotypes.\n",
      "  135 loci matched more than one catalog locus and were excluded.\n",
      "  231 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82886 total haplotypes examined from matching loci, 82331 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019' [40 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88838 stacks compared against the catalog containing 860147 loci.\n",
      "  80959 matching loci, 536 contained no verified haplotypes.\n",
      "  153 loci matched more than one catalog locus and were excluded.\n",
      "  383 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83500 total haplotypes examined from matching loci, 82751 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020' [41 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90049 stacks compared against the catalog containing 860147 loci.\n",
      "  81843 matching loci, 456 contained no verified haplotypes.\n",
      "  161 loci matched more than one catalog locus and were excluded.\n",
      "  295 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84620 total haplotypes examined from matching loci, 83935 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023' [42 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85616 stacks compared against the catalog containing 860147 loci.\n",
      "  80774 matching loci, 364 contained no verified haplotypes.\n",
      "  141 loci matched more than one catalog locus and were excluded.\n",
      "  223 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83478 total haplotypes examined from matching loci, 82910 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024' [43 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94719 stacks compared against the catalog containing 860147 loci.\n",
      "  81275 matching loci, 688 contained no verified haplotypes.\n",
      "  163 loci matched more than one catalog locus and were excluded.\n",
      "  525 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83850 total haplotypes examined from matching loci, 82932 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025' [44 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "99939 stacks compared against the catalog containing 860147 loci.\n",
      "  83873 matching loci, 476 contained no verified haplotypes.\n",
      "  164 loci matched more than one catalog locus and were excluded.\n",
      "  312 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86877 total haplotypes examined from matching loci, 86136 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026' [45 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91751 stacks compared against the catalog containing 860147 loci.\n",
      "  81192 matching loci, 581 contained no verified haplotypes.\n",
      "  136 loci matched more than one catalog locus and were excluded.\n",
      "  446 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83639 total haplotypes examined from matching loci, 82855 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027' [46 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94807 stacks compared against the catalog containing 860147 loci.\n",
      "  81797 matching loci, 708 contained no verified haplotypes.\n",
      "  143 loci matched more than one catalog locus and were excluded.\n",
      "  565 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84335 total haplotypes examined from matching loci, 83389 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029' [47 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96158 stacks compared against the catalog containing 860147 loci.\n",
      "  82207 matching loci, 664 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  498 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84834 total haplotypes examined from matching loci, 83903 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031' [48 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96950 stacks compared against the catalog containing 860147 loci.\n",
      "  82350 matching loci, 684 contained no verified haplotypes.\n",
      "  155 loci matched more than one catalog locus and were excluded.\n",
      "  529 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84945 total haplotypes examined from matching loci, 83973 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032' [49 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92306 stacks compared against the catalog containing 860147 loci.\n",
      "  81802 matching loci, 551 contained no verified haplotypes.\n",
      "  170 loci matched more than one catalog locus and were excluded.\n",
      "  381 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84592 total haplotypes examined from matching loci, 83787 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033' [50 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90208 stacks compared against the catalog containing 860147 loci.\n",
      "  81052 matching loci, 392 contained no verified haplotypes.\n",
      "  130 loci matched more than one catalog locus and were excluded.\n",
      "  262 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83768 total haplotypes examined from matching loci, 83180 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034' [51 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88640 stacks compared against the catalog containing 860147 loci.\n",
      "  80878 matching loci, 407 contained no verified haplotypes.\n",
      "  139 loci matched more than one catalog locus and were excluded.\n",
      "  268 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83517 total haplotypes examined from matching loci, 82908 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035' [52 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90913 stacks compared against the catalog containing 860147 loci.\n",
      "  80791 matching loci, 730 contained no verified haplotypes.\n",
      "  127 loci matched more than one catalog locus and were excluded.\n",
      "  603 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83108 total haplotypes examined from matching loci, 82172 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038' [53 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83203 stacks compared against the catalog containing 860147 loci.\n",
      "  73970 matching loci, 820 contained no verified haplotypes.\n",
      "  136 loci matched more than one catalog locus and were excluded.\n",
      "  684 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  76081 total haplotypes examined from matching loci, 75053 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039' [54 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95639 stacks compared against the catalog containing 860147 loci.\n",
      "  82394 matching loci, 672 contained no verified haplotypes.\n",
      "  146 loci matched more than one catalog locus and were excluded.\n",
      "  526 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85063 total haplotypes examined from matching loci, 84146 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040' [55 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "107231 stacks compared against the catalog containing 860147 loci.\n",
      "  83378 matching loci, 833 contained no verified haplotypes.\n",
      "  173 loci matched more than one catalog locus and were excluded.\n",
      "  661 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86025 total haplotypes examined from matching loci, 84884 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041' [56 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90213 stacks compared against the catalog containing 860147 loci.\n",
      "  81268 matching loci, 522 contained no verified haplotypes.\n",
      "  120 loci matched more than one catalog locus and were excluded.\n",
      "  402 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83823 total haplotypes examined from matching loci, 83115 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047' [57 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96683 stacks compared against the catalog containing 860147 loci.\n",
      "  82078 matching loci, 551 contained no verified haplotypes.\n",
      "  143 loci matched more than one catalog locus and were excluded.\n",
      "  408 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84768 total haplotypes examined from matching loci, 83989 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049' [58 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90522 stacks compared against the catalog containing 860147 loci.\n",
      "  79774 matching loci, 838 contained no verified haplotypes.\n",
      "  189 loci matched more than one catalog locus and were excluded.\n",
      "  649 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86067 total haplotypes examined from matching loci, 84954 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050' [59 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "80467 stacks compared against the catalog containing 860147 loci.\n",
      "  65771 matching loci, 866 contained no verified haplotypes.\n",
      "  106 loci matched more than one catalog locus and were excluded.\n",
      "  760 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  67564 total haplotypes examined from matching loci, 66528 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053' [60 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "80287 stacks compared against the catalog containing 860147 loci.\n",
      "  63725 matching loci, 1021 contained no verified haplotypes.\n",
      "  137 loci matched more than one catalog locus and were excluded.\n",
      "  884 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  65497 total haplotypes examined from matching loci, 64265 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055' [61 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "103528 stacks compared against the catalog containing 860147 loci.\n",
      "  84288 matching loci, 588 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  439 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88328 total haplotypes examined from matching loci, 87508 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056' [62 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95273 stacks compared against the catalog containing 860147 loci.\n",
      "  78320 matching loci, 991 contained no verified haplotypes.\n",
      "  125 loci matched more than one catalog locus and were excluded.\n",
      "  866 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81239 total haplotypes examined from matching loci, 80018 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058' [63 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87202 stacks compared against the catalog containing 860147 loci.\n",
      "  75394 matching loci, 859 contained no verified haplotypes.\n",
      "  146 loci matched more than one catalog locus and were excluded.\n",
      "  713 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78750 total haplotypes examined from matching loci, 77687 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059' [64 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "59972 stacks compared against the catalog containing 860147 loci.\n",
      "  56577 matching loci, 556 contained no verified haplotypes.\n",
      "  104 loci matched more than one catalog locus and were excluded.\n",
      "  452 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  58004 total haplotypes examined from matching loci, 57312 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062' [65 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96631 stacks compared against the catalog containing 860147 loci.\n",
      "  81495 matching loci, 701 contained no verified haplotypes.\n",
      "  192 loci matched more than one catalog locus and were excluded.\n",
      "  509 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85074 total haplotypes examined from matching loci, 84100 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066' [66 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82268 stacks compared against the catalog containing 860147 loci.\n",
      "  70852 matching loci, 1195 contained no verified haplotypes.\n",
      "  178 loci matched more than one catalog locus and were excluded.\n",
      "  1017 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  73299 total haplotypes examined from matching loci, 71843 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067' [67 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87897 stacks compared against the catalog containing 860147 loci.\n",
      "  77043 matching loci, 894 contained no verified haplotypes.\n",
      "  184 loci matched more than one catalog locus and were excluded.\n",
      "  710 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80529 total haplotypes examined from matching loci, 79361 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069' [68 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "98064 stacks compared against the catalog containing 860147 loci.\n",
      "  83683 matching loci, 766 contained no verified haplotypes.\n",
      "  184 loci matched more than one catalog locus and were excluded.\n",
      "  582 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88081 total haplotypes examined from matching loci, 87025 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070' [69 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94031 stacks compared against the catalog containing 860147 loci.\n",
      "  80520 matching loci, 916 contained no verified haplotypes.\n",
      "  170 loci matched more than one catalog locus and were excluded.\n",
      "  746 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83742 total haplotypes examined from matching loci, 82565 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073' [70 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77817 stacks compared against the catalog containing 860147 loci.\n",
      "  70021 matching loci, 793 contained no verified haplotypes.\n",
      "  155 loci matched more than one catalog locus and were excluded.\n",
      "  638 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  72744 total haplotypes examined from matching loci, 71730 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075' [71 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89224 stacks compared against the catalog containing 860147 loci.\n",
      "  77722 matching loci, 1060 contained no verified haplotypes.\n",
      "  146 loci matched more than one catalog locus and were excluded.\n",
      "  914 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80116 total haplotypes examined from matching loci, 78833 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078' [72 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92847 stacks compared against the catalog containing 860147 loci.\n",
      "  80984 matching loci, 803 contained no verified haplotypes.\n",
      "  188 loci matched more than one catalog locus and were excluded.\n",
      "  615 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85457 total haplotypes examined from matching loci, 84379 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101' [73 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "50447 stacks compared against the catalog containing 860147 loci.\n",
      "  48645 matching loci, 396 contained no verified haplotypes.\n",
      "  109 loci matched more than one catalog locus and were excluded.\n",
      "  287 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  49752 total haplotypes examined from matching loci, 49219 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104' [74 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "681 stacks compared against the catalog containing 860147 loci.\n",
      "  46 matching loci, 6 contained no verified haplotypes.\n",
      "  0 loci matched more than one catalog locus and were excluded.\n",
      "  6 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  52 total haplotypes examined from matching loci, 45 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001' [75 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88024 stacks compared against the catalog containing 860147 loci.\n",
      "  82485 matching loci, 356 contained no verified haplotypes.\n",
      "  159 loci matched more than one catalog locus and were excluded.\n",
      "  197 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85274 total haplotypes examined from matching loci, 84689 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002' [76 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92322 stacks compared against the catalog containing 860147 loci.\n",
      "  81427 matching loci, 555 contained no verified haplotypes.\n",
      "  180 loci matched more than one catalog locus and were excluded.\n",
      "  375 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84278 total haplotypes examined from matching loci, 83445 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003' [77 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82844 stacks compared against the catalog containing 860147 loci.\n",
      "  79890 matching loci, 359 contained no verified haplotypes.\n",
      "  168 loci matched more than one catalog locus and were excluded.\n",
      "  191 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82748 total haplotypes examined from matching loci, 82154 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004' [78 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90977 stacks compared against the catalog containing 860147 loci.\n",
      "  83342 matching loci, 381 contained no verified haplotypes.\n",
      "  151 loci matched more than one catalog locus and were excluded.\n",
      "  230 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86338 total haplotypes examined from matching loci, 85729 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005' [79 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "102796 stacks compared against the catalog containing 860147 loci.\n",
      "  86658 matching loci, 551 contained no verified haplotypes.\n",
      "  191 loci matched more than one catalog locus and were excluded.\n",
      "  360 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89697 total haplotypes examined from matching loci, 88843 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006' [80 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90930 stacks compared against the catalog containing 860147 loci.\n",
      "  83344 matching loci, 397 contained no verified haplotypes.\n",
      "  165 loci matched more than one catalog locus and were excluded.\n",
      "  232 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86317 total haplotypes examined from matching loci, 85649 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007' [81 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83416 stacks compared against the catalog containing 860147 loci.\n",
      "  79728 matching loci, 383 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  234 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82347 total haplotypes examined from matching loci, 81752 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008' [82 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94345 stacks compared against the catalog containing 860147 loci.\n",
      "  83978 matching loci, 476 contained no verified haplotypes.\n",
      "  171 loci matched more than one catalog locus and were excluded.\n",
      "  305 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86868 total haplotypes examined from matching loci, 86126 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009' [83 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87941 stacks compared against the catalog containing 860147 loci.\n",
      "  82275 matching loci, 365 contained no verified haplotypes.\n",
      "  169 loci matched more than one catalog locus and were excluded.\n",
      "  196 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85125 total haplotypes examined from matching loci, 84516 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010' [84 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84133 stacks compared against the catalog containing 860147 loci.\n",
      "  80492 matching loci, 348 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  191 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83130 total haplotypes examined from matching loci, 82566 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011' [85 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85617 stacks compared against the catalog containing 860147 loci.\n",
      "  81965 matching loci, 349 contained no verified haplotypes.\n",
      "  171 loci matched more than one catalog locus and were excluded.\n",
      "  178 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84733 total haplotypes examined from matching loci, 84154 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012' [86 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88268 stacks compared against the catalog containing 860147 loci.\n",
      "  82959 matching loci, 364 contained no verified haplotypes.\n",
      "  179 loci matched more than one catalog locus and were excluded.\n",
      "  185 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85845 total haplotypes examined from matching loci, 85224 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013' [87 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "100878 stacks compared against the catalog containing 860147 loci.\n",
      "  86841 matching loci, 564 contained no verified haplotypes.\n",
      "  190 loci matched more than one catalog locus and were excluded.\n",
      "  374 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89839 total haplotypes examined from matching loci, 88957 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014' [88 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "130799 stacks compared against the catalog containing 860147 loci.\n",
      "  87252 matching loci, 885 contained no verified haplotypes.\n",
      "  181 loci matched more than one catalog locus and were excluded.\n",
      "  704 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90321 total haplotypes examined from matching loci, 89103 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015' [89 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "98838 stacks compared against the catalog containing 860147 loci.\n",
      "  84376 matching loci, 499 contained no verified haplotypes.\n",
      "  161 loci matched more than one catalog locus and were excluded.\n",
      "  338 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87268 total haplotypes examined from matching loci, 86538 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016' [90 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "99241 stacks compared against the catalog containing 860147 loci.\n",
      "  85068 matching loci, 502 contained no verified haplotypes.\n",
      "  186 loci matched more than one catalog locus and were excluded.\n",
      "  317 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88041 total haplotypes examined from matching loci, 87239 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017' [91 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "100080 stacks compared against the catalog containing 860147 loci.\n",
      "  84487 matching loci, 580 contained no verified haplotypes.\n",
      "  171 loci matched more than one catalog locus and were excluded.\n",
      "  409 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87207 total haplotypes examined from matching loci, 86323 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018' [92 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94444 stacks compared against the catalog containing 860147 loci.\n",
      "  83882 matching loci, 518 contained no verified haplotypes.\n",
      "  180 loci matched more than one catalog locus and were excluded.\n",
      "  338 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86650 total haplotypes examined from matching loci, 85826 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019' [93 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88474 stacks compared against the catalog containing 860147 loci.\n",
      "  82273 matching loci, 363 contained no verified haplotypes.\n",
      "  167 loci matched more than one catalog locus and were excluded.\n",
      "  196 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85072 total haplotypes examined from matching loci, 84475 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020' [94 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "104514 stacks compared against the catalog containing 860147 loci.\n",
      "  86650 matching loci, 523 contained no verified haplotypes.\n",
      "  183 loci matched more than one catalog locus and were excluded.\n",
      "  340 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89799 total haplotypes examined from matching loci, 89011 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021' [95 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84282 stacks compared against the catalog containing 860147 loci.\n",
      "  80600 matching loci, 346 contained no verified haplotypes.\n",
      "  142 loci matched more than one catalog locus and were excluded.\n",
      "  204 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83195 total haplotypes examined from matching loci, 82654 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022' [96 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90553 stacks compared against the catalog containing 860147 loci.\n",
      "  83940 matching loci, 418 contained no verified haplotypes.\n",
      "  183 loci matched more than one catalog locus and were excluded.\n",
      "  235 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86894 total haplotypes examined from matching loci, 86211 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023' [97 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86314 stacks compared against the catalog containing 860147 loci.\n",
      "  81426 matching loci, 362 contained no verified haplotypes.\n",
      "  154 loci matched more than one catalog locus and were excluded.\n",
      "  208 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84200 total haplotypes examined from matching loci, 83631 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024' [98 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "97615 stacks compared against the catalog containing 860147 loci.\n",
      "  86128 matching loci, 417 contained no verified haplotypes.\n",
      "  176 loci matched more than one catalog locus and were excluded.\n",
      "  241 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89021 total haplotypes examined from matching loci, 88340 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025' [99 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94001 stacks compared against the catalog containing 860147 loci.\n",
      "  84112 matching loci, 443 contained no verified haplotypes.\n",
      "  147 loci matched more than one catalog locus and were excluded.\n",
      "  296 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87035 total haplotypes examined from matching loci, 86349 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026' [100 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78762 stacks compared against the catalog containing 860147 loci.\n",
      "  76611 matching loci, 337 contained no verified haplotypes.\n",
      "  128 loci matched more than one catalog locus and were excluded.\n",
      "  209 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78778 total haplotypes examined from matching loci, 78272 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027' [101 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84388 stacks compared against the catalog containing 860147 loci.\n",
      "  80602 matching loci, 311 contained no verified haplotypes.\n",
      "  142 loci matched more than one catalog locus and were excluded.\n",
      "  169 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83246 total haplotypes examined from matching loci, 82743 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028' [102 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87249 stacks compared against the catalog containing 860147 loci.\n",
      "  81827 matching loci, 387 contained no verified haplotypes.\n",
      "  161 loci matched more than one catalog locus and were excluded.\n",
      "  226 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84805 total haplotypes examined from matching loci, 84187 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029' [103 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "76061 stacks compared against the catalog containing 860147 loci.\n",
      "  73668 matching loci, 496 contained no verified haplotypes.\n",
      "  123 loci matched more than one catalog locus and were excluded.\n",
      "  373 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  75540 total haplotypes examined from matching loci, 74885 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030' [104 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88500 stacks compared against the catalog containing 860147 loci.\n",
      "  83096 matching loci, 373 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  216 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86097 total haplotypes examined from matching loci, 85486 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031' [105 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82727 stacks compared against the catalog containing 860147 loci.\n",
      "  78843 matching loci, 377 contained no verified haplotypes.\n",
      "  125 loci matched more than one catalog locus and were excluded.\n",
      "  252 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81217 total haplotypes examined from matching loci, 80676 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032' [106 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85461 stacks compared against the catalog containing 860147 loci.\n",
      "  81067 matching loci, 337 contained no verified haplotypes.\n",
      "  132 loci matched more than one catalog locus and were excluded.\n",
      "  205 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83867 total haplotypes examined from matching loci, 83333 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033' [107 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85621 stacks compared against the catalog containing 860147 loci.\n",
      "  80837 matching loci, 361 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  204 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83638 total haplotypes examined from matching loci, 83054 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034' [108 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "75694 stacks compared against the catalog containing 860147 loci.\n",
      "  73355 matching loci, 469 contained no verified haplotypes.\n",
      "  124 loci matched more than one catalog locus and were excluded.\n",
      "  345 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  75285 total haplotypes examined from matching loci, 74653 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035' [109 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "47915 stacks compared against the catalog containing 860147 loci.\n",
      "  45971 matching loci, 741 contained no verified haplotypes.\n",
      "  116 loci matched more than one catalog locus and were excluded.\n",
      "  625 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  46939 total haplotypes examined from matching loci, 46054 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036' [110 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84632 stacks compared against the catalog containing 860147 loci.\n",
      "  80524 matching loci, 455 contained no verified haplotypes.\n",
      "  159 loci matched more than one catalog locus and were excluded.\n",
      "  296 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82881 total haplotypes examined from matching loci, 82208 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037' [111 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77371 stacks compared against the catalog containing 860147 loci.\n",
      "  74778 matching loci, 708 contained no verified haplotypes.\n",
      "  125 loci matched more than one catalog locus and were excluded.\n",
      "  583 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  76781 total haplotypes examined from matching loci, 75913 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038' [112 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "46873 stacks compared against the catalog containing 860147 loci.\n",
      "  43689 matching loci, 911 contained no verified haplotypes.\n",
      "  105 loci matched more than one catalog locus and were excluded.\n",
      "  806 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  44671 total haplotypes examined from matching loci, 43620 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039' [113 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "76134 stacks compared against the catalog containing 860147 loci.\n",
      "  74133 matching loci, 636 contained no verified haplotypes.\n",
      "  129 loci matched more than one catalog locus and were excluded.\n",
      "  507 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  76065 total haplotypes examined from matching loci, 75269 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040' [114 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81804 stacks compared against the catalog containing 860147 loci.\n",
      "  78697 matching loci, 849 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  683 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80843 total haplotypes examined from matching loci, 79772 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041' [115 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83349 stacks compared against the catalog containing 860147 loci.\n",
      "  79141 matching loci, 474 contained no verified haplotypes.\n",
      "  129 loci matched more than one catalog locus and were excluded.\n",
      "  345 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81453 total haplotypes examined from matching loci, 80794 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042' [116 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83851 stacks compared against the catalog containing 860147 loci.\n",
      "  79439 matching loci, 461 contained no verified haplotypes.\n",
      "  141 loci matched more than one catalog locus and were excluded.\n",
      "  320 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81846 total haplotypes examined from matching loci, 81194 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043' [117 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "80898 stacks compared against the catalog containing 860147 loci.\n",
      "  77697 matching loci, 569 contained no verified haplotypes.\n",
      "  146 loci matched more than one catalog locus and were excluded.\n",
      "  423 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79944 total haplotypes examined from matching loci, 79177 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044' [118 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81208 stacks compared against the catalog containing 860147 loci.\n",
      "  78037 matching loci, 556 contained no verified haplotypes.\n",
      "  145 loci matched more than one catalog locus and were excluded.\n",
      "  411 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80238 total haplotypes examined from matching loci, 79492 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045' [119 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "62588 stacks compared against the catalog containing 860147 loci.\n",
      "  60773 matching loci, 376 contained no verified haplotypes.\n",
      "  103 loci matched more than one catalog locus and were excluded.\n",
      "  273 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  62104 total haplotypes examined from matching loci, 61593 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046' [120 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82328 stacks compared against the catalog containing 860147 loci.\n",
      "  78358 matching loci, 543 contained no verified haplotypes.\n",
      "  141 loci matched more than one catalog locus and were excluded.\n",
      "  402 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80652 total haplotypes examined from matching loci, 79915 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047' [121 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87491 stacks compared against the catalog containing 860147 loci.\n",
      "  81062 matching loci, 373 contained no verified haplotypes.\n",
      "  129 loci matched more than one catalog locus and were excluded.\n",
      "  244 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83508 total haplotypes examined from matching loci, 82950 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048' [122 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83171 stacks compared against the catalog containing 860147 loci.\n",
      "  79842 matching loci, 432 contained no verified haplotypes.\n",
      "  145 loci matched more than one catalog locus and were excluded.\n",
      "  287 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82170 total haplotypes examined from matching loci, 81544 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049' [123 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84834 stacks compared against the catalog containing 860147 loci.\n",
      "  80273 matching loci, 450 contained no verified haplotypes.\n",
      "  187 loci matched more than one catalog locus and were excluded.\n",
      "  263 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82802 total haplotypes examined from matching loci, 82110 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050' [124 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77722 stacks compared against the catalog containing 860147 loci.\n",
      "  75633 matching loci, 538 contained no verified haplotypes.\n",
      "  119 loci matched more than one catalog locus and were excluded.\n",
      "  419 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77684 total haplotypes examined from matching loci, 76977 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051' [125 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81290 stacks compared against the catalog containing 860147 loci.\n",
      "  78661 matching loci, 434 contained no verified haplotypes.\n",
      "  142 loci matched more than one catalog locus and were excluded.\n",
      "  292 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80946 total haplotypes examined from matching loci, 80323 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052' [126 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90087 stacks compared against the catalog containing 860147 loci.\n",
      "  81746 matching loci, 480 contained no verified haplotypes.\n",
      "  170 loci matched more than one catalog locus and were excluded.\n",
      "  310 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84707 total haplotypes examined from matching loci, 83968 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053' [127 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "75007 stacks compared against the catalog containing 860147 loci.\n",
      "  72560 matching loci, 743 contained no verified haplotypes.\n",
      "  131 loci matched more than one catalog locus and were excluded.\n",
      "  612 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  74439 total haplotypes examined from matching loci, 73510 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054' [128 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85472 stacks compared against the catalog containing 860147 loci.\n",
      "  80812 matching loci, 335 contained no verified haplotypes.\n",
      "  152 loci matched more than one catalog locus and were excluded.\n",
      "  183 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83266 total haplotypes examined from matching loci, 82729 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055' [129 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92985 stacks compared against the catalog containing 860147 loci.\n",
      "  82817 matching loci, 488 contained no verified haplotypes.\n",
      "  178 loci matched more than one catalog locus and were excluded.\n",
      "  310 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85203 total haplotypes examined from matching loci, 84448 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056' [130 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89812 stacks compared against the catalog containing 860147 loci.\n",
      "  82127 matching loci, 435 contained no verified haplotypes.\n",
      "  140 loci matched more than one catalog locus and were excluded.\n",
      "  295 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84549 total haplotypes examined from matching loci, 83908 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057' [131 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86104 stacks compared against the catalog containing 860147 loci.\n",
      "  80988 matching loci, 430 contained no verified haplotypes.\n",
      "  165 loci matched more than one catalog locus and were excluded.\n",
      "  265 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83291 total haplotypes examined from matching loci, 82630 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059' [132 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82320 stacks compared against the catalog containing 860147 loci.\n",
      "  79276 matching loci, 394 contained no verified haplotypes.\n",
      "  144 loci matched more than one catalog locus and were excluded.\n",
      "  250 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81722 total haplotypes examined from matching loci, 81133 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060' [133 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84433 stacks compared against the catalog containing 860147 loci.\n",
      "  79919 matching loci, 436 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  287 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82322 total haplotypes examined from matching loci, 81679 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061' [134 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89170 stacks compared against the catalog containing 860147 loci.\n",
      "  81121 matching loci, 433 contained no verified haplotypes.\n",
      "  151 loci matched more than one catalog locus and were excluded.\n",
      "  282 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83631 total haplotypes examined from matching loci, 82982 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062' [135 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93424 stacks compared against the catalog containing 860147 loci.\n",
      "  83155 matching loci, 490 contained no verified haplotypes.\n",
      "  173 loci matched more than one catalog locus and were excluded.\n",
      "  317 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85768 total haplotypes examined from matching loci, 85000 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063' [136 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96417 stacks compared against the catalog containing 860147 loci.\n",
      "  83754 matching loci, 502 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  336 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86327 total haplotypes examined from matching loci, 85563 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064' [137 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "396 stacks compared against the catalog containing 860147 loci.\n",
      "  77 matching loci, 5 contained no verified haplotypes.\n",
      "  0 loci matched more than one catalog locus and were excluded.\n",
      "  5 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79 total haplotypes examined from matching loci, 74 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065' [138 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90218 stacks compared against the catalog containing 860147 loci.\n",
      "  82402 matching loci, 429 contained no verified haplotypes.\n",
      "  158 loci matched more than one catalog locus and were excluded.\n",
      "  271 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84880 total haplotypes examined from matching loci, 84228 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066' [139 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85486 stacks compared against the catalog containing 860147 loci.\n",
      "  80490 matching loci, 456 contained no verified haplotypes.\n",
      "  164 loci matched more than one catalog locus and were excluded.\n",
      "  292 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83022 total haplotypes examined from matching loci, 82328 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067' [140 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85631 stacks compared against the catalog containing 860147 loci.\n",
      "  80766 matching loci, 413 contained no verified haplotypes.\n",
      "  153 loci matched more than one catalog locus and were excluded.\n",
      "  260 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83197 total haplotypes examined from matching loci, 82561 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068' [141 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88239 stacks compared against the catalog containing 860147 loci.\n",
      "  81980 matching loci, 397 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  240 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84686 total haplotypes examined from matching loci, 84056 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069' [142 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "106056 stacks compared against the catalog containing 860147 loci.\n",
      "  84143 matching loci, 572 contained no verified haplotypes.\n",
      "  154 loci matched more than one catalog locus and were excluded.\n",
      "  419 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86859 total haplotypes examined from matching loci, 86020 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070' [143 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93411 stacks compared against the catalog containing 860147 loci.\n",
      "  83791 matching loci, 473 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  324 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86250 total haplotypes examined from matching loci, 85531 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071' [144 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86532 stacks compared against the catalog containing 860147 loci.\n",
      "  81003 matching loci, 439 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  273 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83331 total haplotypes examined from matching loci, 82639 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072' [145 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96900 stacks compared against the catalog containing 860147 loci.\n",
      "  84221 matching loci, 495 contained no verified haplotypes.\n",
      "  170 loci matched more than one catalog locus and were excluded.\n",
      "  325 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87000 total haplotypes examined from matching loci, 86262 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073' [146 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96421 stacks compared against the catalog containing 860147 loci.\n",
      "  83758 matching loci, 547 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  381 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86386 total haplotypes examined from matching loci, 85576 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074' [147 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90769 stacks compared against the catalog containing 860147 loci.\n",
      "  82874 matching loci, 424 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  258 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85527 total haplotypes examined from matching loci, 84851 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075' [148 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87808 stacks compared against the catalog containing 860147 loci.\n",
      "  81294 matching loci, 466 contained no verified haplotypes.\n",
      "  151 loci matched more than one catalog locus and were excluded.\n",
      "  315 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83688 total haplotypes examined from matching loci, 83000 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076' [149 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96804 stacks compared against the catalog containing 860147 loci.\n",
      "  83576 matching loci, 494 contained no verified haplotypes.\n",
      "  172 loci matched more than one catalog locus and were excluded.\n",
      "  322 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86213 total haplotypes examined from matching loci, 85459 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077' [150 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86600 stacks compared against the catalog containing 860147 loci.\n",
      "  81764 matching loci, 347 contained no verified haplotypes.\n",
      "  139 loci matched more than one catalog locus and were excluded.\n",
      "  208 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84423 total haplotypes examined from matching loci, 83866 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078' [151 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87246 stacks compared against the catalog containing 860147 loci.\n",
      "  81440 matching loci, 385 contained no verified haplotypes.\n",
      "  156 loci matched more than one catalog locus and were excluded.\n",
      "  229 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83893 total haplotypes examined from matching loci, 83267 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079' [152 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93789 stacks compared against the catalog containing 860147 loci.\n",
      "  83044 matching loci, 526 contained no verified haplotypes.\n",
      "  183 loci matched more than one catalog locus and were excluded.\n",
      "  343 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85354 total haplotypes examined from matching loci, 84543 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080' [153 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96341 stacks compared against the catalog containing 860147 loci.\n",
      "  83593 matching loci, 487 contained no verified haplotypes.\n",
      "  172 loci matched more than one catalog locus and were excluded.\n",
      "  315 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86234 total haplotypes examined from matching loci, 85484 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081' [154 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91959 stacks compared against the catalog containing 860147 loci.\n",
      "  83775 matching loci, 454 contained no verified haplotypes.\n",
      "  167 loci matched more than one catalog locus and were excluded.\n",
      "  287 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86606 total haplotypes examined from matching loci, 85884 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082' [155 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89369 stacks compared against the catalog containing 860147 loci.\n",
      "  82174 matching loci, 429 contained no verified haplotypes.\n",
      "  177 loci matched more than one catalog locus and were excluded.\n",
      "  252 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84770 total haplotypes examined from matching loci, 84078 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083' [156 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95827 stacks compared against the catalog containing 860147 loci.\n",
      "  84952 matching loci, 536 contained no verified haplotypes.\n",
      "  184 loci matched more than one catalog locus and were excluded.\n",
      "  352 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87579 total haplotypes examined from matching loci, 86736 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084' [157 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84155 stacks compared against the catalog containing 860147 loci.\n",
      "  79645 matching loci, 892 contained no verified haplotypes.\n",
      "  144 loci matched more than one catalog locus and were excluded.\n",
      "  748 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81829 total haplotypes examined from matching loci, 80728 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085' [158 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81658 stacks compared against the catalog containing 860147 loci.\n",
      "  77874 matching loci, 1134 contained no verified haplotypes.\n",
      "  169 loci matched more than one catalog locus and were excluded.\n",
      "  965 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80008 total haplotypes examined from matching loci, 78653 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086' [159 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78555 stacks compared against the catalog containing 860147 loci.\n",
      "  75690 matching loci, 1183 contained no verified haplotypes.\n",
      "  145 loci matched more than one catalog locus and were excluded.\n",
      "  1039 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77681 total haplotypes examined from matching loci, 76288 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087' [160 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82140 stacks compared against the catalog containing 860147 loci.\n",
      "  79317 matching loci, 642 contained no verified haplotypes.\n",
      "  152 loci matched more than one catalog locus and were excluded.\n",
      "  490 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81438 total haplotypes examined from matching loci, 80572 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088' [161 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78171 stacks compared against the catalog containing 860147 loci.\n",
      "  75263 matching loci, 1110 contained no verified haplotypes.\n",
      "  163 loci matched more than one catalog locus and were excluded.\n",
      "  947 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77327 total haplotypes examined from matching loci, 76005 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089' [162 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "72060 stacks compared against the catalog containing 860147 loci.\n",
      "  69802 matching loci, 1345 contained no verified haplotypes.\n",
      "  135 loci matched more than one catalog locus and were excluded.\n",
      "  1210 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  71455 total haplotypes examined from matching loci, 69932 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090' [163 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "75547 stacks compared against the catalog containing 860147 loci.\n",
      "  72342 matching loci, 1254 contained no verified haplotypes.\n",
      "  135 loci matched more than one catalog locus and were excluded.\n",
      "  1119 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  74239 total haplotypes examined from matching loci, 72794 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091' [164 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85025 stacks compared against the catalog containing 860147 loci.\n",
      "  81020 matching loci, 389 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  240 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83362 total haplotypes examined from matching loci, 82768 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092' [165 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77390 stacks compared against the catalog containing 860147 loci.\n",
      "  74600 matching loci, 1320 contained no verified haplotypes.\n",
      "  138 loci matched more than one catalog locus and were excluded.\n",
      "  1182 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  76595 total haplotypes examined from matching loci, 75087 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093' [166 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82157 stacks compared against the catalog containing 860147 loci.\n",
      "  78582 matching loci, 857 contained no verified haplotypes.\n",
      "  133 loci matched more than one catalog locus and were excluded.\n",
      "  724 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80709 total haplotypes examined from matching loci, 79657 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094' [167 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84542 stacks compared against the catalog containing 860147 loci.\n",
      "  80484 matching loci, 428 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  271 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82726 total haplotypes examined from matching loci, 82085 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095' [168 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81429 stacks compared against the catalog containing 860147 loci.\n",
      "  78119 matching loci, 1044 contained no verified haplotypes.\n",
      "  149 loci matched more than one catalog locus and were excluded.\n",
      "  895 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80210 total haplotypes examined from matching loci, 78955 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096' [169 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82999 stacks compared against the catalog containing 860147 loci.\n",
      "  79042 matching loci, 837 contained no verified haplotypes.\n",
      "  142 loci matched more than one catalog locus and were excluded.\n",
      "  695 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81162 total haplotypes examined from matching loci, 80120 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097' [170 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "42002 stacks compared against the catalog containing 860147 loci.\n",
      "  39810 matching loci, 860 contained no verified haplotypes.\n",
      "  95 loci matched more than one catalog locus and were excluded.\n",
      "  765 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  40630 total haplotypes examined from matching loci, 39651 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098' [171 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "21033 stacks compared against the catalog containing 860147 loci.\n",
      "  18505 matching loci, 384 contained no verified haplotypes.\n",
      "  62 loci matched more than one catalog locus and were excluded.\n",
      "  322 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  18864 total haplotypes examined from matching loci, 18405 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099' [172 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "53927 stacks compared against the catalog containing 860147 loci.\n",
      "  50459 matching loci, 779 contained no verified haplotypes.\n",
      "  121 loci matched more than one catalog locus and were excluded.\n",
      "  658 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  51661 total haplotypes examined from matching loci, 50716 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100' [173 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "42576 stacks compared against the catalog containing 860147 loci.\n",
      "  39614 matching loci, 585 contained no verified haplotypes.\n",
      "  96 loci matched more than one catalog locus and were excluded.\n",
      "  489 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  40693 total haplotypes examined from matching loci, 39985 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101' [174 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "20079 stacks compared against the catalog containing 860147 loci.\n",
      "  18590 matching loci, 363 contained no verified haplotypes.\n",
      "  65 loci matched more than one catalog locus and were excluded.\n",
      "  298 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  18976 total haplotypes examined from matching loci, 18533 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102' [175 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "63003 stacks compared against the catalog containing 860147 loci.\n",
      "  60533 matching loci, 1200 contained no verified haplotypes.\n",
      "  140 loci matched more than one catalog locus and were excluded.\n",
      "  1060 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  62025 total haplotypes examined from matching loci, 60637 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103' [176 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "79366 stacks compared against the catalog containing 860147 loci.\n",
      "  75897 matching loci, 812 contained no verified haplotypes.\n",
      "  146 loci matched more than one catalog locus and were excluded.\n",
      "  666 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78036 total haplotypes examined from matching loci, 77031 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104' [177 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "70017 stacks compared against the catalog containing 860147 loci.\n",
      "  66987 matching loci, 1239 contained no verified haplotypes.\n",
      "  121 loci matched more than one catalog locus and were excluded.\n",
      "  1118 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  68559 total haplotypes examined from matching loci, 67164 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105' [178 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77502 stacks compared against the catalog containing 860147 loci.\n",
      "  73725 matching loci, 788 contained no verified haplotypes.\n",
      "  132 loci matched more than one catalog locus and were excluded.\n",
      "  656 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  75808 total haplotypes examined from matching loci, 74836 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106' [179 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "29393 stacks compared against the catalog containing 860147 loci.\n",
      "  26627 matching loci, 440 contained no verified haplotypes.\n",
      "  70 loci matched more than one catalog locus and were excluded.\n",
      "  370 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  27197 total haplotypes examined from matching loci, 26662 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107' [180 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "56185 stacks compared against the catalog containing 860147 loci.\n",
      "  51086 matching loci, 667 contained no verified haplotypes.\n",
      "  96 loci matched more than one catalog locus and were excluded.\n",
      "  571 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  52297 total haplotypes examined from matching loci, 51507 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109' [181 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "11120 stacks compared against the catalog containing 860147 loci.\n",
      "  9593 matching loci, 161 contained no verified haplotypes.\n",
      "  46 loci matched more than one catalog locus and were excluded.\n",
      "  115 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  9784 total haplotypes examined from matching loci, 9564 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110' [182 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "36876 stacks compared against the catalog containing 860147 loci.\n",
      "  33837 matching loci, 575 contained no verified haplotypes.\n",
      "  104 loci matched more than one catalog locus and were excluded.\n",
      "  471 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  34635 total haplotypes examined from matching loci, 33926 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111' [183 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90811 stacks compared against the catalog containing 860147 loci.\n",
      "  82090 matching loci, 503 contained no verified haplotypes.\n",
      "  158 loci matched more than one catalog locus and were excluded.\n",
      "  345 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84572 total haplotypes examined from matching loci, 83825 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113' [184 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82904 stacks compared against the catalog containing 860147 loci.\n",
      "  79433 matching loci, 537 contained no verified haplotypes.\n",
      "  131 loci matched more than one catalog locus and were excluded.\n",
      "  406 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81706 total haplotypes examined from matching loci, 80998 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114' [185 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "40129 stacks compared against the catalog containing 860147 loci.\n",
      "  37517 matching loci, 494 contained no verified haplotypes.\n",
      "  108 loci matched more than one catalog locus and were excluded.\n",
      "  386 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  38393 total haplotypes examined from matching loci, 37768 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115' [186 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84366 stacks compared against the catalog containing 860147 loci.\n",
      "  76142 matching loci, 588 contained no verified haplotypes.\n",
      "  171 loci matched more than one catalog locus and were excluded.\n",
      "  417 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78766 total haplotypes examined from matching loci, 77941 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116' [187 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "54237 stacks compared against the catalog containing 860147 loci.\n",
      "  50225 matching loci, 741 contained no verified haplotypes.\n",
      "  139 loci matched more than one catalog locus and were excluded.\n",
      "  602 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  51473 total haplotypes examined from matching loci, 50559 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117' [188 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82858 stacks compared against the catalog containing 860147 loci.\n",
      "  76500 matching loci, 751 contained no verified haplotypes.\n",
      "  157 loci matched more than one catalog locus and were excluded.\n",
      "  594 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78855 total haplotypes examined from matching loci, 77875 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122' [189 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "53999 stacks compared against the catalog containing 860147 loci.\n",
      "  50460 matching loci, 538 contained no verified haplotypes.\n",
      "  110 loci matched more than one catalog locus and were excluded.\n",
      "  428 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  51771 total haplotypes examined from matching loci, 51092 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123' [190 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85454 stacks compared against the catalog containing 860147 loci.\n",
      "  79678 matching loci, 659 contained no verified haplotypes.\n",
      "  168 loci matched more than one catalog locus and were excluded.\n",
      "  491 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82119 total haplotypes examined from matching loci, 81217 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124' [191 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90664 stacks compared against the catalog containing 860147 loci.\n",
      "  81597 matching loci, 491 contained no verified haplotypes.\n",
      "  139 loci matched more than one catalog locus and were excluded.\n",
      "  352 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84172 total haplotypes examined from matching loci, 83441 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.matches.tsv.gz\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126' [192 of 192]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "97144 stacks compared against the catalog containing 860147 loci.\n",
      "  83125 matching loci, 585 contained no verified haplotypes.\n",
      "  164 loci matched more than one catalog locus and were excluded.\n",
      "  421 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85734 total haplotypes examined from matching loci, 84864 verified.\n",
      "Outputing to file /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n"
     ]
    }
   ],
   "source": [
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126' \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "populations parameters selected:\n",
      "  Fst kernel smoothing: off\n",
      "  Bootstrap resampling: off\n",
      "  Percent samples limit per population: 0\n",
      "  Locus Population limit: 1\n",
      "  Minimum stack depth: 0\n",
      "  Log liklihood filtering: off; threshold: 0\n",
      "  Minor allele frequency cutoff: 0\n",
      "  Maximum observed heterozygosity cutoff: 1\n",
      "  Applying Fst correction: none.\n",
      "\n",
      "Parsing population map...\n",
      "The population map contained 192 samples, 3 population(s), 1 group(s).\n",
      "Reading the catalog...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "Reading matches to the catalog...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.matches.tsv.gz\n",
      "Catalog is not reference aligned, arbitrarily ordering catalog loci.\n",
      "Working on 192 samples.\n",
      "Working on 3 population(s):\n",
      "    101: PBIRD14X101H_0002, PBIRD14X101H_0003, PBIRD14X101H_0005, PBIRD14X101H_0007, PBIRD14X101H_0009, PBIRD14X101H_0010, PBIRD14X101H_0011, PBIRD14X101H_0012, PBIRD14X101H_0013, PBIRD14X101H_0014, PBIRD14X101H_0015, PBIRD14X101H_0016, PBIRD14X101H_0017, PBIRD14X101H_0018, PBIRD14X101H_0019, PBIRD14X101H_0020, PBIRD14X101H_0023, PBIRD14X101H_0024, PBIRD14X101H_0025, PBIRD14X101H_0026, PBIRD14X101H_0027, PBIRD14X101H_0029, PBIRD14X101H_0031, PBIRD14X101H_0032, PBIRD14X101H_0033, PBIRD14X101H_0034, PBIRD14X101H_0035, PBIRD14X101H_0038, PBIRD14X101H_0039, PBIRD14X101H_0040, PBIRD14X101H_0041, PBIRD14X101H_0047, PBIRD14X101H_0049, PBIRD14X101H_0050, PBIRD14X101H_0053, PBIRD14X101H_0055, PBIRD14X101H_0056, PBIRD14X101H_0058, PBIRD14X101H_0059, PBIRD14X101H_0062, PBIRD14X101H_0066, PBIRD14X101H_0067, PBIRD14X101H_0069, PBIRD14X101H_0070, PBIRD14X101H_0073, PBIRD14X101H_0075, PBIRD14X101H_0078, PBIRD14X101H_0101, PBIRD14X101H_0104, PBIRD14X_0101\n",
      "    107: PBIRD14X107H_0001, PBIRD14X107H_0002, PBIRD14X107H_0003, PBIRD14X107H_0004, PBIRD14X107H_0005, PBIRD14X107H_0006, PBIRD14X107H_0007, PBIRD14X107H_0008, PBIRD14X107H_0009, PBIRD14X107H_0010, PBIRD14X107H_0011, PBIRD14X107H_0012, PBIRD14X107H_0013, PBIRD14X107H_0014, PBIRD14X107H_0015, PBIRD14X107H_0016, PBIRD14X107H_0017, PBIRD14X107H_0018, PBIRD14X107H_0019, PBIRD14X107H_0020, PBIRD14X107H_0021, PBIRD14X107H_0022, PBIRD14X107H_0023, PBIRD14X107H_0024, PBIRD14X107H_0025, PBIRD14X107H_0026, PBIRD14X107H_0027, PBIRD14X107H_0028, PBIRD14X107H_0029, PBIRD14X107H_0030, PBIRD14X107H_0031, PBIRD14X107H_0032, PBIRD14X107H_0033, PBIRD14X107H_0034, PBIRD14X107H_0035, PBIRD14X107H_0036, PBIRD14X107H_0037, PBIRD14X107H_0038, PBIRD14X107H_0039, PBIRD14X107H_0040, PBIRD14X107H_0041, PBIRD14X107H_0042, PBIRD14X107H_0043, PBIRD14X107H_0044, PBIRD14X107H_0045, PBIRD14X107H_0046, PBIRD14X107H_0047, PBIRD14X107H_0048, PBIRD14X107H_0049, PBIRD14X107H_0050, PBIRD14X107H_0051, PBIRD14X107H_0052, PBIRD14X107H_0053, PBIRD14X107H_0054, PBIRD14X107H_0055, PBIRD14X107H_0056, PBIRD14X107H_0057, PBIRD14X107H_0059, PBIRD14X107H_0060, PBIRD14X107H_0061, PBIRD14X107H_0062, PBIRD14X107H_0063, PBIRD14X107H_0064, PBIRD14X107H_0065, PBIRD14X107H_0066, PBIRD14X107H_0067, PBIRD14X107H_0068, PBIRD14X107H_0069, PBIRD14X107H_0070, PBIRD14X107H_0071, PBIRD14X107H_0072, PBIRD14X107H_0073, PBIRD14X107H_0074, PBIRD14X107H_0075, PBIRD14X107H_0076, PBIRD14X107H_0077, PBIRD14X107H_0078, PBIRD14X107H_0079, PBIRD14X107H_0080, PBIRD14X107H_0081, PBIRD14X107H_0082, PBIRD14X107H_0083, PBIRD14X107H_0084, PBIRD14X107H_0085, PBIRD14X107H_0086, PBIRD14X107H_0087, PBIRD14X107H_0088, PBIRD14X107H_0089, PBIRD14X107H_0090, PBIRD14X107H_0091, PBIRD14X107H_0092, PBIRD14X107H_0093, PBIRD14X107H_0094, PBIRD14X107H_0095, PBIRD14X107H_0096, PBIRD14X107H_0097, PBIRD14X107H_0098, PBIRD14X107H_0099, PBIRD14X107H_0100, PBIRD14X107H_0101, PBIRD14X107H_0102, PBIRD14X107H_0103, PBIRD14X107H_0104, PBIRD14X107H_0105, PBIRD14X107H_0106, PBIRD14X107H_0107, PBIRD14X107H_0109, PBIRD14X107H_0110, PBIRD14X107H_0111, PBIRD14X107H_0113, PBIRD14X107H_0114, PBIRD14X107H_0115, PBIRD14X107H_0116, PBIRD14X107H_0117, PBIRD14X107H_0122, PBIRD14X107H_0123, PBIRD14X107H_0124, PBIRD14X107H_0126, PBIRD14X_0107\n",
      "    101G: PBIRD14X101G_0001, PBIRD14X101G_0002, PBIRD14X101G_0003, PBIRD14X101G_0004, PBIRD14X101G_0005, PBIRD14X101G_0006, PBIRD14X101G_0007, PBIRD14X101G_0008, PBIRD14X101G_0009, PBIRD14X101G_0010, PBIRD14X101G_0011, PBIRD14X101G_0012, PBIRD14X101G_0013, PBIRD14X101G_0014, PBIRD14X101G_0015, PBIRD14X101G_0016, PBIRD14X101G_0017, PBIRD14X101G_0018, PBIRD14X101G_0019, PBIRD14X101G_0020, PBIRD14X101G_0021, PBIRD14X101G_0022, PBIRD14X101G_0023\n",
      "Working on 1 group(s) of populations:\n",
      "    defaultgrp: 101, 107, 101G\n",
      "Populating observed haplotypes for 192 samples, 860147 loci.\n",
      "Writing SQL markers file to '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.markers.tsv'\n",
      "Removing 690860 loci that did not pass sample/population constraints... retained 169287 loci.\n",
      "Loading model outputs for 192 samples, 169287 loci.\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0010.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.models.tsv.gz\n",
      "Generating nucleotide-level summary statistics for population '101'\n",
      "Population '101' contained 191 incompatible loci -- more than two alleles present.\n",
      "Generating nucleotide-level summary statistics for population '107'\n",
      "Population '107' contained 451 incompatible loci -- more than two alleles present.\n",
      "Generating nucleotide-level summary statistics for population '101G'\n",
      "Population '101G' contained 69 incompatible loci -- more than two alleles present.\n",
      "Tallying loci across populations...done.\n",
      "Pruned 348 variant sites due to filter constraints (more with --verbose).\n",
      "Removing 60845 additional loci for which all variant sites were filtered... retained 108442 loci.\n",
      "Regenerating nucleotide-level summary statistics for population '101'\n",
      "Population '101' contained 0 incompatible loci -- more than two alleles present.\n",
      "Regenerating nucleotide-level summary statistics for population '107'\n",
      "Population '107' contained 0 incompatible loci -- more than two alleles present.\n",
      "Regenerating nucleotide-level summary statistics for population '101G'\n",
      "Population '101G' contained 0 incompatible loci -- more than two alleles present.\n",
      "Re-tallying loci across populations...done.\n",
      "Generating haplotype-level summary statistics for population '101'\n",
      "Generating haplotype-level summary statistics for population '107'\n",
      "Generating haplotype-level summary statistics for population '101G'\n",
      "Writing 108442 loci to summary statistics file, '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.sumstats.tsv'\n",
      "Writing 108442 loci to observed haplotype file, '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.haplotypes.tsv'\n",
      "Writing population data to GenePop file '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.genepop'\n",
      "Writing population data to PLINK files...done.\n",
      "Populations is done.\n"
     ]
    }
   ],
   "source": [
    "## Running populations without filtering to do all in plink and Waples pipeline\n",
    "! populations -P '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -M '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014PopulationsMap.txt' -b 5 -O '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations' -t 7 -s --genepop --plink --verbose\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Remove consensus and tally missing data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/removeConsensus.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.haplotypes.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensus.tsv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/countHetsMissing.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensus.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HetsMissingCountsInds.txt'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/countHetsMissingLoci.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensus.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HetsMissingCountsLoci.txt'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "It looks like sequencing rounds have been combined correctly based upon the percent of loci that are heterozygous.  \n",
    "Missing data looks a little high, a 40% missing data cut-off will likely remove poor quality individuals.  The rate for offspring from 101H might be lower.\n",
    "\n",
    "I'll filter individuals first and then loci prior to Waples pipeline."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Remove individuals missing 40%+ loci and then reevaluate missingness of loci to filter loci\n",
    "### 40% criterion selected after observing distributions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/filterHaplotypeIndividuals.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird201440PercentMissingWhitelistInds.txt' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensus.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensusFilteredInds.tsv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/countHetsMissingLoci.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensusFilteredInds.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HetsMissingCountsLociPostFilteringInds.txt'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Filtering loci by family based upon 80% rule (only including loci genotyped in 80%+ individuals within a family (101H and 107H)\n",
    "### First subsetting individuals from families"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/filterHaplotypeIndividuals.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family101HWhitelistInds.txt' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensusFilteredInds.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family101HaplotypesNoConsensusFilteredInds.tsv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/filterHaplotypeIndividuals.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family107HWhitelistInds.txt' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensusFilteredInds.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family107HaplotypesNoConsensusFilteredInds.tsv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/filterHaplotypeLoci.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family101HLoci80GenoWhitelist.txt' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family101HaplotypesNoConsensusFilteredInds.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family101HaplotypesNoConsensusFilteredIndsLoci.tsv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/filterHaplotypeLoci.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family107HLoci80GenoWhitelist.txt' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family107HaplotypesNoConsensusFilteredInds.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family107HaplotypesNoConsensusFilteredIndsLoci.tsv'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Also filtering loci for the combined, two-family dataset, including only loci genotyped in 80%+ of all individuals (not family-specific as above)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!perl '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/filterHaplotypeLoci.pl' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/BothFamiliesLoci80GenoWhitelist.txt' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Bird2014HaplotypesNoConsensusFilteredInds.tsv' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/BothFamiliesHaplotypesNoConsensusFilteredIndsLoci.tsv'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prepare files for Ryan's pipeline\n",
    "\n",
    "Exclude gynogenetic diploids (done above), order parents first followed by offspring, add five columns between catalog number and number of offspring, and five between number of offspring and first parent:\n",
    "Catalog ID Annotation Chr BP Consensus Sequence Num Parents Num Progeny Num SNPs SNPs Num Alleles Alleles Deleveraged PBIRD14X_0107\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "os.chdir(\"/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Stacks_mapping/Python\")\n",
    "\n",
    "import _pickle as pickle\n",
    "import numpy\n",
    "\n",
    "import Stacks_SQL_class\n",
    "from models_to_test import models_to_test\n",
    "\n",
    "#Added to attempt phase switching\n",
    "import switch_allele_functions as switchAlleles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "## Move family files into pipeline directory\n",
    "! cp '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family101HapFileFormattedForWaplesPipeline.csv' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Family101HapFileFormattedForWaplesPipeline.csv'\n",
    "! cp '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/Family107HapFileFormattedForWaplesPipeline.csv' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Family107HapFileFormattedForWaplesPipeline.csv'\n",
    "! cp '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/BothFamiliesHapFileFormattedForWaplesPipeline.csv' '/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/BothFamiliesHapFileFormattedForWaplesPipeline.csv'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "unsupported operand type(s) for +: 'zip' and 'list'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-37-9a96c27a613a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      6\u001b[0m     \u001b[0mnum_parents\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m1\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m     \u001b[0mnum_offspring\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m42\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 8\u001b[0;31m     \u001b[0mparent_offspring\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;36m42\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      9\u001b[0m     )\n",
      "\u001b[0;32m/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Stacks_mapping/Python/Stacks_SQL_class.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, path, name, num_parents, num_offspring, parent_offspring)\u001b[0m\n\u001b[1;32m     84\u001b[0m                                                 \u001b[0moffspring_genotypes\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0moffspring_genotype\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     85\u001b[0m                                             \u001b[0mzipped_offspring_genotypes\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0moffspring_of_parent\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mparent_str\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0moffspring_genotypes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 86\u001b[0;31m                                             \u001b[0mcross_genotypes\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mCatID_Cross_Genotypes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcurrent_catID\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzipped_offspring_genotypes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzipped_parent_genotypes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     87\u001b[0m                                             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0madd_catID_cross_genotypes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcurrent_catID\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mparent_str\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcross_genotypes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     88\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Stacks_mapping/Python/CatID_Genotypes_class.py\u001b[0m in \u001b[0;36m__init__\u001b[0;34m(self, catID, zipped_offspring_genotypes, zipped_parent_genotypes, alleles, missing_codes)\u001b[0m\n\u001b[1;32m     19\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mallowed_alleles\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0malleles\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0munion\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"-\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     20\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 21\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mset_alleles\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mzipped_offspring_genotypes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzipped_parent_genotypes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     22\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     23\u001b[0m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0moffspring_genotypes\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mdict\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Stacks_mapping/Python/CatID_Genotypes_class.py\u001b[0m in \u001b[0;36mset_alleles\u001b[0;34m(self, zipped_offspring_genotypes, zipped_parent_genotypes)\u001b[0m\n\u001b[1;32m     96\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     97\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mset_alleles\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzipped_offspring_genotypes\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mzipped_parent_genotypes\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 98\u001b[0;31m         \u001b[0mzipped_genotypes\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mzipped_offspring_genotypes\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mzipped_parent_genotypes\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     99\u001b[0m         \u001b[0mseen\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    100\u001b[0m         \u001b[0;31m#print('set_alleles')\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: unsupported operand type(s) for +: 'zip' and 'list'"
     ]
    }
   ],
   "source": [
    "# Load data created with export_sql.pl \n",
    "\n",
    "Bird101Fam = Stacks_SQL_class.Stacks_SQL(\n",
    "    path = \"/media/tyler/My Book/Stacks/Pink/Haploids2014/Waples_PSV_pipeline_files/Family101HapFileFormattedForWaplesPipeline.csv\", \n",
    "    name = 'Bird101',\n",
    "    num_parents = 1, \n",
    "    num_offspring = 42, \n",
    "    parent_offspring = (42,)\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/usr/bin/python2: No module named install\r\n"
     ]
    }
   ],
   "source": [
    "!python2 -m pip install ipykernel"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
