{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Background on rounds of sequencing"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Two families were chosen for sequencing, offspring from female 101 and female 110.  Both gynogenetic diploids and haploids were included in library prep and sequencing with the following sample sizes: \n",
    "101: haploid=49, gyno diploid=23\n",
    "107: haploid=118\n",
    "Together with dams 101 and 102 we prepared two libraries of 96 individuals each.\n",
    "\n",
    "Libraries were mistakenly swapped on the instrument during the first round of sequencing, resulting in incorrect titrations for the second round of sequencing and inbalanced coverage among individuals.  A third round of sequencing was conducted but didn't result in adequate coverage for the smaller group of 101 haploid individuals to be useful in estimating recombination frequencies.  Finally, we used the instrument training run on ADF&G's NextSeq 500 to try and resurrect 101 haploid individuals and incorporate variation from dam 101 on the linkage map."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Processing .fastq from final round of sequencing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing paired-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 32 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Undetermined_S0_R1_001.fastq.gz]\n",
      "  Reading data from:\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R1_001.fastq.gz and\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R2_001.fastq.gz\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...159M...160M...161M...162M...163M...164M...165M...166M...167M...168M...169M...170M...171M...172M...173M...174M...175M...176M...177M...178M...179M...180M...181M...182M...183M...184M...185M...186M...187M...188M...189M...190M...191M...192M...193M...194M...195M...196M...197M...198M...199M...200M...201M...202M...203M...204M...205M...206M...207M...208M...209M...210M...211M...212M...213M...214M...215M...216M...217M...218M...219M...220M...221M...222M...223M...224M...225M...226M...227M...228M...229M...230M...231M...232M...233M...234M...235M...236M...237M...238M...239M...240M...241M...242M...243M...244M...245M...246M...247M...248M...249M...250M...251M...252M...253M...254M...255M...256M...257M...258M...259M...260M...261M...262M...263M...264M...265M...266M...267M...268M...269M...270M...271M...272M...273M...274M...275M...276M...277M...278M...279M...280M...281M...282M...283M...284M...285M...286M...287M...288M...289M...290M...291M...292M...293M...294M...295M...296M...297M...298M...299M...300M...301M...302M...303M...304M...305M...306M...307M...308M...309M...310M...311M...312M...313M...314M...315M...316M...317M...318M...319M...320M...321M...322M...323M...324M...325M...326M...327M...328M...329M...330M...331M...332M...333M...334M...335M...336M...337M...338M...339M...340M...341M...342M...343M...344M...345M...346M...347M...348M...349M...350M...351M...352M...353M...354M...355M...356M...357M...358M...359M...360M...361M...362M...363M...364M...365M...366M...367M...368M...369M...370M...371M...372M...373M...374M...375M...376M...377M...378M...379M...380M...381M...382M...383M...384M...385M...386M...387M...388M...389M...390M...391M...392M...393M...394M...395M...396M...397M...398M...399M...400M...401M...402M...403M...404M...405M...406M...407M...408M...409M...410M...411M...412M...413M...414M...415M...416M...417M...418M...419M...420M...421M...422M...423M...424M...425M...426M...427M...428M...429M...430M...431M...432M...433M...434M...435M...436M...437M...438M...439M...440M...441M...442M...443M...444M...445M...446M...447M...448M...449M...450M...451M...452M...453M...454M...455M...456M...457M...458M...459M...460M...461M...462M...463M...464M...465M...466M...467M...468M...469M...470M...471M...472M...473M...474M...475M...476M...477M...478M...479M...480M...481M...482M...483M...484M...485M...486M...487M...488M...489M...490M...491M...492M...493M...494M...495M...496M...497M...\n",
      "  995296694 total reads; -0 failed Illumina reads; -789393602 ambiguous barcodes; -73115971 ambiguous RAD-Tags; +132998688 recovered; -755123 low quality reads; 132031998 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/process_radtags.fastq.log'\n",
      "\n",
      "995296694 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      "789393602 ambiguous barcode drops (79.3%)\n",
      "   755123 low quality read drops (0.1%)\n",
      " 73115971 ambiguous RAD-Tag drops (7.3%)\n",
      "132031998 retained reads (13.3%)\n"
     ]
    }
   ],
   "source": [
    "! process_radtags -P -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/Bird2014TrainingRunBarcodes.txt' -E phred33 -e sbfI -i gzfastq -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Same process radtags but trimming to 94 bases to align with previously collected data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing paired-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 32 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Undetermined_S0_R1_001.fastq.gz]\n",
      "  Reading data from:\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R1_001.fastq.gz and\n",
      "  /media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R2_001.fastq.gz\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...159M...160M...161M...162M...163M...164M...165M...166M...167M...168M...169M...170M...171M...172M...173M...174M...175M...176M...177M...178M...179M...180M...181M...182M...183M...184M...185M...186M...187M...188M...189M...190M...191M...192M...193M...194M...195M...196M...197M...198M...199M...200M...201M...202M...203M...204M...205M...206M...207M...208M...209M...210M...211M...212M...213M...214M...215M...216M...217M...218M...219M...220M...221M...222M...223M...224M...225M...226M...227M...228M...229M...230M...231M...232M...233M...234M...235M...236M...237M...238M...239M...240M...241M...242M...243M...244M...245M...246M...247M...248M...249M...250M...251M...252M...253M...254M...255M...256M...257M...258M...259M...260M...261M...262M...263M...264M...265M...266M...267M...268M...269M...270M...271M...272M...273M...274M...275M...276M...277M...278M...279M...280M...281M...282M...283M...284M...285M...286M...287M...288M...289M...290M...291M...292M...293M...294M...295M...296M...297M...298M...299M...300M...301M...302M...303M...304M...305M...306M...307M...308M...309M...310M...311M...312M...313M...314M...315M...316M...317M...318M...319M...320M...321M...322M...323M...324M...325M...326M...327M...328M...329M...330M...331M...332M...333M...334M...335M...336M...337M...338M...339M...340M...341M...342M...343M...344M...345M...346M...347M...348M...349M...350M...351M...352M...353M...354M...355M...356M...357M...358M...359M...360M...361M...362M...363M...364M...365M...366M...367M...368M...369M...370M...371M...372M...373M...374M...375M...376M...377M...378M...379M...380M...381M...382M...383M...384M...385M...386M...387M...388M...389M...390M...391M...392M...393M...394M...395M...396M...397M...398M...399M...400M...401M...402M...403M...404M...405M...406M...407M...408M...409M...410M...411M...412M...413M...414M...415M...416M...417M...418M...419M...420M...421M...422M...423M...424M...425M...426M...427M...428M...429M...430M...431M...432M...433M...434M...435M...436M...437M...438M...439M...440M...441M...442M...443M...444M...445M...446M...447M...448M...449M...450M...451M...452M...453M...454M...455M...456M...457M...458M...459M...460M...461M...462M...463M...464M...465M...466M...467M...468M...469M...470M...471M...472M...473M...474M...475M...476M...477M...478M...479M...480M...481M...482M...483M...484M...485M...486M...487M...488M...489M...490M...491M...492M...493M...494M...495M...496M...497M...\n",
      "  995296694 total reads; -0 failed Illumina reads; -789393602 ambiguous barcodes; -73115971 ambiguous RAD-Tags; +132998688 recovered; -675118 low quality reads; 132112003 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94/process_radtags.fastq.log'\n",
      "\n",
      "995296694 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      "789393602 ambiguous barcode drops (79.3%)\n",
      "   675118 low quality read drops (0.1%)\n",
      " 73115971 ambiguous RAD-Tag drops (7.3%)\n",
      "132112003 retained reads (13.3%)\n"
     ]
    }
   ],
   "source": [
    "! process_radtags -P -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/t94' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/Bird2014TrainingRunBarcodes.txt' -E phred33 -e sbfI -i gzfastq -t 94 -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Description of the four rounds of sequencing for 2014 Bird Creek haploids"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Protocol when the two 2014 Bird Creek libraries (PHap1 and PHap2) were created was to conduct two rounds of sequencing to equalize coverage among individuals, titrating before the second round according to retained reads from the first round.\n",
    "\n",
    "Unfortunately the two libraries were swapped when sequenced at the Core lab for the first round, resulting in incorrect titrations and no equalizing of coverage.  A third round was done on a subset of individuals to rescue them from too few reads.  The final round (unpacked above) was acquired during ADF&G's training run.\n",
    "\n",
    "\n",
    "|         |                        | Barcode key used in: |               |                     |                 |\n",
    "|---------|------------------------|:--------------------:|:-------------:|:-------------------:|:---------------:|\n",
    "| Library | Originally planned key | Round 1              | ReRun         | ReReRun             | Round 4         |\n",
    "| PHap1   | PHap1Barcodes          | PHap2Barcodes        | PHap1Barcodes | PHapReReSeqBarcodes | PHap1 (partial) |\n",
    "| PHap2   | PHap2Barcodes          | PHap1Barcodes        | PHap2Barcodes |                     |                 |"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 59 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Phaps2014_NoIndex_S97_L007_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...\n",
      "  158449390 total reads; -0 failed Illumina reads; -56117100 ambiguous barcodes; -20981990 ambiguous RAD-Tags; +19354516 recovered; -996062 low quality reads; 80354238 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/process_radtags.PHapReReSeq.log'\n",
      "\n",
      "158449390 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      " 56117100 ambiguous barcode drops (35.4%)\n",
      "   996062 low quality read drops (0.6%)\n",
      " 20981990 ambiguous RAD-Tag drops (13.2%)\n",
      " 80354238 retained reads (50.7%)\n"
     ]
    }
   ],
   "source": [
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeq' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqBarcodes.txt' -E phred33 -e sbfI -i gzfastq -t 94 -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 15 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 15 [PHap1_lane5_NoIndex_L005_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1841729 failed Illumina reads; -4861645 ambiguous barcodes; -3275420 ambiguous RAD-Tags; +3998931 recovered; -413159 low quality reads; 5608047 retained reads.\n",
      "Processing file 2 of 15 [PHap1_lane5_NoIndex_L005_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1442672 failed Illumina reads; -4856855 ambiguous barcodes; -3385188 ambiguous RAD-Tags; +3817593 recovered; -507293 low quality reads; 5807992 retained reads.\n",
      "Processing file 3 of 15 [PHap1_lane5_NoIndex_L005_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1768721 failed Illumina reads; -4849842 ambiguous barcodes; -3297460 ambiguous RAD-Tags; +3877673 recovered; -437433 low quality reads; 5646544 retained reads.\n",
      "Processing file 4 of 15 [PHap1_lane5_NoIndex_L005_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1525150 failed Illumina reads; -4870371 ambiguous barcodes; -3362990 ambiguous RAD-Tags; +3910295 recovered; -496853 low quality reads; 5744636 retained reads.\n",
      "Processing file 5 of 15 [PHap1_lane5_NoIndex_L005_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1574810 failed Illumina reads; -4867386 ambiguous barcodes; -3351296 ambiguous RAD-Tags; +3831048 recovered; -447826 low quality reads; 5758682 retained reads.\n",
      "Processing file 6 of 15 [PHap1_lane5_NoIndex_L005_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1750301 failed Illumina reads; -4871056 ambiguous barcodes; -3295240 ambiguous RAD-Tags; +4018245 recovered; -449776 low quality reads; 5633627 retained reads.\n",
      "Processing file 7 of 15 [PHap1_lane5_NoIndex_L005_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1458112 failed Illumina reads; -4859683 ambiguous barcodes; -3387927 ambiguous RAD-Tags; +3818661 recovered; -454998 low quality reads; 5839280 retained reads.\n",
      "Processing file 8 of 15 [PHap1_lane5_NoIndex_L005_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1268941 failed Illumina reads; -4847234 ambiguous barcodes; -3460987 ambiguous RAD-Tags; +3968631 recovered; -423914 low quality reads; 5998924 retained reads.\n",
      "Processing file 9 of 15 [PHap1_lane5_NoIndex_L005_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1630656 failed Illumina reads; -4849660 ambiguous barcodes; -3350808 ambiguous RAD-Tags; +4133675 recovered; -394710 low quality reads; 5774166 retained reads.\n",
      "Processing file 10 of 15 [PHap1_lane5_NoIndex_L005_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1701429 failed Illumina reads; -4868733 ambiguous barcodes; -3313981 ambiguous RAD-Tags; +4164742 recovered; -395174 low quality reads; 5720683 retained reads.\n",
      "Processing file 11 of 15 [PHap1_lane5_NoIndex_L005_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1673024 failed Illumina reads; -4840081 ambiguous barcodes; -3327779 ambiguous RAD-Tags; +3879115 recovered; -502256 low quality reads; 5656860 retained reads.\n",
      "Processing file 12 of 15 [PHap1_lane5_NoIndex_L005_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -4067913 failed Illumina reads; -4852106 ambiguous barcodes; -2491807 ambiguous RAD-Tags; +3955297 recovered; -1973888 low quality reads; 2614286 retained reads.\n",
      "Processing file 13 of 15 [PHap1_lane5_NoIndex_L005_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1274445 failed Illumina reads; -4846972 ambiguous barcodes; -3462894 ambiguous RAD-Tags; +3799797 recovered; -404760 low quality reads; 6010929 retained reads.\n",
      "Processing file 14 of 15 [PHap1_lane5_NoIndex_L005_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1648099 failed Illumina reads; -4853786 ambiguous barcodes; -3342015 ambiguous RAD-Tags; +4134085 recovered; -387209 low quality reads; 5768891 retained reads.\n",
      "Processing file 15 of 15 [PHap1_lane5_NoIndex_L005_R1_015.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...\n",
      "  3486172 total reads; -490779 failed Illumina reads; -1067090 ambiguous barcodes; -681703 ambiguous RAD-Tags; +1053402 recovered; -91708 low quality reads; 1154892 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/process_radtags.PHap1.log'\n",
      "\n",
      "227486172 total sequences\n",
      " 25116781 failed Illumina filtered reads (11.0%)\n",
      " 69062500 ambiguous barcode drops (30.4%)\n",
      "  7780957 low quality read drops (3.4%)\n",
      " 46787495 ambiguous RAD-Tag drops (20.6%)\n",
      " 78738439 retained reads (34.6%)\n",
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 14 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 14 [PHap2_lane6_NoIndex_L006_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1330704 failed Illumina reads; -4599828 ambiguous barcodes; -3268344 ambiguous RAD-Tags; +3626926 recovered; -492588 low quality reads; 6308536 retained reads.\n",
      "Processing file 2 of 14 [PHap2_lane6_NoIndex_L006_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1678389 failed Illumina reads; -4601133 ambiguous barcodes; -3184186 ambiguous RAD-Tags; +3678462 recovered; -436206 low quality reads; 6100086 retained reads.\n",
      "Processing file 3 of 14 [PHap2_lane6_NoIndex_L006_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1436785 failed Illumina reads; -4609882 ambiguous barcodes; -3243865 ambiguous RAD-Tags; +3636098 recovered; -507671 low quality reads; 6201797 retained reads.\n",
      "Processing file 4 of 14 [PHap2_lane6_NoIndex_L006_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1490283 failed Illumina reads; -4603818 ambiguous barcodes; -3231925 ambiguous RAD-Tags; +3635438 recovered; -426563 low quality reads; 6247411 retained reads.\n",
      "Processing file 5 of 14 [PHap2_lane6_NoIndex_L006_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1536959 failed Illumina reads; -4616159 ambiguous barcodes; -3214975 ambiguous RAD-Tags; +3638869 recovered; -467713 low quality reads; 6164194 retained reads.\n",
      "Processing file 6 of 14 [PHap2_lane6_NoIndex_L006_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1455947 failed Illumina reads; -4608911 ambiguous barcodes; -3242646 ambiguous RAD-Tags; +3632713 recovered; -450161 low quality reads; 6242335 retained reads.\n",
      "Processing file 7 of 14 [PHap2_lane6_NoIndex_L006_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1590042 failed Illumina reads; -4594226 ambiguous barcodes; -3215995 ambiguous RAD-Tags; +3749648 recovered; -440666 low quality reads; 6159071 retained reads.\n",
      "Processing file 8 of 14 [PHap2_lane6_NoIndex_L006_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1233899 failed Illumina reads; -4584488 ambiguous barcodes; -3315046 ambiguous RAD-Tags; +3609987 recovered; -405495 low quality reads; 6461072 retained reads.\n",
      "Processing file 9 of 14 [PHap2_lane6_NoIndex_L006_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1673124 failed Illumina reads; -4586831 ambiguous barcodes; -3202959 ambiguous RAD-Tags; +3834485 recovered; -391928 low quality reads; 6145158 retained reads.\n",
      "Processing file 10 of 14 [PHap2_lane6_NoIndex_L006_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1098221 failed Illumina reads; -4579024 ambiguous barcodes; -3361518 ambiguous RAD-Tags; +3616755 recovered; -430492 low quality reads; 6530745 retained reads.\n",
      "Processing file 11 of 14 [PHap2_lane6_NoIndex_L006_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1315593 failed Illumina reads; -4577851 ambiguous barcodes; -3303501 ambiguous RAD-Tags; +3622165 recovered; -396292 low quality reads; 6406763 retained reads.\n",
      "Processing file 12 of 14 [PHap2_lane6_NoIndex_L006_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1390550 failed Illumina reads; -4589564 ambiguous barcodes; -3272464 ambiguous RAD-Tags; +3726163 recovered; -421485 low quality reads; 6325937 retained reads.\n",
      "Processing file 13 of 14 [PHap2_lane6_NoIndex_L006_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1376376 failed Illumina reads; -4589221 ambiguous barcodes; -3280385 ambiguous RAD-Tags; +3678069 recovered; -412692 low quality reads; 6341326 retained reads.\n",
      "Processing file 14 of 14 [PHap2_lane6_NoIndex_L006_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...\n",
      "  11577285 total reads; -1334678 failed Illumina reads; -3324862 ambiguous barcodes; -2278383 ambiguous RAD-Tags; +2842609 recovered; -313524 low quality reads; 4325838 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/process_radtags.PHap2.log'\n",
      "\n",
      "219577285 total sequences\n",
      " 19941550 failed Illumina filtered reads (9.1%)\n",
      " 63065798 ambiguous barcode drops (28.7%)\n",
      "  5993476 low quality read drops (2.7%)\n",
      " 44616192 ambiguous RAD-Tag drops (20.3%)\n",
      " 85960269 retained reads (39.1%)\n",
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 15 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 15 [PHap1rr_lane8_NoIndex_L008_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1602479 failed Illumina reads; -4695437 ambiguous barcodes; -3277284 ambiguous RAD-Tags; +3870641 recovered; -485705 low quality reads; 5939095 retained reads.\n",
      "Processing file 2 of 15 [PHap1rr_lane8_NoIndex_L008_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1612693 failed Illumina reads; -4693554 ambiguous barcodes; -3266974 ambiguous RAD-Tags; +3746812 recovered; -609702 low quality reads; 5817077 retained reads.\n",
      "Processing file 3 of 15 [PHap1rr_lane8_NoIndex_L008_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2021641 failed Illumina reads; -4721926 ambiguous barcodes; -3166730 ambiguous RAD-Tags; +4012619 recovered; -490793 low quality reads; 5598910 retained reads.\n",
      "Processing file 4 of 15 [PHap1rr_lane8_NoIndex_L008_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2008058 failed Illumina reads; -4739371 ambiguous barcodes; -3158722 ambiguous RAD-Tags; +4143499 recovered; -607534 low quality reads; 5486315 retained reads.\n",
      "Processing file 5 of 15 [PHap1rr_lane8_NoIndex_L008_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1875257 failed Illumina reads; -4709012 ambiguous barcodes; -3200580 ambiguous RAD-Tags; +3878907 recovered; -571554 low quality reads; 5643597 retained reads.\n",
      "Processing file 6 of 15 [PHap1rr_lane8_NoIndex_L008_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2276737 failed Illumina reads; -4739102 ambiguous barcodes; -3088549 ambiguous RAD-Tags; +4396773 recovered; -503786 low quality reads; 5391826 retained reads.\n",
      "Processing file 7 of 15 [PHap1rr_lane8_NoIndex_L008_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1642162 failed Illumina reads; -4689234 ambiguous barcodes; -3265012 ambiguous RAD-Tags; +3738107 recovered; -590719 low quality reads; 5812873 retained reads.\n",
      "Processing file 8 of 15 [PHap1rr_lane8_NoIndex_L008_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2059459 failed Illumina reads; -4718699 ambiguous barcodes; -3152874 ambiguous RAD-Tags; +4007598 recovered; -492553 low quality reads; 5576415 retained reads.\n",
      "Processing file 9 of 15 [PHap1rr_lane8_NoIndex_L008_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1632234 failed Illumina reads; -4702433 ambiguous barcodes; -3277596 ambiguous RAD-Tags; +3748186 recovered; -434261 low quality reads; 5953476 retained reads.\n",
      "Processing file 10 of 15 [PHap1rr_lane8_NoIndex_L008_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1944027 failed Illumina reads; -4723975 ambiguous barcodes; -3190628 ambiguous RAD-Tags; +4100146 recovered; -468172 low quality reads; 5673198 retained reads.\n",
      "Processing file 11 of 15 [PHap1rr_lane8_NoIndex_L008_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1418732 failed Illumina reads; -4682878 ambiguous barcodes; -3339668 ambiguous RAD-Tags; +3703497 recovered; -509951 low quality reads; 6048771 retained reads.\n",
      "Processing file 12 of 15 [PHap1rr_lane8_NoIndex_L008_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1822112 failed Illumina reads; -4710022 ambiguous barcodes; -3230969 ambiguous RAD-Tags; +4060340 recovered; -437409 low quality reads; 5799488 retained reads.\n",
      "Processing file 13 of 15 [PHap1rr_lane8_NoIndex_L008_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1738507 failed Illumina reads; -4701593 ambiguous barcodes; -3241922 ambiguous RAD-Tags; +3955733 recovered; -490667 low quality reads; 5827311 retained reads.\n",
      "Processing file 14 of 15 [PHap1rr_lane8_NoIndex_L008_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1586142 failed Illumina reads; -4691086 ambiguous barcodes; -3292685 ambiguous RAD-Tags; +3739036 recovered; -453503 low quality reads; 5976584 retained reads.\n",
      "Processing file 15 of 15 [PHap1rr_lane8_NoIndex_L008_R1_015.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...\n",
      "  14018093 total reads; -1873726 failed Illumina reads; -4139698 ambiguous barcodes; -2744203 ambiguous RAD-Tags; +3952326 recovered; -361325 low quality reads; 4899141 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/process_radtags.PHap1Rerun.log'\n",
      "\n",
      "238018093 total sequences\n",
      " 27113966 failed Illumina filtered reads (11.4%)\n",
      " 70058020 ambiguous barcode drops (29.4%)\n",
      "  7507634 low quality read drops (3.2%)\n",
      " 47894396 ambiguous RAD-Tag drops (20.1%)\n",
      " 85444077 retained reads (35.9%)\n",
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads will be truncated to 94bp\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 15 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 96 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 15 [PHap2rr_lane7_NoIndex_L007_R1_008.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1227217 failed Illumina reads; -4737586 ambiguous barcodes; -3412125 ambiguous RAD-Tags; +3821279 recovered; -382618 low quality reads; 6240454 retained reads.\n",
      "Processing file 2 of 15 [PHap2rr_lane7_NoIndex_L007_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1401461 failed Illumina reads; -4758477 ambiguous barcodes; -3355864 ambiguous RAD-Tags; +3769144 recovered; -446761 low quality reads; 6037437 retained reads.\n",
      "Processing file 3 of 15 [PHap2rr_lane7_NoIndex_L007_R1_002.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1742549 failed Illumina reads; -4768938 ambiguous barcodes; -3243897 ambiguous RAD-Tags; +4143079 recovered; -407387 low quality reads; 5837229 retained reads.\n",
      "Processing file 4 of 15 [PHap2rr_lane7_NoIndex_L007_R1_003.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1624298 failed Illumina reads; -4752055 ambiguous barcodes; -3280535 ambiguous RAD-Tags; +3913643 recovered; -491493 low quality reads; 5851619 retained reads.\n",
      "Processing file 5 of 15 [PHap2rr_lane7_NoIndex_L007_R1_004.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1764312 failed Illumina reads; -4760553 ambiguous barcodes; -3237523 ambiguous RAD-Tags; +4024161 recovered; -452630 low quality reads; 5784982 retained reads.\n",
      "Processing file 6 of 15 [PHap2rr_lane7_NoIndex_L007_R1_005.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1887797 failed Illumina reads; -4789054 ambiguous barcodes; -3188939 ambiguous RAD-Tags; +4276285 recovered; -434640 low quality reads; 5699570 retained reads.\n",
      "Processing file 7 of 15 [PHap2rr_lane7_NoIndex_L007_R1_006.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1663021 failed Illumina reads; -4752589 ambiguous barcodes; -3265568 ambiguous RAD-Tags; +3920300 recovered; -458684 low quality reads; 5860138 retained reads.\n",
      "Processing file 8 of 15 [PHap2rr_lane7_NoIndex_L007_R1_007.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -2019554 failed Illumina reads; -4804656 ambiguous barcodes; -3143677 ambiguous RAD-Tags; +4455055 recovered; -411885 low quality reads; 5620228 retained reads.\n",
      "Processing file 9 of 15 [PHap2rr_lane7_NoIndex_L007_R1_009.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1528303 failed Illumina reads; -4753294 ambiguous barcodes; -3318134 ambiguous RAD-Tags; +4017673 recovered; -354918 low quality reads; 6045351 retained reads.\n",
      "Processing file 10 of 15 [PHap2rr_lane7_NoIndex_L007_R1_010.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1494370 failed Illumina reads; -4782647 ambiguous barcodes; -3305228 ambiguous RAD-Tags; +4104806 recovered; -410521 low quality reads; 6007234 retained reads.\n",
      "Processing file 11 of 15 [PHap2rr_lane7_NoIndex_L007_R1_011.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1422756 failed Illumina reads; -4743479 ambiguous barcodes; -3349364 ambiguous RAD-Tags; +3891987 recovered; -394776 low quality reads; 6089625 retained reads.\n",
      "Processing file 12 of 15 [PHap2rr_lane7_NoIndex_L007_R1_012.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1899278 failed Illumina reads; -4857943 ambiguous barcodes; -3161427 ambiguous RAD-Tags; +4813193 recovered; -366728 low quality reads; 5714624 retained reads.\n",
      "Processing file 13 of 15 [PHap2rr_lane7_NoIndex_L007_R1_013.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1386825 failed Illumina reads; -4740097 ambiguous barcodes; -3355731 ambiguous RAD-Tags; +3799814 recovered; -395816 low quality reads; 6121531 retained reads.\n",
      "Processing file 14 of 15 [PHap2rr_lane7_NoIndex_L007_R1_014.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...\n",
      "  16000000 total reads; -1882720 failed Illumina reads; -4786366 ambiguous barcodes; -3192114 ambiguous RAD-Tags; +4553515 recovered; -379263 low quality reads; 5759537 retained reads.\n",
      "Processing file 15 of 15 [PHap2rr_lane7_NoIndex_L007_R1_015.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...\n",
      "  3096382 total reads; -460699 failed Illumina reads; -931010 ambiguous barcodes; -583537 ambiguous RAD-Tags; +923079 recovered; -66931 low quality reads; 1054205 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/process_radtags.PHap2Rerun.log'\n",
      "\n",
      "227096382 total sequences\n",
      " 23405160 failed Illumina filtered reads (10.3%)\n",
      " 67718744 ambiguous barcode drops (29.8%)\n",
      "  5855051 low quality read drops (2.6%)\n",
      " 46393663 ambiguous RAD-Tag drops (20.4%)\n",
      " 83723764 retained reads (36.9%)\n"
     ]
    }
   ],
   "source": [
    "## Also rerunning process radtags on initial and resequence rounds for both libraries so that everything starts fresh from stacks 1.46\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Rerun' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94\n",
    "! process_radtags -p '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Rerun' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Barcodes.txt' -e sbfI -c -r -q -E phred33 --filter_illumina -i gzfastq -t 94"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing single-end data.\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Discarding reads marked as 'failed' by Illumina's chastity/purity filters.\n",
      "Found 1 input file(s).\n",
      "Searching for single-end, inlined barcodes.\n",
      "Loaded 32 barcodes (6bp).\n",
      "Will attempt to recover barcodes with at most 1 mismatches.\n",
      "Processing file 1 of 1 [Undetermined_S0_R1_001.fastq.gz]\n",
      "  Processing RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...12M...13M...14M...15M...16M...17M...18M...19M...20M...21M...22M...23M...24M...25M...26M...27M...28M...29M...30M...31M...32M...33M...34M...35M...36M...37M...38M...39M...40M...41M...42M...43M...44M...45M...46M...47M...48M...49M...50M...51M...52M...53M...54M...55M...56M...57M...58M...59M...60M...61M...62M...63M...64M...65M...66M...67M...68M...69M...70M...71M...72M...73M...74M...75M...76M...77M...78M...79M...80M...81M...82M...83M...84M...85M...86M...87M...88M...89M...90M...91M...92M...93M...94M...95M...96M...97M...98M...99M...100M...101M...102M...103M...104M...105M...106M...107M...108M...109M...110M...111M...112M...113M...114M...115M...116M...117M...118M...119M...120M...121M...122M...123M...124M...125M...126M...127M...128M...129M...130M...131M...132M...133M...134M...135M...136M...137M...138M...139M...140M...141M...142M...143M...144M...145M...146M...147M...148M...149M...150M...151M...152M...153M...154M...155M...156M...157M...158M...159M...160M...161M...162M...163M...164M...165M...166M...167M...168M...169M...170M...171M...172M...173M...174M...175M...176M...177M...178M...179M...180M...181M...182M...183M...184M...185M...186M...187M...188M...189M...190M...191M...192M...193M...194M...195M...196M...197M...198M...199M...200M...201M...202M...203M...204M...205M...206M...207M...208M...209M...210M...211M...212M...213M...214M...215M...216M...217M...218M...219M...220M...221M...222M...223M...224M...225M...226M...227M...228M...229M...230M...231M...232M...233M...234M...235M...236M...237M...238M...239M...240M...241M...242M...243M...244M...245M...246M...247M...248M...249M...250M...251M...252M...253M...254M...255M...256M...257M...258M...259M...260M...261M...262M...263M...264M...265M...266M...267M...268M...269M...270M...271M...272M...273M...274M...275M...276M...277M...278M...279M...280M...281M...282M...283M...284M...285M...286M...287M...288M...289M...290M...291M...292M...293M...294M...295M...296M...297M...298M...299M...300M...301M...302M...303M...304M...305M...306M...307M...308M...309M...310M...311M...312M...313M...314M...315M...316M...317M...318M...319M...320M...321M...322M...323M...324M...325M...326M...327M...328M...329M...330M...331M...332M...333M...334M...335M...336M...337M...338M...339M...340M...341M...342M...343M...344M...345M...346M...347M...348M...349M...350M...351M...352M...353M...354M...355M...356M...357M...358M...359M...360M...361M...362M...363M...364M...365M...366M...367M...368M...369M...370M...371M...372M...373M...374M...375M...376M...377M...378M...379M...380M...381M...382M...383M...384M...385M...386M...387M...388M...389M...390M...391M...392M...393M...394M...395M...396M...397M...398M...399M...400M...401M...402M...403M...404M...405M...406M...407M...408M...409M...410M...411M...412M...413M...414M...415M...416M...417M...418M...419M...420M...421M...422M...423M...424M...425M...426M...427M...428M...429M...430M...431M...432M...433M...434M...435M...436M...437M...438M...439M...440M...441M...442M...443M...444M...445M...446M...447M...448M...449M...450M...451M...452M...453M...454M...455M...456M...457M...458M...459M...460M...461M...462M...463M...464M...465M...466M...467M...468M...469M...470M...471M...472M...473M...474M...475M...476M...477M...478M...479M...480M...481M...482M...483M...484M...485M...486M...487M...488M...489M...490M...491M...492M...493M...494M...495M...496M...497M...\n",
      "  497648347 total reads; -0 failed Illumina reads; -394696801 ambiguous barcodes; -73115971 ambiguous RAD-Tags; +66959453 recovered; -124959 low quality reads; 29710616 retained reads.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/SingleEndt94/process_radtags.fastq.log'\n",
      "\n",
      "497648347 total sequences\n",
      "        0 failed Illumina filtered reads (0.0%)\n",
      "394696801 ambiguous barcode drops (79.3%)\n",
      "   124959 low quality read drops (0.0%)\n",
      " 73115971 ambiguous RAD-Tag drops (14.7%)\n",
      " 29710616 retained reads (6.0%)\n"
     ]
    }
   ],
   "source": [
    "## Paired end reads have two .fq along with .rem for each sample corresponding to read 1 and 2 (and reads thrown out .rem).\n",
    "## Running process_radtags with single end to only look at read 1 for comparison purposes and maybe make concatenation easier.\n",
    "! process_radtags -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/fastq/Undetermined_S0_R1_001.fastq.gz' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/SingleEndt94' -b '/media/tyler/My Book/Stacks/Pink/Haploids2014/Training Ogo Run/Bird2014TrainingRunBarcodes.txt' -E phred33 -e sbfI -i gzfastq -c -r -q --filter_illumina "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Concatenate multiple rounds of sequence - paired-end data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X_0101.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0013.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0014.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0015.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0016.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0017.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0018.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0019.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0020.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0021.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0021.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0022.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0022.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101G_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101G_0023.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0002.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0003.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0005.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0007.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0009.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0010.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0010.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0011.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0012.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0012.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0013.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0013.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0014.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0015.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0016.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0017.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0017.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0018.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0019.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0019.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0020.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0020.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0023.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0024.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0024.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0024.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0024.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101H_0025.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0026.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0026.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0026.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0026.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0027.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0027.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0027.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0027.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0029.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0029.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0029.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0029.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0031.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0031.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0031.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0031.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0032.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0032.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0033.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0033.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0034.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0034.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0035.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0035.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0035.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0035.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0038.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0038.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0038.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0038.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0039.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0039.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0039.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0039.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0040.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0040.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0040.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0040.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0041.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0041.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0041.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0041.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0047.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0047.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0047.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0047.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0049.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0049.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0050.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0050.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0050.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0050.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0053.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0053.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0053.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0053.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0056.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0056.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0056.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0056.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0058.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0058.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0058.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0058.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0059.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0059.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0059.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0059.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0062.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0062.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0066.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0066.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0066.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0066.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0067.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0067.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0067.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0067.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0069.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0069.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0070.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0070.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0073.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0073.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0075.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0075.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0075.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0075.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0078.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0078.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0078.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0078.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0101.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X101H_0101.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0101.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X101H_0104.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X101H_0104.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0001.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0001.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0002.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0003.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0004.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0004.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0004.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0005.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0006.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0006.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0006.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0007.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0008.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0009.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0010.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0011.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0012.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0012.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0013.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0013.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0014.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0014.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0015.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0015.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0016.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0016.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0017.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0017.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0018.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0018.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0019.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0019.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0020.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0020.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0021.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0021.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0021.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0022.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0022.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0022.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0023.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0023.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0024.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0024.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0025.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0025.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0026.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0026.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0027.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0027.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0028.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0028.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0028.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0029.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0029.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0030.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0030.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0030.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0031.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0031.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0032.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0032.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0032.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0033.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0033.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0034.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0034.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1Stacks/PBIRD14X107H_0045.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap1RerunStacks/PBIRD14X107H_0045.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X_0107.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X_0107.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X_0107.1.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/OgoTrainingStacks/PBIRD14X_0107.2.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0001.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0001.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0002.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0002.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0003.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0003.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0004.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0004.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0005.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0005.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0006.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0006.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0007.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0007.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0008.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0008.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0009.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0009.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0010.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0010.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0011.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X101G_0011.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X101G_0012.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X101G_0012.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0035.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0035.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0036.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0036.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0037.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0037.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0038.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0038.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0039.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0039.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0040.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0040.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0041.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0041.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0042.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0042.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0043.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0043.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0044.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0044.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0046.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0046.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0047.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0047.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0048.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0048.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0049.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0049.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0050.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0050.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0051.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0051.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0052.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0052.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0052.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0053.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0053.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0054.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0054.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0055.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0055.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0056.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0056.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0057.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0057.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0059.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0059.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0060.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0060.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0061.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0061.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0062.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0062.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0063.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0063.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0063.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0064.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0064.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0064.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0065.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0065.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0065.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0066.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0066.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0067.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0067.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0068.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0068.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0068.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0069.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0069.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0070.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0070.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0071.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0071.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0072.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0072.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0072.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0073.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0073.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0074.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0074.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0074.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0075.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0075.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0076.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0076.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0076.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0077.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0077.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0077.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0078.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0078.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0079.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0079.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0079.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0080.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0080.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0080.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0081.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0081.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0081.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0082.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0082.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0082.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0083.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0083.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0083.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0084.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0084.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0085.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0085.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0086.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0086.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0087.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0087.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0088.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0088.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0089.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0089.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0090.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0090.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0091.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0091.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0092.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0092.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0093.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0093.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0094.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0094.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0095.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0095.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0096.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0096.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0097.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0097.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0098.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0098.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0099.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0099.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0100.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0100.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0101.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0101.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0102.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0102.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0103.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0103.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0104.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0104.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0105.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0105.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0106.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0106.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0107.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0107.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0109.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0109.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0110.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0110.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0111.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0111.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0111.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0113.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0113.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0114.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0114.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0115.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0115.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0115.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0116.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0116.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0117.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0117.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0122.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0122.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0123.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0123.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0124.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0124.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0124.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz'\n",
    "! cat '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2Stacks/PBIRD14X107H_0126.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHap2RerunStacks/PBIRD14X107H_0126.fq.gz' '/media/tyler/My Book/Stacks/Pink/Haploids2014/PHapReReSeqStacks/PBIRD14X107H_0126.fq.gz' > '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## ustacks with paired data - using standard parameter values for m (min depth coverage to create a stack) and M (maximum distance allowed between stacks) and enabling the deleveraging algorithm to remove highly repetitive stacks."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ustacks parameters selected:\n",
      "  Sample ID: 5000\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3059977 RAD-Tags.\n",
      "  Inserted 353231 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110646 initial stacks were populated; 242585 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.4631; Std Dev: 74.6973; Max: 11914\n",
      "Deleveraging trigger: 100; Removal trigger: 175\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4044 stacks.\n",
      "  106731 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.1421; Std Dev: 15.6575; Max: 175\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  106731 stacks merged into 90523 loci; deleveraged 596 loci; blacklisted 437 loci.\n",
      "After merging, coverage depth Mean: 28.1299; Std Dev: 16.0715; Max: 287\n",
      "Merging remainder radtags\n",
      "  242585 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 201457 remainder reads; unable to match 41128 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 30.3694; Std Dev: 17.0504; Max: 310\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3018849\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz... read 3059977 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5001\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1031745 RAD-Tags.\n",
      "  Inserted 177370 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "91641 initial stacks were populated; 85729 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.3231; Std Dev: 28.5769; Max: 4881\n",
      "Deleveraging trigger: 39; Removal trigger: 67\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2516 stacks.\n",
      "  89236 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.71501; Std Dev: 5.4505; Max: 67\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  89236 stacks merged into 81740 loci; deleveraged 314 loci; blacklisted 182 loci.\n",
      "After merging, coverage depth Mean: 10.5075; Std Dev: 5.85133; Max: 107\n",
      "Merging remainder radtags\n",
      "  85729 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 70515 remainder reads; unable to match 15214 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.3733; Std Dev: 6.2136; Max: 118\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1016531\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz... read 1031745 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5002\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4374437 RAD-Tags.\n",
      "  Inserted 448121 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114369 initial stacks were populated; 333752 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 35.3302; Std Dev: 109.286; Max: 16613\n",
      "Deleveraging trigger: 145; Removal trigger: 254\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4931 stacks.\n",
      "  109572 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 33.5856; Std Dev: 23.3247; Max: 254\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109572 stacks merged into 89960 loci; deleveraged 704 loci; blacklisted 669 loci.\n",
      "After merging, coverage depth Mean: 40.41; Std Dev: 22.9035; Max: 395\n",
      "Merging remainder radtags\n",
      "  333752 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 285837 remainder reads; unable to match 47915 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 43.616; Std Dev: 24.2747; Max: 412\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4326522\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz... read 4374437 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5003\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 978484 RAD-Tags.\n",
      "  Inserted 181740 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "92480 initial stacks were populated; 89260 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.61531; Std Dev: 25.0123; Max: 3557\n",
      "Deleveraging trigger: 35; Removal trigger: 60\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2577 stacks.\n",
      "  90029 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.03532; Std Dev: 5.27153; Max: 60\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  90029 stacks merged into 82157 loci; deleveraged 337 loci; blacklisted 183 loci.\n",
      "After merging, coverage depth Mean: 9.80866; Std Dev: 5.68461; Max: 140\n",
      "Merging remainder radtags\n",
      "  89260 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 68591 remainder reads; unable to match 20669 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 10.6467; Std Dev: 6.07891; Max: 162\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 957815\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz... read 978484 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5004\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1956763 RAD-Tags.\n",
      "  Inserted 254123 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100295 initial stacks were populated; 153828 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.9763; Std Dev: 59.8584; Max: 10840\n",
      "Deleveraging trigger: 78; Removal trigger: 138\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3022 stacks.\n",
      "  97381 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.8841; Std Dev: 10.5931; Max: 134\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97381 stacks merged into 86320 loci; deleveraged 481 loci; blacklisted 288 loci.\n",
      "After merging, coverage depth Mean: 18.8264; Std Dev: 11.0647; Max: 196\n",
      "Merging remainder radtags\n",
      "  153828 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 127017 remainder reads; unable to match 26811 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.3047; Std Dev: 11.7513; Max: 223\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1929952\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz... read 1956763 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5005\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 506333 RAD-Tags.\n",
      "  Inserted 131793 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "76331 initial stacks were populated; 55462 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.90679; Std Dev: 15.2129; Max: 2259\n",
      "Deleveraging trigger: 21; Removal trigger: 36\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1943 stacks.\n",
      "  74499 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.52228; Std Dev: 3.32774; Max: 36\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  74499 stacks merged into 69751 loci; deleveraged 225 loci; blacklisted 156 loci.\n",
      "After merging, coverage depth Mean: 5.83193; Std Dev: 3.60955; Max: 67\n",
      "Merging remainder radtags\n",
      "  55462 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 34486 remainder reads; unable to match 20976 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.32825; Std Dev: 3.87144; Max: 73\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 485357\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz... read 506333 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5006\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1900416 RAD-Tags.\n",
      "  Inserted 248943 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99705 initial stacks were populated; 149238 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.5636; Std Dev: 55.7311; Max: 9852\n",
      "Deleveraging trigger: 73; Removal trigger: 129\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3035 stacks.\n",
      "  96776 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.5224; Std Dev: 10.2821; Max: 129\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96776 stacks merged into 85473 loci; deleveraged 469 loci; blacklisted 306 loci.\n",
      "After merging, coverage depth Mean: 18.489; Std Dev: 10.7828; Max: 212\n",
      "Merging remainder radtags\n",
      "  149238 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 126700 remainder reads; unable to match 22538 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.9785; Std Dev: 11.5087; Max: 256\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1877878\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz... read 1900416 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5007\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 836692 RAD-Tags.\n",
      "  Inserted 160895 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88546 initial stacks were populated; 72349 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.63216; Std Dev: 23.124; Max: 3316\n",
      "Deleveraging trigger: 32; Removal trigger: 55\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2309 stacks.\n",
      "  86341 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.12498; Std Dev: 4.58888; Max: 54\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  86341 stacks merged into 79628 loci; deleveraged 271 loci; blacklisted 174 loci.\n",
      "After merging, coverage depth Mean: 8.72776; Std Dev: 4.95807; Max: 87\n",
      "Merging remainder radtags\n",
      "  72349 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 58586 remainder reads; unable to match 13763 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.46609; Std Dev: 5.32155; Max: 166\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 822929\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz... read 836692 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5008\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 485790 RAD-Tags.\n",
      "  Inserted 130995 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "78795 initial stacks were populated; 52200 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.50276; Std Dev: 14.329; Max: 1874\n",
      "Deleveraging trigger: 20; Removal trigger: 34\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1924 stacks.\n",
      "  76974 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.16711; Std Dev: 2.87281; Max: 34\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  76974 stacks merged into 72362 loci; deleveraged 206 loci; blacklisted 142 loci.\n",
      "After merging, coverage depth Mean: 5.44121; Std Dev: 3.09766; Max: 56\n",
      "Merging remainder radtags\n",
      "  52200 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 33618 remainder reads; unable to match 18582 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.90737; Std Dev: 3.32726; Max: 74\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 467208\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz... read 485790 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5009\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 768704 RAD-Tags.\n",
      "  Inserted 170315 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88079 initial stacks were populated; 82236 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.79378; Std Dev: 23.8505; Max: 3971\n",
      "Deleveraging trigger: 32; Removal trigger: 55\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2255 stacks.\n",
      "  85930 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.28483; Std Dev: 4.63917; Max: 54\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  85930 stacks merged into 78359 loci; deleveraged 312 loci; blacklisted 215 loci.\n",
      "After merging, coverage depth Mean: 7.89691; Std Dev: 5.01688; Max: 84\n",
      "Merging remainder radtags\n",
      "  82236 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 56816 remainder reads; unable to match 25420 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.62496; Std Dev: 5.3876; Max: 95\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 743284\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz... read 768704 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5010\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 742970 RAD-Tags.\n",
      "  Inserted 161982 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "85796 initial stacks were populated; 76186 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.77174; Std Dev: 20.8619; Max: 3276\n",
      "Deleveraging trigger: 29; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2208 stacks.\n",
      "  83693 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.26026; Std Dev: 4.60837; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83693 stacks merged into 77020 loci; deleveraged 316 loci; blacklisted 188 loci.\n",
      "After merging, coverage depth Mean: 7.80274; Std Dev: 4.95678; Max: 84\n",
      "Merging remainder radtags\n",
      "  76186 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 51747 remainder reads; unable to match 24439 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.47717; Std Dev: 5.34163; Max: 119\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 718531\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz... read 742970 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5011\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 490221 RAD-Tags.\n",
      "  Inserted 134844 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "74131 initial stacks were populated; 60713 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.79391; Std Dev: 15.5144; Max: 2642\n",
      "Deleveraging trigger: 21; Removal trigger: 37\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1799 stacks.\n",
      "  72424 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.41283; Std Dev: 3.49579; Max: 37\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  72424 stacks merged into 67430 loci; deleveraged 251 loci; blacklisted 165 loci.\n",
      "After merging, coverage depth Mean: 5.74319; Std Dev: 3.80621; Max: 101\n",
      "Merging remainder radtags\n",
      "  60713 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 33095 remainder reads; unable to match 27618 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.23587; Std Dev: 4.09109; Max: 106\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 462603\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz... read 490221 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5012\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2179812 RAD-Tags.\n",
      "  Inserted 285013 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102970 initial stacks were populated; 182043 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4015; Std Dev: 55.4339; Max: 9215\n",
      "Deleveraging trigger: 75; Removal trigger: 130\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3362 stacks.\n",
      "  99743 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.2712; Std Dev: 12.7894; Max: 130\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  99743 stacks merged into 88774 loci; deleveraged 520 loci; blacklisted 350 loci.\n",
      "After merging, coverage depth Mean: 20.2969; Std Dev: 13.2871; Max: 181\n",
      "Merging remainder radtags\n",
      "  182043 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 144459 remainder reads; unable to match 37584 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.9331; Std Dev: 14.171; Max: 189\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2142228\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz... read 2179812 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5013\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7611497 RAD-Tags.\n",
      "  Inserted 749141 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "147044 initial stacks were populated; 602097 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 47.6687; Std Dev: 158.222; Max: 25063\n",
      "Deleveraging trigger: 206; Removal trigger: 364\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6852 stacks.\n",
      "  140344 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 45.4758; Std Dev: 41.7967; Max: 364\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  140344 stacks merged into 103885 loci; deleveraged 1179 loci; blacklisted 2022 loci.\n",
      "After merging, coverage depth Mean: 60.2303; Std Dev: 40.8484; Max: 538\n",
      "Merging remainder radtags\n",
      "  602097 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 467490 remainder reads; unable to match 134607 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 64.8266; Std Dev: 43.3373; Max: 567\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7476890\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz... read 7611497 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5014\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1852808 RAD-Tags.\n",
      "  Inserted 260332 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99405 initial stacks were populated; 160927 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.0201; Std Dev: 44.5494; Max: 6704\n",
      "Deleveraging trigger: 62; Removal trigger: 106\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3123 stacks.\n",
      "  96423 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.0783; Std Dev: 9.92684; Max: 106\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96423 stacks merged into 86724 loci; deleveraged 414 loci; blacklisted 266 loci.\n",
      "After merging, coverage depth Mean: 17.6946; Std Dev: 10.2409; Max: 157\n",
      "Merging remainder radtags\n",
      "  160927 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 130325 remainder reads; unable to match 30602 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.2044; Std Dev: 10.9425; Max: 179\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1822206\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz... read 1852808 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5015\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4015590 RAD-Tags.\n",
      "  Inserted 456199 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "119820 initial stacks were populated; 336379 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.7062; Std Dev: 105.345; Max: 18819\n",
      "Deleveraging trigger: 136; Removal trigger: 241\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4855 stacks.\n",
      "  115092 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 29.036; Std Dev: 21.4321; Max: 241\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  115092 stacks merged into 95054 loci; deleveraged 724 loci; blacklisted 717 loci.\n",
      "After merging, coverage depth Mean: 34.6381; Std Dev: 20.9899; Max: 309\n",
      "Merging remainder radtags\n",
      "  336379 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 266882 remainder reads; unable to match 69497 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 37.4709; Std Dev: 22.3435; Max: 333\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3946093\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz... read 4015590 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5016\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6098729 RAD-Tags.\n",
      "  Inserted 641107 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "137160 initial stacks were populated; 503947 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 40.7902; Std Dev: 133.061; Max: 23319\n",
      "Deleveraging trigger: 174; Removal trigger: 307\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6080 stacks.\n",
      "  131241 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 38.7999; Std Dev: 33.7687; Max: 307\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  131241 stacks merged into 101358 loci; deleveraged 1079 loci; blacklisted 1479 loci.\n",
      "After merging, coverage depth Mean: 49.4093; Std Dev: 33.0635; Max: 514\n",
      "Merging remainder radtags\n",
      "  503947 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 394205 remainder reads; unable to match 109742 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 53.3625; Std Dev: 35.1735; Max: 538\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5988987\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz... read 6098729 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5017\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2256767 RAD-Tags.\n",
      "  Inserted 507650 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "155739 initial stacks were populated; 351911 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.2311; Std Dev: 40.19; Max: 7768\n",
      "Deleveraging trigger: 52; Removal trigger: 93\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 71 (75 k-mers per read); 4 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3599 stacks.\n",
      "  152319 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.4127; Std Dev: 10.3966; Max: 92\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  152319 stacks merged into 142364 loci; deleveraged 467 loci; blacklisted 283 loci.\n",
      "After merging, coverage depth Mean: 12.0934; Std Dev: 11.0895; Max: 182\n",
      "Merging remainder radtags\n",
      "  351911 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': free(): corrupted unsorted chunks: 0x00007f88849bdaa0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5018\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 4112 RAD-Tags.\n",
      "  Inserted 3258 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "459 initial stacks were populated; 2799 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.86057; Std Dev: 10.7957; Max: 232\n",
      "Deleveraging trigger: 14; Removal trigger: 24\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 23 stacks.\n",
      "  437 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.22706; Std Dev: 0.583647; Max: 8\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  437 stacks merged into 425 loci; deleveraged 4 loci; blacklisted 0 loci.\n",
      "After merging, coverage depth Mean: 2.29009; Std Dev: 0.820317; Max: 10\n",
      "Merging remainder radtags\n",
      "  2799 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 307 remainder reads; unable to match 2492 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.01415; Std Dev: 4.91002; Max: 77\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1620\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz... read 4112 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5019\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2030819 RAD-Tags.\n",
      "  Inserted 259829 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98522 initial stacks were populated; 161307 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.9756; Std Dev: 53.9717; Max: 7248\n",
      "Deleveraging trigger: 73; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3087 stacks.\n",
      "  95551 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.9409; Std Dev: 10.5636; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95551 stacks merged into 86137 loci; deleveraged 384 loci; blacklisted 259 loci.\n",
      "After merging, coverage depth Mean: 19.6864; Std Dev: 10.8925; Max: 212\n",
      "Merging remainder radtags\n",
      "  161307 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 131062 remainder reads; unable to match 30245 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.2146; Std Dev: 11.5694; Max: 224\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2000574\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz... read 2030819 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5020\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1709295 RAD-Tags.\n",
      "  Inserted 461868 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "151286 initial stacks were populated; 310582 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.24549; Std Dev: 33.3498; Max: 7035\n",
      "Deleveraging trigger: 43; Removal trigger: 76\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 71 (75 k-mers per read); 4 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3196 stacks.\n",
      "  148239 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.61969; Std Dev: 7.4635; Max: 76\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  148239 stacks merged into 138651 loci; deleveraged 438 loci; blacklisted 273 loci.\n",
      "After merging, coverage depth Mean: 9.11985; Std Dev: 7.95009; Max: 145\n",
      "Merging remainder radtags\n",
      "  310582 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x0000000043f9ce30 ***\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007ff744972ba0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5021\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1904041 RAD-Tags.\n",
      "  Inserted 274090 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103262 initial stacks were populated; 170828 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.7846; Std Dev: 53.4207; Max: 8922\n",
      "Deleveraging trigger: 70; Removal trigger: 124\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3038 stacks.\n",
      "  100327 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.8296; Std Dev: 10.0212; Max: 124\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100327 stacks merged into 90239 loci; deleveraged 416 loci; blacklisted 283 loci.\n",
      "After merging, coverage depth Mean: 17.3866; Std Dev: 10.4067; Max: 188\n",
      "Merging remainder radtags\n",
      "  170828 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 127729 remainder reads; unable to match 43099 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.8082; Std Dev: 11.1454; Max: 233\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1860942\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz... read 1904041 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5022\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2216770 RAD-Tags.\n",
      "  Inserted 292651 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "104551 initial stacks were populated; 188100 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4036; Std Dev: 56.7134; Max: 9707\n",
      "Deleveraging trigger: 76; Removal trigger: 133\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3305 stacks.\n",
      "  101373 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.3334; Std Dev: 11.9307; Max: 131\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101373 stacks merged into 90260 loci; deleveraged 471 loci; blacklisted 330 loci.\n",
      "After merging, coverage depth Mean: 20.3619; Std Dev: 12.3098; Max: 243\n",
      "Merging remainder radtags\n",
      "  188100 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 146815 remainder reads; unable to match 41285 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.9968; Std Dev: 13.1126; Max: 259\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2175485\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz... read 2216770 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5023\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1681593 RAD-Tags.\n",
      "  Inserted 248411 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100543 initial stacks were populated; 147868 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.2544; Std Dev: 44.2099; Max: 6938\n",
      "Deleveraging trigger: 59; Removal trigger: 104\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3032 stacks.\n",
      "  97630 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.3549; Std Dev: 8.96744; Max: 103\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97630 stacks merged into 88061 loci; deleveraged 407 loci; blacklisted 263 loci.\n",
      "After merging, coverage depth Mean: 15.7413; Std Dev: 9.34848; Max: 160\n",
      "Merging remainder radtags\n",
      "  147868 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 112176 remainder reads; unable to match 35692 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.0207; Std Dev: 9.9588; Max: 177\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1645901\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz... read 1681593 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5024\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2413271 RAD-Tags.\n",
      "  Inserted 954830 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "257959 initial stacks were populated; 696871 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.65377; Std Dev: 21.3345; Max: 4619\n",
      "Deleveraging trigger: 28; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4028 stacks.\n",
      "  254252 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.21904; Std Dev: 5.91901; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  254252 stacks merged into 241560 loci; deleveraged 568 loci; blacklisted 343 loci.\n",
      "After merging, coverage depth Mean: 6.49078; Std Dev: 6.31036; Max: 107\n",
      "Merging remainder radtags\n",
      "  696871 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f9ee10f0c60 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5025\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1554895 RAD-Tags.\n",
      "  Inserted 225273 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95448 initial stacks were populated; 129825 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.9303; Std Dev: 45.0373; Max: 7331\n",
      "Deleveraging trigger: 60; Removal trigger: 105\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2814 stacks.\n",
      "  92745 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.0744; Std Dev: 8.10533; Max: 105\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92745 stacks merged into 84560 loci; deleveraged 379 loci; blacklisted 219 loci.\n",
      "After merging, coverage depth Mean: 15.2778; Std Dev: 8.48449; Max: 164\n",
      "Merging remainder radtags\n",
      "  129825 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 105079 remainder reads; unable to match 24746 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.5254; Std Dev: 9.03189; Max: 176\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1530149\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz... read 1554895 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5026\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2614878 RAD-Tags.\n",
      "  Inserted 1187359 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "298628 initial stacks were populated; 888731 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.78026; Std Dev: 16.2716; Max: 3268\n",
      "Deleveraging trigger: 22; Removal trigger: 38\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4163 stacks.\n",
      "  294802 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.47223; Std Dev: 4.4142; Max: 38\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  294802 stacks merged into 282169 loci; deleveraged 484 loci; blacklisted 305 loci.\n",
      "After merging, coverage depth Mean: 5.67667; Std Dev: 4.6885; Max: 65\n",
      "Merging remainder radtags\n",
      "  888731 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f7cad3f6230 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5027\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1995033 RAD-Tags.\n",
      "  Inserted 427784 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "135685 initial stacks were populated; 292099 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.5506; Std Dev: 42.3982; Max: 8242\n",
      "Deleveraging trigger: 55; Removal trigger: 97\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3488 stacks.\n",
      "  132351 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7418; Std Dev: 9.60702; Max: 97\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  132351 stacks merged into 121301 loci; deleveraged 511 loci; blacklisted 316 loci.\n",
      "After merging, coverage depth Mean: 12.6618; Std Dev: 10.2466; Max: 187\n",
      "Merging remainder radtags\n",
      "  292099 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f35e4820cf0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5028\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2064354 RAD-Tags.\n",
      "  Inserted 266712 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98713 initial stacks were populated; 167999 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.2108; Std Dev: 53.8159; Max: 7855\n",
      "Deleveraging trigger: 73; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3132 stacks.\n",
      "  95697 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.1581; Std Dev: 11.2448; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95697 stacks merged into 85986 loci; deleveraged 403 loci; blacklisted 265 loci.\n",
      "After merging, coverage depth Mean: 19.9976; Std Dev: 11.5714; Max: 191\n",
      "Merging remainder radtags\n",
      "  167999 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 139461 remainder reads; unable to match 28538 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.6267; Std Dev: 12.3152; Max: 198\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2035816\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz... read 2064354 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5029\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 3231685 RAD-Tags.\n",
      "  Inserted 1678480 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "343858 initial stacks were populated; 1334622 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.517; Std Dev: 38.994; Max: 20729\n",
      "Deleveraging trigger: 45; Removal trigger: 84\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 71 (75 k-mers per read); 4 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3757 stacks.\n",
      "  340319 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.1878; Std Dev: 4.59172; Max: 84\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  340319 stacks merged into 321737 loci; deleveraged 813 loci; blacklisted 530 loci.\n",
      "After merging, coverage depth Mean: 5.4177; Std Dev: 4.86054; Max: 115\n",
      "Merging remainder radtags\n",
      "  1334622 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 259296 remainder reads; unable to match 1075326 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.22551; Std Dev: 5.69148; Max: 198\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2156359\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz... read 3231685 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5030\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3068096 RAD-Tags.\n",
      "  Inserted 405660 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121379 initial stacks were populated; 284281 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.9349; Std Dev: 81.4306; Max: 13949\n",
      "Deleveraging trigger: 104; Removal trigger: 186\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4162 stacks.\n",
      "  117339 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.5759; Std Dev: 16.1339; Max: 185\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117339 stacks merged into 100620 loci; deleveraged 704 loci; blacklisted 554 loci.\n",
      "After merging, coverage depth Mean: 24.8094; Std Dev: 16.4533; Max: 283\n",
      "Merging remainder radtags\n",
      "  284281 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 207027 remainder reads; unable to match 77254 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 26.8809; Std Dev: 17.6269; Max: 306\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2990842\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz... read 3068096 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5031\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2864022 RAD-Tags.\n",
      "  Inserted 1301902 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "320506 initial stacks were populated; 981396 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.87392; Std Dev: 19.1669; Max: 4638\n",
      "Deleveraging trigger: 25; Removal trigger: 44\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4190 stacks.\n",
      "  316614 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.54676; Std Dev: 4.89065; Max: 44\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  316614 stacks merged into 300421 loci; deleveraged 741 loci; blacklisted 402 loci.\n",
      "After merging, coverage depth Mean: 5.8035; Std Dev: 5.22653; Max: 80\n",
      "Merging remainder radtags\n",
      "  981396 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': free(): corrupted unsorted chunks: 0x00007f3745515680 ***\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f375552b1a0 ***\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x000000004b284d40 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5032\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 4195070 RAD-Tags.\n",
      "  Inserted 2144237 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "459196 initial stacks were populated; 1685041 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.46614; Std Dev: 17.6764; Max: 4735\n",
      "Deleveraging trigger: 23; Removal trigger: 41\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5008 stacks.\n",
      "  454704 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.20283; Std Dev: 5.36733; Max: 41\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  454704 stacks merged into 432613 loci; deleveraged 1018 loci; blacklisted 557 loci.\n",
      "After merging, coverage depth Mean: 5.43228; Std Dev: 5.71833; Max: 86\n",
      "Merging remainder radtags\n",
      "  1685041 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': *** Error in `ustacks': malloc(): memory corruption: 0x00007fae9df2c690 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5033\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 3168502 RAD-Tags.\n",
      "  Inserted 1509483 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "355953 initial stacks were populated; 1153530 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.66078; Std Dev: 36.7605; Max: 19053\n",
      "Deleveraging trigger: 42; Removal trigger: 79\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3927 stacks.\n",
      "  352257 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.30936; Std Dev: 4.67194; Max: 79\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  352257 stacks merged into 334201 loci; deleveraged 795 loci; blacklisted 513 loci.\n",
      "After merging, coverage depth Mean: 5.52972; Std Dev: 4.94845; Max: 127\n",
      "Merging remainder radtags\n",
      "  1153530 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 221853 remainder reads; unable to match 931677 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.19504; Std Dev: 5.58105; Max: 199\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2236825\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz... read 3168502 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5034\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 4950033 RAD-Tags.\n",
      "  Inserted 2392989 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "482258 initial stacks were populated; 1910731 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.30223; Std Dev: 21.6679; Max: 5557\n",
      "Deleveraging trigger: 28; Removal trigger: 50\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5619 stacks.\n",
      "  477084 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.97922; Std Dev: 6.29993; Max: 50\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  477084 stacks merged into 450572 loci; deleveraged 1221 loci; blacklisted 728 loci.\n",
      "After merging, coverage depth Mean: 6.28341; Std Dev: 6.72607; Max: 101\n",
      "Merging remainder radtags\n",
      "  1910731 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 370330 remainder reads; unable to match 1540401 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.10746; Std Dev: 7.74496; Max: 303\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3409632\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz... read 4950033 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5035\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1704806 RAD-Tags.\n",
      "  Inserted 261857 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "105852 initial stacks were populated; 156005 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6318; Std Dev: 42.0104; Max: 6832\n",
      "Deleveraging trigger: 57; Removal trigger: 99\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3025 stacks.\n",
      "  102957 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.7113; Std Dev: 9.57485; Max: 99\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  102957 stacks merged into 92747 loci; deleveraged 517 loci; blacklisted 317 loci.\n",
      "After merging, coverage depth Mean: 15.036; Std Dev: 10.0885; Max: 169\n",
      "Merging remainder radtags\n",
      "  156005 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 111593 remainder reads; unable to match 44412 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.2451; Std Dev: 10.7898; Max: 176\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1660394\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz... read 1704806 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5036\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1814074 RAD-Tags.\n",
      "  Inserted 268407 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102937 initial stacks were populated; 165470 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.0157; Std Dev: 50.1006; Max: 9176\n",
      "Deleveraging trigger: 66; Removal trigger: 116\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2928 stacks.\n",
      "  100121 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.0732; Std Dev: 9.57085; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100121 stacks merged into 90597 loci; deleveraged 402 loci; blacklisted 277 loci.\n",
      "After merging, coverage depth Mean: 16.4552; Std Dev: 9.97421; Max: 194\n",
      "Merging remainder radtags\n",
      "  165470 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 119636 remainder reads; unable to match 45834 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.7814; Std Dev: 10.6349; Max: 208\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1768240\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz... read 1814074 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5037\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1586004 RAD-Tags.\n",
      "  Inserted 239398 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99643 initial stacks were populated; 139755 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.5143; Std Dev: 46.8163; Max: 7434\n",
      "Deleveraging trigger: 61; Removal trigger: 108\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2886 stacks.\n",
      "  96872 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.6067; Std Dev: 8.33083; Max: 108\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  96872 stacks merged into 88994 loci; deleveraged 356 loci; blacklisted 235 loci.\n",
      "After merging, coverage depth Mean: 14.6526; Std Dev: 8.82497; Max: 185\n",
      "Merging remainder radtags\n",
      "  139755 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 101497 remainder reads; unable to match 38258 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.7976; Std Dev: 9.40022; Max: 193\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1547746\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz... read 1586004 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5038\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2356641 RAD-Tags.\n",
      "  Inserted 1254020 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "316035 initial stacks were populated; 937985 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.48892; Std Dev: 13.8658; Max: 3346\n",
      "Deleveraging trigger: 18; Removal trigger: 32\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3727 stacks.\n",
      "  312575 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.27024; Std Dev: 3.23117; Max: 32\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  312575 stacks merged into 299082 loci; deleveraged 605 loci; blacklisted 325 loci.\n",
      "After merging, coverage depth Mean: 4.43327; Std Dev: 3.46037; Max: 55\n",
      "Merging remainder radtags\n",
      "  937985 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 170695 remainder reads; unable to match 767290 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.00513; Std Dev: 4.04546; Max: 212\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1589351\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz... read 2356641 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5039\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1831298 RAD-Tags.\n",
      "  Inserted 1004366 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "260405 initial stacks were populated; 743961 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.17556; Std Dev: 10.9583; Max: 2117\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3236 stacks.\n",
      "  257519 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.95276; Std Dev: 2.99824; Max: 26\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  257519 stacks merged into 246867 loci; deleveraged 506 loci; blacklisted 280 loci.\n",
      "After merging, coverage depth Mean: 4.09716; Std Dev: 3.21549; Max: 54\n",
      "Merging remainder radtags\n",
      "  743961 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007fb695131390 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5040\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 4063779 RAD-Tags.\n",
      "  Inserted 2045764 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "434243 initial stacks were populated; 1611521 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.6472; Std Dev: 19.0443; Max: 4220\n",
      "Deleveraging trigger: 25; Removal trigger: 44\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5203 stacks.\n",
      "  429449 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.34922; Std Dev: 5.1404; Max: 44\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  429449 stacks merged into 406528 loci; deleveraged 1044 loci; blacklisted 585 loci.\n",
      "After merging, coverage depth Mean: 5.60576; Std Dev: 5.4982; Max: 83\n",
      "Merging remainder radtags\n",
      "  1611521 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 322749 remainder reads; unable to match 1288772 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.40162; Std Dev: 6.44772; Max: 182\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2775007\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz... read 4063779 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5041\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 4529686 RAD-Tags.\n",
      "  Inserted 2328403 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "495602 initial stacks were populated; 1832801 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.44163; Std Dev: 46.505; Max: 29224\n",
      "Deleveraging trigger: 52; Removal trigger: 98\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4217 stacks.\n",
      "  491615 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.10918; Std Dev: 5.36884; Max: 98\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  491615 stacks merged into 464753 loci; deleveraged 1430 loci; blacklisted 912 loci.\n",
      "After merging, coverage depth Mean: 5.32574; Std Dev: 5.67344; Max: 125\n",
      "Merging remainder radtags\n",
      "  1832801 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (123 k-mers per read); 7 k-mer hits required.\n",
      "  Matched 314380 remainder reads; unable to match 1518421 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.00385; Std Dev: 6.57177; Max: 146\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3011265\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz... read 4529686 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5042\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2373889 RAD-Tags.\n",
      "  Inserted 1005650 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "277027 initial stacks were populated; 728623 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.93901; Std Dev: 19.4497; Max: 4926\n",
      "Deleveraging trigger: 25; Removal trigger: 45\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3962 stacks.\n",
      "  273338 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.578; Std Dev: 4.84362; Max: 45\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  273338 stacks merged into 259945 loci; deleveraged 599 loci; blacklisted 344 loci.\n",
      "After merging, coverage depth Mean: 5.81218; Std Dev: 5.14576; Max: 78\n",
      "Merging remainder radtags\n",
      "  728623 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f9e9d2454f0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5043\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2313190 RAD-Tags.\n",
      "  Inserted 907488 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "259926 initial stacks were populated; 647562 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.40809; Std Dev: 21.3073; Max: 4835\n",
      "Deleveraging trigger: 28; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3982 stacks.\n",
      "  256247 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.98976; Std Dev: 5.69932; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  256247 stacks merged into 243813 loci; deleveraged 618 loci; blacklisted 323 loci.\n",
      "After merging, coverage depth Mean: 6.2436; Std Dev: 6.07973; Max: 100\n",
      "Merging remainder radtags\n",
      "  647562 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007fa38d1210e0 ***\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007fa37d11a900 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5044\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2962843 RAD-Tags.\n",
      "  Inserted 1435922 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "351261 initial stacks were populated; 1084661 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.34697; Std Dev: 16.0963; Max: 4414\n",
      "Deleveraging trigger: 21; Removal trigger: 38\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4447 stacks.\n",
      "  347198 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.05772; Std Dev: 4.60931; Max: 38\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  347198 stacks merged into 322057 loci; deleveraged 1062 loci; blacklisted 492 loci.\n",
      "After merging, coverage depth Mean: 5.4103; Std Dev: 5.06821; Max: 79\n",
      "Merging remainder radtags\n",
      "  1084661 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f26956dd6d0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5045\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1461829 RAD-Tags.\n",
      "  Inserted 870492 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "213183 initial stacks were populated; 657309 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.77385; Std Dev: 8.53966; Max: 1729\n",
      "Deleveraging trigger: 12; Removal trigger: 21\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2646 stacks.\n",
      "  210916 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.58549; Std Dev: 2.54084; Max: 21\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  210916 stacks merged into 203456 loci; deleveraged 356 loci; blacklisted 197 loci.\n",
      "After merging, coverage depth Mean: 3.69732; Std Dev: 2.72309; Max: 49\n",
      "Merging remainder radtags\n",
      "  657309 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (123 k-mers per read); 7 k-mer hits required.\n",
      "  Matched 88701 remainder reads; unable to match 568608 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.13453; Std Dev: 3.19206; Max: 116\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 893221\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz... read 1461829 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5046\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1553199 RAD-Tags.\n",
      "  Inserted 959977 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "225079 initial stacks were populated; 734898 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.63562; Std Dev: 8.78361; Max: 2022\n",
      "Deleveraging trigger: 12; Removal trigger: 21\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2903 stacks.\n",
      "  222675 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.44044; Std Dev: 2.56069; Max: 21\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  222675 stacks merged into 213717 loci; deleveraged 519 loci; blacklisted 266 loci.\n",
      "After merging, coverage depth Mean: 3.56071; Std Dev: 2.76964; Max: 48\n",
      "Merging remainder radtags\n",
      "  734898 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 96543 remainder reads; unable to match 638355 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.01407; Std Dev: 3.32423; Max: 121\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 914844\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz... read 1553199 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5047\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 3127373 RAD-Tags.\n",
      "  Inserted 1101672 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "273701 initial stacks were populated; 827971 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.40115; Std Dev: 29.3629; Max: 6952\n",
      "Deleveraging trigger: 38; Removal trigger: 67\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4473 stacks.\n",
      "  269530 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.84865; Std Dev: 8.45611; Max: 67\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  269530 stacks merged into 254123 loci; deleveraged 726 loci; blacklisted 405 loci.\n",
      "After merging, coverage depth Mean: 8.25457; Std Dev: 8.9947; Max: 118\n",
      "Merging remainder radtags\n",
      "  827971 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 206139 remainder reads; unable to match 621832 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.06801; Std Dev: 9.68581; Max: 124\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2505541\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz... read 3127373 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5048\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2495655 RAD-Tags.\n",
      "  Inserted 1449990 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "322614 initial stacks were populated; 1127376 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.24123; Std Dev: 12.4869; Max: 2951\n",
      "Deleveraging trigger: 17; Removal trigger: 29\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3285 stacks.\n",
      "  319728 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.03759; Std Dev: 3.5024; Max: 29\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 49 (103 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  319728 stacks merged into 305744 loci; deleveraged 688 loci; blacklisted 373 loci.\n",
      "After merging, coverage depth Mean: 4.19652; Std Dev: 3.75588; Max: 60\n",
      "Merging remainder radtags\n",
      "  1127376 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (123 k-mers per read); 7 k-mer hits required.\n",
      "  Matched 178100 remainder reads; unable to match 949276 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.78051; Std Dev: 4.48685; Max: 90\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1546379\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz... read 2495655 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5049\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1755078 RAD-Tags.\n",
      "  Inserted 985022 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "255798 initial stacks were populated; 729224 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.01041; Std Dev: 10.1056; Max: 2133\n",
      "Deleveraging trigger: 14; Removal trigger: 24\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3159 stacks.\n",
      "  253011 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.80719; Std Dev: 2.76698; Max: 24\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  253011 stacks merged into 243210 loci; deleveraged 452 loci; blacklisted 251 loci.\n",
      "After merging, coverage depth Mean: 3.93595; Std Dev: 2.9654; Max: 49\n",
      "Merging remainder radtags\n",
      "  729224 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (123 k-mers per read); 7 k-mer hits required.\n",
      "  Matched 114394 remainder reads; unable to match 614830 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.40751; Std Dev: 3.46883; Max: 128\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1140248\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz... read 1755078 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5050\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 431328 RAD-Tags.\n",
      "  Inserted 166131 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "72086 initial stacks were populated; 94045 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.6789; Std Dev: 10.7362; Max: 1704\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1747 stacks.\n",
      "  70478 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.37068; Std Dev: 2.79598; Max: 26\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  70478 stacks merged into 67320 loci; deleveraged 183 loci; blacklisted 137 loci.\n",
      "After merging, coverage depth Mean: 4.52522; Std Dev: 3.004; Max: 49\n",
      "Merging remainder radtags\n",
      "  94045 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f5b2c4a2970 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5051\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2406063 RAD-Tags.\n",
      "  Inserted 818037 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "230344 initial stacks were populated; 587693 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.89415; Std Dev: 54.8944; Max: 22643\n",
      "Deleveraging trigger: 63; Removal trigger: 118\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3648 stacks.\n",
      "  226903 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.2295; Std Dev: 8.02578; Max: 118\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  226903 stacks merged into 213810 loci; deleveraged 666 loci; blacklisted 438 loci.\n",
      "After merging, coverage depth Mean: 7.56172; Std Dev: 8.49691; Max: 192\n",
      "Merging remainder radtags\n",
      "  587693 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 152944 remainder reads; unable to match 434749 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.27921; Std Dev: 9.18032; Max: 201\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1971314\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz... read 2406063 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5052\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1533313 RAD-Tags.\n",
      "  Inserted 917147 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "221931 initial stacks were populated; 695216 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.77639; Std Dev: 72.9264; Max: 33966\n",
      "Deleveraging trigger: 77; Removal trigger: 150\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1338 stacks.\n",
      "  220720 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.40575; Std Dev: 2.99171; Max: 148\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  220720 stacks merged into 210667 loci; deleveraged 475 loci; blacklisted 390 loci.\n",
      "After merging, coverage depth Mean: 3.49641; Std Dev: 2.98452; Max: 158\n",
      "Merging remainder radtags\n",
      "  695216 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (123 k-mers per read); 7 k-mer hits required.\n",
      "  Matched 97158 remainder reads; unable to match 598058 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.95873; Std Dev: 3.58748; Max: 182\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 935255\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz... read 1533313 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5053\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2151396 RAD-Tags.\n",
      "  Inserted 1158087 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "284453 initial stacks were populated; 873634 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.492; Std Dev: 28.2306; Max: 13578\n",
      "Deleveraging trigger: 33; Removal trigger: 61\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2792 stacks.\n",
      "  281866 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.21984; Std Dev: 3.48429; Max: 61\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  281866 stacks merged into 268619 loci; deleveraged 607 loci; blacklisted 381 loci.\n",
      "After merging, coverage depth Mean: 4.37957; Std Dev: 3.72657; Max: 124\n",
      "Merging remainder radtags\n",
      "  873634 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 153292 remainder reads; unable to match 720342 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.95149; Std Dev: 4.34845; Max: 151\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1431054\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz... read 2151396 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5054\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 3221085 RAD-Tags.\n",
      "  Inserted 1521571 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "360727 initial stacks were populated; 1160844 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.71136; Std Dev: 18.1965; Max: 5037\n",
      "Deleveraging trigger: 24; Removal trigger: 42\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4716 stacks.\n",
      "  356428 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.38115; Std Dev: 4.98437; Max: 42\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  356428 stacks merged into 338128 loci; deleveraged 863 loci; blacklisted 440 loci.\n",
      "After merging, coverage depth Mean: 5.63108; Std Dev: 5.32848; Max: 87\n",
      "Merging remainder radtags\n",
      "  1160844 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks*** Error in `ustacks': malloc(): memory corruption: 0x00007f44cd82f8f0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5055\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 3094883 RAD-Tags.\n",
      "  Inserted 1749352 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "379838 initial stacks were populated; 1369514 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.54238; Std Dev: 13.863; Max: 3649\n",
      "Deleveraging trigger: 18; Removal trigger: 32\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4336 stacks.\n",
      "  376262 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.29897; Std Dev: 4.34951; Max: 32\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  376262 stacks merged into 358279 loci; deleveraged 884 loci; blacklisted 443 loci.\n",
      "After merging, coverage depth Mean: 4.48682; Std Dev: 4.65278; Max: 71\n",
      "Merging remainder radtags\n",
      "  1369514 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 229294 remainder reads; unable to match 1140220 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.12896; Std Dev: 5.55365; Max: 199\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1954663\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz... read 3094883 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5056\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1269594 RAD-Tags.\n",
      "  Inserted 566984 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "169840 initial stacks were populated; 397144 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.13689; Std Dev: 115.074; Max: 40201\n",
      "Deleveraging trigger: 120; Removal trigger: 235\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2099 stacks.\n",
      "  167869 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.39894; Std Dev: 6.03505; Max: 235\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  167869 stacks merged into 159360 loci; deleveraged 342 loci; blacklisted 467 loci.\n",
      "After merging, coverage depth Mean: 4.44404; Std Dev: 5.32794; Max: 323\n",
      "Merging remainder radtags\n",
      "  397144 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 73373 remainder reads; unable to match 323771 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.90619; Std Dev: 5.99159; Max: 348\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 945823\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz... read 1269594 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5057\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2172704 RAD-Tags.\n",
      "  Inserted 1311826 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "279356 initial stacks were populated; 1032470 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.08165; Std Dev: 13.031; Max: 3089\n",
      "Deleveraging trigger: 17; Removal trigger: 30\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2294 stacks.\n",
      "  277328 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.89612; Std Dev: 3.33508; Max: 30\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  277328 stacks merged into 265096 loci; deleveraged 583 loci; blacklisted 318 loci.\n",
      "After merging, coverage depth Mean: 4.04969; Std Dev: 3.57849; Max: 51\n",
      "Merging remainder radtags\n",
      "  1032470 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 158317 remainder reads; unable to match 874153 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.64822; Std Dev: 4.35671; Max: 254\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1298551\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz... read 2172704 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5058\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2932390 RAD-Tags.\n",
      "  Inserted 1472306 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "342201 initial stacks were populated; 1130105 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.26674; Std Dev: 15.9147; Max: 3446\n",
      "Deleveraging trigger: 21; Removal trigger: 37\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4352 stacks.\n",
      "  338301 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.97505; Std Dev: 4.56443; Max: 37\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  338301 stacks merged into 321267 loci; deleveraged 824 loci; blacklisted 451 loci.\n",
      "After merging, coverage depth Mean: 5.20005; Std Dev: 4.88495; Max: 64\n",
      "Merging remainder radtags\n",
      "  1130105 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': free(): corrupted unsorted chunks: 0x00007fac796b3130 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5059\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 298885 RAD-Tags.\n",
      "  Inserted 99767 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "54344 initial stacks were populated; 45423 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.66403; Std Dev: 10.4604; Max: 1706\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1329 stacks.\n",
      "  53116 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.36533; Std Dev: 2.61869; Max: 26\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  53116 stacks merged into 50655 loci; deleveraged 133 loci; blacklisted 110 loci.\n",
      "After merging, coverage depth Mean: 4.52763; Std Dev: 2.84739; Max: 56\n",
      "Merging remainder radtags\n",
      "  45423 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 19868 remainder reads; unable to match 25555 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.9215; Std Dev: 3.10701; Max: 62\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 273330\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz... read 298885 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5060\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 6958 RAD-Tags.\n",
      "  Inserted 5698 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "766 initial stacks were populated; 4932 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.64491; Std Dev: 3.40201; Max: 63\n",
      "Deleveraging trigger: 6; Removal trigger: 9\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 57 stacks.\n",
      "  718 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.31876; Std Dev: 0.805447; Max: 9\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  718 stacks merged into 691 loci; deleveraged 2 loci; blacklisted 1 loci.\n",
      "After merging, coverage depth Mean: 2.37885; Std Dev: 0.954359; Max: 12\n",
      "Merging remainder radtags\n",
      "  4932 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 463 remainder reads; unable to match 4469 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.05874; Std Dev: 3.09081; Max: 45\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2489\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz... read 6958 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5061\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3081893 RAD-Tags.\n",
      "  Inserted 347536 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "107445 initial stacks were populated; 240091 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 26.4489; Std Dev: 103.673; Max: 18824\n",
      "Deleveraging trigger: 130; Removal trigger: 234\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3824 stacks.\n",
      "  103709 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.9993; Std Dev: 16.6233; Max: 232\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  103709 stacks merged into 88600 loci; deleveraged 525 loci; blacklisted 491 loci.\n",
      "After merging, coverage depth Mean: 28.8067; Std Dev: 16.2128; Max: 348\n",
      "Merging remainder radtags\n",
      "  240091 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 201292 remainder reads; unable to match 38799 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 31.0936; Std Dev: 17.2257; Max: 364\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3043094\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz... read 3081893 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5062\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4604151 RAD-Tags.\n",
      "  Inserted 508362 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "124870 initial stacks were populated; 383492 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 33.8004; Std Dev: 102.245; Max: 22016\n",
      "Deleveraging trigger: 136; Removal trigger: 238\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5427 stacks.\n",
      "  119627 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 32.0211; Std Dev: 30.9846; Max: 238\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  119627 stacks merged into 93769 loci; deleveraged 972 loci; blacklisted 1256 loci.\n",
      "After merging, coverage depth Mean: 40.0231; Std Dev: 31.8271; Max: 338\n",
      "Merging remainder radtags\n",
      "  383492 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 306701 remainder reads; unable to match 76791 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 43.345; Std Dev: 34.0203; Max: 398\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4527360\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz... read 4604151 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5063\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1919446 RAD-Tags.\n",
      "  Inserted 260778 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95281 initial stacks were populated; 165497 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.4082; Std Dev: 55.6882; Max: 7712\n",
      "Deleveraging trigger: 74; Removal trigger: 130\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3003 stacks.\n",
      "  92398 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.3291; Std Dev: 10.8348; Max: 130\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92398 stacks merged into 83242 loci; deleveraged 394 loci; blacklisted 284 loci.\n",
      "After merging, coverage depth Mean: 18.9993; Std Dev: 11.1896; Max: 245\n",
      "Merging remainder radtags\n",
      "  165497 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 139359 remainder reads; unable to match 26138 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.6816; Std Dev: 12.029; Max: 265\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1893308\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz... read 1919446 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5064\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4543578 RAD-Tags.\n",
      "  Inserted 472914 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "115267 initial stacks were populated; 357647 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 36.3151; Std Dev: 107.57; Max: 16956\n",
      "Deleveraging trigger: 144; Removal trigger: 251\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4624 stacks.\n",
      "  110784 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 34.6901; Std Dev: 24.3695; Max: 251\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  110784 stacks merged into 91844 loci; deleveraged 657 loci; blacklisted 718 loci.\n",
      "After merging, coverage depth Mean: 41.3188; Std Dev: 23.5094; Max: 423\n",
      "Merging remainder radtags\n",
      "  357647 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 295142 remainder reads; unable to match 62505 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 44.5627; Std Dev: 24.9389; Max: 482\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4481073\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz... read 4543578 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5065\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6825612 RAD-Tags.\n",
      "  Inserted 650052 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "136654 initial stacks were populated; 513398 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 46.1912; Std Dev: 187.723; Max: 34416\n",
      "Deleveraging trigger: 234; Removal trigger: 422\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5694 stacks.\n",
      "  131084 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 43.852; Std Dev: 37.4926; Max: 418\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  131084 stacks merged into 104275 loci; deleveraged 906 loci; blacklisted 1365 loci.\n",
      "After merging, coverage depth Mean: 54.0731; Std Dev: 36.7669; Max: 490\n",
      "Merging remainder radtags\n",
      "  513398 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 402664 remainder reads; unable to match 110734 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 57.9906; Std Dev: 38.9147; Max: 515\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6714878\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz... read 6825612 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5066\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4911562 RAD-Tags.\n",
      "  Inserted 490647 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "116358 initial stacks were populated; 374289 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 38.9941; Std Dev: 146.935; Max: 24573\n",
      "Deleveraging trigger: 186; Removal trigger: 333\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4626 stacks.\n",
      "  111839 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 37.0535; Std Dev: 27.0819; Max: 331\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111839 stacks merged into 91876 loci; deleveraged 654 loci; blacklisted 835 loci.\n",
      "After merging, coverage depth Mean: 44.3281; Std Dev: 25.8383; Max: 523\n",
      "Merging remainder radtags\n",
      "  374289 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 308931 remainder reads; unable to match 65358 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 47.7254; Std Dev: 27.3609; Max: 550\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4846204\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz... read 4911562 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5067\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1425539 RAD-Tags.\n",
      "  Inserted 216515 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94123 initial stacks were populated; 122392 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.8451; Std Dev: 50.4703; Max: 8345\n",
      "Deleveraging trigger: 64; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2597 stacks.\n",
      "  91610 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.0328; Std Dev: 7.56272; Max: 114\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91610 stacks merged into 83734 loci; deleveraged 315 loci; blacklisted 241 loci.\n",
      "After merging, coverage depth Mean: 14.0668; Std Dev: 7.89051; Max: 194\n",
      "Merging remainder radtags\n",
      "  122392 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 98285 remainder reads; unable to match 24107 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.2451; Std Dev: 8.42123; Max: 224\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1401432\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz... read 1425539 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5068\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6407382 RAD-Tags.\n",
      "  Inserted 618927 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "129756 initial stacks were populated; 489171 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 45.6103; Std Dev: 145.409; Max: 24193\n",
      "Deleveraging trigger: 191; Removal trigger: 336\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5764 stacks.\n",
      "  124138 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 43.6747; Std Dev: 35.2806; Max: 335\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  124138 stacks merged into 95832 loci; deleveraged 937 loci; blacklisted 1344 loci.\n",
      "After merging, coverage depth Mean: 55.6353; Std Dev: 33.3817; Max: 483\n",
      "Merging remainder radtags\n",
      "  489171 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 399005 remainder reads; unable to match 90166 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 59.8647; Std Dev: 35.3439; Max: 511\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6317216\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz... read 6407382 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5069\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2933020 RAD-Tags.\n",
      "  Inserted 334889 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103691 initial stacks were populated; 231198 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 26.0565; Std Dev: 79.0776; Max: 13954\n",
      "Deleveraging trigger: 105; Removal trigger: 184\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3559 stacks.\n",
      "  100246 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.7426; Std Dev: 15.3207; Max: 183\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100246 stacks merged into 88418 loci; deleveraged 462 loci; blacklisted 367 loci.\n",
      "After merging, coverage depth Mean: 27.7522; Std Dev: 15.4656; Max: 266\n",
      "Merging remainder radtags\n",
      "  231198 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 191134 remainder reads; unable to match 40064 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 29.9258; Std Dev: 16.4276; Max: 280\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2892956\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz... read 2933020 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5070\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1600529 RAD-Tags.\n",
      "  Inserted 234550 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95419 initial stacks were populated; 139131 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.3156; Std Dev: 48.8172; Max: 9052\n",
      "Deleveraging trigger: 64; Removal trigger: 113\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2784 stacks.\n",
      "  92743 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.4462; Std Dev: 8.15225; Max: 113\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92743 stacks merged into 84476 loci; deleveraged 380 loci; blacklisted 240 loci.\n",
      "After merging, coverage depth Mean: 15.6861; Std Dev: 8.48509; Max: 157\n",
      "Merging remainder radtags\n",
      "  139131 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 114235 remainder reads; unable to match 24896 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.044; Std Dev: 9.08407; Max: 167\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1575633\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz... read 1600529 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5071\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2689632 RAD-Tags.\n",
      "  Inserted 305197 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98817 initial stacks were populated; 206380 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.1298; Std Dev: 77.388; Max: 13476\n",
      "Deleveraging trigger: 103; Removal trigger: 180\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3357 stacks.\n",
      "  95567 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.7787; Std Dev: 13.9319; Max: 179\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95567 stacks merged into 86044 loci; deleveraged 383 loci; blacklisted 322 loci.\n",
      "After merging, coverage depth Mean: 26.0956; Std Dev: 14.1024; Max: 284\n",
      "Merging remainder radtags\n",
      "  206380 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 175178 remainder reads; unable to match 31202 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 28.1418; Std Dev: 14.9736; Max: 310\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2658430\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz... read 2689632 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5072\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3369853 RAD-Tags.\n",
      "  Inserted 370431 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "105063 initial stacks were populated; 265368 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 29.5488; Std Dev: 113.087; Max: 18278\n",
      "Deleveraging trigger: 143; Removal trigger: 256\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3734 stacks.\n",
      "  101425 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 27.9825; Std Dev: 17.7199; Max: 256\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101425 stacks merged into 88813 loci; deleveraged 470 loci; blacklisted 449 loci.\n",
      "After merging, coverage depth Mean: 31.4884; Std Dev: 17.6177; Max: 340\n",
      "Merging remainder radtags\n",
      "  265368 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 219544 remainder reads; unable to match 45824 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 33.9756; Std Dev: 18.7117; Max: 366\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3324029\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz... read 3369853 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5073\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7142863 RAD-Tags.\n",
      "  Inserted 707813 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "141175 initial stacks were populated; 566638 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 46.5821; Std Dev: 188.617; Max: 34227\n",
      "Deleveraging trigger: 235; Removal trigger: 424\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5965 stacks.\n",
      "  135321 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 44.3693; Std Dev: 39.0858; Max: 423\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  135321 stacks merged into 102706 loci; deleveraged 1006 loci; blacklisted 1724 loci.\n",
      "After merging, coverage depth Mean: 57.2171; Std Dev: 37.6417; Max: 560\n",
      "Merging remainder radtags\n",
      "  566638 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 448302 remainder reads; unable to match 118336 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 61.6614; Std Dev: 39.941; Max: 588\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7024527\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz... read 7142863 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5074\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...10M...11M...done\n",
      "Loaded 11307187 RAD-Tags.\n",
      "  Inserted 1126269 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "202368 initial stacks were populated; 923901 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 51.3089; Std Dev: 268.191; Max: 60988\n",
      "Deleveraging trigger: 319; Removal trigger: 588\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 8715 stacks.\n",
      "  193797 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 48.4024; Std Dev: 58.9723; Max: 584\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  193797 stacks merged into 135490 loci; deleveraged 1741 loci; blacklisted 4557 loci.\n",
      "After merging, coverage depth Mean: 66.026; Std Dev: 61.3313; Max: 723\n",
      "Merging remainder radtags\n",
      "  923901 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 636965 remainder reads; unable to match 286936 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 70.8962; Std Dev: 65.2224; Max: 723\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 11020251\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz... read 11307187 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5075\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4420847 RAD-Tags.\n",
      "  Inserted 507345 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "128242 initial stacks were populated; 379103 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 31.5165; Std Dev: 118.744; Max: 24827\n",
      "Deleveraging trigger: 150; Removal trigger: 269\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4958 stacks.\n",
      "  123401 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 29.8442; Std Dev: 23.4157; Max: 268\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  123401 stacks merged into 99886 loci; deleveraged 864 loci; blacklisted 936 loci.\n",
      "After merging, coverage depth Mean: 36.2278; Std Dev: 22.9065; Max: 346\n",
      "Merging remainder radtags\n",
      "  379103 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 286819 remainder reads; unable to match 92284 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 39.1298; Std Dev: 24.3698; Max: 369\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4328563\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz... read 4420847 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5076\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5119263 RAD-Tags.\n",
      "  Inserted 580668 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "134482 initial stacks were populated; 446186 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 34.7487; Std Dev: 130.43; Max: 27745\n",
      "Deleveraging trigger: 165; Removal trigger: 296\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5369 stacks.\n",
      "  129241 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 32.9112; Std Dev: 27.3141; Max: 295\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  129241 stacks merged into 100671 loci; deleveraged 938 loci; blacklisted 1285 loci.\n",
      "After merging, coverage depth Mean: 41.4147; Std Dev: 26.4527; Max: 380\n",
      "Merging remainder radtags\n",
      "  446186 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 341485 remainder reads; unable to match 104701 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 44.8551; Std Dev: 28.2087; Max: 398\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5014562\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz... read 5119263 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5077\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...done\n",
      "Loaded 8012485 RAD-Tags.\n",
      "  Inserted 781283 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "151707 initial stacks were populated; 629576 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 48.6656; Std Dev: 184.009; Max: 35942\n",
      "Deleveraging trigger: 233; Removal trigger: 417\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 7058 stacks.\n",
      "  144797 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 46.4397; Std Dev: 43.7197; Max: 417\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  144797 stacks merged into 102828 loci; deleveraged 1251 loci; blacklisted 2616 loci.\n",
      "After merging, coverage depth Mean: 63.6748; Std Dev: 41.5402; Max: 585\n",
      "Merging remainder radtags\n",
      "  629576 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 488157 remainder reads; unable to match 141419 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 68.5533; Std Dev: 44.0984; Max: 618\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7871066\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz... read 8012485 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5078\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...8M...9M...done\n",
      "Loaded 9213735 RAD-Tags.\n",
      "  Inserted 810326 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143200 initial stacks were populated; 667126 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 59.683; Std Dev: 202.853; Max: 35621\n",
      "Deleveraging trigger: 263; Removal trigger: 465\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 7371 stacks.\n",
      "  135979 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 57.2662; Std Dev: 51.3889; Max: 459\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  135979 stacks merged into 97045 loci; deleveraged 1060 loci; blacklisted 2443 loci.\n",
      "After merging, coverage depth Mean: 78.544; Std Dev: 47.5487; Max: 606\n",
      "Merging remainder radtags\n",
      "  667126 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 539253 remainder reads; unable to match 127873 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 84.2533; Std Dev: 50.1754; Max: 625\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 9085862\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz... read 9213735 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5079\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2512638 RAD-Tags.\n",
      "  Inserted 322546 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "105469 initial stacks were populated; 217077 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 21.7653; Std Dev: 66.8036; Max: 11828\n",
      "Deleveraging trigger: 89; Removal trigger: 155\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3480 stacks.\n",
      "  102101 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 20.6337; Std Dev: 12.9555; Max: 153\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  102101 stacks merged into 88966 loci; deleveraged 511 loci; blacklisted 378 loci.\n",
      "After merging, coverage depth Mean: 23.3724; Std Dev: 12.8754; Max: 241\n",
      "Merging remainder radtags\n",
      "  217077 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 175073 remainder reads; unable to match 42004 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 25.3512; Std Dev: 13.7651; Max: 257\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2470634\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz... read 2512638 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5080\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6037770 RAD-Tags.\n",
      "  Inserted 624757 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "134193 initial stacks were populated; 490564 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 41.3375; Std Dev: 139.399; Max: 23549\n",
      "Deleveraging trigger: 181; Removal trigger: 320\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5394 stacks.\n",
      "  128939 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 39.3008; Std Dev: 32.8999; Max: 315\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128939 stacks merged into 105800 loci; deleveraged 847 loci; blacklisted 1130 loci.\n",
      "After merging, coverage depth Mean: 47.1295; Std Dev: 33.0583; Max: 447\n",
      "Merging remainder radtags\n",
      "  490564 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 372229 remainder reads; unable to match 118335 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 50.6905; Std Dev: 35.1219; Max: 472\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5919435\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz... read 6037770 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5081\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1656408 RAD-Tags.\n",
      "  Inserted 231201 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94922 initial stacks were populated; 136279 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.0145; Std Dev: 55.3617; Max: 8234\n",
      "Deleveraging trigger: 71; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2728 stacks.\n",
      "  92284 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.0729; Std Dev: 8.7188; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92284 stacks merged into 84611 loci; deleveraged 341 loci; blacklisted 238 loci.\n",
      "After merging, coverage depth Mean: 16.2184; Std Dev: 9.09348; Max: 228\n",
      "Merging remainder radtags\n",
      "  136279 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 108958 remainder reads; unable to match 27321 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.5111; Std Dev: 9.6783; Max: 240\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1629087\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz... read 1656408 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5082\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3920230 RAD-Tags.\n",
      "  Inserted 423322 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111932 initial stacks were populated; 311390 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 32.2414; Std Dev: 122.192; Max: 19579\n",
      "Deleveraging trigger: 154; Removal trigger: 277\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4164 stacks.\n",
      "  107867 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 30.5559; Std Dev: 20.7521; Max: 277\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107867 stacks merged into 91275 loci; deleveraged 582 loci; blacklisted 627 loci.\n",
      "After merging, coverage depth Mean: 35.549; Std Dev: 20.1359; Max: 355\n",
      "Merging remainder radtags\n",
      "  311390 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 251716 remainder reads; unable to match 59674 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 38.3289; Std Dev: 21.3483; Max: 383\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3860556\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz... read 3920230 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5083\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1949083 RAD-Tags.\n",
      "  Inserted 266387 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98552 initial stacks were populated; 167835 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.0742; Std Dev: 49.8637; Max: 7676\n",
      "Deleveraging trigger: 68; Removal trigger: 118\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2975 stacks.\n",
      "  95700 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.1139; Std Dev: 9.87616; Max: 118\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95700 stacks merged into 86689 loci; deleveraged 381 loci; blacklisted 248 loci.\n",
      "After merging, coverage depth Mean: 18.6868; Std Dev: 10.2166; Max: 197\n",
      "Merging remainder radtags\n",
      "  167835 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 132999 remainder reads; unable to match 34836 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.2276; Std Dev: 10.9017; Max: 265\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1914247\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz... read 1949083 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5084\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5815230 RAD-Tags.\n",
      "  Inserted 560435 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "127358 initial stacks were populated; 433077 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 42.26; Std Dev: 147.621; Max: 27899\n",
      "Deleveraging trigger: 190; Removal trigger: 338\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5133 stacks.\n",
      "  122346 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 40.2691; Std Dev: 32.4043; Max: 337\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  122346 stacks merged into 98791 loci; deleveraged 746 loci; blacklisted 1054 loci.\n",
      "After merging, coverage depth Mean: 49.0254; Std Dev: 31.6793; Max: 462\n",
      "Merging remainder radtags\n",
      "  433077 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 355396 remainder reads; unable to match 77681 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 52.6661; Std Dev: 33.6273; Max: 489\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5737549\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz... read 5815230 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5085\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5136394 RAD-Tags.\n",
      "  Inserted 512901 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "122283 initial stacks were populated; 390618 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 38.8098; Std Dev: 122.022; Max: 20973\n",
      "Deleveraging trigger: 161; Removal trigger: 283\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4955 stacks.\n",
      "  117470 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 36.9897; Std Dev: 27.5257; Max: 282\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117470 stacks merged into 95026 loci; deleveraged 833 loci; blacklisted 897 loci.\n",
      "After merging, coverage depth Mean: 45.1068; Std Dev: 26.3634; Max: 411\n",
      "Merging remainder radtags\n",
      "  390618 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 312960 remainder reads; unable to match 77658 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 48.4366; Std Dev: 27.8308; Max: 448\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5058736\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz... read 5136394 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5086\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 793285 RAD-Tags.\n",
      "  Inserted 155600 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "85379 initial stacks were populated; 70221 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.46887; Std Dev: 27.5428; Max: 4089\n",
      "Deleveraging trigger: 36; Removal trigger: 64\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2060 stacks.\n",
      "  83403 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.97205; Std Dev: 4.4201; Max: 64\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83403 stacks merged into 79022 loci; deleveraged 201 loci; blacklisted 174 loci.\n",
      "After merging, coverage depth Mean: 8.3065; Std Dev: 4.71167; Max: 97\n",
      "Merging remainder radtags\n",
      "  70221 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 52952 remainder reads; unable to match 17269 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.97878; Std Dev: 5.07956; Max: 160\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 776016\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz... read 793285 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5087\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1814441 RAD-Tags.\n",
      "  Inserted 243104 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95239 initial stacks were populated; 147865 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.4989; Std Dev: 51.9822; Max: 8892\n",
      "Deleveraging trigger: 69; Removal trigger: 121\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2866 stacks.\n",
      "  92478 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.5369; Std Dev: 9.34765; Max: 121\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92478 stacks merged into 84720 loci; deleveraged 309 loci; blacklisted 225 loci.\n",
      "After merging, coverage depth Mean: 17.8508; Std Dev: 9.71112; Max: 214\n",
      "Merging remainder radtags\n",
      "  147865 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 123030 remainder reads; unable to match 24835 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.3087; Std Dev: 10.3332; Max: 229\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1789606\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz... read 1814441 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5088\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1997292 RAD-Tags.\n",
      "  Inserted 276469 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99953 initial stacks were populated; 176516 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.2163; Std Dev: 65.6011; Max: 9934\n",
      "Deleveraging trigger: 84; Removal trigger: 149\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2936 stacks.\n",
      "  97098 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.1735; Std Dev: 10.2774; Max: 149\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97098 stacks merged into 87644 loci; deleveraged 375 loci; blacklisted 315 loci.\n",
      "After merging, coverage depth Mean: 18.7539; Std Dev: 10.6001; Max: 234\n",
      "Merging remainder radtags\n",
      "  176516 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 132675 remainder reads; unable to match 43841 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.2746; Std Dev: 11.277; Max: 247\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1953451\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz... read 1997292 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5089\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 622668 RAD-Tags.\n",
      "  Inserted 142149 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "82427 initial stacks were populated; 59722 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.82963; Std Dev: 18.401; Max: 2791\n",
      "Deleveraging trigger: 25; Removal trigger: 44\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1940 stacks.\n",
      "  80585 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.4294; Std Dev: 3.72493; Max: 44\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80585 stacks merged into 76305 loci; deleveraged 212 loci; blacklisted 151 loci.\n",
      "After merging, coverage depth Mean: 6.7175; Std Dev: 3.99697; Max: 73\n",
      "Merging remainder radtags\n",
      "  59722 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 42310 remainder reads; unable to match 17412 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.2738; Std Dev: 4.26651; Max: 76\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 605256\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz... read 622668 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5090\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2493911 RAD-Tags.\n",
      "  Inserted 316013 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "102533 initial stacks were populated; 213480 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.2409; Std Dev: 76.5681; Max: 12427\n",
      "Deleveraging trigger: 99; Removal trigger: 175\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3325 stacks.\n",
      "  99308 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.0153; Std Dev: 12.8196; Max: 173\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  99308 stacks merged into 88933 loci; deleveraged 436 loci; blacklisted 325 loci.\n",
      "After merging, coverage depth Mean: 23.165; Std Dev: 13.088; Max: 278\n",
      "Merging remainder radtags\n",
      "  213480 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 167578 remainder reads; unable to match 45902 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 25.0584; Std Dev: 13.927; Max: 298\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2448009\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz... read 2493911 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5091\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1087223 RAD-Tags.\n",
      "  Inserted 187887 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "90926 initial stacks were populated; 96961 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.8909; Std Dev: 35.687; Max: 5512\n",
      "Deleveraging trigger: 47; Removal trigger: 82\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2343 stacks.\n",
      "  88684 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.2473; Std Dev: 5.91571; Max: 81\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  88684 stacks merged into 83027 loci; deleveraged 261 loci; blacklisted 196 loci.\n",
      "After merging, coverage depth Mean: 10.8214; Std Dev: 6.2776; Max: 122\n",
      "Merging remainder radtags\n",
      "  96961 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 73886 remainder reads; unable to match 23075 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.7145; Std Dev: 6.68625; Max: 130\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1064148\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz... read 1087223 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5092\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1896600 RAD-Tags.\n",
      "  Inserted 256419 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "97152 initial stacks were populated; 159267 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.8826; Std Dev: 48.5869; Max: 7312\n",
      "Deleveraging trigger: 66; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3015 stacks.\n",
      "  94260 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.912; Std Dev: 9.61107; Max: 115\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  94260 stacks merged into 85830 loci; deleveraged 344 loci; blacklisted 247 loci.\n",
      "After merging, coverage depth Mean: 18.3859; Std Dev: 9.99085; Max: 182\n",
      "Merging remainder radtags\n",
      "  159267 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 126162 remainder reads; unable to match 33105 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.8621; Std Dev: 10.5996; Max: 189\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1863495\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz... read 1896600 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5093\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1870044 RAD-Tags.\n",
      "  Inserted 254371 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "97502 initial stacks were populated; 156869 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.5707; Std Dev: 63.1856; Max: 9762\n",
      "Deleveraging trigger: 81; Removal trigger: 144\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2797 stacks.\n",
      "  94784 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.5826; Std Dev: 9.81023; Max: 142\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  94784 stacks merged into 85964 loci; deleveraged 388 loci; blacklisted 272 loci.\n",
      "After merging, coverage depth Mean: 18.0337; Std Dev: 10.112; Max: 210\n",
      "Merging remainder radtags\n",
      "  156869 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 126444 remainder reads; unable to match 30425 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.5106; Std Dev: 10.7527; Max: 229\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1839619\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz... read 1870044 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5094\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 576672 RAD-Tags.\n",
      "  Inserted 142211 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "81921 initial stacks were populated; 60290 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.30341; Std Dev: 22.0349; Max: 3203\n",
      "Deleveraging trigger: 28; Removal trigger: 50\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1879 stacks.\n",
      "  80120 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.89893; Std Dev: 3.35509; Max: 50\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80120 stacks merged into 75928 loci; deleveraged 203 loci; blacklisted 155 loci.\n",
      "After merging, coverage depth Mean: 6.1499; Std Dev: 3.62301; Max: 76\n",
      "Merging remainder radtags\n",
      "  60290 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 40998 remainder reads; unable to match 19292 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.69153; Std Dev: 3.9399; Max: 155\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 557380\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz... read 576672 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5095\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 345734 RAD-Tags.\n",
      "  Inserted 114093 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "66986 initial stacks were populated; 47107 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.45805; Std Dev: 11.4881; Max: 1630\n",
      "Deleveraging trigger: 16; Removal trigger: 27\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1588 stacks.\n",
      "  65487 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.18582; Std Dev: 2.27503; Max: 27\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  65487 stacks merged into 62794 loci; deleveraged 140 loci; blacklisted 116 loci.\n",
      "After merging, coverage depth Mean: 4.31919; Std Dev: 2.44507; Max: 43\n",
      "Merging remainder radtags\n",
      "  47107 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 22590 remainder reads; unable to match 24517 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.68012; Std Dev: 2.68456; Max: 84\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 321217\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz... read 345734 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5096\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 6519826 RAD-Tags.\n",
      "  Inserted 3331617 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "599521 initial stacks were populated; 2732096 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.31793; Std Dev: 29.4303; Max: 9571\n",
      "Deleveraging trigger: 36; Removal trigger: 65\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6731 stacks.\n",
      "  593271 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.9923; Std Dev: 7.92524; Max: 65\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  593271 stacks merged into 540118 loci; deleveraged 2034 loci; blacklisted 1662 loci.\n",
      "After merging, coverage depth Mean: 6.48688; Std Dev: 8.77099; Max: 122\n",
      "Merging remainder radtags\n",
      "  2732096 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (117 k-mers per read); 1 k-mer hits required.\n",
      "  Matched 560807 remainder reads; unable to match 2171289 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.52932; Std Dev: 10.3454; Max: 417\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4348537\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz... read 6519826 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5097\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3710826 RAD-Tags.\n",
      "  Inserted 536075 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "159995 initial stacks were populated; 376080 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 20.8428; Std Dev: 88.1965; Max: 19858\n",
      "Deleveraging trigger: 109; Removal trigger: 197\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5213 stacks.\n",
      "  154916 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.4755; Std Dev: 18.775; Max: 197\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  154916 stacks merged into 94459 loci; deleveraged 1431 loci; blacklisted 3193 loci.\n",
      "After merging, coverage depth Mean: 30.3969; Std Dev: 17.9202; Max: 210\n",
      "Merging remainder radtags\n",
      "  376080 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 311034 remainder reads; unable to match 65046 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 33.8099; Std Dev: 19.3356; Max: 274\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3645780\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz... read 3710826 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5098\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 763190 RAD-Tags.\n",
      "  Inserted 207860 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "86454 initial stacks were populated; 121406 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.42342; Std Dev: 19.9638; Max: 3534\n",
      "Deleveraging trigger: 27; Removal trigger: 47\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2738 stacks.\n",
      "  83841 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.8825; Std Dev: 4.91692; Max: 47\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83841 stacks merged into 69701 loci; deleveraged 474 loci; blacklisted 344 loci.\n",
      "After merging, coverage depth Mean: 8.13131; Std Dev: 5.37466; Max: 70\n",
      "Merging remainder radtags\n",
      "  121406 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 94749 remainder reads; unable to match 26657 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.49988; Std Dev: 6.03519; Max: 82\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 736533\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz... read 763190 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5099\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3082824 RAD-Tags.\n",
      "  Inserted 468282 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "141961 initial stacks were populated; 326321 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4173; Std Dev: 68.1116; Max: 12736\n",
      "Deleveraging trigger: 88; Removal trigger: 156\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4730 stacks.\n",
      "  137372 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.2244; Std Dev: 15.9704; Max: 155\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  137372 stacks merged into 93230 loci; deleveraged 1133 loci; blacklisted 1866 loci.\n",
      "After merging, coverage depth Mean: 25.9608; Std Dev: 15.4798; Max: 190\n",
      "Merging remainder radtags\n",
      "  326321 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 268860 remainder reads; unable to match 57461 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 28.9081; Std Dev: 16.747; Max: 211\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3025363\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz... read 3082824 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5100\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1586287 RAD-Tags.\n",
      "  Inserted 301800 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111749 initial stacks were populated; 190051 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.4944; Std Dev: 39.7975; Max: 7403\n",
      "Deleveraging trigger: 52; Removal trigger: 92\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3351 stacks.\n",
      "  108515 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7385; Std Dev: 7.93013; Max: 92\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  108515 stacks merged into 87313 loci; deleveraged 662 loci; blacklisted 486 loci.\n",
      "After merging, coverage depth Mean: 14.3724; Std Dev: 8.03518; Max: 121\n",
      "Merging remainder radtags\n",
      "  190051 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 159772 remainder reads; unable to match 30279 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.215; Std Dev: 8.74965; Max: 154\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1556008\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz... read 1586287 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5101\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5002086 RAD-Tags.\n",
      "  Inserted 607446 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "148120 initial stacks were populated; 459326 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.6695; Std Dev: 120.787; Max: 27181\n",
      "Deleveraging trigger: 151; Removal trigger: 272\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5762 stacks.\n",
      "  142502 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.8762; Std Dev: 26.2133; Max: 272\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  142502 stacks merged into 98569 loci; deleveraged 1263 loci; blacklisted 2088 loci.\n",
      "After merging, coverage depth Mean: 40.439; Std Dev: 25.1122; Max: 386\n",
      "Merging remainder radtags\n",
      "  459326 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 368749 remainder reads; unable to match 90577 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 44.2667; Std Dev: 26.9253; Max: 469\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4911509\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz... read 5002086 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5102\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1623565 RAD-Tags.\n",
      "  Inserted 308907 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114507 initial stacks were populated; 194400 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.481; Std Dev: 40.0923; Max: 7685\n",
      "Deleveraging trigger: 53; Removal trigger: 93\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3359 stacks.\n",
      "  111265 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7479; Std Dev: 8.1716; Max: 93\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111265 stacks merged into 85130 loci; deleveraged 716 loci; blacklisted 711 loci.\n",
      "After merging, coverage depth Mean: 15.0455; Std Dev: 8.03825; Max: 108\n",
      "Merging remainder radtags\n",
      "  194400 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 171365 remainder reads; unable to match 23035 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.0782; Std Dev: 8.76853; Max: 165\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1600530\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz... read 1623565 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5103\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2896047 RAD-Tags.\n",
      "  Inserted 408188 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121649 initial stacks were populated; 286539 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 21.4511; Std Dev: 74.1222; Max: 14447\n",
      "Deleveraging trigger: 96; Removal trigger: 170\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4446 stacks.\n",
      "  117317 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 20.2031; Std Dev: 15.0121; Max: 170\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117317 stacks merged into 89692 loci; deleveraged 833 loci; blacklisted 848 loci.\n",
      "After merging, coverage depth Mean: 25.9147; Std Dev: 14.3313; Max: 241\n",
      "Merging remainder radtags\n",
      "  286539 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 243766 remainder reads; unable to match 42773 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 28.662; Std Dev: 15.4148; Max: 259\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2853274\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz... read 2896047 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5104\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6303429 RAD-Tags.\n",
      "  Inserted 714652 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "159674 initial stacks were populated; 554978 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 36.0012; Std Dev: 125.21; Max: 21973\n",
      "Deleveraging trigger: 161; Removal trigger: 286\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6273 stacks.\n",
      "  153550 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 34.2091; Std Dev: 32.7654; Max: 286\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  153550 stacks merged into 98402 loci; deleveraged 1410 loci; blacklisted 3117 loci.\n",
      "After merging, coverage depth Mean: 51.483; Std Dev: 30.5115; Max: 401\n",
      "Merging remainder radtags\n",
      "  554978 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 449981 remainder reads; unable to match 104997 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 56.2129; Std Dev: 32.5782; Max: 421\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6198432\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz... read 6303429 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5105\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1203467 RAD-Tags.\n",
      "  Inserted 247055 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95946 initial stacks were populated; 151109 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.9682; Std Dev: 31.8843; Max: 5845\n",
      "Deleveraging trigger: 43; Removal trigger: 75\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2956 stacks.\n",
      "  93123 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.2174; Std Dev: 7.72888; Max: 75\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  93123 stacks merged into 81175 loci; deleveraged 516 loci; blacklisted 365 loci.\n",
      "After merging, coverage depth Mean: 11.5437; Std Dev: 8.24687; Max: 102\n",
      "Merging remainder radtags\n",
      "  151109 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 110836 remainder reads; unable to match 40273 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.9175; Std Dev: 9.07123; Max: 166\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1163194\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz... read 1203467 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5106\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3914709 RAD-Tags.\n",
      "  Inserted 531747 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "142489 initial stacks were populated; 389258 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 24.7419; Std Dev: 88.74; Max: 16115\n",
      "Deleveraging trigger: 113; Removal trigger: 202\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5098 stacks.\n",
      "  137518 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.3242; Std Dev: 20.0302; Max: 202\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  137518 stacks merged into 96903 loci; deleveraged 1195 loci; blacklisted 1696 loci.\n",
      "After merging, coverage depth Mean: 32.1719; Std Dev: 19.186; Max: 257\n",
      "Merging remainder radtags\n",
      "  389258 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 314775 remainder reads; unable to match 74483 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 35.4825; Std Dev: 20.6618; Max: 336\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3840226\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz... read 3914709 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5107\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3379814 RAD-Tags.\n",
      "  Inserted 454560 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "127610 initial stacks were populated; 326950 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 23.9234; Std Dev: 81.022; Max: 13757\n",
      "Deleveraging trigger: 105; Removal trigger: 186\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4728 stacks.\n",
      "  123009 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 22.4512; Std Dev: 18.7677; Max: 185\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  123009 stacks merged into 95676 loci; deleveraged 954 loci; blacklisted 1017 loci.\n",
      "After merging, coverage depth Mean: 28.2934; Std Dev: 19.0737; Max: 246\n",
      "Merging remainder radtags\n",
      "  326950 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 257562 remainder reads; unable to match 69388 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 31.018; Std Dev: 20.523; Max: 264\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3310426\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz... read 3379814 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5108\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1275204 RAD-Tags.\n",
      "  Inserted 261937 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "103697 initial stacks were populated; 158240 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.7714; Std Dev: 35.9146; Max: 6726\n",
      "Deleveraging trigger: 47; Removal trigger: 83\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3072 stacks.\n",
      "  100741 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.0429; Std Dev: 6.93387; Max: 83\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  100741 stacks merged into 84422 loci; deleveraged 566 loci; blacklisted 397 loci.\n",
      "After merging, coverage depth Mean: 11.7854; Std Dev: 7.23683; Max: 121\n",
      "Merging remainder radtags\n",
      "  158240 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 127606 remainder reads; unable to match 30634 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.3062; Std Dev: 7.9358; Max: 137\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1244570\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz... read 1275204 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5109\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 260255 RAD-Tags.\n",
      "  Inserted 117227 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "52078 initial stacks were populated; 65149 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.74642; Std Dev: 9.22062; Max: 1261\n",
      "Deleveraging trigger: 13; Removal trigger: 22\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1376 stacks.\n",
      "  50773 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.51041; Std Dev: 1.84097; Max: 22\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  50773 stacks merged into 48074 loci; deleveraged 132 loci; blacklisted 86 loci.\n",
      "After merging, coverage depth Mean: 3.66793; Std Dev: 2.02147; Max: 46\n",
      "Merging remainder radtags\n",
      "  65149 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 29708 remainder reads; unable to match 35441 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.28791; Std Dev: 2.4469; Max: 73\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 224814\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz... read 260255 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5110\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1835238 RAD-Tags.\n",
      "  Inserted 322067 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110959 initial stacks were populated; 211108 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6372; Std Dev: 53.3996; Max: 8901\n",
      "Deleveraging trigger: 68; Removal trigger: 121\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3188 stacks.\n",
      "  107867 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.7938; Std Dev: 9.54429; Max: 121\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107867 stacks merged into 85343 loci; deleveraged 604 loci; blacklisted 625 loci.\n",
      "After merging, coverage depth Mean: 17.0706; Std Dev: 9.18173; Max: 140\n",
      "Merging remainder radtags\n",
      "  211108 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 182381 remainder reads; unable to match 28727 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.2258; Std Dev: 10.0029; Max: 154\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1806511\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz... read 1835238 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5111\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 694417 RAD-Tags.\n",
      "  Inserted 182702 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "87040 initial stacks were populated; 95662 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.87908; Std Dev: 20.4777; Max: 3562\n",
      "Deleveraging trigger: 27; Removal trigger: 48\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2432 stacks.\n",
      "  84705 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.44133; Std Dev: 3.81094; Max: 48\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  84705 stacks merged into 77648 loci; deleveraged 282 loci; blacklisted 188 loci.\n",
      "After merging, coverage depth Mean: 6.9439; Std Dev: 4.07257; Max: 68\n",
      "Merging remainder radtags\n",
      "  95662 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 74779 remainder reads; unable to match 20883 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.9105; Std Dev: 4.53217; Max: 123\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 673534\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz... read 694417 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5112\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 275462 RAD-Tags.\n",
      "  Inserted 120774 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "51654 initial stacks were populated; 69120 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.9947; Std Dev: 8.30906; Max: 1176\n",
      "Deleveraging trigger: 12; Removal trigger: 21\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1466 stacks.\n",
      "  50297 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.73213; Std Dev: 2.11554; Max: 21\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  50297 stacks merged into 47087 loci; deleveraged 178 loci; blacklisted 105 loci.\n",
      "After merging, coverage depth Mean: 3.94308; Std Dev: 2.33254; Max: 35\n",
      "Merging remainder radtags\n",
      "  69120 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 32086 remainder reads; unable to match 37034 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.62761; Std Dev: 2.76902; Max: 43\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 238428\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz... read 275462 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5113\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 668697 RAD-Tags.\n",
      "  Inserted 165274 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "84264 initial stacks were populated; 81010 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.97435; Std Dev: 21.0159; Max: 3718\n",
      "Deleveraging trigger: 28; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2145 stacks.\n",
      "  82206 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.55506; Std Dev: 3.80895; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  82206 stacks merged into 76391 loci; deleveraged 233 loci; blacklisted 165 loci.\n",
      "After merging, coverage depth Mean: 6.97779; Std Dev: 4.0976; Max: 85\n",
      "Merging remainder radtags\n",
      "  81010 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 64103 remainder reads; unable to match 16907 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.81971; Std Dev: 4.48498; Max: 96\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 651790\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz... read 668697 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5114\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1302591 RAD-Tags.\n",
      "  Inserted 291653 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "110588 initial stacks were populated; 181065 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.1415; Std Dev: 38.4471; Max: 6147\n",
      "Deleveraging trigger: 49; Removal trigger: 87\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2989 stacks.\n",
      "  107692 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.48069; Std Dev: 6.5784; Max: 87\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107692 stacks merged into 82605 loci; deleveraged 615 loci; blacklisted 725 loci.\n",
      "After merging, coverage depth Mean: 12.0648; Std Dev: 6.58828; Max: 101\n",
      "Merging remainder radtags\n",
      "  181065 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 158359 remainder reads; unable to match 22706 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.0011; Std Dev: 7.26075; Max: 133\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1279885\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz... read 1302591 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5115\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1212671 RAD-Tags.\n",
      "  Inserted 241754 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "98459 initial stacks were populated; 143295 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.8611; Std Dev: 33.2779; Max: 5850\n",
      "Deleveraging trigger: 44; Removal trigger: 77\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2819 stacks.\n",
      "  95744 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.228; Std Dev: 6.30104; Max: 76\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  95744 stacks merged into 83726 loci; deleveraged 427 loci; blacklisted 260 loci.\n",
      "After merging, coverage depth Mean: 11.5357; Std Dev: 6.4892; Max: 125\n",
      "Merging remainder radtags\n",
      "  143295 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 118167 remainder reads; unable to match 25128 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.9533; Std Dev: 7.07774; Max: 141\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1187543\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz... read 1212671 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5116\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1239350 RAD-Tags.\n",
      "  Inserted 238590 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "96957 initial stacks were populated; 141633 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.3217; Std Dev: 39.3885; Max: 5946\n",
      "Deleveraging trigger: 51; Removal trigger: 90\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2710 stacks.\n",
      "  94345 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.6453; Std Dev: 6.49649; Max: 90\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  94345 stacks merged into 84198 loci; deleveraged 384 loci; blacklisted 248 loci.\n",
      "After merging, coverage depth Mean: 11.7669; Std Dev: 6.69296; Max: 117\n",
      "Merging remainder radtags\n",
      "  141633 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 114492 remainder reads; unable to match 27141 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.1323; Std Dev: 7.31289; Max: 236\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1212209\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz... read 1239350 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5117\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 902867 RAD-Tags.\n",
      "  Inserted 199315 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "91448 initial stacks were populated; 107867 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.69347; Std Dev: 31.0241; Max: 4784\n",
      "Deleveraging trigger: 40; Removal trigger: 71\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2376 stacks.\n",
      "  89160 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.14547; Std Dev: 4.9087; Max: 71\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  89160 stacks merged into 81216 loci; deleveraged 314 loci; blacklisted 234 loci.\n",
      "After merging, coverage depth Mean: 8.812; Std Dev: 5.1732; Max: 111\n",
      "Merging remainder radtags\n",
      "  107867 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 85671 remainder reads; unable to match 22196 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.87105; Std Dev: 5.66799; Max: 131\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 880671\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz... read 902867 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5118\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1047551 RAD-Tags.\n",
      "  Inserted 220253 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "93765 initial stacks were populated; 126488 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.8231; Std Dev: 32.8457; Max: 4921\n",
      "Deleveraging trigger: 43; Removal trigger: 76\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2557 stacks.\n",
      "  91300 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.23249; Std Dev: 5.62677; Max: 76\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91300 stacks merged into 81564 loci; deleveraged 317 loci; blacklisted 257 loci.\n",
      "After merging, coverage depth Mean: 10.174; Std Dev: 5.83718; Max: 127\n",
      "Merging remainder radtags\n",
      "  126488 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 104506 remainder reads; unable to match 21982 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.4607; Std Dev: 6.39472; Max: 160\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1025569\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz... read 1047551 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5119\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1031154 RAD-Tags.\n",
      "  Inserted 214529 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "93616 initial stacks were populated; 120913 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.72313; Std Dev: 31.1092; Max: 5571\n",
      "Deleveraging trigger: 41; Removal trigger: 72\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2591 stacks.\n",
      "  91128 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.10687; Std Dev: 5.44326; Max: 72\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91128 stacks merged into 82631 loci; deleveraged 332 loci; blacklisted 193 loci.\n",
      "After merging, coverage depth Mean: 9.92619; Std Dev: 5.73846; Max: 111\n",
      "Merging remainder radtags\n",
      "  120913 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 96885 remainder reads; unable to match 24028 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.1029; Std Dev: 6.27078; Max: 122\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1007126\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz... read 1031154 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5120\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2016001 RAD-Tags.\n",
      "  Inserted 334983 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111053 initial stacks were populated; 223930 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.1371; Std Dev: 49.6815; Max: 8427\n",
      "Deleveraging trigger: 66; Removal trigger: 116\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3530 stacks.\n",
      "  107647 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.2311; Std Dev: 10.4329; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  107647 stacks merged into 88110 loci; deleveraged 603 loci; blacklisted 485 loci.\n",
      "After merging, coverage depth Mean: 18.3183; Std Dev: 10.2396; Max: 167\n",
      "Merging remainder radtags\n",
      "  223930 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 186839 remainder reads; unable to match 37091 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 20.4536; Std Dev: 11.1421; Max: 188\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1978910\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz... read 2016001 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5121\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1568923 RAD-Tags.\n",
      "  Inserted 271328 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101652 initial stacks were populated; 169676 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.7651; Std Dev: 47.8531; Max: 7420\n",
      "Deleveraging trigger: 62; Removal trigger: 109\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2988 stacks.\n",
      "  98760 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.9389; Std Dev: 8.29948; Max: 107\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98760 stacks merged into 83665 loci; deleveraged 502 loci; blacklisted 378 loci.\n",
      "After merging, coverage depth Mean: 15.0177; Std Dev: 8.26716; Max: 145\n",
      "Merging remainder radtags\n",
      "  169676 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 146269 remainder reads; unable to match 23407 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.776; Std Dev: 8.97074; Max: 199\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1545516\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz... read 1568923 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5122\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1559365 RAD-Tags.\n",
      "  Inserted 266430 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100108 initial stacks were populated; 166322 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.9154; Std Dev: 50.505; Max: 8502\n",
      "Deleveraging trigger: 64; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3022 stacks.\n",
      "  97175 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.0503; Std Dev: 8.32486; Max: 115\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97175 stacks merged into 85251 loci; deleveraged 422 loci; blacklisted 328 loci.\n",
      "After merging, coverage depth Mean: 14.6406; Std Dev: 8.41182; Max: 155\n",
      "Merging remainder radtags\n",
      "  166322 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 134598 remainder reads; unable to match 31724 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.2272; Std Dev: 9.10572; Max: 162\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1527641\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz... read 1559365 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5123\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 753640 RAD-Tags.\n",
      "  Inserted 170680 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "86064 initial stacks were populated; 84616 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.77356; Std Dev: 22.0022; Max: 3458\n",
      "Deleveraging trigger: 30; Removal trigger: 52\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2268 stacks.\n",
      "  83890 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.31924; Std Dev: 4.22418; Max: 52\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  83890 stacks merged into 77981 loci; deleveraged 268 loci; blacklisted 163 loci.\n",
      "After merging, coverage depth Mean: 7.7919; Std Dev: 4.4965; Max: 83\n",
      "Merging remainder radtags\n",
      "  84616 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 67830 remainder reads; unable to match 16786 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.66461; Std Dev: 4.92796; Max: 124\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 736854\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz... read 753640 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5124\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1193901 RAD-Tags.\n",
      "  Inserted 227708 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94427 initial stacks were populated; 133281 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.2322; Std Dev: 33.6524; Max: 5904\n",
      "Deleveraging trigger: 45; Removal trigger: 79\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2711 stacks.\n",
      "  91811 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.5808; Std Dev: 6.28818; Max: 79\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  91811 stacks merged into 81640 loci; deleveraged 349 loci; blacklisted 256 loci.\n",
      "After merging, coverage depth Mean: 11.7493; Std Dev: 6.52433; Max: 112\n",
      "Merging remainder radtags\n",
      "  133281 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 114994 remainder reads; unable to match 18287 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.164; Std Dev: 7.12647; Max: 127\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1175614\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz... read 1193901 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5125\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2852852 RAD-Tags.\n",
      "  Inserted 464194 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121948 initial stacks were populated; 342246 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 20.5875; Std Dev: 66.3196; Max: 12308\n",
      "Deleveraging trigger: 87; Removal trigger: 153\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4644 stacks.\n",
      "  117453 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.4649; Std Dev: 14.8362; Max: 152\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117453 stacks merged into 91115 loci; deleveraged 764 loci; blacklisted 891 loci.\n",
      "After merging, coverage depth Mean: 24.6137; Std Dev: 14.3093; Max: 241\n",
      "Merging remainder radtags\n",
      "  342246 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 276753 remainder reads; unable to match 65493 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.6862; Std Dev: 15.6362; Max: 265\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2787359\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz... read 2852852 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5126\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 599052 RAD-Tags.\n",
      "  Inserted 162372 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "82840 initial stacks were populated; 79532 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.27137; Std Dev: 20.3208; Max: 2748\n",
      "Deleveraging trigger: 27; Removal trigger: 47\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2083 stacks.\n",
      "  80838 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.88135; Std Dev: 3.49327; Max: 47\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80838 stacks merged into 75255 loci; deleveraged 235 loci; blacklisted 165 loci.\n",
      "After merging, coverage depth Mean: 6.23527; Std Dev: 3.74399; Max: 96\n",
      "Merging remainder radtags\n",
      "  79532 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 59636 remainder reads; unable to match 19896 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.03032; Std Dev: 4.1904; Max: 161\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 579156\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz... read 599052 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5127\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2153734 RAD-Tags.\n",
      "  Inserted 324477 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "107301 initial stacks were populated; 217176 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.0479; Std Dev: 68.1696; Max: 10604\n",
      "Deleveraging trigger: 86; Removal trigger: 154\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3377 stacks.\n",
      "  104012 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.9957; Std Dev: 11.3124; Max: 154\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  104012 stacks merged into 86070 loci; deleveraged 569 loci; blacklisted 513 loci.\n",
      "After merging, coverage depth Mean: 20.1758; Std Dev: 10.8602; Max: 167\n",
      "Merging remainder radtags\n",
      "  217176 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 184999 remainder reads; unable to match 32177 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.3403; Std Dev: 11.724; Max: 180\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2121557\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz... read 2153734 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5128\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4331420 RAD-Tags.\n",
      "  Inserted 559654 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143612 initial stacks were populated; 416042 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 27.2636; Std Dev: 118.108; Max: 21468\n",
      "Deleveraging trigger: 145; Removal trigger: 263\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5109 stacks.\n",
      "  138613 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 25.657; Std Dev: 22.8069; Max: 262\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  138613 stacks merged into 95128 loci; deleveraged 1044 loci; blacklisted 2024 loci.\n",
      "After merging, coverage depth Mean: 36.1297; Std Dev: 20.9421; Max: 356\n",
      "Merging remainder radtags\n",
      "  416042 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 338839 remainder reads; unable to match 77203 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 39.7734; Std Dev: 22.5297; Max: 378\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4254217\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz... read 4331420 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5129\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2481542 RAD-Tags.\n",
      "  Inserted 382670 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "121438 initial stacks were populated; 261232 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.2835; Std Dev: 68.769; Max: 12100\n",
      "Deleveraging trigger: 87; Removal trigger: 156\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3773 stacks.\n",
      "  117775 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.1421; Std Dev: 13.418; Max: 156\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  117775 stacks merged into 90812 loci; deleveraged 757 loci; blacklisted 888 loci.\n",
      "After merging, coverage depth Mean: 21.7096; Std Dev: 13.0331; Max: 227\n",
      "Merging remainder radtags\n",
      "  261232 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 213495 remainder reads; unable to match 47737 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.0867; Std Dev: 14.1138; Max: 239\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2433805\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz... read 2481542 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5130\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2367282 RAD-Tags.\n",
      "  Inserted 367507 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "119022 initial stacks were populated; 248485 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 17.8017; Std Dev: 69.8902; Max: 11856\n",
      "Deleveraging trigger: 88; Removal trigger: 158\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3592 stacks.\n",
      "  115531 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 16.7122; Std Dev: 12.4615; Max: 158\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  115531 stacks merged into 87087 loci; deleveraged 720 loci; blacklisted 898 loci.\n",
      "After merging, coverage depth Mean: 21.6391; Std Dev: 11.6486; Max: 190\n",
      "Merging remainder radtags\n",
      "  248485 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 210595 remainder reads; unable to match 37890 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.0854; Std Dev: 12.5774; Max: 201\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2329392\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz... read 2367282 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5131\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1333709 RAD-Tags.\n",
      "  Inserted 236639 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95971 initial stacks were populated; 140668 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.4313; Std Dev: 33.6965; Max: 5138\n",
      "Deleveraging trigger: 46; Removal trigger: 80\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2910 stacks.\n",
      "  93179 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7447; Std Dev: 6.97781; Max: 80\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  93179 stacks merged into 82666 loci; deleveraged 355 loci; blacklisted 221 loci.\n",
      "After merging, coverage depth Mean: 13.0893; Std Dev: 7.22291; Max: 136\n",
      "Merging remainder radtags\n",
      "  140668 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 120414 remainder reads; unable to match 20254 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.5519; Std Dev: 7.82939; Max: 147\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1313455\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz... read 1333709 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5132\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1491573 RAD-Tags.\n",
      "  Inserted 264713 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "99946 initial stacks were populated; 164767 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.2752; Std Dev: 48.862; Max: 7978\n",
      "Deleveraging trigger: 62; Removal trigger: 111\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2830 stacks.\n",
      "  97196 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.4822; Std Dev: 7.7658; Max: 111\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97196 stacks merged into 84827 loci; deleveraged 423 loci; blacklisted 303 loci.\n",
      "After merging, coverage depth Mean: 14.0899; Std Dev: 7.89631; Max: 146\n",
      "Merging remainder radtags\n",
      "  164767 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 133561 remainder reads; unable to match 31206 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 15.6716; Std Dev: 8.56026; Max: 169\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1460367\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz... read 1491573 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5133\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1899144 RAD-Tags.\n",
      "  Inserted 334275 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114520 initial stacks were populated; 219755 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 14.6646; Std Dev: 49.6495; Max: 9270\n",
      "Deleveraging trigger: 64; Removal trigger: 114\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3730 stacks.\n",
      "  110918 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 13.7629; Std Dev: 9.7135; Max: 114\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  110918 stacks merged into 89889 loci; deleveraged 666 loci; blacklisted 589 loci.\n",
      "After merging, coverage depth Mean: 16.6644; Std Dev: 9.61252; Max: 123\n",
      "Merging remainder radtags\n",
      "  219755 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 176695 remainder reads; unable to match 43060 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.646; Std Dev: 10.4774; Max: 135\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1856084\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz... read 1899144 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5134\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4112057 RAD-Tags.\n",
      "  Inserted 522182 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "132820 initial stacks were populated; 389362 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 28.0281; Std Dev: 117.959; Max: 20916\n",
      "Deleveraging trigger: 146; Removal trigger: 264\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4857 stacks.\n",
      "  128074 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 26.4128; Std Dev: 21.7374; Max: 260\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128074 stacks merged into 94934 loci; deleveraged 953 loci; blacklisted 1392 loci.\n",
      "After merging, coverage depth Mean: 34.6991; Std Dev: 20.1052; Max: 317\n",
      "Merging remainder radtags\n",
      "  389362 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 314807 remainder reads; unable to match 74555 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 38.0685; Std Dev: 21.6139; Max: 337\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4037502\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz... read 4112057 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5135\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4679003 RAD-Tags.\n",
      "  Inserted 587341 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "140135 initial stacks were populated; 447206 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.198; Std Dev: 119.487; Max: 26462\n",
      "Deleveraging trigger: 150; Removal trigger: 269\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5379 stacks.\n",
      "  134873 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.5301; Std Dev: 24.6159; Max: 267\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  134873 stacks merged into 98161 loci; deleveraged 1087 loci; blacklisted 1650 loci.\n",
      "After merging, coverage depth Mean: 38.1711; Std Dev: 23.1359; Max: 310\n",
      "Merging remainder radtags\n",
      "  447206 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 355164 remainder reads; unable to match 92042 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 41.8556; Std Dev: 24.8878; Max: 326\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4586961\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz... read 4679003 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5136\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 4223 RAD-Tags.\n",
      "  Inserted 3562 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "418 initial stacks were populated; 3144 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.58134; Std Dev: 3.15536; Max: 47\n",
      "Deleveraging trigger: 6; Removal trigger: 9\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 13 stacks.\n",
      "  410 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.25679; Std Dev: 0.742525; Max: 9\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  410 stacks merged into 401 loci; deleveraged 1 loci; blacklisted 0 loci.\n",
      "After merging, coverage depth Mean: 2.30808; Std Dev: 0.937995; Max: 11\n",
      "Merging remainder radtags\n",
      "  3144 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 182 remainder reads; unable to match 2962 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 2.76768; Std Dev: 1.94795; Max: 21\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1261\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz... read 4223 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5137\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3895374 RAD-Tags.\n",
      "  Inserted 504099 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "135941 initial stacks were populated; 368158 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.9467; Std Dev: 105.884; Max: 18152\n",
      "Deleveraging trigger: 132; Removal trigger: 238\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4806 stacks.\n",
      "  131259 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 24.3801; Std Dev: 20.8246; Max: 235\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  131259 stacks merged into 91974 loci; deleveraged 929 loci; blacklisted 1615 loci.\n",
      "After merging, coverage depth Mean: 33.7542; Std Dev: 19.0884; Max: 266\n",
      "Merging remainder radtags\n",
      "  368158 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 304061 remainder reads; unable to match 64097 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 37.1239; Std Dev: 20.4624; Max: 282\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3831277\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz... read 3895374 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5138\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1553798 RAD-Tags.\n",
      "  Inserted 267297 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101411 initial stacks were populated; 165886 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 13.686; Std Dev: 43.9009; Max: 6049\n",
      "Deleveraging trigger: 58; Removal trigger: 101\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2995 stacks.\n",
      "  98517 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 12.8775; Std Dev: 8.57162; Max: 101\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98517 stacks merged into 85892 loci; deleveraged 487 loci; blacklisted 311 loci.\n",
      "After merging, coverage depth Mean: 14.5588; Std Dev: 8.76555; Max: 148\n",
      "Merging remainder radtags\n",
      "  165886 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 137454 remainder reads; unable to match 28432 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 16.1668; Std Dev: 9.5114; Max: 155\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1525366\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz... read 1553798 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5139\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1896666 RAD-Tags.\n",
      "  Inserted 312963 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "109379 initial stacks were populated; 203584 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.479; Std Dev: 46.8864; Max: 7783\n",
      "Deleveraging trigger: 62; Removal trigger: 109\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3488 stacks.\n",
      "  106028 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.557; Std Dev: 9.98119; Max: 108\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  106028 stacks merged into 86286 loci; deleveraged 578 loci; blacklisted 512 loci.\n",
      "After merging, coverage depth Mean: 17.6022; Std Dev: 9.8168; Max: 135\n",
      "Merging remainder radtags\n",
      "  203584 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 172705 remainder reads; unable to match 30879 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 19.6189; Std Dev: 10.6576; Max: 143\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1865787\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz... read 1896666 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5140\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2764667 RAD-Tags.\n",
      "  Inserted 403461 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "117398 initial stacks were populated; 286063 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 21.1128; Std Dev: 88.4809; Max: 15607\n",
      "Deleveraging trigger: 110; Removal trigger: 198\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3964 stacks.\n",
      "  113521 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.8342; Std Dev: 14.5182; Max: 197\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  113521 stacks merged into 89114 loci; deleveraged 707 loci; blacklisted 805 loci.\n",
      "After merging, coverage depth Mean: 24.7046; Std Dev: 13.4742; Max: 207\n",
      "Merging remainder radtags\n",
      "  286063 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 238983 remainder reads; unable to match 47080 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.4135; Std Dev: 14.5731; Max: 223\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2717587\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz... read 2764667 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5141\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5096752 RAD-Tags.\n",
      "  Inserted 658811 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "153255 initial stacks were populated; 505556 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 29.9579; Std Dev: 114.796; Max: 23027\n",
      "Deleveraging trigger: 145; Removal trigger: 260\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5810 stacks.\n",
      "  147583 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.2726; Std Dev: 26.8445; Max: 257\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  147583 stacks merged into 108157 loci; deleveraged 1255 loci; blacklisted 1972 loci.\n",
      "After merging, coverage depth Mean: 37.4241; Std Dev: 26.57; Max: 463\n",
      "Merging remainder radtags\n",
      "  505556 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 373355 remainder reads; unable to match 132201 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 40.9447; Std Dev: 28.6497; Max: 492\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4964551\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz... read 5096752 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5142\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6508843 RAD-Tags.\n",
      "  Inserted 709441 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "147534 initial stacks were populated; 561907 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 40.3089; Std Dev: 139.292; Max: 25625\n",
      "Deleveraging trigger: 180; Removal trigger: 319\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6401 stacks.\n",
      "  141283 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 38.3861; Std Dev: 35.2331; Max: 319\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  141283 stacks merged into 96043 loci; deleveraged 1144 loci; blacklisted 2489 loci.\n",
      "After merging, coverage depth Mean: 54.82; Std Dev: 32.1646; Max: 403\n",
      "Merging remainder radtags\n",
      "  561907 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 458313 remainder reads; unable to match 103594 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 59.7268; Std Dev: 34.3848; Max: 540\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6405249\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz... read 6508843 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5143\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2167327 RAD-Tags.\n",
      "  Inserted 352872 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "115801 initial stacks were populated; 237071 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 16.6687; Std Dev: 49.2712; Max: 8111\n",
      "Deleveraging trigger: 66; Removal trigger: 115\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3640 stacks.\n",
      "  112285 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 15.7721; Std Dev: 11.3; Max: 115\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112285 stacks merged into 87323 loci; deleveraged 689 loci; blacklisted 681 loci.\n",
      "After merging, coverage depth Mean: 19.8987; Std Dev: 10.806; Max: 176\n",
      "Merging remainder radtags\n",
      "  237071 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 200205 remainder reads; unable to match 36866 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 22.2127; Std Dev: 11.7051; Max: 193\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2130461\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz... read 2167327 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5144\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4120581 RAD-Tags.\n",
      "  Inserted 530652 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "132617 initial stacks were populated; 398035 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 28.0699; Std Dev: 122.732; Max: 22011\n",
      "Deleveraging trigger: 151; Removal trigger: 274\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4709 stacks.\n",
      "  128006 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 26.4284; Std Dev: 21.7059; Max: 272\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128006 stacks merged into 98171 loci; deleveraged 884 loci; blacklisted 1186 loci.\n",
      "After merging, coverage depth Mean: 33.6231; Std Dev: 20.5526; Max: 328\n",
      "Merging remainder radtags\n",
      "  398035 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 311681 remainder reads; unable to match 86354 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 36.8401; Std Dev: 22.1259; Max: 359\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4034227\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz... read 4120581 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5145\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5980737 RAD-Tags.\n",
      "  Inserted 684038 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "153132 initial stacks were populated; 530906 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 35.5891; Std Dev: 134.919; Max: 28034\n",
      "Deleveraging trigger: 171; Removal trigger: 305\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6235 stacks.\n",
      "  147045 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 33.6914; Std Dev: 31.9874; Max: 301\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  147045 stacks merged into 99247 loci; deleveraged 1210 loci; blacklisted 2671 loci.\n",
      "After merging, coverage depth Mean: 48.2261; Std Dev: 29.8569; Max: 356\n",
      "Merging remainder radtags\n",
      "  530906 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 420913 remainder reads; unable to match 109993 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 52.5911; Std Dev: 31.9201; Max: 369\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5870744\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz... read 5980737 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5146\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3835136 RAD-Tags.\n",
      "  Inserted 481311 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "123696 initial stacks were populated; 357615 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 28.1134; Std Dev: 109.236; Max: 18584\n",
      "Deleveraging trigger: 137; Removal trigger: 247\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4562 stacks.\n",
      "  119236 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 26.5693; Std Dev: 20.263; Max: 246\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  119236 stacks merged into 91867 loci; deleveraged 793 loci; blacklisted 1000 loci.\n",
      "After merging, coverage depth Mean: 33.7359; Std Dev: 18.6918; Max: 328\n",
      "Merging remainder radtags\n",
      "  357615 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 295266 remainder reads; unable to match 62349 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 36.989; Std Dev: 20.065; Max: 376\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3772787\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz... read 3835136 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5147\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1895004 RAD-Tags.\n",
      "  Inserted 325940 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "111724 initial stacks were populated; 214216 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.0441; Std Dev: 50.751; Max: 8030\n",
      "Deleveraging trigger: 66; Removal trigger: 117\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3409 stacks.\n",
      "  108424 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.1436; Std Dev: 10.2125; Max: 116\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  108424 stacks merged into 88510 loci; deleveraged 667 loci; blacklisted 596 loci.\n",
      "After merging, coverage depth Mean: 16.9719; Std Dev: 10.1366; Max: 127\n",
      "Merging remainder radtags\n",
      "  214216 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 175605 remainder reads; unable to match 38611 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.9719; Std Dev: 11.0266; Max: 135\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1856393\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz... read 1895004 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5148\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...done\n",
      "Loaded 5683067 RAD-Tags.\n",
      "  Inserted 657970 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "146837 initial stacks were populated; 511133 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 35.2223; Std Dev: 135.524; Max: 29735\n",
      "Deleveraging trigger: 171; Removal trigger: 306\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6114 stacks.\n",
      "  140853 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 33.3666; Std Dev: 30.4553; Max: 306\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  140853 stacks merged into 99044 loci; deleveraged 1157 loci; blacklisted 2101 loci.\n",
      "After merging, coverage depth Mean: 46.0786; Std Dev: 28.6185; Max: 378\n",
      "Merging remainder radtags\n",
      "  511133 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 407482 remainder reads; unable to match 103651 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 50.2875; Std Dev: 30.6756; Max: 400\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 5579416\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz... read 5683067 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5149\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2639636 RAD-Tags.\n",
      "  Inserted 353907 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "107472 initial stacks were populated; 246435 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 22.2681; Std Dev: 80.8767; Max: 13567\n",
      "Deleveraging trigger: 103; Removal trigger: 184\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3558 stacks.\n",
      "  104005 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 21.0395; Std Dev: 13.65; Max: 184\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  104005 stacks merged into 87174 loci; deleveraged 582 loci; blacklisted 485 loci.\n",
      "After merging, coverage depth Mean: 24.6932; Std Dev: 12.9901; Max: 221\n",
      "Merging remainder radtags\n",
      "  246435 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 206931 remainder reads; unable to match 39504 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 27.0828; Std Dev: 13.9367; Max: 240\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2600132\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz... read 2639636 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5150\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2467187 RAD-Tags.\n",
      "  Inserted 363074 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "113842 initial stacks were populated; 249232 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 19.4827; Std Dev: 73.9901; Max: 12625\n",
      "Deleveraging trigger: 93; Removal trigger: 167\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3438 stacks.\n",
      "  110495 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 18.3734; Std Dev: 13.1227; Max: 167\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  110495 stacks merged into 88026 loci; deleveraged 666 loci; blacklisted 700 loci.\n",
      "After merging, coverage depth Mean: 22.5769; Std Dev: 12.3821; Max: 192\n",
      "Merging remainder radtags\n",
      "  249232 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 211374 remainder reads; unable to match 37858 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.9999; Std Dev: 13.3773; Max: 244\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2429329\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz... read 2467187 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5151\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7233016 RAD-Tags.\n",
      "  Inserted 774730 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "160090 initial stacks were populated; 614640 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 41.3416; Std Dev: 189.588; Max: 36908\n",
      "Deleveraging trigger: 231; Removal trigger: 421\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6450 stacks.\n",
      "  153743 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 39.148; Std Dev: 39.1268; Max: 418\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  153743 stacks merged into 97386 loci; deleveraged 1281 loci; blacklisted 3502 loci.\n",
      "After merging, coverage depth Mean: 59.2057; Std Dev: 35.1872; Max: 401\n",
      "Merging remainder radtags\n",
      "  614640 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 496718 remainder reads; unable to match 117922 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 64.5023; Std Dev: 37.5821; Max: 427\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 7115094\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz... read 7233016 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5152\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4714496 RAD-Tags.\n",
      "  Inserted 596188 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143427 initial stacks were populated; 452761 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 29.7136; Std Dev: 131.396; Max: 24672\n",
      "Deleveraging trigger: 161; Removal trigger: 293\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5208 stacks.\n",
      "  138334 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.0044; Std Dev: 25.0002; Max: 293\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  138334 stacks merged into 98384 loci; deleveraged 1012 loci; blacklisted 1914 loci.\n",
      "After merging, coverage depth Mean: 38.1057; Std Dev: 23.3617; Max: 335\n",
      "Merging remainder radtags\n",
      "  452761 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 360593 remainder reads; unable to match 92168 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 41.848; Std Dev: 25.1544; Max: 379\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4622328\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz... read 4714496 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5153\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4140989 RAD-Tags.\n",
      "  Inserted 500613 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "125033 initial stacks were populated; 375580 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 30.1153; Std Dev: 99.9064; Max: 18720\n",
      "Deleveraging trigger: 130; Removal trigger: 230\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4945 stacks.\n",
      "  120220 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 28.5077; Std Dev: 22.5491; Max: 230\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  120220 stacks merged into 93103 loci; deleveraged 847 loci; blacklisted 1004 loci.\n",
      "After merging, coverage depth Mean: 36.0921; Std Dev: 21.4006; Max: 259\n",
      "Merging remainder radtags\n",
      "  375580 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 307804 remainder reads; unable to match 67776 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 39.439; Std Dev: 22.9163; Max: 288\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4073213\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz... read 4140989 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5154\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2627262 RAD-Tags.\n",
      "  Inserted 388524 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "116085 initial stacks were populated; 272439 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 20.2853; Std Dev: 72.6414; Max: 11107\n",
      "Deleveraging trigger: 93; Removal trigger: 166\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3997 stacks.\n",
      "  112206 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 19.0887; Std Dev: 13.9354; Max: 165\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112206 stacks merged into 90159 loci; deleveraged 701 loci; blacklisted 676 loci.\n",
      "After merging, coverage depth Mean: 23.3129; Std Dev: 13.4901; Max: 187\n",
      "Merging remainder radtags\n",
      "  272439 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 223365 remainder reads; unable to match 49074 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 25.8124; Std Dev: 14.5805; Max: 206\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2578188\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz... read 2627262 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5155\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...done\n",
      "Loaded 6540120 RAD-Tags.\n",
      "  Inserted 721798 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "154107 initial stacks were populated; 567691 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 38.7551; Std Dev: 182.968; Max: 34767\n",
      "Deleveraging trigger: 222; Removal trigger: 405\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6250 stacks.\n",
      "  147975 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 36.5664; Std Dev: 35.1331; Max: 404\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  147975 stacks merged into 98808 loci; deleveraged 1178 loci; blacklisted 2891 loci.\n",
      "After merging, coverage depth Mean: 52.7749; Std Dev: 32.3145; Max: 410\n",
      "Merging remainder radtags\n",
      "  567691 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 453024 remainder reads; unable to match 114667 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 57.5038; Std Dev: 34.5662; Max: 489\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6425453\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz... read 6540120 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5156\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1307424 RAD-Tags.\n",
      "  Inserted 295472 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "112919 initial stacks were populated; 182553 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.96175; Std Dev: 29.435; Max: 4891\n",
      "Deleveraging trigger: 39; Removal trigger: 69\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3244 stacks.\n",
      "  109800 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.35522; Std Dev: 6.59143; Max: 69\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109800 stacks merged into 84914 loci; deleveraged 680 loci; blacklisted 639 loci.\n",
      "After merging, coverage depth Mean: 11.8489; Std Dev: 6.7738; Max: 98\n",
      "Merging remainder radtags\n",
      "  182553 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 154113 remainder reads; unable to match 28440 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.6803; Std Dev: 7.46403; Max: 132\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1278984\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz... read 1307424 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5157\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 956204 RAD-Tags.\n",
      "  Inserted 245955 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101342 initial stacks were populated; 144613 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.00844; Std Dev: 31.355; Max: 5324\n",
      "Deleveraging trigger: 39; Removal trigger: 71\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2688 stacks.\n",
      "  98742 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.44267; Std Dev: 4.91297; Max: 70\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98742 stacks merged into 82154 loci; deleveraged 509 loci; blacklisted 408 loci.\n",
      "After merging, coverage depth Mean: 8.76164; Std Dev: 5.11853; Max: 81\n",
      "Merging remainder radtags\n",
      "  144613 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 119572 remainder reads; unable to match 25041 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 10.2259; Std Dev: 5.74541; Max: 117\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 931163\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz... read 956204 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5158\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 802147 RAD-Tags.\n",
      "  Inserted 219034 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "94331 initial stacks were populated; 124703 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.18156; Std Dev: 26.7488; Max: 4199\n",
      "Deleveraging trigger: 34; Removal trigger: 61\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2362 stacks.\n",
      "  92046 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.69084; Std Dev: 4.26211; Max: 61\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92046 stacks merged into 78987 loci; deleveraged 384 loci; blacklisted 355 loci.\n",
      "After merging, coverage depth Mean: 7.64646; Std Dev: 4.50183; Max: 75\n",
      "Merging remainder radtags\n",
      "  124703 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 102353 remainder reads; unable to match 22350 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.94941; Std Dev: 5.04374; Max: 132\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 779797\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz... read 802147 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5159\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1634885 RAD-Tags.\n",
      "  Inserted 326835 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "119647 initial stacks were populated; 207188 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.9326; Std Dev: 42.4978; Max: 7329\n",
      "Deleveraging trigger: 54; Removal trigger: 97\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3293 stacks.\n",
      "  116470 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.1749; Std Dev: 8.38054; Max: 95\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  116470 stacks merged into 83229 loci; deleveraged 703 loci; blacklisted 979 loci.\n",
      "After merging, coverage depth Mean: 15.1978; Std Dev: 8.14055; Max: 118\n",
      "Merging remainder radtags\n",
      "  207188 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 183060 remainder reads; unable to match 24128 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 17.4266; Std Dev: 8.85099; Max: 125\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1610757\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz... read 1634885 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5160\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 859782 RAD-Tags.\n",
      "  Inserted 226459 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95450 initial stacks were populated; 131009 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.63513; Std Dev: 23.3481; Max: 3242\n",
      "Deleveraging trigger: 31; Removal trigger: 54\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2591 stacks.\n",
      "  92967 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.12536; Std Dev: 4.76055; Max: 54\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  92967 stacks merged into 78612 loci; deleveraged 430 loci; blacklisted 332 loci.\n",
      "After merging, coverage depth Mean: 8.28959; Std Dev: 5.10709; Max: 70\n",
      "Merging remainder radtags\n",
      "  131009 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 108618 remainder reads; unable to match 22391 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.67907; Std Dev: 5.73871; Max: 146\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 837391\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz... read 859782 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5161\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 564008 RAD-Tags.\n",
      "  Inserted 177164 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "83003 initial stacks were populated; 94161 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.6606; Std Dev: 19.6591; Max: 3134\n",
      "Deleveraging trigger: 25; Removal trigger: 45\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2110 stacks.\n",
      "  80973 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.2691; Std Dev: 3.23826; Max: 45\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  80973 stacks merged into 72353 loci; deleveraged 282 loci; blacklisted 220 loci.\n",
      "After merging, coverage depth Mean: 5.80134; Std Dev: 3.51168; Max: 84\n",
      "Merging remainder radtags\n",
      "  94161 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 72249 remainder reads; unable to match 21912 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.80406; Std Dev: 4.00431; Max: 93\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 542096\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz... read 564008 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5162\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 660511 RAD-Tags.\n",
      "  Inserted 197572 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88463 initial stacks were populated; 109109 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.23314; Std Dev: 20.9213; Max: 3312\n",
      "Deleveraging trigger: 27; Removal trigger: 48\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2418 stacks.\n",
      "  86144 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.79288; Std Dev: 3.6575; Max: 48\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  86144 stacks merged into 75893 loci; deleveraged 326 loci; blacklisted 249 loci.\n",
      "After merging, coverage depth Mean: 6.47554; Std Dev: 3.97101; Max: 98\n",
      "Merging remainder radtags\n",
      "  109109 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 84309 remainder reads; unable to match 24800 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 7.59155; Std Dev: 4.4772; Max: 123\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 635711\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz... read 660511 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5163\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2361466 RAD-Tags.\n",
      "  Inserted 354753 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "114950 initial stacks were populated; 239803 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 18.4573; Std Dev: 54.4792; Max: 9208\n",
      "Deleveraging trigger: 73; Removal trigger: 127\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3830 stacks.\n",
      "  111264 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 17.4266; Std Dev: 12.6506; Max: 127\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  111264 stacks merged into 85882 loci; deleveraged 702 loci; blacklisted 698 loci.\n",
      "After merging, coverage depth Mean: 22.1752; Std Dev: 12.0612; Max: 183\n",
      "Merging remainder radtags\n",
      "  239803 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 209042 remainder reads; unable to match 30761 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 24.6334; Std Dev: 13.0058; Max: 245\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 2330705\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz... read 2361466 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5164\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 727486 RAD-Tags.\n",
      "  Inserted 207800 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "92037 initial stacks were populated; 115763 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.64649; Std Dev: 20.4026; Max: 3792\n",
      "Deleveraging trigger: 27; Removal trigger: 47\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2490 stacks.\n",
      "  89650 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.21273; Std Dev: 3.86293; Max: 47\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  89650 stacks merged into 77785 loci; deleveraged 369 loci; blacklisted 293 loci.\n",
      "After merging, coverage depth Mean: 7.0459; Std Dev: 4.15602; Max: 87\n",
      "Merging remainder radtags\n",
      "  115763 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 93962 remainder reads; unable to match 21801 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 8.26005; Std Dev: 4.67993; Max: 117\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 705685\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz... read 727486 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5165\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1204794 RAD-Tags.\n",
      "  Inserted 277751 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "108361 initial stacks were populated; 169390 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.55514; Std Dev: 28.5506; Max: 5136\n",
      "Deleveraging trigger: 38; Removal trigger: 67\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3060 stacks.\n",
      "  105417 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.95443; Std Dev: 6.26871; Max: 67\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  105417 stacks merged into 82880 loci; deleveraged 581 loci; blacklisted 611 loci.\n",
      "After merging, coverage depth Mean: 11.1371; Std Dev: 6.46485; Max: 86\n",
      "Merging remainder radtags\n",
      "  169390 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 143962 remainder reads; unable to match 25428 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 12.8895; Std Dev: 7.13025; Max: 98\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1179366\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz... read 1204794 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5166\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Loaded 2025667 RAD-Tags.\n",
      "  Inserted 344768 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "115923 initial stacks were populated; 228845 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.5001; Std Dev: 64.3174; Max: 11570\n",
      "Deleveraging trigger: 80; Removal trigger: 144\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3511 stacks.\n",
      "  112508 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.5131; Std Dev: 10.4138; Max: 143\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  112508 stacks merged into 85474 loci; deleveraged 685 loci; blacklisted 829 loci.\n",
      "After merging, coverage depth Mean: 18.6488; Std Dev: 9.72997; Max: 141\n",
      "Merging remainder radtags\n",
      "  228845 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 199299 remainder reads; unable to match 29546 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 21.006; Std Dev: 10.5692; Max: 156\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1996121\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz... read 2025667 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5167\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1062265 RAD-Tags.\n",
      "  Inserted 261092 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "104448 initial stacks were populated; 156644 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 8.67054; Std Dev: 26.1257; Max: 4593\n",
      "Deleveraging trigger: 35; Removal trigger: 61\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2989 stacks.\n",
      "  101580 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.13468; Std Dev: 5.33628; Max: 61\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  101580 stacks merged into 82009 loci; deleveraged 539 loci; blacklisted 469 loci.\n",
      "After merging, coverage depth Mean: 9.89139; Std Dev: 5.5346; Max: 71\n",
      "Merging remainder radtags\n",
      "  156644 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 134344 remainder reads; unable to match 22300 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.5414; Std Dev: 6.1544; Max: 132\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1039965\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz... read 1062265 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5168\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1334222 RAD-Tags.\n",
      "  Inserted 301046 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "112904 initial stacks were populated; 188142 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 10.1509; Std Dev: 30.6625; Max: 5778\n",
      "Deleveraging trigger: 41; Removal trigger: 71\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3330 stacks.\n",
      "  109709 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 9.53168; Std Dev: 6.73823; Max: 71\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  109709 stacks merged into 83803 loci; deleveraged 710 loci; blacklisted 676 loci.\n",
      "After merging, coverage depth Mean: 12.2164; Std Dev: 6.85359; Max: 81\n",
      "Merging remainder radtags\n",
      "  188142 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 160919 remainder reads; unable to match 27223 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.1553; Std Dev: 7.55027; Max: 114\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1306999\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz... read 1334222 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5169\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 230003 RAD-Tags.\n",
      "  Inserted 108588 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "45962 initial stacks were populated; 62626 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.64164; Std Dev: 7.58952; Max: 867\n",
      "Deleveraging trigger: 11; Removal trigger: 19\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1266 stacks.\n",
      "  44789 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.40127; Std Dev: 1.77912; Max: 19\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  44789 stacks merged into 42188 loci; deleveraged 125 loci; blacklisted 90 loci.\n",
      "After merging, coverage depth Mean: 3.56762; Std Dev: 1.96369; Max: 32\n",
      "Merging remainder radtags\n",
      "  62626 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 25855 remainder reads; unable to match 36771 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.18314; Std Dev: 2.4361; Max: 75\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 193232\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz... read 230003 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5170\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 117340 RAD-Tags.\n",
      "  Inserted 66247 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "22757 initial stacks were populated; 43490 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.24516; Std Dev: 18.8866; Max: 1947\n",
      "Deleveraging trigger: 22; Removal trigger: 41\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 397 stacks.\n",
      "  22383 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.91418; Std Dev: 1.71198; Max: 41\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  22383 stacks merged into 21132 loci; deleveraged 73 loci; blacklisted 76 loci.\n",
      "After merging, coverage depth Mean: 2.98902; Std Dev: 1.7161; Max: 44\n",
      "Merging remainder radtags\n",
      "  43490 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 10155 remainder reads; unable to match 33335 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.47183; Std Dev: 2.23256; Max: 52\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 84005\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz... read 117340 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5171\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 333059 RAD-Tags.\n",
      "  Inserted 124806 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "59185 initial stacks were populated; 65621 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.51868; Std Dev: 10.0473; Max: 1410\n",
      "Deleveraging trigger: 15; Removal trigger: 25\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1626 stacks.\n",
      "  57657 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.23049; Std Dev: 2.52857; Max: 25\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  57657 stacks merged into 54146 loci; deleveraged 172 loci; blacklisted 121 loci.\n",
      "After merging, coverage depth Mean: 4.45384; Std Dev: 2.76685; Max: 42\n",
      "Merging remainder radtags\n",
      "  65621 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 32689 remainder reads; unable to match 32932 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.06001; Std Dev: 3.15049; Max: 48\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 300127\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz... read 333059 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5172\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 302940 RAD-Tags.\n",
      "  Inserted 107650 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "47416 initial stacks were populated; 60234 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.11865; Std Dev: 15.8535; Max: 2043\n",
      "Deleveraging trigger: 21; Removal trigger: 37\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1351 stacks.\n",
      "  46152 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.70628; Std Dev: 3.20968; Max: 37\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  46152 stacks merged into 42800 loci; deleveraged 216 loci; blacklisted 136 loci.\n",
      "After merging, coverage depth Mean: 4.99253; Std Dev: 3.48149; Max: 49\n",
      "Merging remainder radtags\n",
      "  60234 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 30816 remainder reads; unable to match 29418 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.7163; Std Dev: 4.05405; Max: 139\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 273522\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz... read 302940 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5173\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 107578 RAD-Tags.\n",
      "  Inserted 61780 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "21615 initial stacks were populated; 40165 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.11881; Std Dev: 5.29906; Max: 497\n",
      "Deleveraging trigger: 8; Removal trigger: 14\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 663 stacks.\n",
      "  21020 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.89514; Std Dev: 1.33206; Max: 14\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  21020 stacks merged into 20176 loci; deleveraged 56 loci; blacklisted 29 loci.\n",
      "After merging, coverage depth Mean: 2.99407; Std Dev: 1.50501; Max: 20\n",
      "Merging remainder radtags\n",
      "  40165 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 10047 remainder reads; unable to match 30118 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.49445; Std Dev: 2.04054; Max: 42\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 77460\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz... read 107578 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5174\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 438876 RAD-Tags.\n",
      "  Inserted 155487 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "71462 initial stacks were populated; 84025 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.96559; Std Dev: 13.7633; Max: 2043\n",
      "Deleveraging trigger: 19; Removal trigger: 32\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2058 stacks.\n",
      "  69505 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.61401; Std Dev: 2.74787; Max: 32\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  69505 stacks merged into 63261 loci; deleveraged 254 loci; blacklisted 157 loci.\n",
      "After merging, coverage depth Mean: 5.0067; Std Dev: 3.0386; Max: 48\n",
      "Merging remainder radtags\n",
      "  84025 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 56300 remainder reads; unable to match 27725 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.90031; Std Dev: 3.52745; Max: 105\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 411151\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz... read 438876 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5175\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 833361 RAD-Tags.\n",
      "  Inserted 204266 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "91094 initial stacks were populated; 113172 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.906; Std Dev: 24.1186; Max: 4218\n",
      "Deleveraging trigger: 32; Removal trigger: 56\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2459 stacks.\n",
      "  88739 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.41294; Std Dev: 4.51588; Max: 56\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  88739 stacks merged into 79685 loci; deleveraged 335 loci; blacklisted 218 loci.\n",
      "After merging, coverage depth Mean: 8.15107; Std Dev: 4.82302; Max: 74\n",
      "Merging remainder radtags\n",
      "  113172 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 89297 remainder reads; unable to match 23875 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.27624; Std Dev: 5.32554; Max: 81\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 809486\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz... read 833361 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5176\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 535704 RAD-Tags.\n",
      "  Inserted 177923 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "80275 initial stacks were populated; 97648 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.45694; Std Dev: 14.8148; Max: 1818\n",
      "Deleveraging trigger: 20; Removal trigger: 35\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2194 stacks.\n",
      "  78192 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.10187; Std Dev: 3.12626; Max: 35\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  78192 stacks merged into 70320 loci; deleveraged 270 loci; blacklisted 191 loci.\n",
      "After merging, coverage depth Mean: 5.5981; Std Dev: 3.41962; Max: 50\n",
      "Merging remainder radtags\n",
      "  97648 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 69239 remainder reads; unable to match 28409 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.58698; Std Dev: 3.931; Max: 108\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 507295\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz... read 535704 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5177\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 802239 RAD-Tags.\n",
      "  Inserted 192538 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "88309 initial stacks were populated; 104229 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 7.90418; Std Dev: 23.8235; Max: 3674\n",
      "Deleveraging trigger: 32; Removal trigger: 56\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2452 stacks.\n",
      "  85973 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 7.37324; Std Dev: 4.88195; Max: 56\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  85973 stacks merged into 77864 loci; deleveraged 331 loci; blacklisted 239 loci.\n",
      "After merging, coverage depth Mean: 8.02414; Std Dev: 5.22322; Max: 112\n",
      "Merging remainder radtags\n",
      "  104229 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 78532 remainder reads; unable to match 25697 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 9.03734; Std Dev: 5.77509; Max: 125\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 776542\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz... read 802239 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5178\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 162073 RAD-Tags.\n",
      "  Inserted 75360 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "31750 initial stacks were populated; 43610 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.73112; Std Dev: 7.40212; Max: 934\n",
      "Deleveraging trigger: 11; Removal trigger: 19\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 865 stacks.\n",
      "  30983 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.478; Std Dev: 1.95656; Max: 19\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  30983 stacks merged into 29548 loci; deleveraged 101 loci; blacklisted 62 loci.\n",
      "After merging, coverage depth Mean: 3.60868; Std Dev: 2.13697; Max: 27\n",
      "Merging remainder radtags\n",
      "  43610 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 14083 remainder reads; unable to match 29527 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.08789; Std Dev: 2.53825; Max: 51\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 132546\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz... read 162073 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5179\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 392299 RAD-Tags.\n",
      "  Inserted 130893 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "61840 initial stacks were populated; 69053 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.22713; Std Dev: 12.7484; Max: 2149\n",
      "Deleveraging trigger: 18; Removal trigger: 31\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1769 stacks.\n",
      "  60183 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.8739; Std Dev: 3.21028; Max: 31\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  60183 stacks merged into 56431 loci; deleveraged 214 loci; blacklisted 131 loci.\n",
      "After merging, coverage depth Mean: 5.13821; Std Dev: 3.47361; Max: 54\n",
      "Merging remainder radtags\n",
      "  69053 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 35567 remainder reads; unable to match 33486 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.77121; Std Dev: 3.88408; Max: 58\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 358813\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz... read 392299 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5180\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 63151 RAD-Tags.\n",
      "  Inserted 40664 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "11919 initial stacks were populated; 28745 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 2.88665; Std Dev: 4.45767; Max: 327\n",
      "Deleveraging trigger: 7; Removal trigger: 12\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 388 stacks.\n",
      "  11583 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 2.66395; Std Dev: 1.11497; Max: 12\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  11583 stacks merged into 11187 loci; deleveraged 39 loci; blacklisted 15 loci.\n",
      "After merging, coverage depth Mean: 2.73885; Std Dev: 1.28577; Max: 18\n",
      "Merging remainder radtags\n",
      "  28745 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 4962 remainder reads; unable to match 23783 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 3.18507; Std Dev: 1.87791; Max: 32\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 39368\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz... read 63151 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5181\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 212945 RAD-Tags.\n",
      "  Inserted 91820 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "40275 initial stacks were populated; 51545 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.00745; Std Dev: 8.90369; Max: 1284\n",
      "Deleveraging trigger: 13; Removal trigger: 22\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1126 stacks.\n",
      "  39249 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.72127; Std Dev: 2.15912; Max: 22\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  39249 stacks merged into 37068 loci; deleveraged 111 loci; blacklisted 93 loci.\n",
      "After merging, coverage depth Mean: 3.89049; Std Dev: 2.36862; Max: 45\n",
      "Merging remainder radtags\n",
      "  51545 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 19704 remainder reads; unable to match 31841 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.42484; Std Dev: 2.77343; Max: 49\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 181104\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz... read 212945 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5182\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...done\n",
      "Loaded 3710092 RAD-Tags.\n",
      "  Inserted 498792 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "133466 initial stacks were populated; 365326 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 25.0608; Std Dev: 94.6474; Max: 15352\n",
      "Deleveraging trigger: 120; Removal trigger: 214\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4892 stacks.\n",
      "  128706 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 23.5574; Std Dev: 20.837; Max: 212\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  128706 stacks merged into 92517 loci; deleveraged 987 loci; blacklisted 1563 loci.\n",
      "After merging, coverage depth Mean: 31.7641; Std Dev: 19.9287; Max: 222\n",
      "Merging remainder radtags\n",
      "  365326 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 299228 remainder reads; unable to match 66098 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 35.0587; Std Dev: 21.5034; Max: 246\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 3643994\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz... read 3710092 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5183\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1284589 RAD-Tags.\n",
      "  Inserted 255372 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "100124 initial stacks were populated; 155248 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.2794; Std Dev: 35.9385; Max: 6430\n",
      "Deleveraging trigger: 47; Removal trigger: 83\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2983 stacks.\n",
      "  97246 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.5969; Std Dev: 6.5846; Max: 82\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  97246 stacks merged into 83350 loci; deleveraged 477 loci; blacklisted 334 loci.\n",
      "After merging, coverage depth Mean: 12.1771; Std Dev: 6.66045; Max: 122\n",
      "Merging remainder radtags\n",
      "  155248 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 132640 remainder reads; unable to match 22608 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 13.7769; Std Dev: 7.28741; Max: 131\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1261981\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz... read 1284589 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5184\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 220105 RAD-Tags.\n",
      "  Inserted 96337 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "43546 initial stacks were populated; 52791 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 3.84224; Std Dev: 8.6024; Max: 1184\n",
      "Deleveraging trigger: 12; Removal trigger: 21\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1240 stacks.\n",
      "  42389 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 3.57921; Std Dev: 2.00283; Max: 21\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  42389 stacks merged into 40306 loci; deleveraged 122 loci; blacklisted 92 loci.\n",
      "After merging, coverage depth Mean: 3.72151; Std Dev: 2.21267; Max: 42\n",
      "Merging remainder radtags\n",
      "  52791 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 19282 remainder reads; unable to match 33509 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 4.20199; Std Dev: 2.61302; Max: 57\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 186596\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz... read 220105 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5185\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1796847 RAD-Tags.\n",
      "  Inserted 299399 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "101961 initial stacks were populated; 197438 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 15.6865; Std Dev: 43.6576; Max: 8693\n",
      "Deleveraging trigger: 59; Removal trigger: 103\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3465 stacks.\n",
      "  98650 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 14.7159; Std Dev: 12.3597; Max: 103\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  98650 stacks merged into 85014 loci; deleveraged 620 loci; blacklisted 500 loci.\n",
      "After merging, coverage depth Mean: 16.8041; Std Dev: 12.9902; Max: 145\n",
      "Merging remainder radtags\n",
      "  197438 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 147014 remainder reads; unable to match 50424 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 18.5468; Std Dev: 14.1158; Max: 152\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1746423\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz... read 1796847 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5186\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 338291 RAD-Tags.\n",
      "  Inserted 123920 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "59362 initial stacks were populated; 64558 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.61125; Std Dev: 10.6282; Max: 1635\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1688 stacks.\n",
      "  57783 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 4.30801; Std Dev: 2.61315; Max: 26\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  57783 stacks merged into 54470 loci; deleveraged 169 loci; blacklisted 122 loci.\n",
      "After merging, coverage depth Mean: 4.5194; Std Dev: 2.85499; Max: 58\n",
      "Merging remainder radtags\n",
      "  64558 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 30854 remainder reads; unable to match 33704 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 5.08826; Std Dev: 3.2315; Max: 71\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 304587\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz... read 338291 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5187\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1052123 RAD-Tags.\n",
      "  Inserted 230779 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "95965 initial stacks were populated; 134814 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.55879; Std Dev: 29.6576; Max: 4367\n",
      "Deleveraging trigger: 39; Removal trigger: 69\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 2737 stacks.\n",
      "  93339 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.93712; Std Dev: 6.2416; Max: 69\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  93339 stacks merged into 83257 loci; deleveraged 408 loci; blacklisted 291 loci.\n",
      "After merging, coverage depth Mean: 9.86332; Std Dev: 6.60193; Max: 106\n",
      "Merging remainder radtags\n",
      "  134814 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 100046 remainder reads; unable to match 34768 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 11.0708; Std Dev: 7.27233; Max: 112\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1017355\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz... read 1052123 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5188\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Loaded 428009 RAD-Tags.\n",
      "  Inserted 125872 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "59824 initial stacks were populated; 66048 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.05043; Std Dev: 14.5745; Max: 2580\n",
      "Deleveraging trigger: 21; Removal trigger: 35\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 1758 stacks.\n",
      "  58194 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.65701; Std Dev: 3.96925; Max: 35\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  58194 stacks merged into 54269 loci; deleveraged 231 loci; blacklisted 147 loci.\n",
      "After merging, coverage depth Mean: 5.99426; Std Dev: 4.27226; Max: 57\n",
      "Merging remainder radtags\n",
      "  66048 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 37828 remainder reads; unable to match 28220 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 6.69485; Std Dev: 4.73921; Max: 75\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 399789\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz... read 428009 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5189\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Loaded 1385987 RAD-Tags.\n",
      "  Inserted 284446 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "106557 initial stacks were populated; 177889 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 11.3376; Std Dev: 34.0343; Max: 5535\n",
      "Deleveraging trigger: 45; Removal trigger: 79\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3326 stacks.\n",
      "  103374 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 10.5978; Std Dev: 7.58167; Max: 79\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  103374 stacks merged into 86040 loci; deleveraged 566 loci; blacklisted 448 loci.\n",
      "After merging, coverage depth Mean: 12.5264; Std Dev: 7.87537; Max: 104\n",
      "Merging remainder radtags\n",
      "  177889 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 144148 remainder reads; unable to match 33741 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 14.2134; Std Dev: 8.6842; Max: 115\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 1352246\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz... read 1385987 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5190\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Loaded 4945581 RAD-Tags.\n",
      "  Inserted 608778 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "143599 initial stacks were populated; 465179 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 31.2008; Std Dev: 103.623; Max: 19446\n",
      "Deleveraging trigger: 135; Removal trigger: 238\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 6097 stacks.\n",
      "  137665 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 29.4956; Std Dev: 29.0856; Max: 238\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  137665 stacks merged into 93262 loci; deleveraged 1135 loci; blacklisted 2435 loci.\n",
      "After merging, coverage depth Mean: 41.958; Std Dev: 28.2164; Max: 281\n",
      "Merging remainder radtags\n",
      "  465179 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 378789 remainder reads; unable to match 86390 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 46.1359; Std Dev: 30.3813; Max: 310\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 4859191\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz... read 4945581 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5191\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...5M...6M...7M...done\n",
      "Loaded 7125415 RAD-Tags.\n",
      "  Inserted 801140 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "162653 initial stacks were populated; 638487 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 39.882; Std Dev: 184.125; Max: 35655\n",
      "Deleveraging trigger: 224; Removal trigger: 408\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 7001 stacks.\n",
      "  155777 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 37.7556; Std Dev: 38.333; Max: 408\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  155777 stacks merged into 100832 loci; deleveraged 1398 loci; blacklisted 3556 loci.\n",
      "After merging, coverage depth Mean: 55.8454; Std Dev: 35.6513; Max: 557\n",
      "Merging remainder radtags\n",
      "  638487 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "  Matched 499418 remainder reads; unable to match 139069 remainder reads.\n",
      "After remainders merged, coverage depth Mean: 60.9861; Std Dev: 38.2024; Max: 685\n",
      "Calling final consensus sequences, invoking SNP-calling model...\n",
      "Number of utilized reads: 6986346\n",
      "Writing loci, SNPs, and alleles to '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'...\n",
      "  Refetching sequencing IDs from /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz... read 7125415 sequence IDs.\n",
      "done.\n",
      "ustacks is done.\n"
     ]
    }
   ],
   "source": [
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0101.fq.gz' -i  5000 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0013.fq.gz' -i  5001 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0014.fq.gz' -i  5002 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0015.fq.gz' -i  5003 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0016.fq.gz' -i  5004 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0017.fq.gz' -i  5005 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0018.fq.gz' -i  5006 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0019.fq.gz' -i  5007 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0020.fq.gz' -i  5008 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0021.fq.gz' -i  5009 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0022.fq.gz' -i  5010 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0023.fq.gz' -i  5011 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0002.fq.gz' -i  5012 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0003.fq.gz' -i  5013 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0005.fq.gz' -i  5014 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0007.fq.gz' -i  5015 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0009.fq.gz' -i  5016 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0010.fq.gz' -i  5017 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0011.fq.gz' -i  5018 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0012.fq.gz' -i  5019 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz' -i  5020 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0014.fq.gz' -i  5021 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0015.fq.gz' -i  5022 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0016.fq.gz' -i  5023 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz' -i  5024 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0018.fq.gz' -i  5025 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz' -i  5026 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz' -i  5027 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0023.fq.gz' -i  5028 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0024.fq.gz' -i  5029 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0025.fq.gz' -i  5030 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz' -i  5031 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz' -i  5032 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0029.fq.gz' -i  5033 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0031.fq.gz' -i  5034 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0032.fq.gz' -i  5035 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0033.fq.gz' -i  5036 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0034.fq.gz' -i  5037 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0035.fq.gz' -i  5038 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz' -i  5039 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0039.fq.gz' -i  5040 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0040.fq.gz' -i  5041 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz' -i  5042 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz' -i  5043 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz' -i  5044 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0050.fq.gz' -i  5045 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0053.fq.gz' -i  5046 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0055.fq.gz' -i  5047 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0056.fq.gz' -i  5048 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0058.fq.gz' -i  5049 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz' -i  5050 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0062.fq.gz' -i  5051 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0066.fq.gz' -i  5052 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0067.fq.gz' -i  5053 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz' -i  5054 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0070.fq.gz' -i  5055 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0073.fq.gz' -i  5056 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0075.fq.gz' -i  5057 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz' -i  5058 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0101.fq.gz' -i  5059 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0104.fq.gz' -i  5060 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0001.fq.gz' -i  5061 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0002.fq.gz' -i  5062 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0003.fq.gz' -i  5063 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0004.fq.gz' -i  5064 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0005.fq.gz' -i  5065 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0006.fq.gz' -i  5066 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0007.fq.gz' -i  5067 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0008.fq.gz' -i  5068 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0009.fq.gz' -i  5069 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0010.fq.gz' -i  5070 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0011.fq.gz' -i  5071 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0012.fq.gz' -i  5072 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0013.fq.gz' -i  5073 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0014.fq.gz' -i  5074 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0015.fq.gz' -i  5075 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0016.fq.gz' -i  5076 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0017.fq.gz' -i  5077 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0018.fq.gz' -i  5078 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0019.fq.gz' -i  5079 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0020.fq.gz' -i  5080 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0021.fq.gz' -i  5081 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0022.fq.gz' -i  5082 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0023.fq.gz' -i  5083 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0024.fq.gz' -i  5084 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0025.fq.gz' -i  5085 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0026.fq.gz' -i  5086 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0027.fq.gz' -i  5087 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0028.fq.gz' -i  5088 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0029.fq.gz' -i  5089 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0030.fq.gz' -i  5090 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0031.fq.gz' -i  5091 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0032.fq.gz' -i  5092 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0033.fq.gz' -i  5093 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0034.fq.gz' -i  5094 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0045.fq.gz' -i  5095 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X_0107.fq.gz' -i  5096 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0001.fq.gz' -i  5097 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0002.fq.gz' -i  5098 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0003.fq.gz' -i  5099 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0004.fq.gz' -i  5100 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0005.fq.gz' -i  5101 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0006.fq.gz' -i  5102 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0007.fq.gz' -i  5103 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0008.fq.gz' -i  5104 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0009.fq.gz' -i  5105 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0010.fq.gz' -i  5106 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0011.fq.gz' -i  5107 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101G_0012.fq.gz' -i  5108 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0035.fq.gz' -i  5109 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0036.fq.gz' -i  5110 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0037.fq.gz' -i  5111 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0038.fq.gz' -i  5112 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0039.fq.gz' -i  5113 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0040.fq.gz' -i  5114 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0041.fq.gz' -i  5115 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0042.fq.gz' -i  5116 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0043.fq.gz' -i  5117 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0044.fq.gz' -i  5118 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0046.fq.gz' -i  5119 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0047.fq.gz' -i  5120 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0048.fq.gz' -i  5121 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0049.fq.gz' -i  5122 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0050.fq.gz' -i  5123 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0051.fq.gz' -i  5124 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0052.fq.gz' -i  5125 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0053.fq.gz' -i  5126 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0054.fq.gz' -i  5127 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0055.fq.gz' -i  5128 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0056.fq.gz' -i  5129 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0057.fq.gz' -i  5130 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0059.fq.gz' -i  5131 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0060.fq.gz' -i  5132 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0061.fq.gz' -i  5133 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0062.fq.gz' -i  5134 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0063.fq.gz' -i  5135 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0064.fq.gz' -i  5136 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0065.fq.gz' -i  5137 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0066.fq.gz' -i  5138 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0067.fq.gz' -i  5139 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0068.fq.gz' -i  5140 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0069.fq.gz' -i  5141 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0070.fq.gz' -i  5142 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0071.fq.gz' -i  5143 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0072.fq.gz' -i  5144 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0073.fq.gz' -i  5145 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0074.fq.gz' -i  5146 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0075.fq.gz' -i  5147 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0076.fq.gz' -i  5148 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0077.fq.gz' -i  5149 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0078.fq.gz' -i  5150 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0079.fq.gz' -i  5151 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0080.fq.gz' -i  5152 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0081.fq.gz' -i  5153 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0082.fq.gz' -i  5154 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0083.fq.gz' -i  5155 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0084.fq.gz' -i  5156 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0085.fq.gz' -i  5157 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0086.fq.gz' -i  5158 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0087.fq.gz' -i  5159 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0088.fq.gz' -i  5160 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0089.fq.gz' -i  5161 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0090.fq.gz' -i  5162 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0091.fq.gz' -i  5163 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0092.fq.gz' -i  5164 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0093.fq.gz' -i  5165 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0094.fq.gz' -i  5166 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0095.fq.gz' -i  5167 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0096.fq.gz' -i  5168 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0097.fq.gz' -i  5169 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0098.fq.gz' -i  5170 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0099.fq.gz' -i  5171 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0100.fq.gz' -i  5172 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0101.fq.gz' -i  5173 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0102.fq.gz' -i  5174 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0103.fq.gz' -i  5175 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0104.fq.gz' -i  5176 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0105.fq.gz' -i  5177 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0106.fq.gz' -i  5178 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0107.fq.gz' -i  5179 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0109.fq.gz' -i  5180 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0110.fq.gz' -i  5181 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0111.fq.gz' -i  5182 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0113.fq.gz' -i  5183 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0114.fq.gz' -i  5184 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0115.fq.gz' -i  5185 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0116.fq.gz' -i  5186 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0117.fq.gz' -i  5187 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0122.fq.gz' -i  5188 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0123.fq.gz' -i  5189 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0124.fq.gz' -i  5190 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X107H_0126.fq.gz' -i  5191 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## I'd already added variation from populations running cstacks before (catalog 3 -> catalog 4). Since we've added sequence to even-year parents I'm going to add them to the catalog again (catalog 4 -> 5) just to be safe."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cstacks parameters selected:\n",
      "  Database/Batch ID: 5\n",
      "  Loci matched based on sequence identity.\n",
      "  Number of mismatches allowed between stacks: 2\n",
      "  Gapped alignments: disabled\n",
      "Constructing catalog from 2 samples.\n",
      "\n",
      "Initializing existing catalog...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4.catalog.alleles.tsv\n",
      "\n",
      "Processing sample /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101 [1 of 2]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "  854646 loci in the catalog, 37732116 kmers in the catalog hash.\n",
      "Merging matches into catalog...\n",
      "  88604 loci were matched to a catalog locus.\n",
      "  0 loci were matched to a catalog locus using gapped alignments.\n",
      "  273 loci were newly added to the catalog.\n",
      "  1080 loci matched more than one catalog locus and were excluded.\n",
      "\n",
      "Processing sample /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107 [2 of 2]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "  854919 loci in the catalog, 38169561 kmers in the catalog hash.\n",
      "Merging matches into catalog...\n",
      "  172805 loci were matched to a catalog locus.\n",
      "  0 loci were matched to a catalog locus using gapped alignments.\n",
      "  362298 loci were newly added to the catalog.\n",
      "  2872 loci matched more than one catalog locus and were excluded.\n",
      "\n",
      "Writing catalog in directory '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'.\n",
      "cstacks is done.\n"
     ]
    }
   ],
   "source": [
    "! cstacks -p 7 -b 5 -n 2 -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' --catalog '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_4'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## sstacks to match sets of stacks to the catalog\n",
    "####A number of individuals failed in ustacks due to a memory dump(see above), so running sstacks individual by individual"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "537975 stacks compared against the catalog containing 1217217 loci.\n",
      "  535821 matching loci, 49643 contained no verified haplotypes.\n",
      "  2313 loci matched more than one catalog locus and were excluded.\n",
      "  18 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  560286 total haplotypes examined from matching loci, 496845 verified.\n",
      "Outputing to file ./PBIRD14X_0107.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91132 stacks compared against the catalog containing 1217217 loci.\n",
      "  83013 matching loci, 14835 contained no verified haplotypes.\n",
      "  290 loci matched more than one catalog locus and were excluded.\n",
      "  324 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87724 total haplotypes examined from matching loci, 71345 verified.\n",
      "Outputing to file ./PBIRD14X101G_0001.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "69232 stacks compared against the catalog containing 1217217 loci.\n",
      "  66555 matching loci, 13202 contained no verified haplotypes.\n",
      "  296 loci matched more than one catalog locus and were excluded.\n",
      "  790 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  69635 total haplotypes examined from matching loci, 55003 verified.\n",
      "Outputing to file ./PBIRD14X101G_0002.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91223 stacks compared against the catalog containing 1217217 loci.\n",
      "  83465 matching loci, 15004 contained no verified haplotypes.\n",
      "  317 loci matched more than one catalog locus and were excluded.\n",
      "  313 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88594 total haplotypes examined from matching loci, 71759 verified.\n",
      "Outputing to file ./PBIRD14X101G_0003.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86710 stacks compared against the catalog containing 1217217 loci.\n",
      "  82130 matching loci, 15308 contained no verified haplotypes.\n",
      "  311 loci matched more than one catalog locus and were excluded.\n",
      "  339 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86737 total haplotypes examined from matching loci, 69507 verified.\n",
      "Outputing to file ./PBIRD14X101G_0004.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96337 stacks compared against the catalog containing 1217217 loci.\n",
      "  85106 matching loci, 15241 contained no verified haplotypes.\n",
      "  326 loci matched more than one catalog locus and were excluded.\n",
      "  338 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90252 total haplotypes examined from matching loci, 73236 verified.\n",
      "Outputing to file ./PBIRD14X101G_0005.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84302 stacks compared against the catalog containing 1217217 loci.\n",
      "  81579 matching loci, 15235 contained no verified haplotypes.\n",
      "  309 loci matched more than one catalog locus and were excluded.\n",
      "  340 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86292 total haplotypes examined from matching loci, 69134 verified.\n",
      "Outputing to file ./PBIRD14X101G_0006.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88730 stacks compared against the catalog containing 1217217 loci.\n",
      "  83722 matching loci, 15265 contained no verified haplotypes.\n",
      "  330 loci matched more than one catalog locus and were excluded.\n",
      "  213 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88871 total haplotypes examined from matching loci, 71647 verified.\n",
      "Outputing to file ./PBIRD14X101G_0007.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95136 stacks compared against the catalog containing 1217217 loci.\n",
      "  85081 matching loci, 15047 contained no verified haplotypes.\n",
      "  315 loci matched more than one catalog locus and were excluded.\n",
      "  351 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90303 total haplotypes examined from matching loci, 73558 verified.\n",
      "Outputing to file ./PBIRD14X101G_0008.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "80677 stacks compared against the catalog containing 1217217 loci.\n",
      "  74805 matching loci, 13879 contained no verified haplotypes.\n",
      "  297 loci matched more than one catalog locus and were excluded.\n",
      "  365 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79054 total haplotypes examined from matching loci, 63392 verified.\n",
      "Outputing to file ./PBIRD14X101G_0009.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95080 stacks compared against the catalog containing 1217217 loci.\n",
      "  85201 matching loci, 15243 contained no verified haplotypes.\n",
      "  293 loci matched more than one catalog locus and were excluded.\n",
      "  305 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90561 total haplotypes examined from matching loci, 73465 verified.\n",
      "Outputing to file ./PBIRD14X101G_0010.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94532 stacks compared against the catalog containing 1217217 loci.\n",
      "  84544 matching loci, 15080 contained no verified haplotypes.\n",
      "  323 loci matched more than one catalog locus and were excluded.\n",
      "  288 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90332 total haplotypes examined from matching loci, 73207 verified.\n",
      "Outputing to file ./PBIRD14X101G_0011.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83909 stacks compared against the catalog containing 1217217 loci.\n",
      "  79447 matching loci, 14960 contained no verified haplotypes.\n",
      "  340 loci matched more than one catalog locus and were excluded.\n",
      "  400 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83700 total haplotypes examined from matching loci, 66872 verified.\n",
      "Outputing to file ./PBIRD14X101G_0012.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "47917 stacks compared against the catalog containing 1217217 loci.\n",
      "  46243 matching loci, 10320 contained no verified haplotypes.\n",
      "  209 loci matched more than one catalog locus and were excluded.\n",
      "  593 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  47273 total haplotypes examined from matching loci, 36164 verified.\n",
      "Outputing to file ./PBIRD14X107H_0035.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84622 stacks compared against the catalog containing 1217217 loci.\n",
      "  80855 matching loci, 16122 contained no verified haplotypes.\n",
      "  320 loci matched more than one catalog locus and were excluded.\n",
      "  251 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83312 total haplotypes examined from matching loci, 65714 verified.\n",
      "Outputing to file ./PBIRD14X107H_0036.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77363 stacks compared against the catalog containing 1217217 loci.\n",
      "  75135 matching loci, 15586 contained no verified haplotypes.\n",
      "  282 loci matched more than one catalog locus and were excluded.\n",
      "  528 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77247 total haplotypes examined from matching loci, 60271 verified.\n",
      "Outputing to file ./PBIRD14X107H_0037.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "46873 stacks compared against the catalog containing 1217217 loci.\n",
      "  43928 matching loci, 9831 contained no verified haplotypes.\n",
      "  186 loci matched more than one catalog locus and were excluded.\n",
      "  782 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  44961 total haplotypes examined from matching loci, 34413 verified.\n",
      "Outputing to file ./PBIRD14X107H_0038.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "76139 stacks compared against the catalog containing 1217217 loci.\n",
      "  74500 matching loci, 15430 contained no verified haplotypes.\n",
      "  289 loci matched more than one catalog locus and were excluded.\n",
      "  457 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  76541 total haplotypes examined from matching loci, 59700 verified.\n",
      "Outputing to file ./PBIRD14X107H_0039.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81787 stacks compared against the catalog containing 1217217 loci.\n",
      "  79028 matching loci, 16211 contained no verified haplotypes.\n",
      "  279 loci matched more than one catalog locus and were excluded.\n",
      "  661 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81271 total haplotypes examined from matching loci, 63683 verified.\n",
      "Outputing to file ./PBIRD14X107H_0040.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83362 stacks compared against the catalog containing 1217217 loci.\n",
      "  79521 matching loci, 16046 contained no verified haplotypes.\n",
      "  290 loci matched more than one catalog locus and were excluded.\n",
      "  304 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81953 total haplotypes examined from matching loci, 64362 verified.\n",
      "Outputing to file ./PBIRD14X107H_0041.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83852 stacks compared against the catalog containing 1217217 loci.\n",
      "  79831 matching loci, 16054 contained no verified haplotypes.\n",
      "  310 loci matched more than one catalog locus and were excluded.\n",
      "  262 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82378 total haplotypes examined from matching loci, 64755 verified.\n",
      "Outputing to file ./PBIRD14X107H_0042.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "80894 stacks compared against the catalog containing 1217217 loci.\n",
      "  78062 matching loci, 15938 contained no verified haplotypes.\n",
      "  306 loci matched more than one catalog locus and were excluded.\n",
      "  378 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80434 total haplotypes examined from matching loci, 62975 verified.\n",
      "Outputing to file ./PBIRD14X107H_0043.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81215 stacks compared against the catalog containing 1217217 loci.\n",
      "  78431 matching loci, 15964 contained no verified haplotypes.\n",
      "  301 loci matched more than one catalog locus and were excluded.\n",
      "  358 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80750 total haplotypes examined from matching loci, 63230 verified.\n",
      "Outputing to file ./PBIRD14X107H_0044.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82335 stacks compared against the catalog containing 1217217 loci.\n",
      "  78724 matching loci, 15996 contained no verified haplotypes.\n",
      "  306 loci matched more than one catalog locus and were excluded.\n",
      "  342 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81132 total haplotypes examined from matching loci, 63576 verified.\n",
      "Outputing to file ./PBIRD14X107H_0046.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87501 stacks compared against the catalog containing 1217217 loci.\n",
      "  81449 matching loci, 16119 contained no verified haplotypes.\n",
      "  290 loci matched more than one catalog locus and were excluded.\n",
      "  196 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84007 total haplotypes examined from matching loci, 66411 verified.\n",
      "Outputing to file ./PBIRD14X107H_0047.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83191 stacks compared against the catalog containing 1217217 loci.\n",
      "  80213 matching loci, 16072 contained no verified haplotypes.\n",
      "  308 loci matched more than one catalog locus and were excluded.\n",
      "  228 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82689 total haplotypes examined from matching loci, 65091 verified.\n",
      "Outputing to file ./PBIRD14X107H_0048.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84834 stacks compared against the catalog containing 1217217 loci.\n",
      "  80645 matching loci, 16014 contained no verified haplotypes.\n",
      "  307 loci matched more than one catalog locus and were excluded.\n",
      "  240 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83269 total haplotypes examined from matching loci, 65651 verified.\n",
      "Outputing to file ./PBIRD14X107H_0049.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77724 stacks compared against the catalog containing 1217217 loci.\n",
      "  76006 matching loci, 15675 contained no verified haplotypes.\n",
      "  281 loci matched more than one catalog locus and were excluded.\n",
      "  361 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78164 total haplotypes examined from matching loci, 61007 verified.\n",
      "Outputing to file ./PBIRD14X107H_0050.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81289 stacks compared against the catalog containing 1217217 loci.\n",
      "  79006 matching loci, 15984 contained no verified haplotypes.\n",
      "  310 loci matched more than one catalog locus and were excluded.\n",
      "  236 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81418 total haplotypes examined from matching loci, 63872 verified.\n",
      "Outputing to file ./PBIRD14X107H_0051.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90075 stacks compared against the catalog containing 1217217 loci.\n",
      "  82109 matching loci, 16060 contained no verified haplotypes.\n",
      "  317 loci matched more than one catalog locus and were excluded.\n",
      "  262 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85159 total haplotypes examined from matching loci, 67528 verified.\n",
      "Outputing to file ./PBIRD14X107H_0052.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "75009 stacks compared against the catalog containing 1217217 loci.\n",
      "  72918 matching loci, 15332 contained no verified haplotypes.\n",
      "  282 loci matched more than one catalog locus and were excluded.\n",
      "  589 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  74891 total haplotypes examined from matching loci, 58231 verified.\n",
      "Outputing to file ./PBIRD14X107H_0053.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85469 stacks compared against the catalog containing 1217217 loci.\n",
      "  81205 matching loci, 16077 contained no verified haplotypes.\n",
      "  268 loci matched more than one catalog locus and were excluded.\n",
      "  152 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83720 total haplotypes examined from matching loci, 66177 verified.\n",
      "Outputing to file ./PBIRD14X107H_0054.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92994 stacks compared against the catalog containing 1217217 loci.\n",
      "  83162 matching loci, 16086 contained no verified haplotypes.\n",
      "  293 loci matched more than one catalog locus and were excluded.\n",
      "  279 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85602 total haplotypes examined from matching loci, 68259 verified.\n",
      "Outputing to file ./PBIRD14X107H_0055.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89814 stacks compared against the catalog containing 1217217 loci.\n",
      "  82488 matching loci, 16164 contained no verified haplotypes.\n",
      "  296 loci matched more than one catalog locus and were excluded.\n",
      "  257 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85010 total haplotypes examined from matching loci, 67481 verified.\n",
      "Outputing to file ./PBIRD14X107H_0056.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86088 stacks compared against the catalog containing 1217217 loci.\n",
      "  81291 matching loci, 16050 contained no verified haplotypes.\n",
      "  296 loci matched more than one catalog locus and were excluded.\n",
      "  211 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83686 total haplotypes examined from matching loci, 66255 verified.\n",
      "Outputing to file ./PBIRD14X107H_0057.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82327 stacks compared against the catalog containing 1217217 loci.\n",
      "  79663 matching loci, 15971 contained no verified haplotypes.\n",
      "  294 loci matched more than one catalog locus and were excluded.\n",
      "  209 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82203 total haplotypes examined from matching loci, 64613 verified.\n",
      "Outputing to file ./PBIRD14X107H_0059.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84444 stacks compared against the catalog containing 1217217 loci.\n",
      "  80296 matching loci, 16060 contained no verified haplotypes.\n",
      "  311 loci matched more than one catalog locus and were excluded.\n",
      "  238 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82823 total haplotypes examined from matching loci, 65171 verified.\n",
      "Outputing to file ./PBIRD14X107H_0060.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89172 stacks compared against the catalog containing 1217217 loci.\n",
      "  81480 matching loci, 16106 contained no verified haplotypes.\n",
      "  286 loci matched more than one catalog locus and were excluded.\n",
      "  242 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84076 total haplotypes examined from matching loci, 66504 verified.\n",
      "Outputing to file ./PBIRD14X107H_0061.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93431 stacks compared against the catalog containing 1217217 loci.\n",
      "  83508 matching loci, 16228 contained no verified haplotypes.\n",
      "  310 loci matched more than one catalog locus and were excluded.\n",
      "  285 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86210 total haplotypes examined from matching loci, 68599 verified.\n",
      "Outputing to file ./PBIRD14X107H_0062.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96394 stacks compared against the catalog containing 1217217 loci.\n",
      "  84097 matching loci, 16264 contained no verified haplotypes.\n",
      "  303 loci matched more than one catalog locus and were excluded.\n",
      "  322 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86752 total haplotypes examined from matching loci, 69170 verified.\n",
      "Outputing to file ./PBIRD14X107H_0063.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "396 stacks compared against the catalog containing 1217217 loci.\n",
      "  77 matching loci, 10 contained no verified haplotypes.\n",
      "  0 loci matched more than one catalog locus and were excluded.\n",
      "  5 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79 total haplotypes examined from matching loci, 69 verified.\n",
      "Outputing to file ./PBIRD14X107H_0064.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90235 stacks compared against the catalog containing 1217217 loci.\n",
      "  82732 matching loci, 16016 contained no verified haplotypes.\n",
      "  276 loci matched more than one catalog locus and were excluded.\n",
      "  244 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85313 total haplotypes examined from matching loci, 68023 verified.\n",
      "Outputing to file ./PBIRD14X107H_0065.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85480 stacks compared against the catalog containing 1217217 loci.\n",
      "  80867 matching loci, 16061 contained no verified haplotypes.\n",
      "  301 loci matched more than one catalog locus and were excluded.\n",
      "  254 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83476 total haplotypes examined from matching loci, 65852 verified.\n",
      "Outputing to file ./PBIRD14X107H_0066.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85637 stacks compared against the catalog containing 1217217 loci.\n",
      "  81114 matching loci, 16047 contained no verified haplotypes.\n",
      "  291 loci matched more than one catalog locus and were excluded.\n",
      "  236 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83638 total haplotypes examined from matching loci, 66084 verified.\n",
      "Outputing to file ./PBIRD14X107H_0067.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88222 stacks compared against the catalog containing 1217217 loci.\n",
      "  82378 matching loci, 16148 contained no verified haplotypes.\n",
      "  329 loci matched more than one catalog locus and were excluded.\n",
      "  185 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85209 total haplotypes examined from matching loci, 67527 verified.\n",
      "Outputing to file ./PBIRD14X107H_0068.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "106047 stacks compared against the catalog containing 1217217 loci.\n",
      "  84502 matching loci, 16239 contained no verified haplotypes.\n",
      "  298 loci matched more than one catalog locus and were excluded.\n",
      "  383 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87278 total haplotypes examined from matching loci, 69758 verified.\n",
      "Outputing to file ./PBIRD14X107H_0069.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93404 stacks compared against the catalog containing 1217217 loci.\n",
      "  84153 matching loci, 16092 contained no verified haplotypes.\n",
      "  299 loci matched more than one catalog locus and were excluded.\n",
      "  292 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86706 total haplotypes examined from matching loci, 69398 verified.\n",
      "Outputing to file ./PBIRD14X107H_0070.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86518 stacks compared against the catalog containing 1217217 loci.\n",
      "  81335 matching loci, 16088 contained no verified haplotypes.\n",
      "  309 loci matched more than one catalog locus and were excluded.\n",
      "  216 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83750 total haplotypes examined from matching loci, 66212 verified.\n",
      "Outputing to file ./PBIRD14X107H_0071.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96887 stacks compared against the catalog containing 1217217 loci.\n",
      "  84552 matching loci, 16294 contained no verified haplotypes.\n",
      "  289 loci matched more than one catalog locus and were excluded.\n",
      "  276 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87391 total haplotypes examined from matching loci, 69738 verified.\n",
      "Outputing to file ./PBIRD14X107H_0072.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96428 stacks compared against the catalog containing 1217217 loci.\n",
      "  84119 matching loci, 16053 contained no verified haplotypes.\n",
      "  296 loci matched more than one catalog locus and were excluded.\n",
      "  364 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86823 total haplotypes examined from matching loci, 69551 verified.\n",
      "Outputing to file ./PBIRD14X107H_0073.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90765 stacks compared against the catalog containing 1217217 loci.\n",
      "  83258 matching loci, 16202 contained no verified haplotypes.\n",
      "  310 loci matched more than one catalog locus and were excluded.\n",
      "  223 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86017 total haplotypes examined from matching loci, 68346 verified.\n",
      "Outputing to file ./PBIRD14X107H_0074.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87805 stacks compared against the catalog containing 1217217 loci.\n",
      "  81653 matching loci, 16120 contained no verified haplotypes.\n",
      "  306 loci matched more than one catalog locus and were excluded.\n",
      "  254 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84150 total haplotypes examined from matching loci, 66566 verified.\n",
      "Outputing to file ./PBIRD14X107H_0075.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96813 stacks compared against the catalog containing 1217217 loci.\n",
      "  83918 matching loci, 16115 contained no verified haplotypes.\n",
      "  298 loci matched more than one catalog locus and were excluded.\n",
      "  293 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86594 total haplotypes examined from matching loci, 69249 verified.\n",
      "Outputing to file ./PBIRD14X107H_0076.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86598 stacks compared against the catalog containing 1217217 loci.\n",
      "  82164 matching loci, 16145 contained no verified haplotypes.\n",
      "  292 loci matched more than one catalog locus and were excluded.\n",
      "  159 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84959 total haplotypes examined from matching loci, 67257 verified.\n",
      "Outputing to file ./PBIRD14X107H_0077.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87235 stacks compared against the catalog containing 1217217 loci.\n",
      "  81766 matching loci, 16114 contained no verified haplotypes.\n",
      "  285 loci matched more than one catalog locus and were excluded.\n",
      "  201 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84307 total haplotypes examined from matching loci, 66711 verified.\n",
      "Outputing to file ./PBIRD14X107H_0078.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93781 stacks compared against the catalog containing 1217217 loci.\n",
      "  83365 matching loci, 15901 contained no verified haplotypes.\n",
      "  309 loci matched more than one catalog locus and were excluded.\n",
      "  315 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85742 total haplotypes examined from matching loci, 68707 verified.\n",
      "Outputing to file ./PBIRD14X107H_0079.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "96355 stacks compared against the catalog containing 1217217 loci.\n",
      "  83950 matching loci, 16155 contained no verified haplotypes.\n",
      "  312 loci matched more than one catalog locus and were excluded.\n",
      "  300 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86695 total haplotypes examined from matching loci, 69243 verified.\n",
      "Outputing to file ./PBIRD14X107H_0080.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "91967 stacks compared against the catalog containing 1217217 loci.\n",
      "  84175 matching loci, 16257 contained no verified haplotypes.\n",
      "  313 loci matched more than one catalog locus and were excluded.\n",
      "  266 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87099 total haplotypes examined from matching loci, 69358 verified.\n",
      "Outputing to file ./PBIRD14X107H_0081.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89365 stacks compared against the catalog containing 1217217 loci.\n",
      "  82524 matching loci, 16186 contained no verified haplotypes.\n",
      "  325 loci matched more than one catalog locus and were excluded.\n",
      "  219 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85224 total haplotypes examined from matching loci, 67545 verified.\n",
      "Outputing to file ./PBIRD14X107H_0082.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "95799 stacks compared against the catalog containing 1217217 loci.\n",
      "  85311 matching loci, 16080 contained no verified haplotypes.\n",
      "  292 loci matched more than one catalog locus and were excluded.\n",
      "  301 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87967 total haplotypes examined from matching loci, 70699 verified.\n",
      "Outputing to file ./PBIRD14X107H_0083.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84150 stacks compared against the catalog containing 1217217 loci.\n",
      "  80009 matching loci, 16338 contained no verified haplotypes.\n",
      "  280 loci matched more than one catalog locus and were excluded.\n",
      "  725 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82265 total haplotypes examined from matching loci, 64602 verified.\n",
      "Outputing to file ./PBIRD14X107H_0084.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81658 stacks compared against the catalog containing 1217217 loci.\n",
      "  78247 matching loci, 16341 contained no verified haplotypes.\n",
      "  292 loci matched more than one catalog locus and were excluded.\n",
      "  921 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80455 total haplotypes examined from matching loci, 62704 verified.\n",
      "Outputing to file ./PBIRD14X107H_0085.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78555 stacks compared against the catalog containing 1217217 loci.\n",
      "  76066 matching loci, 16114 contained no verified haplotypes.\n",
      "  279 loci matched more than one catalog locus and were excluded.\n",
      "  981 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78169 total haplotypes examined from matching loci, 60649 verified.\n",
      "Outputing to file ./PBIRD14X107H_0086.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82134 stacks compared against the catalog containing 1217217 loci.\n",
      "  79656 matching loci, 16048 contained no verified haplotypes.\n",
      "  283 loci matched more than one catalog locus and were excluded.\n",
      "  456 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81860 total haplotypes examined from matching loci, 64491 verified.\n",
      "Outputing to file ./PBIRD14X107H_0087.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78172 stacks compared against the catalog containing 1217217 loci.\n",
      "  75612 matching loci, 15909 contained no verified haplotypes.\n",
      "  310 loci matched more than one catalog locus and were excluded.\n",
      "  895 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77779 total haplotypes examined from matching loci, 60500 verified.\n",
      "Outputing to file ./PBIRD14X107H_0088.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "72053 stacks compared against the catalog containing 1217217 loci.\n",
      "  70158 matching loci, 15237 contained no verified haplotypes.\n",
      "  266 loci matched more than one catalog locus and were excluded.\n",
      "  1157 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  71899 total haplotypes examined from matching loci, 55450 verified.\n",
      "Outputing to file ./PBIRD14X107H_0089.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "75545 stacks compared against the catalog containing 1217217 loci.\n",
      "  72695 matching loci, 15579 contained no verified haplotypes.\n",
      "  268 loci matched more than one catalog locus and were excluded.\n",
      "  1065 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  74692 total haplotypes examined from matching loci, 57812 verified.\n",
      "Outputing to file ./PBIRD14X107H_0090.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85040 stacks compared against the catalog containing 1217217 loci.\n",
      "  81386 matching loci, 16110 contained no verified haplotypes.\n",
      "  294 loci matched more than one catalog locus and were excluded.\n",
      "  205 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83862 total haplotypes examined from matching loci, 66336 verified.\n",
      "Outputing to file ./PBIRD14X107H_0091.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77389 stacks compared against the catalog containing 1217217 loci.\n",
      "  74954 matching loci, 15978 contained no verified haplotypes.\n",
      "  272 loci matched more than one catalog locus and were excluded.\n",
      "  1137 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77053 total haplotypes examined from matching loci, 59693 verified.\n",
      "Outputing to file ./PBIRD14X107H_0092.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82153 stacks compared against the catalog containing 1217217 loci.\n",
      "  78916 matching loci, 16191 contained no verified haplotypes.\n",
      "  274 loci matched more than one catalog locus and were excluded.\n",
      "  658 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81139 total haplotypes examined from matching loci, 63607 verified.\n",
      "Outputing to file ./PBIRD14X107H_0093.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84549 stacks compared against the catalog containing 1217217 loci.\n",
      "  80854 matching loci, 16076 contained no verified haplotypes.\n",
      "  304 loci matched more than one catalog locus and were excluded.\n",
      "  217 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83201 total haplotypes examined from matching loci, 65744 verified.\n",
      "Outputing to file ./PBIRD14X107H_0094.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81419 stacks compared against the catalog containing 1217217 loci.\n",
      "  78441 matching loci, 16271 contained no verified haplotypes.\n",
      "  290 loci matched more than one catalog locus and were excluded.\n",
      "  842 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  80645 total haplotypes examined from matching loci, 62979 verified.\n",
      "Outputing to file ./PBIRD14X107H_0095.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82992 stacks compared against the catalog containing 1217217 loci.\n",
      "  79352 matching loci, 16284 contained no verified haplotypes.\n",
      "  284 loci matched more than one catalog locus and were excluded.\n",
      "  670 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81566 total haplotypes examined from matching loci, 63970 verified.\n",
      "Outputing to file ./PBIRD14X107H_0096.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "42005 stacks compared against the catalog containing 1217217 loci.\n",
      "  40050 matching loci, 9127 contained no verified haplotypes.\n",
      "  185 loci matched more than one catalog locus and were excluded.\n",
      "  734 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  40940 total haplotypes examined from matching loci, 31176 verified.\n",
      "Outputing to file ./PBIRD14X107H_0097.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "21033 stacks compared against the catalog containing 1217217 loci.\n",
      "  18622 matching loci, 4175 contained no verified haplotypes.\n",
      "  86 loci matched more than one catalog locus and were excluded.\n",
      "  313 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  19000 total haplotypes examined from matching loci, 14576 verified.\n",
      "Outputing to file ./PBIRD14X107H_0098.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "53927 stacks compared against the catalog containing 1217217 loci.\n",
      "  50715 matching loci, 10938 contained no verified haplotypes.\n",
      "  217 loci matched more than one catalog locus and were excluded.\n",
      "  636 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  51972 total haplotypes examined from matching loci, 40173 verified.\n",
      "Outputing to file ./PBIRD14X107H_0099.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "42577 stacks compared against the catalog containing 1217217 loci.\n",
      "  39848 matching loci, 8460 contained no verified haplotypes.\n",
      "  166 loci matched more than one catalog locus and were excluded.\n",
      "  471 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  40982 total haplotypes examined from matching loci, 31814 verified.\n",
      "Outputing to file ./PBIRD14X107H_0100.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "20079 stacks compared against the catalog containing 1217217 loci.\n",
      "  18714 matching loci, 4261 contained no verified haplotypes.\n",
      "  102 loci matched more than one catalog locus and were excluded.\n",
      "  274 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  19139 total haplotypes examined from matching loci, 14573 verified.\n",
      "Outputing to file ./PBIRD14X107H_0101.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "63003 stacks compared against the catalog containing 1217217 loci.\n",
      "  60880 matching loci, 13427 contained no verified haplotypes.\n",
      "  264 loci matched more than one catalog locus and were excluded.\n",
      "  1011 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  62444 total haplotypes examined from matching loci, 47975 verified.\n",
      "Outputing to file ./PBIRD14X107H_0102.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "79363 stacks compared against the catalog containing 1217217 loci.\n",
      "  76288 matching loci, 15765 contained no verified haplotypes.\n",
      "  282 loci matched more than one catalog locus and were excluded.\n",
      "  633 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78520 total haplotypes examined from matching loci, 61328 verified.\n",
      "Outputing to file ./PBIRD14X107H_0103.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "70018 stacks compared against the catalog containing 1217217 loci.\n",
      "  67321 matching loci, 14584 contained no verified haplotypes.\n",
      "  243 loci matched more than one catalog locus and were excluded.\n",
      "  1085 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  68958 total haplotypes examined from matching loci, 53276 verified.\n",
      "Outputing to file ./PBIRD14X107H_0104.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "77509 stacks compared against the catalog containing 1217217 loci.\n",
      "  74101 matching loci, 15311 contained no verified haplotypes.\n",
      "  270 loci matched more than one catalog locus and were excluded.\n",
      "  613 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  76281 total haplotypes examined from matching loci, 59598 verified.\n",
      "Outputing to file ./PBIRD14X107H_0105.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "29388 stacks compared against the catalog containing 1217217 loci.\n",
      "  26765 matching loci, 5844 contained no verified haplotypes.\n",
      "  115 loci matched more than one catalog locus and were excluded.\n",
      "  346 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  27360 total haplotypes examined from matching loci, 21131 verified.\n",
      "Outputing to file ./PBIRD14X107H_0106.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "56188 stacks compared against the catalog containing 1217217 loci.\n",
      "  51383 matching loci, 10886 contained no verified haplotypes.\n",
      "  219 loci matched more than one catalog locus and were excluded.\n",
      "  530 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  52688 total haplotypes examined from matching loci, 40943 verified.\n",
      "Outputing to file ./PBIRD14X107H_0107.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "11120 stacks compared against the catalog containing 1217217 loci.\n",
      "  9661 matching loci, 2111 contained no verified haplotypes.\n",
      "  63 loci matched more than one catalog locus and were excluded.\n",
      "  106 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  9872 total haplotypes examined from matching loci, 7603 verified.\n",
      "Outputing to file ./PBIRD14X107H_0109.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "36875 stacks compared against the catalog containing 1217217 loci.\n",
      "  34037 matching loci, 7449 contained no verified haplotypes.\n",
      "  169 loci matched more than one catalog locus and were excluded.\n",
      "  441 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  34879 total haplotypes examined from matching loci, 26869 verified.\n",
      "Outputing to file ./PBIRD14X107H_0110.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90822 stacks compared against the catalog containing 1217217 loci.\n",
      "  82452 matching loci, 16004 contained no verified haplotypes.\n",
      "  270 loci matched more than one catalog locus and were excluded.\n",
      "  338 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84998 total haplotypes examined from matching loci, 67741 verified.\n",
      "Outputing to file ./PBIRD14X107H_0111.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82911 stacks compared against the catalog containing 1217217 loci.\n",
      "  79800 matching loci, 16142 contained no verified haplotypes.\n",
      "  289 loci matched more than one catalog locus and were excluded.\n",
      "  365 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82186 total haplotypes examined from matching loci, 64568 verified.\n",
      "Outputing to file ./PBIRD14X107H_0113.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "40131 stacks compared against the catalog containing 1217217 loci.\n",
      "  37730 matching loci, 8202 contained no verified haplotypes.\n",
      "  186 loci matched more than one catalog locus and were excluded.\n",
      "  367 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  38658 total haplotypes examined from matching loci, 29805 verified.\n",
      "Outputing to file ./PBIRD14X107H_0114.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84360 stacks compared against the catalog containing 1217217 loci.\n",
      "  76488 matching loci, 15121 contained no verified haplotypes.\n",
      "  302 loci matched more than one catalog locus and were excluded.\n",
      "  377 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79195 total haplotypes examined from matching loci, 62653 verified.\n",
      "Outputing to file ./PBIRD14X107H_0115.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "54239 stacks compared against the catalog containing 1217217 loci.\n",
      "  50497 matching loci, 10950 contained no verified haplotypes.\n",
      "  238 loci matched more than one catalog locus and were excluded.\n",
      "  581 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  51806 total haplotypes examined from matching loci, 39994 verified.\n",
      "Outputing to file ./PBIRD14X107H_0116.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82855 stacks compared against the catalog containing 1217217 loci.\n",
      "  76877 matching loci, 15581 contained no verified haplotypes.\n",
      "  287 loci matched more than one catalog locus and were excluded.\n",
      "  561 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79314 total haplotypes examined from matching loci, 62240 verified.\n",
      "Outputing to file ./PBIRD14X107H_0117.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "53994 stacks compared against the catalog containing 1217217 loci.\n",
      "  50744 matching loci, 10634 contained no verified haplotypes.\n",
      "  226 loci matched more than one catalog locus and were excluded.\n",
      "  402 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  52121 total haplotypes examined from matching loci, 40599 verified.\n",
      "Outputing to file ./PBIRD14X107H_0122.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85449 stacks compared against the catalog containing 1217217 loci.\n",
      "  80011 matching loci, 16041 contained no verified haplotypes.\n",
      "  310 loci matched more than one catalog locus and were excluded.\n",
      "  446 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82561 total haplotypes examined from matching loci, 64992 verified.\n",
      "Outputing to file ./PBIRD14X107H_0123.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90664 stacks compared against the catalog containing 1217217 loci.\n",
      "  81939 matching loci, 15783 contained no verified haplotypes.\n",
      "  275 loci matched more than one catalog locus and were excluded.\n",
      "  322 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84570 total haplotypes examined from matching loci, 67556 verified.\n",
      "Outputing to file ./PBIRD14X107H_0124.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "97151 stacks compared against the catalog containing 1217217 loci.\n",
      "  83480 matching loci, 15956 contained no verified haplotypes.\n",
      "  299 loci matched more than one catalog locus and were excluded.\n",
      "  397 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86169 total haplotypes examined from matching loci, 69022 verified.\n",
      "Outputing to file ./PBIRD14X107H_0126.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89957 stacks compared against the catalog containing 1217217 loci.\n",
      "  89343 matching loci, 14852 contained no verified haplotypes.\n",
      "  259 loci matched more than one catalog locus and were excluded.\n",
      "  28 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  97825 total haplotypes examined from matching loci, 80088 verified.\n",
      "Outputing to file ./PBIRD14X_0101.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81447 stacks compared against the catalog containing 1217217 loci.\n",
      "  80050 matching loci, 14850 contained no verified haplotypes.\n",
      "  309 loci matched more than one catalog locus and were excluded.\n",
      "  131 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84664 total haplotypes examined from matching loci, 67756 verified.\n",
      "Outputing to file ./PBIRD14X101G_0013.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89157 stacks compared against the catalog containing 1217217 loci.\n",
      "  84948 matching loci, 15198 contained no verified haplotypes.\n",
      "  361 loci matched more than one catalog locus and were excluded.\n",
      "  177 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90552 total haplotypes examined from matching loci, 73127 verified.\n",
      "Outputing to file ./PBIRD14X101G_0014.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "81848 stacks compared against the catalog containing 1217217 loci.\n",
      "  79774 matching loci, 14812 contained no verified haplotypes.\n",
      "  307 loci matched more than one catalog locus and were excluded.\n",
      "  186 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84462 total haplotypes examined from matching loci, 67623 verified.\n",
      "Outputing to file ./PBIRD14X101G_0015.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85924 stacks compared against the catalog containing 1217217 loci.\n",
      "  82379 matching loci, 15032 contained no verified haplotypes.\n",
      "  314 loci matched more than one catalog locus and were excluded.\n",
      "  173 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87634 total haplotypes examined from matching loci, 70429 verified.\n",
      "Outputing to file ./PBIRD14X101G_0016.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "69484 stacks compared against the catalog containing 1217217 loci.\n",
      "  67915 matching loci, 13141 contained no verified haplotypes.\n",
      "  262 loci matched more than one catalog locus and were excluded.\n",
      "  250 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  70637 total haplotypes examined from matching loci, 56093 verified.\n",
      "Outputing to file ./PBIRD14X101G_0017.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85061 stacks compared against the catalog containing 1217217 loci.\n",
      "  82515 matching loci, 15046 contained no verified haplotypes.\n",
      "  347 loci matched more than one catalog locus and were excluded.\n",
      "  173 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87774 total haplotypes examined from matching loci, 70528 verified.\n",
      "Outputing to file ./PBIRD14X101G_0018.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "79350 stacks compared against the catalog containing 1217217 loci.\n",
      "  78063 matching loci, 14721 contained no verified haplotypes.\n",
      "  301 loci matched more than one catalog locus and were excluded.\n",
      "  185 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82013 total haplotypes examined from matching loci, 65437 verified.\n",
      "Outputing to file ./PBIRD14X101G_0019.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "72117 stacks compared against the catalog containing 1217217 loci.\n",
      "  70826 matching loci, 13738 contained no verified haplotypes.\n",
      "  243 loci matched more than one catalog locus and were excluded.\n",
      "  191 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  73573 total haplotypes examined from matching loci, 58410 verified.\n",
      "Outputing to file ./PBIRD14X101G_0020.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78038 stacks compared against the catalog containing 1217217 loci.\n",
      "  74766 matching loci, 14216 contained no verified haplotypes.\n",
      "  298 loci matched more than one catalog locus and were excluded.\n",
      "  396 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  78602 total haplotypes examined from matching loci, 62651 verified.\n",
      "Outputing to file ./PBIRD14X101G_0021.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "76727 stacks compared against the catalog containing 1217217 loci.\n",
      "  73973 matching loci, 14015 contained no verified haplotypes.\n",
      "  281 loci matched more than one catalog locus and were excluded.\n",
      "  260 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  77617 total haplotypes examined from matching loci, 61938 verified.\n",
      "Outputing to file ./PBIRD14X101G_0022.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "67173 stacks compared against the catalog containing 1217217 loci.\n",
      "  63865 matching loci, 12497 contained no verified haplotypes.\n",
      "  248 loci matched more than one catalog locus and were excluded.\n",
      "  402 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  66362 total haplotypes examined from matching loci, 52658 verified.\n",
      "Outputing to file ./PBIRD14X101G_0023.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88289 stacks compared against the catalog containing 1217217 loci.\n",
      "  81744 matching loci, 15037 contained no verified haplotypes.\n",
      "  286 loci matched more than one catalog locus and were excluded.\n",
      "  218 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84730 total haplotypes examined from matching loci, 68070 verified.\n",
      "Outputing to file ./PBIRD14X101H_0002.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "101711 stacks compared against the catalog containing 1217217 loci.\n",
      "  85213 matching loci, 15444 contained no verified haplotypes.\n",
      "  283 loci matched more than one catalog locus and were excluded.\n",
      "  383 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88222 total haplotypes examined from matching loci, 71427 verified.\n",
      "Outputing to file ./PBIRD14X101H_0003.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86317 stacks compared against the catalog containing 1217217 loci.\n",
      "  81442 matching loci, 15100 contained no verified haplotypes.\n",
      "  269 loci matched more than one catalog locus and were excluded.\n",
      "  175 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84212 total haplotypes examined from matching loci, 67540 verified.\n",
      "Outputing to file ./PBIRD14X101H_0005.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94210 stacks compared against the catalog containing 1217217 loci.\n",
      "  84198 matching loci, 15263 contained no verified haplotypes.\n",
      "  298 loci matched more than one catalog locus and were excluded.\n",
      "  207 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87179 total haplotypes examined from matching loci, 70385 verified.\n",
      "Outputing to file ./PBIRD14X101H_0007.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "99718 stacks compared against the catalog containing 1217217 loci.\n",
      "  85201 matching loci, 15405 contained no verified haplotypes.\n",
      "  292 loci matched more than one catalog locus and were excluded.\n",
      "  316 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88254 total haplotypes examined from matching loci, 71472 verified.\n",
      "Outputing to file ./PBIRD14X101H_0009.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "424 stacks compared against the catalog containing 1217217 loci.\n",
      "  33 matching loci, 5 contained no verified haplotypes.\n",
      "  0 loci matched more than one catalog locus and were excluded.\n",
      "  3 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  37 total haplotypes examined from matching loci, 32 verified.\n",
      "Outputing to file ./PBIRD14X101H_0011.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85762 stacks compared against the catalog containing 1217217 loci.\n",
      "  81214 matching loci, 15086 contained no verified haplotypes.\n",
      "  260 loci matched more than one catalog locus and were excluded.\n",
      "  143 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84011 total haplotypes examined from matching loci, 67322 verified.\n",
      "Outputing to file ./PBIRD14X101H_0012.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89853 stacks compared against the catalog containing 1217217 loci.\n",
      "  82336 matching loci, 15168 contained no verified haplotypes.\n",
      "  285 loci matched more than one catalog locus and were excluded.\n",
      "  197 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85254 total haplotypes examined from matching loci, 68407 verified.\n",
      "Outputing to file ./PBIRD14X101H_0014.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "89803 stacks compared against the catalog containing 1217217 loci.\n",
      "  81982 matching loci, 15104 contained no verified haplotypes.\n",
      "  270 loci matched more than one catalog locus and were excluded.\n",
      "  202 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84819 total haplotypes examined from matching loci, 68095 verified.\n",
      "Outputing to file ./PBIRD14X101H_0015.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87679 stacks compared against the catalog containing 1217217 loci.\n",
      "  81058 matching loci, 15090 contained no verified haplotypes.\n",
      "  249 loci matched more than one catalog locus and were excluded.\n",
      "  240 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83716 total haplotypes examined from matching loci, 67022 verified.\n",
      "Outputing to file ./PBIRD14X101H_0016.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84230 stacks compared against the catalog containing 1217217 loci.\n",
      "  80752 matching loci, 15102 contained no verified haplotypes.\n",
      "  278 loci matched more than one catalog locus and were excluded.\n",
      "  167 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83433 total haplotypes examined from matching loci, 66713 verified.\n",
      "Outputing to file ./PBIRD14X101H_0018.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85605 stacks compared against the catalog containing 1217217 loci.\n",
      "  81206 matching loci, 15066 contained no verified haplotypes.\n",
      "  261 loci matched more than one catalog locus and were excluded.\n",
      "  181 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84014 total haplotypes examined from matching loci, 67318 verified.\n",
      "Outputing to file ./PBIRD14X101H_0023.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "320989 stacks compared against the catalog containing 1217217 loci.\n",
      "  173002 matching loci, 38424 contained no verified haplotypes.\n",
      "  1028 loci matched more than one catalog locus and were excluded.\n",
      "  1185 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  179938 total haplotypes examined from matching loci, 135989 verified.\n",
      "Outputing to file ./PBIRD14X101H_0024.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "99944 stacks compared against the catalog containing 1217217 loci.\n",
      "  84311 matching loci, 15339 contained no verified haplotypes.\n",
      "  276 loci matched more than one catalog locus and were excluded.\n",
      "  263 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87413 total haplotypes examined from matching loci, 70478 verified.\n",
      "Outputing to file ./PBIRD14X101H_0025.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "333457 stacks compared against the catalog containing 1217217 loci.\n",
      "  177072 matching loci, 39039 contained no verified haplotypes.\n",
      "  997 loci matched more than one catalog locus and were excluded.\n",
      "  1168 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  183768 total haplotypes examined from matching loci, 139399 verified.\n",
      "Outputing to file ./PBIRD14X101H_0029.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "449399 stacks compared against the catalog containing 1217217 loci.\n",
      "  187958 matching loci, 41449 contained no verified haplotypes.\n",
      "  1163 loci matched more than one catalog locus and were excluded.\n",
      "  1228 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  195858 total haplotypes examined from matching loci, 147931 verified.\n",
      "Outputing to file ./PBIRD14X101H_0031.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92300 stacks compared against the catalog containing 1217217 loci.\n",
      "  82204 matching loci, 15217 contained no verified haplotypes.\n",
      "  294 loci matched more than one catalog locus and were excluded.\n",
      "  311 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85104 total haplotypes examined from matching loci, 68265 verified.\n",
      "Outputing to file ./PBIRD14X101H_0032.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90208 stacks compared against the catalog containing 1217217 loci.\n",
      "  81472 matching loci, 15132 contained no verified haplotypes.\n",
      "  247 loci matched more than one catalog locus and were excluded.\n",
      "  224 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84296 total haplotypes examined from matching loci, 67539 verified.\n",
      "Outputing to file ./PBIRD14X101H_0033.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88644 stacks compared against the catalog containing 1217217 loci.\n",
      "  81306 matching loci, 15122 contained no verified haplotypes.\n",
      "  275 loci matched more than one catalog locus and were excluded.\n",
      "  223 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84070 total haplotypes examined from matching loci, 67270 verified.\n",
      "Outputing to file ./PBIRD14X101H_0034.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "298490 stacks compared against the catalog containing 1217217 loci.\n",
      "  162096 matching loci, 36170 contained no verified haplotypes.\n",
      "  926 loci matched more than one catalog locus and were excluded.\n",
      "  1189 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  167728 total haplotypes examined from matching loci, 126868 verified.\n",
      "Outputing to file ./PBIRD14X101H_0035.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "405534 stacks compared against the catalog containing 1217217 loci.\n",
      "  182657 matching loci, 40544 contained no verified haplotypes.\n",
      "  1044 loci matched more than one catalog locus and were excluded.\n",
      "  1185 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  190322 total haplotypes examined from matching loci, 143542 verified.\n",
      "Outputing to file ./PBIRD14X101H_0039.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "463611 stacks compared against the catalog containing 1217217 loci.\n",
      "  195291 matching loci, 43560 contained no verified haplotypes.\n",
      "  1218 loci matched more than one catalog locus and were excluded.\n",
      "  1363 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  202852 total haplotypes examined from matching loci, 153157 verified.\n",
      "Outputing to file ./PBIRD14X101H_0040.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "202880 stacks compared against the catalog containing 1217217 loci.\n",
      "  110645 matching loci, 25290 contained no verified haplotypes.\n",
      "  633 loci matched more than one catalog locus and were excluded.\n",
      "  977 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  114036 total haplotypes examined from matching loci, 85936 verified.\n",
      "Outputing to file ./PBIRD14X101H_0050.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "212952 stacks compared against the catalog containing 1217217 loci.\n",
      "  106280 matching loci, 24719 contained no verified haplotypes.\n",
      "  648 loci matched more than one catalog locus and were excluded.\n",
      "  1099 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  109714 total haplotypes examined from matching loci, 82088 verified.\n",
      "Outputing to file ./PBIRD14X101H_0053.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "253416 stacks compared against the catalog containing 1217217 loci.\n",
      "  150259 matching loci, 32037 contained no verified haplotypes.\n",
      "  743 loci matched more than one catalog locus and were excluded.\n",
      "  895 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  155802 total haplotypes examined from matching loci, 119739 verified.\n",
      "Outputing to file ./PBIRD14X101H_0055.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "304972 stacks compared against the catalog containing 1217217 loci.\n",
      "  139930 matching loci, 32494 contained no verified haplotypes.\n",
      "  875 loci matched more than one catalog locus and were excluded.\n",
      "  1247 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  145192 total haplotypes examined from matching loci, 108240 verified.\n",
      "Outputing to file ./PBIRD14X101H_0056.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "242587 stacks compared against the catalog containing 1217217 loci.\n",
      "  135976 matching loci, 30588 contained no verified haplotypes.\n",
      "  746 loci matched more than one catalog locus and were excluded.\n",
      "  1071 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  140257 total haplotypes examined from matching loci, 106159 verified.\n",
      "Outputing to file ./PBIRD14X101H_0058.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "213165 stacks compared against the catalog containing 1217217 loci.\n",
      "  138889 matching loci, 29650 contained no verified haplotypes.\n",
      "  662 loci matched more than one catalog locus and were excluded.\n",
      "  905 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  143480 total haplotypes examined from matching loci, 110512 verified.\n",
      "Outputing to file ./PBIRD14X101H_0062.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "210150 stacks compared against the catalog containing 1217217 loci.\n",
      "  108556 matching loci, 25814 contained no verified haplotypes.\n",
      "  659 loci matched more than one catalog locus and were excluded.\n",
      "  1091 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  111991 total haplotypes examined from matching loci, 83176 verified.\n",
      "Outputing to file ./PBIRD14X101H_0066.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "268033 stacks compared against the catalog containing 1217217 loci.\n",
      "  146141 matching loci, 33202 contained no verified haplotypes.\n",
      "  817 loci matched more than one catalog locus and were excluded.\n",
      "  1375 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  151147 total haplotypes examined from matching loci, 113865 verified.\n",
      "Outputing to file ./PBIRD14X101H_0067.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "357076 stacks compared against the catalog containing 1217217 loci.\n",
      "  158717 matching loci, 36362 contained no verified haplotypes.\n",
      "  1010 loci matched more than one catalog locus and were excluded.\n",
      "  1321 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  164352 total haplotypes examined from matching loci, 123178 verified.\n",
      "Outputing to file ./PBIRD14X101H_0070.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "158765 stacks compared against the catalog containing 1217217 loci.\n",
      "  108783 matching loci, 23583 contained no verified haplotypes.\n",
      "  539 loci matched more than one catalog locus and were excluded.\n",
      "  871 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  111795 total haplotypes examined from matching loci, 85897 verified.\n",
      "Outputing to file ./PBIRD14X101H_0073.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "264512 stacks compared against the catalog containing 1217217 loci.\n",
      "  119299 matching loci, 28485 contained no verified haplotypes.\n",
      "  825 loci matched more than one catalog locus and were excluded.\n",
      "  1078 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  123693 total haplotypes examined from matching loci, 91343 verified.\n",
      "Outputing to file ./PBIRD14X101H_0075.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078' [1 of 1]\n",
      " Unable to open '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078'\n",
      "Unable to parse '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078'\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "50444 stacks compared against the catalog containing 1217217 loci.\n",
      "  48949 matching loci, 9724 contained no verified haplotypes.\n",
      "  194 loci matched more than one catalog locus and were excluded.\n",
      "  265 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  50108 total haplotypes examined from matching loci, 39639 verified.\n",
      "Outputing to file ./PBIRD14X101H_0101.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "681 stacks compared against the catalog containing 1217217 loci.\n",
      "  46 matching loci, 7 contained no verified haplotypes.\n",
      "  0 loci matched more than one catalog locus and were excluded.\n",
      "  6 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  52 total haplotypes examined from matching loci, 43 verified.\n",
      "Outputing to file ./PBIRD14X101H_0104.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88021 stacks compared against the catalog containing 1217217 loci.\n",
      "  82856 matching loci, 16056 contained no verified haplotypes.\n",
      "  302 loci matched more than one catalog locus and were excluded.\n",
      "  152 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85743 total haplotypes examined from matching loci, 68016 verified.\n",
      "Outputing to file ./PBIRD14X107H_0001.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "92329 stacks compared against the catalog containing 1217217 loci.\n",
      "  81781 matching loci, 15754 contained no verified haplotypes.\n",
      "  314 loci matched more than one catalog locus and were excluded.\n",
      "  343 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84699 total haplotypes examined from matching loci, 67472 verified.\n",
      "Outputing to file ./PBIRD14X107H_0002.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82838 stacks compared against the catalog containing 1217217 loci.\n",
      "  80232 matching loci, 15755 contained no verified haplotypes.\n",
      "  307 loci matched more than one catalog locus and were excluded.\n",
      "  131 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83205 total haplotypes examined from matching loci, 65747 verified.\n",
      "Outputing to file ./PBIRD14X107H_0003.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90985 stacks compared against the catalog containing 1217217 loci.\n",
      "  83719 matching loci, 16223 contained no verified haplotypes.\n",
      "  308 loci matched more than one catalog locus and were excluded.\n",
      "  195 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86806 total haplotypes examined from matching loci, 68975 verified.\n",
      "Outputing to file ./PBIRD14X107H_0004.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "102786 stacks compared against the catalog containing 1217217 loci.\n",
      "  87027 matching loci, 16364 contained no verified haplotypes.\n",
      "  338 loci matched more than one catalog locus and were excluded.\n",
      "  322 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90153 total haplotypes examined from matching loci, 72253 verified.\n",
      "Outputing to file ./PBIRD14X107H_0005.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90934 stacks compared against the catalog containing 1217217 loci.\n",
      "  83732 matching loci, 16150 contained no verified haplotypes.\n",
      "  309 loci matched more than one catalog locus and were excluded.\n",
      "  205 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86783 total haplotypes examined from matching loci, 69012 verified.\n",
      "Outputing to file ./PBIRD14X107H_0006.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "83409 stacks compared against the catalog containing 1217217 loci.\n",
      "  80089 matching loci, 15956 contained no verified haplotypes.\n",
      "  281 loci matched more than one catalog locus and were excluded.\n",
      "  190 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  82794 total haplotypes examined from matching loci, 65199 verified.\n",
      "Outputing to file ./PBIRD14X107H_0007.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94342 stacks compared against the catalog containing 1217217 loci.\n",
      "  84329 matching loci, 16267 contained no verified haplotypes.\n",
      "  314 loci matched more than one catalog locus and were excluded.\n",
      "  256 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87309 total haplotypes examined from matching loci, 69582 verified.\n",
      "Outputing to file ./PBIRD14X107H_0008.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87937 stacks compared against the catalog containing 1217217 loci.\n",
      "  82650 matching loci, 16075 contained no verified haplotypes.\n",
      "  322 loci matched more than one catalog locus and were excluded.\n",
      "  160 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85605 total haplotypes examined from matching loci, 67799 verified.\n",
      "Outputing to file ./PBIRD14X107H_0009.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84128 stacks compared against the catalog containing 1217217 loci.\n",
      "  80884 matching loci, 16012 contained no verified haplotypes.\n",
      "  303 loci matched more than one catalog locus and were excluded.\n",
      "  146 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83613 total haplotypes examined from matching loci, 65963 verified.\n",
      "Outputing to file ./PBIRD14X107H_0010.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85615 stacks compared against the catalog containing 1217217 loci.\n",
      "  82335 matching loci, 15991 contained no verified haplotypes.\n",
      "  313 loci matched more than one catalog locus and were excluded.\n",
      "  120 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85247 total haplotypes examined from matching loci, 67549 verified.\n",
      "Outputing to file ./PBIRD14X107H_0011.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88268 stacks compared against the catalog containing 1217217 loci.\n",
      "  83322 matching loci, 16064 contained no verified haplotypes.\n",
      "  301 loci matched more than one catalog locus and were excluded.\n",
      "  137 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86304 total haplotypes examined from matching loci, 68537 verified.\n",
      "Outputing to file ./PBIRD14X107H_0012.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "100871 stacks compared against the catalog containing 1217217 loci.\n",
      "  87220 matching loci, 16368 contained no verified haplotypes.\n",
      "  343 loci matched more than one catalog locus and were excluded.\n",
      "  336 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90291 total haplotypes examined from matching loci, 72449 verified.\n",
      "Outputing to file ./PBIRD14X107H_0013.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "130789 stacks compared against the catalog containing 1217217 loci.\n",
      "  87589 matching loci, 16261 contained no verified haplotypes.\n",
      "  315 loci matched more than one catalog locus and were excluded.\n",
      "  685 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90729 total haplotypes examined from matching loci, 73226 verified.\n",
      "Outputing to file ./PBIRD14X107H_0014.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "98833 stacks compared against the catalog containing 1217217 loci.\n",
      "  84768 matching loci, 16365 contained no verified haplotypes.\n",
      "  287 loci matched more than one catalog locus and were excluded.\n",
      "  309 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87742 total haplotypes examined from matching loci, 69915 verified.\n",
      "Outputing to file ./PBIRD14X107H_0015.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "99258 stacks compared against the catalog containing 1217217 loci.\n",
      "  85477 matching loci, 16236 contained no verified haplotypes.\n",
      "  337 loci matched more than one catalog locus and were excluded.\n",
      "  305 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  88531 total haplotypes examined from matching loci, 70781 verified.\n",
      "Outputing to file ./PBIRD14X107H_0016.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "100064 stacks compared against the catalog containing 1217217 loci.\n",
      "  84850 matching loci, 16211 contained no verified haplotypes.\n",
      "  294 loci matched more than one catalog locus and were excluded.\n",
      "  371 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87637 total haplotypes examined from matching loci, 70116 verified.\n",
      "Outputing to file ./PBIRD14X107H_0017.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "94452 stacks compared against the catalog containing 1217217 loci.\n",
      "  84298 matching loci, 16178 contained no verified haplotypes.\n",
      "  341 loci matched more than one catalog locus and were excluded.\n",
      "  324 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87196 total haplotypes examined from matching loci, 69571 verified.\n",
      "Outputing to file ./PBIRD14X107H_0018.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88476 stacks compared against the catalog containing 1217217 loci.\n",
      "  82640 matching loci, 16096 contained no verified haplotypes.\n",
      "  288 loci matched more than one catalog locus and were excluded.\n",
      "  159 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85519 total haplotypes examined from matching loci, 67791 verified.\n",
      "Outputing to file ./PBIRD14X107H_0019.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "104530 stacks compared against the catalog containing 1217217 loci.\n",
      "  87047 matching loci, 16306 contained no verified haplotypes.\n",
      "  325 loci matched more than one catalog locus and were excluded.\n",
      "  303 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  90307 total haplotypes examined from matching loci, 72446 verified.\n",
      "Outputing to file ./PBIRD14X107H_0020.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84283 stacks compared against the catalog containing 1217217 loci.\n",
      "  80977 matching loci, 15958 contained no verified haplotypes.\n",
      "  291 loci matched more than one catalog locus and were excluded.\n",
      "  159 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83698 total haplotypes examined from matching loci, 66066 verified.\n",
      "Outputing to file ./PBIRD14X107H_0021.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "90549 stacks compared against the catalog containing 1217217 loci.\n",
      "  84326 matching loci, 16203 contained no verified haplotypes.\n",
      "  329 loci matched more than one catalog locus and were excluded.\n",
      "  186 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87415 total haplotypes examined from matching loci, 69540 verified.\n",
      "Outputing to file ./PBIRD14X107H_0022.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "86318 stacks compared against the catalog containing 1217217 loci.\n",
      "  81788 matching loci, 16008 contained no verified haplotypes.\n",
      "  278 loci matched more than one catalog locus and were excluded.\n",
      "  145 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84635 total haplotypes examined from matching loci, 66962 verified.\n",
      "Outputing to file ./PBIRD14X107H_0023.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "97616 stacks compared against the catalog containing 1217217 loci.\n",
      "  86491 matching loci, 16156 contained no verified haplotypes.\n",
      "  319 loci matched more than one catalog locus and were excluded.\n",
      "  182 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  89475 total haplotypes examined from matching loci, 71749 verified.\n",
      "Outputing to file ./PBIRD14X107H_0024.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "93987 stacks compared against the catalog containing 1217217 loci.\n",
      "  84508 matching loci, 16325 contained no verified haplotypes.\n",
      "  289 loci matched more than one catalog locus and were excluded.\n",
      "  254 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  87535 total haplotypes examined from matching loci, 69667 verified.\n",
      "Outputing to file ./PBIRD14X107H_0025.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "78764 stacks compared against the catalog containing 1217217 loci.\n",
      "  76971 matching loci, 15638 contained no verified haplotypes.\n",
      "  278 loci matched more than one catalog locus and were excluded.\n",
      "  159 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  79237 total haplotypes examined from matching loci, 62124 verified.\n",
      "Outputing to file ./PBIRD14X107H_0026.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "84390 stacks compared against the catalog containing 1217217 loci.\n",
      "  80960 matching loci, 15895 contained no verified haplotypes.\n",
      "  294 loci matched more than one catalog locus and were excluded.\n",
      "  128 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  83702 total haplotypes examined from matching loci, 66094 verified.\n",
      "Outputing to file ./PBIRD14X107H_0027.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "87248 stacks compared against the catalog containing 1217217 loci.\n",
      "  82206 matching loci, 16024 contained no verified haplotypes.\n",
      "  309 loci matched more than one catalog locus and were excluded.\n",
      "  186 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  85295 total haplotypes examined from matching loci, 67513 verified.\n",
      "Outputing to file ./PBIRD14X107H_0028.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "76056 stacks compared against the catalog containing 1217217 loci.\n",
      "  74034 matching loci, 15303 contained no verified haplotypes.\n",
      "  264 loci matched more than one catalog locus and were excluded.\n",
      "  341 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  75998 total haplotypes examined from matching loci, 59370 verified.\n",
      "Outputing to file ./PBIRD14X107H_0029.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "88508 stacks compared against the catalog containing 1217217 loci.\n",
      "  83472 matching loci, 16062 contained no verified haplotypes.\n",
      "  306 loci matched more than one catalog locus and were excluded.\n",
      "  163 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  86582 total haplotypes examined from matching loci, 68799 verified.\n",
      "Outputing to file ./PBIRD14X107H_0030.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "82730 stacks compared against the catalog containing 1217217 loci.\n",
      "  79207 matching loci, 15817 contained no verified haplotypes.\n",
      "  265 loci matched more than one catalog locus and were excluded.\n",
      "  212 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  81691 total haplotypes examined from matching loci, 64294 verified.\n",
      "Outputing to file ./PBIRD14X107H_0031.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85460 stacks compared against the catalog containing 1217217 loci.\n",
      "  81423 matching loci, 15980 contained no verified haplotypes.\n",
      "  267 loci matched more than one catalog locus and were excluded.\n",
      "  163 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84326 total haplotypes examined from matching loci, 66665 verified.\n",
      "Outputing to file ./PBIRD14X107H_0032.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "85613 stacks compared against the catalog containing 1217217 loci.\n",
      "  81187 matching loci, 15973 contained no verified haplotypes.\n",
      "  317 loci matched more than one catalog locus and were excluded.\n",
      "  139 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  84104 total haplotypes examined from matching loci, 66423 verified.\n",
      "Outputing to file ./PBIRD14X107H_0033.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "75695 stacks compared against the catalog containing 1217217 loci.\n",
      "  73717 matching loci, 15097 contained no verified haplotypes.\n",
      "  278 loci matched more than one catalog locus and were excluded.\n",
      "  288 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  75748 total haplotypes examined from matching loci, 59282 verified.\n",
      "Outputing to file ./PBIRD14X107H_0034.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "Searching for matches by sequence identity...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "\n",
      "Processing sample '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045' [1 of 1]\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.tags.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.snps.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045.alleles.tsv.gz\n",
      "Searching for sequence matches...\n",
      "62589 stacks compared against the catalog containing 1217217 loci.\n",
      "  61078 matching loci, 12747 contained no verified haplotypes.\n",
      "  207 loci matched more than one catalog locus and were excluded.\n",
      "  243 loci contained SNPs unaccounted for in the catalog and were excluded.\n",
      "  62467 total haplotypes examined from matching loci, 48735 verified.\n",
      "Outputing to file ./PBIRD14X107H_0045.matches.tsv.gz\n",
      "\n",
      "sstacks is done.\n",
      "You must specify at least one sample file.\n",
      "sstacks 1.46\n",
      "sstacks [--aligned] -P dir [-b batch_id] -M popmap [-p n_threads]\n",
      "sstacks [--aligned] -c catalog_path -s sample_path [-s sample_path ...] -o path [-p n_threads]\n",
      "  b: database/batch ID of the catalog to consider (default: guess).\n",
      "  P: path to the directory containing Stacks files.\n",
      "  M: path to a population map file from which to take sample names.\n",
      "  s: filename prefix from which to load sample loci.\n",
      "  c: path to the catalog.\n",
      "  g,--aligned: base matching on alignment position, not sequence identity.\n",
      "  p: enable parallel execution with num_threads threads.\n",
      "  o: output path to write results.\n",
      "  x: don't verify haplotype of matching locus.\n",
      "\n",
      "Gapped assembly options:\n",
      "  --gapped: preform gapped alignments between stacks.\n"
     ]
    }
   ],
   "source": [
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0001'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0002'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0003'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0004'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0005'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0006'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0007'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0008'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0009'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0010'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0011'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0012'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0035'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0037'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0038'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0039'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0043' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0050' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0053' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0064' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0085' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0086'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0088'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0089'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0090' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0092'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0097' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0098'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0099'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0100' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0101'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0102' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0103' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0104' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0105' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0106'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0107' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0109'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0110'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0114'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0116'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0122' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0013' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0014'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0015'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0016'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0017'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0018'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0019'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0020' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0021'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0022'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101G_0023'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0011' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0013' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0014'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0015' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0016' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0017' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0018'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0019' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0020'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0023'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0024' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0025' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0026'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0027' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0029'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0031' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0032' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0033'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0034'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0035'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0038'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0039' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0040'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0041'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0047'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0049'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0050'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0053' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0055'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0056' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0058' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0059'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0062'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0066'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0067' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0069'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0070' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0073' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0075' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0078' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0101' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0104'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0001' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0002' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0003' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0004' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0005'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0006'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0007' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0008' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0009' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0010'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0011'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0012'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0013' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0014'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0015' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0016' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0017'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0018' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0019'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0020' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0021'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0022'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0023' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0024'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0025' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0026' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0027' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0028' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0029' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0030' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0031'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0032' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0033'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0034' \n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -s '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0045'\n",
    "! sstacks -b 5 -p 7 -c '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5' -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "These individuals not in MapStacks, failed in ustacks so trying again\n",
    "PBIRD14X101H_0013\n",
    "PBIRD14X101H_0017\n",
    "PBIRD14X101H_0019\n",
    "PBIRD14X101H_0020\n",
    "PBIRD14X101H_0026\n",
    "PBIRD14X101H_0027\n",
    "PBIRD14X101H_0038\n",
    "PBIRD14X101H_0041\n",
    "PBIRD14X101H_0047\n",
    "PBIRD14X101H_0049\n",
    "PBIRD14X101H_0059\n",
    "PBIRD14X101H_0069\n",
    "PBIRD14X101H_0078"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ustacks parameters selected:\n",
      "  Sample ID: 5020\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1709295 RAD-Tags.\n",
      "  Inserted 461868 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "151286 initial stacks were populated; 310582 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 9.24549; Std Dev: 33.3498; Max: 7035\n",
      "Deleveraging trigger: 43; Removal trigger: 76\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 71 (75 k-mers per read); 4 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3196 stacks.\n",
      "  148239 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 8.61969; Std Dev: 7.4635; Max: 76\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  148239 stacks merged into 138651 loci; deleveraged 438 loci; blacklisted 279 loci.\n",
      "After merging, coverage depth Mean: 9.11976; Std Dev: 7.94749; Max: 145\n",
      "Merging remainder radtags\n",
      "  310582 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x000000005805a890 ***\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007f4b1091dfa0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5024\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2413271 RAD-Tags.\n",
      "  Inserted 954830 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "257959 initial stacks were populated; 696871 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 6.65377; Std Dev: 21.3345; Max: 4619\n",
      "Deleveraging trigger: 28; Removal trigger: 49\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4028 stacks.\n",
      "  254252 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 6.21904; Std Dev: 5.91901; Max: 49\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  254252 stacks merged into 241560 loci; deleveraged 568 loci; blacklisted 346 loci.\n",
      "After merging, coverage depth Mean: 6.48982; Std Dev: 6.3073; Max: 107\n",
      "Merging remainder radtags\n",
      "  696871 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007fce0d0ff2c0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5026\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2614878 RAD-Tags.\n",
      "  Inserted 1187359 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "298628 initial stacks were populated; 888731 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.78026; Std Dev: 16.2716; Max: 3268\n",
      "Deleveraging trigger: 22; Removal trigger: 38\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4163 stacks.\n",
      "  294802 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.47223; Std Dev: 4.4142; Max: 38\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  294802 stacks merged into 282169 loci; deleveraged 484 loci; blacklisted 300 loci.\n",
      "After merging, coverage depth Mean: 5.67657; Std Dev: 4.6872; Max: 65\n",
      "Merging remainder radtags\n",
      "  888731 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00000000b57b6ca0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5027\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz\n",
      "Loading RAD-Tags...1M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 1995033 RAD-Tags.\n",
      "  Inserted 427784 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "135685 initial stacks were populated; 292099 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 12.5506; Std Dev: 42.3982; Max: 8242\n",
      "Deleveraging trigger: 55; Removal trigger: 97\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3488 stacks.\n",
      "  132351 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 11.7418; Std Dev: 9.60702; Max: 97\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  132351 stacks merged into 121301 loci; deleveraged 511 loci; blacklisted 314 loci.\n",
      "After merging, coverage depth Mean: 12.6637; Std Dev: 10.2509; Max: 187\n",
      "Merging remainder radtags\n",
      "  292099 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007fb7a4825af0 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5031\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2864022 RAD-Tags.\n",
      "  Inserted 1301902 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "320506 initial stacks were populated; 981396 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.87392; Std Dev: 19.1669; Max: 4638\n",
      "Deleveraging trigger: 25; Removal trigger: 44\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 4190 stacks.\n",
      "  316614 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.54676; Std Dev: 4.89065; Max: 44\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  316614 stacks merged into 300421 loci; deleveraged 741 loci; blacklisted 404 loci.\n",
      "After merging, coverage depth Mean: 5.80366; Std Dev: 5.22573; Max: 80\n",
      "Merging remainder radtags\n",
      "  981396 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 17 (78 k-mers per read); 10 k-mer hits required.\n",
      "*** Error in `ustacks': free(): corrupted unsorted chunks: 0x00007fed11537150 ***\n",
      "*** Error in `ustacks': malloc(): memory corruption: 0x00007fed21508200 ***\n",
      "Aborted (core dumped)\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5032\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz\n",
      "Loading RAD-Tags...1M...2M...3M...4M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 4195070 RAD-Tags.\n",
      "  Inserted 2144237 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "459196 initial stacks were populated; 1685041 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.46614; Std Dev: 17.6764; Max: 4735\n",
      "Deleveraging trigger: 23; Removal trigger: 41\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 75 (77 k-mers per read); 2 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 5008 stacks.\n",
      "  454704 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.20283; Std Dev: 5.36733; Max: 41\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 47 (99 k-mers per read); 5 k-mer hits required.\n",
      "Merging stacks, maximum allowed distance: 2 nucleotide(s)\n",
      "  454704 stacks merged into 432612 loci; deleveraged 1018 loci; blacklisted 567 loci.\n",
      "After merging, coverage depth Mean: 5.43188; Std Dev: 5.71758; Max: 86\n",
      "Merging remainder radtags\n",
      "  1685041 remainder sequences left to merge.\n",
      "  Distance allowed between stacks: 4; searching with a k-mer length of 29 (123 k-mers per read); 7 k-mer hits required.\n",
      "^C\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5039\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz\n",
      "Loading RAD-Tags...1M...^C\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5042\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz\n",
      "Loading RAD-Tags...1M...2M...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 2373889 RAD-Tags.\n",
      "  Inserted 1005650 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "277027 initial stacks were populated; 728623 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 5.93901; Std Dev: 19.4497; Max: 4926\n",
      "Deleveraging trigger: 25; Removal trigger: 45\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "Removing repetitive stacks.\n",
      "  Removed 3962 stacks.\n",
      "  273338 stacks remain for merging.\n",
      "Post-Repeat Removal, coverage depth Mean: 5.578; Std Dev: 4.84362; Max: 45\n",
      "Calculating distance between stacks...\n",
      "  Distance allowed between stacks: 2; searching with a k-mer length of 31 (64 k-mers per read); 2 k-mer hits required.\n",
      "^C\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5043\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz\n",
      "Loading RAD-Tags...^C\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5044\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz\n",
      "Loading RAD-Tags...1M...^C\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5050\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz\n",
      "Loading RAD-Tags...done\n",
      "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
      "Loaded 431328 RAD-Tags.\n",
      "  Inserted 166131 elements into the RAD-Tags hash map.\n",
      "  0 reads contained uncalled nucleotides that were modified.\n",
      "72086 initial stacks were populated; 94045 stacks were set aside as secondary reads.\n",
      "Initial coverage mean: 4.6789; Std Dev: 10.7362; Max: 1704\n",
      "Deleveraging trigger: 15; Removal trigger: 26\n",
      "Calculating distance for removing repetitive stacks.\n",
      "  Distance allowed between stacks: 1; searching with a k-mer length of 47 (48 k-mers per read); 1 k-mer hits required.\n",
      "^C\n",
      "ustacks parameters selected:\n",
      "  Sample ID: 5054\n",
      "  Min depth of coverage to create a stack: 2\n",
      "  Max distance allowed between stacks: 2\n",
      "  Max distance allowed to align secondary reads: 4\n",
      "  Max number of stacks allowed per de novo locus: 3\n",
      "  Deleveraging algorithm: enabled\n",
      "  Removal algorithm: enabled\n",
      "  Model type: SNP\n",
      "  Alpha significance level for model: 0.05\n",
      "  Gapped alignments: disabled\n",
      "Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz\n",
      "Loading RAD-Tags...1M..."
     ]
    }
   ],
   "source": [
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0013.fq.gz' -i 5020 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0017.fq.gz' -i 5024 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0019.fq.gz' -i 5026 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0020.fq.gz' -i 5027 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0026.fq.gz' -i 5031 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0027.fq.gz' -i 5032 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0038.fq.gz' -i 5039 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0041.fq.gz' -i 5042 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0047.fq.gz' -i 5043 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0049.fq.gz' -i 5044 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0059.fq.gz' -i 5050 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0069.fq.gz' -i 5054 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n",
    "! ustacks -f '/media/tyler/My Book/Stacks/Pink/Haploids2014/Bird2014Stacks/PBIRD14X101H_0078.fq.gz' -i 5058 -o '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -m 2 -M 2 -t gzfastq -p 7 -d \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Warning: different sequence lengths detected, this will interfere with Stacks algorithms.\n",
    "So it goes, removing those individuals.  May end up running single read stacks only..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "populations parameters selected:\n",
      "  Fst kernel smoothing: off\n",
      "  Bootstrap resampling: off\n",
      "  Percent samples limit per population: 0\n",
      "  Locus Population limit: 1\n",
      "  Minimum stack depth: 0\n",
      "  Log liklihood filtering: off; threshold: 0\n",
      "  Minor allele frequency cutoff: 0\n",
      "  Maximum observed heterozygosity cutoff: 1\n",
      "  Applying Fst correction: none.\n",
      "\n",
      "Parsing population map...\n",
      "The population map contained 120 samples, 2 population(s), 1 group(s).\n",
      "Reading the catalog...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.tags.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.snps.tsv\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/batch_5.catalog.alleles.tsv\n",
      "Reading matches to the catalog...\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.matches.tsv.gz\n",
      "Warning: File 'PBIRD14X101H_0010.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.matches.tsv.gz\n",
      "Warning: File 'PBIRD14X101H_0014.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0015.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0016.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0018.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0023.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0024.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0025.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0029.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0031.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0032.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0033.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0034.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0035.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0039.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0040.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0050.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0053.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0055.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0056.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0058.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0062.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0066.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0067.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0070.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0073.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0075.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X101H_0101.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.matches.tsv.gz\n",
      "Warning: File 'PBIRD14X107H_0001.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0002.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0003.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0004.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0005.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0006.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0007.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0008.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0009.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0010.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0011.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0012.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0013.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0014.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0015.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0016.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0017.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0018.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0019.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0020.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0021.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0022.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0023.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0024.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0025.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0027.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0028.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0030.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0031.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0032.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "Warning: File 'PBIRD14X107H_0033.matches.tsv(.gz)' is absent, malformed, or does not match the catalog batch ID. Excluding this sample from population analysis.\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.matches.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.matches.tsv.gz\n",
      "Catalog is not reference aligned, arbitrarily ordering catalog loci.\n",
      "Working on 61 samples.\n",
      "Working on 2 population(s):\n",
      "    101: PBIRD14X101H_0002, PBIRD14X101H_0003, PBIRD14X101H_0005, PBIRD14X101H_0007, PBIRD14X101H_0009, PBIRD14X101H_0012, PBIRD14X_0101\n",
      "    107: PBIRD14X107H_0036, PBIRD14X107H_0040, PBIRD14X107H_0041, PBIRD14X107H_0042, PBIRD14X107H_0044, PBIRD14X107H_0046, PBIRD14X107H_0047, PBIRD14X107H_0048, PBIRD14X107H_0049, PBIRD14X107H_0051, PBIRD14X107H_0052, PBIRD14X107H_0054, PBIRD14X107H_0055, PBIRD14X107H_0056, PBIRD14X107H_0057, PBIRD14X107H_0059, PBIRD14X107H_0060, PBIRD14X107H_0061, PBIRD14X107H_0062, PBIRD14X107H_0063, PBIRD14X107H_0065, PBIRD14X107H_0066, PBIRD14X107H_0067, PBIRD14X107H_0068, PBIRD14X107H_0069, PBIRD14X107H_0070, PBIRD14X107H_0071, PBIRD14X107H_0072, PBIRD14X107H_0073, PBIRD14X107H_0074, PBIRD14X107H_0075, PBIRD14X107H_0076, PBIRD14X107H_0077, PBIRD14X107H_0078, PBIRD14X107H_0079, PBIRD14X107H_0080, PBIRD14X107H_0081, PBIRD14X107H_0082, PBIRD14X107H_0083, PBIRD14X107H_0084, PBIRD14X107H_0087, PBIRD14X107H_0091, PBIRD14X107H_0093, PBIRD14X107H_0094, PBIRD14X107H_0095, PBIRD14X107H_0096, PBIRD14X107H_0111, PBIRD14X107H_0113, PBIRD14X107H_0115, PBIRD14X107H_0117, PBIRD14X107H_0123, PBIRD14X107H_0124, PBIRD14X107H_0126, PBIRD14X_0107\n",
      "Working on 1 group(s) of populations:\n",
      "    defaultgrp: 101, 107\n",
      "Populating observed haplotypes for 61 samples, 1217217 loci.\n",
      "Writing SQL markers file to '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.markers.tsv'\n",
      "Removing 741289 loci that did not pass sample/population constraints... retained 475928 loci.\n",
      "Loading model outputs for 61 samples, 475928 loci.\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0002.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0003.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0005.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0007.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0009.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X101H_0012.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0101.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0036.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0040.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0041.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0042.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0044.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0046.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0047.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0048.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0049.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0051.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0052.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0054.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0055.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0056.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0057.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0059.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0060.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0061.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0062.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0063.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0065.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0066.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0067.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0068.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0069.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0070.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0071.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0072.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0073.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0074.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0075.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0076.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0077.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0078.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0079.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0080.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0081.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0082.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0083.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0084.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0087.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0091.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0093.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0094.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0095.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0096.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0111.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0113.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0115.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0117.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0123.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0124.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X107H_0126.models.tsv.gz\n",
      "  Parsing /media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks/PBIRD14X_0107.models.tsv.gz\n",
      "Generating nucleotide-level summary statistics for population '101'\n",
      "Population '101' contained 17 incompatible loci -- more than two alleles present.\n",
      "Generating nucleotide-level summary statistics for population '107'\n",
      "Population '107' contained 228 incompatible loci -- more than two alleles present.\n",
      "Tallying loci across populations...done.\n",
      "Pruned 59 variant sites due to filter constraints (more with --verbose).\n",
      "Removing 53788 additional loci for which all variant sites were filtered... retained 422140 loci.\n",
      "Regenerating nucleotide-level summary statistics for population '101'\n",
      "Population '101' contained 0 incompatible loci -- more than two alleles present.\n",
      "Regenerating nucleotide-level summary statistics for population '107'\n",
      "Population '107' contained 0 incompatible loci -- more than two alleles present.\n",
      "Re-tallying loci across populations...done.\n",
      "Generating haplotype-level summary statistics for population '101'\n",
      "Generating haplotype-level summary statistics for population '107'\n",
      "Writing 422140 loci to summary statistics file, '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.sumstats.tsv'\n",
      "Writing 422140 loci to observed haplotype file, '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.haplotypes.tsv'\n",
      "Writing population data to GenePop file '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations/batch_5.genepop'\n",
      "Writing population data to PLINK files...done.\n",
      "Populations is done.\n"
     ]
    }
   ],
   "source": [
    "## Running populations without filtering to do all in plink and Waples pipeline\n",
    "! populations -P '/media/tyler/My Book/Stacks/Pink/Haploids2014/MapStacks' -M '/media/tyler/My Book/Stacks/Pink/Haploids2014/BirdCreek2014MapStacksPopulationsMapParentsHapsPassedustacks.txt' -b 5 -O '/media/tyler/My Book/Stacks/Pink/Haploids2014/Populations' -t 7 -s --genepop --plink --verbose"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
